{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# AMR Analysis training\n",
    "## Prerequisite\n",
    " * blast (known to work with 2.10.1+)\n",
    " * samtools (1.11)\n",
    " * skesa\n",
    " * prokka (1.14.6)\n",
    " * mlst (2.19.6)\n",
    " * abricate (1.0.1 | Database: vfdb ecoli_vf ecoh card megares resfinder argannot ncbi plasmidfinder)\n",
    " * parsnp \n",
    " ### Data\n",
    " 1. Reference genome: sample/GCF_000005845.2_ASM584v2_genomic.fna\n",
    " 2. Sample to analysis: Download [here](https://drive.google.com/drive/folders/1P7_un6I2HzBkObztqbBEffM19ESG2w-u?usp=sharing) and move to samples/ folder "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Download IGV"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Archive:  IGV.zip\n",
      "   creating: IGV_Linux_2.14.1/\n",
      "   creating: IGV_Linux_2.14.1/jdk-11/\n",
      "   creating: IGV_Linux_2.14.1/jdk-11/lib/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libawt.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libmanagement_agent.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libsctp.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libverify.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libawt_xawt.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libinstrument.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libjava.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libfontmanager.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libsplashscreen.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libjawt.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libjdwp.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libprefs.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libsunec.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libj2pkcs11.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/liblcms.so  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/lib/jfr/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/jfr/profile.jfc  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/jfr/default.jfc  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libzip.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/psfontj2d.properties  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libdt_socket.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libjaas.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libj2pcsc.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/jvm.cfg  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libnio.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/librmi.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/jexec  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/lib/server/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/server/Xusage.txt  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/server/libjsig.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/server/libjvm.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libjsig.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libjimage.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libmanagement_ext.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libjsound.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/tzdb.dat  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libjavajpeg.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libnet.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libextnet.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libmanagement.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libj2gss.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libunpack.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/classlist  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libawt_headless.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/modules  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/libmlib_image.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/jrt-fs.jar  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/jspawnhelper  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/psfont.properties.ja  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/lib/security/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/security/public_suffix_list.dat  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/security/cacerts  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/security/blacklisted.certs  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/security/default.policy  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/lib/jli/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/lib/jli/libjli.so  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/release  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/bin/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/bin/pack200  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/bin/jfr  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/bin/keytool  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/bin/unpack200  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/bin/jjs  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/bin/jrunscript  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/bin/java  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/bin/jaotc  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/bin/rmid  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/bin/rmiregistry  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/man/\n",
      "   creating: IGV_Linux_2.14.1/jdk-11/man/ja_JP.UTF-8/\n",
      "   creating: IGV_Linux_2.14.1/jdk-11/man/ja_JP.UTF-8/man1/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/man/ja_JP.UTF-8/man1/rmiregistry.1  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/man/ja_JP.UTF-8/man1/keytool.1  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/man/ja_JP.UTF-8/man1/jjs.1  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/man/ja_JP.UTF-8/man1/rmid.1  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/man/ja_JP.UTF-8/man1/unpack200.1  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/man/ja_JP.UTF-8/man1/java.1  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/man/ja_JP.UTF-8/man1/pack200.1  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/man/ja/\n",
      "   creating: IGV_Linux_2.14.1/jdk-11/man/ja/man1/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/man/ja/man1/rmiregistry.1  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/man/ja/man1/keytool.1  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/man/ja/man1/jjs.1  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/man/ja/man1/rmid.1  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/man/ja/man1/unpack200.1  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/man/ja/man1/java.1  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/man/ja/man1/pack200.1  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/man/man1/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/man/man1/rmiregistry.1  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/man/man1/keytool.1  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/man/man1/jjs.1  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/man/man1/rmid.1  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/man/man1/unpack200.1  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/man/man1/java.1  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/man/man1/pack200.1  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/\n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.dynalink/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.dynalink/dynalink.md  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.dynalink/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.dynalink/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.dynalink/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/java.sql/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.sql/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.sql/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.sql/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.jsobject/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.jsobject/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.jsobject/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.jsobject/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.net/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.net/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.net/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.net/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/java.compiler/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.compiler/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.compiler/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.compiler/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.aot/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.aot/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.aot/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.aot/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/java.net.http/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.net.http/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.net.http/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.net.http/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.sctp/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.sctp/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.sctp/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.sctp/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/java.prefs/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.prefs/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.prefs/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.prefs/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/java.security.jgss/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.security.jgss/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.security.jgss/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.security.jgss/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/java.base/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.base/unicode.md  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.base/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.base/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.base/ASSEMBLY_EXCEPTION  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.base/aes.md  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.base/cldr.md  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.base/icu.md  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.base/public_suffix.md  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.base/asm.md  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.base/c-libutl.md  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.scripting.nashorn.shell/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.scripting.nashorn.shell/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.scripting.nashorn.shell/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.scripting.nashorn.shell/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/java.security.sasl/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.security.sasl/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.security.sasl/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.security.sasl/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/java.desktop/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.desktop/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.desktop/lcms.md  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.desktop/giflib.md  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.desktop/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.desktop/ASSEMBLY_EXCEPTION  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.desktop/colorimaging.md  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.desktop/mesa3d.md  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.desktop/harfbuzz.md  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.desktop/libpng.md  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.desktop/jpeg.md  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.desktop/xwd.md  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.internal.vm.compiler.management/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.internal.vm.compiler.management/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.internal.vm.compiler.management/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.internal.vm.compiler.management/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.security.auth/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.security.auth/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.security.auth/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.security.auth/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.zipfs/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.zipfs/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.zipfs/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.zipfs/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.unsupported/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.unsupported/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.unsupported/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.unsupported/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.httpserver/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.httpserver/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.httpserver/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.httpserver/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.jfr/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.jfr/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.jfr/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.jfr/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.internal.le/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.internal.le/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.internal.le/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.internal.le/ASSEMBLY_EXCEPTION  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.internal.le/jline.md  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.jdwp.agent/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.jdwp.agent/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.jdwp.agent/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.jdwp.agent/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.management.jfr/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.management.jfr/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.management.jfr/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.management.jfr/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.naming.rmi/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.naming.rmi/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.naming.rmi/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.naming.rmi/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/java.management/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.management/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.management/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.management/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.crypto.ec/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.crypto.ec/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.crypto.ec/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.crypto.ec/ASSEMBLY_EXCEPTION  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.crypto.ec/ecc.md  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.scripting.nashorn/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.scripting.nashorn/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.scripting.nashorn/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.scripting.nashorn/ASSEMBLY_EXCEPTION  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.scripting.nashorn/joni.md  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.scripting.nashorn/double-conversion.md  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.naming.dns/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.naming.dns/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.naming.dns/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.naming.dns/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.charsets/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.charsets/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.charsets/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.charsets/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/java.management.rmi/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.management.rmi/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.management.rmi/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.management.rmi/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.internal.vm.ci/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.internal.vm.ci/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.internal.vm.ci/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.internal.vm.ci/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/java.rmi/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.rmi/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.rmi/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.rmi/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/java.transaction.xa/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.transaction.xa/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.transaction.xa/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.transaction.xa/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/java.sql.rowset/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.sql.rowset/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.sql.rowset/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.sql.rowset/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/java.scripting/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.scripting/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.scripting/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.scripting/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.security.jgss/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.security.jgss/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.security.jgss/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.security.jgss/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/java.xml/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.xml/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.xml/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.xml/ASSEMBLY_EXCEPTION  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.xml/bcel.md  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.xml/dom.md  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.xml/xerces.md  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.xml/jcup.md  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.xml/xalan.md  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/java.logging/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.logging/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.logging/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.logging/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.accessibility/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.accessibility/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.accessibility/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.accessibility/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/java.smartcardio/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.smartcardio/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.smartcardio/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.smartcardio/ASSEMBLY_EXCEPTION  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.smartcardio/pcsclite.md  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.management/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.management/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.management/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.management/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.pack/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.pack/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.pack/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.pack/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.internal.ed/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.internal.ed/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.internal.ed/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.internal.ed/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/java.se/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.se/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.se/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.se/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.xml.dom/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.xml.dom/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.xml.dom/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.xml.dom/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.management.agent/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.management.agent/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.management.agent/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.management.agent/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.crypto.cryptoki/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.crypto.cryptoki/pkcs11wrapper.md  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.crypto.cryptoki/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.crypto.cryptoki/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.crypto.cryptoki/ASSEMBLY_EXCEPTION  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.crypto.cryptoki/pkcs11cryptotoken.md  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/java.instrument/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.instrument/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.instrument/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.instrument/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.internal.vm.compiler/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.internal.vm.compiler/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.internal.vm.compiler/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.internal.vm.compiler/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.naming.ldap/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.naming.ldap/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.naming.ldap/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.naming.ldap/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/java.xml.crypto/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.xml.crypto/santuario.md  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.xml.crypto/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.xml.crypto/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.xml.crypto/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/jdk.localedata/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.localedata/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.localedata/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.localedata/ASSEMBLY_EXCEPTION  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.localedata/cldr.md  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/jdk.localedata/thaidict.md  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/java.datatransfer/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.datatransfer/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.datatransfer/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.datatransfer/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/legal/java.naming/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.naming/LICENSE  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.naming/ADDITIONAL_LICENSE_INFO  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/legal/java.naming/ASSEMBLY_EXCEPTION  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/conf/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/conf/sound.properties  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/conf/net.properties  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/conf/management/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/conf/management/jmxremote.password.template  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/conf/management/jmxremote.access  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/conf/management/management.properties  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/conf/logging.properties  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/conf/security/\n",
      "   creating: IGV_Linux_2.14.1/jdk-11/conf/security/policy/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/conf/security/policy/README.txt  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/conf/security/policy/limited/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/conf/security/policy/limited/exempt_local.policy  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/conf/security/policy/limited/default_US_export.policy  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/conf/security/policy/limited/default_local.policy  \n",
      "   creating: IGV_Linux_2.14.1/jdk-11/conf/security/policy/unlimited/\n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/conf/security/policy/unlimited/default_US_export.policy  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/conf/security/policy/unlimited/default_local.policy  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/conf/security/java.security  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/conf/security/java.policy  \n",
      "  inflating: IGV_Linux_2.14.1/jdk-11/NOTICE  \n",
      "   creating: IGV_Linux_2.14.1/lib/\n",
      "  inflating: IGV_Linux_2.14.1/lib/sts-2.16.7.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/httpclient-4.5.13.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/jackson-annotations-2.12.1.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/commons-jexl-2.1.1.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/cognitoidentity-2.16.7.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/aws-xml-protocol-2.16.7.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/batik-constants-1.11.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/netty-handler-4.1.59.Final.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/commons-logging-1.2.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/netty-codec-http-4.1.59.Final.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/batik-codec-1.11.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/ngs-java-2.9.0.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/netty-common-4.1.59.Final.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/batik-awt-util-1.11.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/protocol-core-2.16.7.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/jackson-core-2.12.1.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/slf4j-api-1.7.28.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/j2objc-annotations-1.1.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/jackson-databind-2.12.1.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/aws-query-protocol-2.16.7.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/AbsoluteLayout-RELEASE110.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/gson-2.8.5.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/netty-transport-native-unix-common-4.1.59.Final.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/netty-resolver-4.1.59.Final.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/batik-xml-1.11.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/auth-2.16.7.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/commons-lang3-3.9.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/reactive-streams-1.0.3.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/htsjdk-3.0.0.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/annotations-2.16.7.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/netty-codec-4.1.59.Final.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/snappy-java-1.1.8.4.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/sdk-core-2.16.7.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/batik-transcoder-1.11.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/netty-nio-client-2.16.7.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/netty-transport-4.1.59.Final.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/profiles-2.16.7.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/guava-27.0.1-jre.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/igv.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/s3-2.16.7.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/jide-common-3.7.3.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/mjson-1.4.1.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/batik-util-1.11.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/netty-transport-native-epoll-4.1.59.Final-linux-x86_64.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/batik-dom-1.11.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/commons-math3-3.6.1.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/netty-reactive-streams-http-2.0.4.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/regions-2.16.7.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/aws-core-2.16.7.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/apache-client-2.16.7.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/batik-svggen-1.11.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/utils-2.16.7.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/http-client-spi-2.16.7.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/xmlgraphics-commons-2.3.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/commons-io-2.6.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/netty-buffer-4.1.59.Final.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/netty-reactive-streams-2.0.4.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/httpcore-4.4.13.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/metrics-spi-2.16.7.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/commons-codec-1.11.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/arns-2.16.7.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/aws-json-protocol-2.16.7.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/swing-layout-1.0.3.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/commons-compress-1.19.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/batik-ext-1.11.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/batik-i18n-1.11.jar  \n",
      "  inflating: IGV_Linux_2.14.1/lib/eventstream-1.0.1.jar  \n",
      "  inflating: IGV_Linux_2.14.1/igv.args  \n",
      "  inflating: IGV_Linux_2.14.1/igv_hidpi.sh  \n",
      "  inflating: IGV_Linux_2.14.1/readme.txt  \n",
      "  inflating: IGV_Linux_2.14.1/igv.sh  \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "--2022-09-22 17:44:08--  https://data.broadinstitute.org/igv/projects/downloads/2.14/IGV_Linux_2.14.1_WithJava.zip\n",
      "Resolving data.broadinstitute.org (data.broadinstitute.org)... 69.173.92.29\n",
      "Connecting to data.broadinstitute.org (data.broadinstitute.org)|69.173.92.29|:443... connected.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 71472977 (68M) [application/zip]\n",
      "Saving to: ‘IGV.zip’\n",
      "\n",
      "     0K .......... .......... .......... .......... ..........  0% 17.8M 4s\n",
      "    50K .......... .......... .......... .......... ..........  0% 82.4K 7m5s\n",
      "   100K .......... .......... .......... .......... ..........  0% 89.3K 9m3s\n",
      "   150K .......... .......... .......... .......... ..........  0% 1.55M 6m58s\n",
      "   200K .......... .......... .......... .......... ..........  0%  151K 7m6s\n",
      "   250K .......... .......... .......... .......... ..........  0% 34.6M 5m55s\n",
      "   300K .......... .......... .......... .......... ..........  0%  164K 6m5s\n",
      "   350K .......... .......... .......... .......... ..........  0% 23.5M 5m19s\n",
      "   400K .......... .......... .......... .......... ..........  0%  161K 5m31s\n",
      "   450K .......... .......... .......... .......... ..........  0% 44.2M 4m58s\n",
      "   500K .......... .......... .......... .......... ..........  0%  166K 5m9s\n",
      "   550K .......... .......... .......... .......... ..........  0% 51.8M 4m43s\n",
      "   600K .......... .......... .......... .......... ..........  0%  186K 4m50s\n",
      "   650K .......... .......... .......... .......... ..........  1%  172K 4m57s\n",
      "   700K .......... .......... .......... .......... ..........  1% 22.6M 4m38s\n",
      "   750K .......... .......... .......... .......... ..........  1%  179K 4m44s\n",
      "   800K .......... .......... .......... .......... ..........  1%  214K 4m46s\n",
      "   850K .......... .......... .......... .......... ..........  1% 3.00M 4m31s\n",
      "   900K .......... .......... .......... .......... ..........  1%  505K 4m24s\n",
      "   950K .......... .......... .......... .......... ..........  1%  358K 4m20s\n",
      "  1000K .......... .......... .......... .......... ..........  1% 61.0K 5m1s\n",
      "  1050K .......... .......... .......... .......... ..........  1% 79.2M 4m47s\n",
      "  1100K .......... .......... .......... .......... ..........  1%  109M 4m35s\n",
      "  1150K .......... .......... .......... .......... ..........  1% 77.3M 4m23s\n",
      "  1200K .......... .......... .......... .......... ..........  1% 81.5K 4m46s\n",
      "  1250K .......... .......... .......... .......... ..........  1%  101M 4m35s\n",
      "  1300K .......... .......... .......... .......... ..........  1%  124M 4m25s\n",
      "  1350K .......... .......... .......... .......... ..........  2%  117K 4m36s\n",
      "  1400K .......... .......... .......... .......... ..........  2% 67.5M 4m26s\n",
      "  1450K .......... .......... .......... .......... ..........  2%  487K 4m22s\n",
      "  1500K .......... .......... .......... .......... ..........  2%  250K 4m22s\n",
      "  1550K .......... .......... .......... .......... ..........  2%  933K 4m16s\n",
      "  1600K .......... .......... .......... .......... ..........  2%  257K 4m16s\n",
      "  1650K .......... .......... .......... .......... ..........  2% 1.02M 4m10s\n",
      "  1700K .......... .......... .......... .......... ..........  2%  321K 4m9s\n",
      "  1750K .......... .......... .......... .......... ..........  2%  329K 4m8s\n",
      "  1800K .......... .......... .......... .......... ..........  2%  329K 4m6s\n",
      "  1850K .......... .......... .......... .......... ..........  2%  289K 4m6s\n",
      "  1900K .......... .......... .......... .......... ..........  2%  261K 4m6s\n",
      "  1950K .......... .......... .......... .......... ..........  2%  217K 4m7s\n",
      "  2000K .......... .......... .......... .......... ..........  2%  289K 4m7s\n",
      "  2050K .......... .......... .......... .......... ..........  3%  620K 4m3s\n",
      "  2100K .......... .......... .......... .......... ..........  3%  551K 4m1s\n",
      "  2150K .......... .......... .......... .......... ..........  3%  498K 3m58s\n",
      "  2200K .......... .......... .......... .......... ..........  3%  501K 3m55s\n",
      "  2250K .......... .......... .......... .......... ..........  3%  370K 3m54s\n",
      "  2300K .......... .......... .......... .......... ..........  3% 2.72M 3m50s\n",
      "  2350K .......... .......... .......... .......... ..........  3%  501K 3m47s\n",
      "  2400K .......... .......... .......... .......... ..........  3%  222K 3m49s\n",
      "  2450K .......... .......... .......... .......... ..........  3%  643K 3m46s\n",
      "  2500K .......... .......... .......... .......... ..........  3%  349K 3m45s\n",
      "  2550K .......... .......... .......... .......... ..........  3%  942K 3m42s\n",
      "  2600K .......... .......... .......... .......... ..........  3%  497K 3m40s\n",
      "  2650K .......... .......... .......... .......... ..........  3%  503K 3m39s\n",
      "  2700K .......... .......... .......... .......... ..........  3%  963K 3m36s\n",
      "  2750K .......... .......... .......... .......... ..........  4%  969K 3m33s\n",
      "  2800K .......... .......... .......... .......... ..........  4%  472K 3m32s\n",
      "  2850K .......... .......... .......... .......... ..........  4%  572K 3m30s\n",
      "  2900K .......... .......... .......... .......... ..........  4% 1005K 3m27s\n",
      "  2950K .......... .......... .......... .......... ..........  4%  998K 3m25s\n",
      "  3000K .......... .......... .......... .......... ..........  4%  469K 3m24s\n",
      "  3050K .......... .......... .......... .......... ..........  4% 1.06M 3m21s\n",
      "  3100K .......... .......... .......... .......... ..........  4% 1.03M 3m19s\n",
      "  3150K .......... .......... .......... .......... ..........  4% 2.36M 3m16s\n",
      "  3200K .......... .......... .......... .......... ..........  4%  481K 3m15s\n",
      "  3250K .......... .......... .......... .......... ..........  4% 1.37M 3m13s\n",
      "  3300K .......... .......... .......... .......... ..........  4% 1.03M 3m10s\n",
      "  3350K .......... .......... .......... .......... ..........  4%  499K 3m9s\n",
      "  3400K .......... .......... .......... .......... ..........  4%  339K 3m9s\n",
      "  3450K .......... .......... .......... .......... ..........  5%  877K 3m8s\n",
      "  3500K .......... .......... .......... .......... ..........  5%  424K 3m7s\n",
      "  3550K .......... .......... .......... .......... ..........  5%  899K 3m5s\n",
      "  3600K .......... .......... .......... .......... ..........  5%  572K 3m4s\n",
      "  3650K .......... .......... .......... .......... ..........  5%  993K 3m3s\n",
      "  3700K .......... .......... .......... .......... ..........  5%  546K 3m2s\n",
      "  3750K .......... .......... .......... .......... ..........  5%  459K 3m1s\n",
      "  3800K .......... .......... .......... .......... ..........  5%  292K 3m1s\n",
      "  3850K .......... .......... .......... .......... ..........  5%  557K 3m0s\n",
      "  3900K .......... .......... .......... .......... ..........  5%  506K 3m0s\n",
      "  3950K .......... .......... .......... .......... ..........  5%  749K 2m58s\n",
      "  4000K .......... .......... .......... .......... ..........  5%  354K 2m58s\n",
      "  4050K .......... .......... .......... .......... ..........  5%  286K 2m59s\n",
      "  4100K .......... .......... .......... .......... ..........  5%  656K 2m58s\n",
      "  4150K .......... .......... .......... .......... ..........  6%  284K 2m58s\n",
      "  4200K .......... .......... .......... .......... ..........  6%  172K 3m0s\n",
      "  4250K .......... .......... .......... .......... ..........  6% 67.5M 2m58s\n",
      "  4300K .......... .......... .......... .......... ..........  6%  477K 2m58s\n",
      "  4350K .......... .......... .......... .......... ..........  6%  249K 2m58s\n",
      "  4400K .......... .......... .......... .......... ..........  6%  299K 2m59s\n",
      "  4450K .......... .......... .......... .......... ..........  6%  598K 2m58s\n",
      "  4500K .......... .......... .......... .......... ..........  6%  655K 2m57s\n",
      "  4550K .......... .......... .......... .......... ..........  6%  394K 2m57s\n",
      "  4600K .......... .......... .......... .......... ..........  6%  601K 2m56s\n",
      "  4650K .......... .......... .......... .......... ..........  6%  612K 2m55s\n",
      "  4700K .......... .......... .......... .......... ..........  6%  412K 2m55s\n",
      "  4750K .......... .......... .......... .......... ..........  6%  454K 2m54s\n",
      "  4800K .......... .......... .......... .......... ..........  6% 1.29M 2m53s\n",
      "  4850K .......... .......... .......... .......... ..........  7%  350K 2m53s\n",
      "  4900K .......... .......... .......... .......... ..........  7%  460K 2m52s\n",
      "  4950K .......... .......... .......... .......... ..........  7%  347K 2m52s\n",
      "  5000K .......... .......... .......... .......... ..........  7%  497K 2m52s\n",
      "  5050K .......... .......... .......... .......... ..........  7%  341K 2m52s\n",
      "  5100K .......... .......... .......... .......... ..........  7%  689K 2m51s\n",
      "  5150K .......... .......... .......... .......... ..........  7% 1.39M 2m50s\n",
      "  5200K .......... .......... .......... .......... ..........  7%  346K 2m50s\n",
      "  5250K .......... .......... .......... .......... ..........  7%  696K 2m49s\n",
      "  5300K .......... .......... .......... .......... ..........  7% 1.35M 2m47s\n",
      "  5350K .......... .......... .......... .......... ..........  7%  895K 2m46s\n",
      "  5400K .......... .......... .......... .......... ..........  7%  495K 2m46s\n",
      "  5450K .......... .......... .......... .......... ..........  7%  727K 2m45s\n",
      "  5500K .......... .......... .......... .......... ..........  7%  155K 2m47s\n",
      "  5550K .......... .......... .......... .......... ..........  8% 86.1M 2m46s\n",
      "  5600K .......... .......... .......... .......... ..........  8%  297K 2m46s\n",
      "  5650K .......... .......... .......... .......... ..........  8% 63.6M 2m44s\n",
      "  5700K .......... .......... .......... .......... ..........  8% 77.3M 2m43s\n",
      "  5750K .......... .......... .......... .......... ..........  8%  613K 2m42s\n",
      "  5800K .......... .......... .......... .......... ..........  8%  536K 2m42s\n",
      "  5850K .......... .......... .......... .......... ..........  8%  332K 2m42s\n",
      "  5900K .......... .......... .......... .......... ..........  8% 2.03M 2m41s\n",
      "  5950K .......... .......... .......... .......... ..........  8%  275K 2m41s\n",
      "  6000K .......... .......... .......... .......... ..........  8%  981K 2m40s\n",
      "  6050K .......... .......... .......... .......... ..........  8%  195K 2m41s\n",
      "  6100K .......... .......... .......... .......... ..........  8% 1.34M 2m40s\n",
      "  6150K .......... .......... .......... .......... ..........  8%  232K 2m41s\n",
      "  6200K .......... .......... .......... .......... ..........  8% 1.36M 2m40s\n",
      "  6250K .......... .......... .......... .......... ..........  9%  208K 2m41s\n",
      "  6300K .......... .......... .......... .......... ..........  9%  272K 2m42s\n",
      "  6350K .......... .......... .......... .......... ..........  9%  592K 2m41s\n",
      "  6400K .......... .......... .......... .......... ..........  9%  298K 2m41s\n",
      "  6450K .......... .......... .......... .......... ..........  9%  689K 2m41s\n",
      "  6500K .......... .......... .......... .......... ..........  9%  743K 2m40s\n",
      "  6550K .......... .......... .......... .......... ..........  9%  291K 2m40s\n",
      "  6600K .......... .......... .......... .......... ..........  9% 1.97M 2m39s\n",
      "  6650K .......... .......... .......... .......... ..........  9%  766K 2m38s\n",
      "  6700K .......... .......... .......... .......... ..........  9%  481K 2m38s\n",
      "  6750K .......... .......... .......... .......... ..........  9%  714K 2m37s\n",
      "  6800K .......... .......... .......... .......... ..........  9%  277K 2m38s\n",
      "  6850K .......... .......... .......... .......... ..........  9%  592K 2m37s\n",
      "  6900K .......... .......... .......... .......... ..........  9%  419K 2m37s\n",
      "  6950K .......... .......... .......... .......... .......... 10%  400K 2m37s\n",
      "  7000K .......... .......... .......... .......... .......... 10%  633K 2m37s\n",
      "  7050K .......... .......... .......... .......... .......... 10%  478K 2m36s\n",
      "  7100K .......... .......... .......... .......... .......... 10%  465K 2m36s\n",
      "  7150K .......... .......... .......... .......... .......... 10%  227K 2m37s\n",
      "  7200K .......... .......... .......... .......... .......... 10%  627K 2m36s\n",
      "  7250K .......... .......... .......... .......... .......... 10%  291K 2m36s\n",
      "  7300K .......... .......... .......... .......... .......... 10%  917K 2m36s\n",
      "  7350K .......... .......... .......... .......... .......... 10%  412K 2m36s\n",
      "  7400K .......... .......... .......... .......... .......... 10%  482K 2m35s\n",
      "  7450K .......... .......... .......... .......... .......... 10%  605K 2m35s\n",
      "  7500K .......... .......... .......... .......... .......... 10%  157K 2m36s\n",
      "  7550K .......... .......... .......... .......... .......... 10%  106M 2m35s\n",
      "  7600K .......... .......... .......... .......... .......... 10%  547K 2m35s\n",
      "  7650K .......... .......... .......... .......... .......... 11%  835K 2m34s\n",
      "  7700K .......... .......... .......... .......... .......... 11%  259K 2m34s\n",
      "  7750K .......... .......... .......... .......... .......... 11%  924K 2m34s\n",
      "  7800K .......... .......... .......... .......... .......... 11%  243K 2m34s\n",
      "  7850K .......... .......... .......... .......... .......... 11%  554K 2m34s\n",
      "  7900K .......... .......... .......... .......... .......... 11%  296K 2m34s\n",
      "  7950K .......... .......... .......... .......... .......... 11%  597K 2m34s\n",
      "  8000K .......... .......... .......... .......... .......... 11%  298K 2m34s\n",
      "  8050K .......... .......... .......... .......... .......... 11% 1.50M 2m33s\n",
      "  8100K .......... .......... .......... .......... .......... 11%  519K 2m33s\n",
      "  8150K .......... .......... .......... .......... .......... 11%  406K 2m33s\n",
      "  8200K .......... .......... .......... .......... .......... 11%  649K 2m32s\n",
      "  8250K .......... .......... .......... .......... .......... 11%  990K 2m31s\n",
      "  8300K .......... .......... .......... .......... .......... 11%  426K 2m31s\n",
      "  8350K .......... .......... .......... .......... .......... 12% 1.37M 2m31s\n",
      "  8400K .......... .......... .......... .......... .......... 12%  420K 2m30s\n",
      "  8450K .......... .......... .......... .......... .......... 12%  338K 2m30s\n",
      "  8500K .......... .......... .......... .......... .......... 12%  595K 2m30s\n",
      "  8550K .......... .......... .......... .......... .......... 12%  346K 2m30s\n",
      "  8600K .......... .......... .......... .......... .......... 12%  663K 2m30s\n",
      "  8650K .......... .......... .......... .......... .......... 12% 1.37M 2m29s\n",
      "  8700K .......... .......... .......... .......... .......... 12%  346K 2m29s\n",
      "  8750K .......... .......... .......... .......... .......... 12% 1.01M 2m28s\n",
      "  8800K .......... .......... .......... .......... .......... 12%  901K 2m28s\n",
      "  8850K .......... .......... .......... .......... .......... 12%  417K 2m28s\n",
      "  8900K .......... .......... .......... .......... .......... 12%  655K 2m27s\n",
      "  8950K .......... .......... .......... .......... .......... 12%  291K 2m27s\n",
      "  9000K .......... .......... .......... .......... .......... 12%  573K 2m27s\n",
      "  9050K .......... .......... .......... .......... .......... 13%  996K 2m26s\n",
      "  9100K .......... .......... .......... .......... .......... 13%  327K 2m27s\n",
      "  9150K .......... .......... .......... .......... .......... 13%  953K 2m26s\n",
      "  9200K .......... .......... .......... .......... .......... 13%  589K 2m26s\n",
      "  9250K .......... .......... .......... .......... .......... 13%  235K 2m26s\n",
      "  9300K .......... .......... .......... .......... .......... 13% 29.0M 2m25s\n",
      "  9350K .......... .......... .......... .......... .......... 13%  594K 2m25s\n",
      "  9400K .......... .......... .......... .......... .......... 13%  102K 2m27s\n",
      "  9450K .......... .......... .......... .......... .......... 13% 33.7M 2m26s\n",
      "  9500K .......... .......... .......... .......... .......... 13% 98.2K 2m29s\n",
      "  9550K .......... .......... .......... .......... .......... 13%  198K 2m29s\n",
      "  9600K .......... .......... .......... .......... .......... 13% 63.6M 2m28s\n",
      "  9650K .......... .......... .......... .......... .......... 13% 3.56M 2m28s\n",
      "  9700K .......... .......... .......... .......... .......... 13%  382K 2m27s\n",
      "  9750K .......... .......... .......... .......... .......... 14%  434K 2m27s\n",
      "  9800K .......... .......... .......... .......... .......... 14%  364K 2m27s\n",
      "  9850K .......... .......... .......... .......... .......... 14%  418K 2m27s\n",
      "  9900K .......... .......... .......... .......... .......... 14% 1.53M 2m26s\n",
      "  9950K .......... .......... .......... .......... .......... 14%  236K 2m27s\n",
      " 10000K .......... .......... .......... .......... .......... 14%  342K 2m27s\n",
      " 10050K .......... .......... .......... .......... .......... 14%  434K 2m27s\n",
      " 10100K .......... .......... .......... .......... .......... 14%  372K 2m27s\n",
      " 10150K .......... .......... .......... .......... .......... 14%  432K 2m26s\n",
      " 10200K .......... .......... .......... .......... .......... 14%  624K 2m26s\n",
      " 10250K .......... .......... .......... .......... .......... 14%  295K 2m26s\n",
      " 10300K .......... .......... .......... .......... .......... 14% 2.16M 2m26s\n",
      " 10350K .......... .......... .......... .......... .......... 14%  885K 2m25s\n",
      " 10400K .......... .......... .......... .......... .......... 14%  278K 2m25s\n",
      " 10450K .......... .......... .......... .......... .......... 15% 3.15M 2m25s\n",
      " 10500K .......... .......... .......... .......... .......... 15%  386K 2m24s\n",
      " 10550K .......... .......... .......... .......... .......... 15% 1.24M 2m24s\n",
      " 10600K .......... .......... .......... .......... .......... 15%  363K 2m24s\n",
      " 10650K .......... .......... .......... .......... .......... 15%  612K 2m24s\n",
      " 10700K .......... .......... .......... .......... .......... 15%  291K 2m24s\n",
      " 10750K .......... .......... .......... .......... .......... 15%  885K 2m23s\n",
      " 10800K .......... .......... .......... .......... .......... 15%  348K 2m23s\n",
      " 10850K .......... .......... .......... .......... .......... 15%  925K 2m23s\n",
      " 10900K .......... .......... .......... .......... .......... 15%  607K 2m22s\n",
      " 10950K .......... .......... .......... .......... .......... 15%  398K 2m22s\n",
      " 11000K .......... .......... .......... .......... .......... 15% 1.11M 2m22s\n",
      " 11050K .......... .......... .......... .......... .......... 15%  613K 2m21s\n",
      " 11100K .......... .......... .......... .......... .......... 15%  846K 2m21s\n",
      " 11150K .......... .......... .......... .......... .......... 16%  458K 2m21s\n",
      " 11200K .......... .......... .......... .......... .......... 16%  698K 2m20s\n",
      " 11250K .......... .......... .......... .......... .......... 16% 3.13M 2m20s\n",
      " 11300K .......... .......... .......... .......... .......... 16%  542K 2m20s\n",
      " 11350K .......... .......... .......... .......... .......... 16%  690K 2m19s\n",
      " 11400K .......... .......... .......... .......... .......... 16%  700K 2m19s\n",
      " 11450K .......... .......... .......... .......... .......... 16% 2.30M 2m18s\n",
      " 11500K .......... .......... .......... .......... .......... 16%  315K 2m18s\n",
      " 11550K .......... .......... .......... .......... .......... 16%  627K 2m18s\n",
      " 11600K .......... .......... .......... .......... .......... 16%  368K 2m18s\n",
      " 11650K .......... .......... .......... .......... .......... 16% 1.07M 2m17s\n",
      " 11700K .......... .......... .......... .......... .......... 16%  603K 2m17s\n",
      " 11750K .......... .......... .......... .......... .......... 16%  265K 2m17s\n",
      " 11800K .......... .......... .......... .......... .......... 16%  331K 2m17s\n",
      " 11850K .......... .......... .......... .......... .......... 17%  493K 2m17s\n",
      " 11900K .......... .......... .......... .......... .......... 17%  469K 2m17s\n",
      " 11950K .......... .......... .......... .......... .......... 17%  379K 2m17s\n",
      " 12000K .......... .......... .......... .......... .......... 17%  724K 2m17s\n",
      " 12050K .......... .......... .......... .......... .......... 17%  565K 2m16s\n",
      " 12100K .......... .......... .......... .......... .......... 17%  485K 2m16s\n",
      " 12150K .......... .......... .......... .......... .......... 17%  735K 2m16s\n",
      " 12200K .......... .......... .......... .......... .......... 17%  908K 2m15s\n",
      " 12250K .......... .......... .......... .......... .......... 17%  543K 2m15s\n",
      " 12300K .......... .......... .......... .......... .......... 17%  874K 2m15s\n",
      " 12350K .......... .......... .......... .......... .......... 17%  719K 2m14s\n",
      " 12400K .......... .......... .......... .......... .......... 17%  744K 2m14s\n",
      " 12450K .......... .......... .......... .......... .......... 17%  486K 2m14s\n",
      " 12500K .......... .......... .......... .......... .......... 17% 1.62M 2m13s\n",
      " 12550K .......... .......... .......... .......... .......... 18% 1010K 2m13s\n",
      " 12600K .......... .......... .......... .......... .......... 18%  737K 2m13s\n",
      " 12650K .......... .......... .......... .......... .......... 18%  481K 2m12s\n",
      " 12700K .......... .......... .......... .......... .......... 18% 1.71M 2m12s\n",
      " 12750K .......... .......... .......... .......... .......... 18% 1.32M 2m11s\n",
      " 12800K .......... .......... .......... .......... .......... 18%  161K 2m12s\n",
      " 12850K .......... .......... .......... .......... .......... 18% 2.01M 2m12s\n",
      " 12900K .......... .......... .......... .......... .......... 18% 94.8M 2m11s\n",
      " 12950K .......... .......... .......... .......... .......... 18%  100M 2m10s\n",
      " 13000K .......... .......... .......... .......... .......... 18%  370K 2m10s\n",
      " 13050K .......... .......... .......... .......... .......... 18%  504K 2m10s\n",
      " 13100K .......... .......... .......... .......... .......... 18% 1.14M 2m10s\n",
      " 13150K .......... .......... .......... .......... .......... 18%  344K 2m10s\n",
      " 13200K .......... .......... .......... .......... .......... 18% 3.05M 2m9s\n",
      " 13250K .......... .......... .......... .......... .......... 19%  486K 2m9s\n",
      " 13300K .......... .......... .......... .......... .......... 19%  350K 2m9s\n",
      " 13350K .......... .......... .......... .......... .......... 19% 1.13M 2m9s\n",
      " 13400K .......... .......... .......... .......... .......... 19%  789K 2m8s\n",
      " 13450K .......... .......... .......... .......... .......... 19%  529K 2m8s\n",
      " 13500K .......... .......... .......... .......... .......... 19% 1008K 2m8s\n",
      " 13550K .......... .......... .......... .......... .......... 19%  989K 2m7s\n",
      " 13600K .......... .......... .......... .......... .......... 19%  331K 2m8s\n",
      " 13650K .......... .......... .......... .......... .......... 19% 1.14M 2m7s\n",
      " 13700K .......... .......... .......... .......... .......... 19%  632K 2m7s\n",
      " 13750K .......... .......... .......... .......... .......... 19%  362K 2m7s\n",
      " 13800K .......... .......... .......... .......... .......... 19% 1.41M 2m6s\n",
      " 13850K .......... .......... .......... .......... .......... 19%  219K 2m7s\n",
      " 13900K .......... .......... .......... .......... .......... 19% 2.45M 2m6s\n",
      " 13950K .......... .......... .......... .......... .......... 20% 8.16M 2m6s\n",
      " 14000K .......... .......... .......... .......... .......... 20%  598K 2m6s\n",
      " 14050K .......... .......... .......... .......... .......... 20%  336K 2m6s\n",
      " 14100K .......... .......... .......... .......... .......... 20% 3.17M 2m5s\n",
      " 14150K .......... .......... .......... .......... .......... 20%  214K 2m5s\n",
      " 14200K .......... .......... .......... .......... .......... 20% 1.73M 2m5s\n",
      " 14250K .......... .......... .......... .......... .......... 20%  198K 2m5s\n",
      " 14300K .......... .......... .......... .......... .......... 20% 83.6M 2m5s\n",
      " 14350K .......... .......... .......... .......... .......... 20% 1.27M 2m4s\n",
      " 14400K .......... .......... .......... .......... .......... 20%  240K 2m5s\n",
      " 14450K .......... .......... .......... .......... .......... 20%  163K 2m5s\n",
      " 14500K .......... .......... .......... .......... .......... 20% 77.8M 2m5s\n",
      " 14550K .......... .......... .......... .......... .......... 20% 81.6K 2m7s\n",
      " 14600K .......... .......... .......... .......... .......... 20% 69.7M 2m6s\n",
      " 14650K .......... .......... .......... .......... .......... 21%  164K 2m7s\n",
      " 14700K .......... .......... .......... .......... .......... 21% 36.3M 2m6s\n",
      " 14750K .......... .......... .......... .......... .......... 21%  161K 2m7s\n",
      " 14800K .......... .......... .......... .......... .......... 21% 26.1M 2m6s\n",
      " 14850K .......... .......... .......... .......... .......... 21%  352K 2m6s\n",
      " 14900K .......... .......... .......... .......... .......... 21%  250K 2m6s\n",
      " 14950K .......... .......... .......... .......... .......... 21%  307K 2m6s\n",
      " 15000K .......... .......... .......... .......... .......... 21%  274K 2m7s\n",
      " 15050K .......... .......... .......... .......... .......... 21%  305K 2m7s\n",
      " 15100K .......... .......... .......... .......... .......... 21%  245K 2m7s\n",
      " 15150K .......... .......... .......... .......... .......... 21%  576K 2m7s\n",
      " 15200K .......... .......... .......... .......... .......... 21%  327K 2m7s\n",
      " 15250K .......... .......... .......... .......... .......... 21%  955K 2m6s\n",
      " 15300K .......... .......... .......... .......... .......... 21%  518K 2m6s\n",
      " 15350K .......... .......... .......... .......... .......... 22%  421K 2m6s\n",
      " 15400K .......... .......... .......... .......... .......... 22%  198K 2m6s\n",
      " 15450K .......... .......... .......... .......... .......... 22%  596K 2m6s\n",
      " 15500K .......... .......... .......... .......... .......... 22%  286K 2m6s\n",
      " 15550K .......... .......... .......... .......... .......... 22% 1.69M 2m6s\n",
      " 15600K .......... .......... .......... .......... .......... 22%  518K 2m6s\n",
      " 15650K .......... .......... .......... .......... .......... 22%  387K 2m6s\n",
      " 15700K .......... .......... .......... .......... .......... 22%  749K 2m5s\n",
      " 15750K .......... .......... .......... .......... .......... 22%  895K 2m5s\n",
      " 15800K .......... .......... .......... .......... .......... 22%  368K 2m5s\n",
      " 15850K .......... .......... .......... .......... .......... 22%  893K 2m5s\n",
      " 15900K .......... .......... .......... .......... .......... 22%  962K 2m4s\n",
      " 15950K .......... .......... .......... .......... .......... 22%  462K 2m4s\n",
      " 16000K .......... .......... .......... .......... .......... 22%  878K 2m4s\n",
      " 16050K .......... .......... .......... .......... .......... 23% 1.04M 2m3s\n",
      " 16100K .......... .......... .......... .......... .......... 23% 1.21M 2m3s\n",
      " 16150K .......... .......... .......... .......... .......... 23%  427K 2m3s\n",
      " 16200K .......... .......... .......... .......... .......... 23% 1.05M 2m3s\n",
      " 16250K .......... .......... .......... .......... .......... 23% 1.36M 2m2s\n",
      " 16300K .......... .......... .......... .......... .......... 23% 1012K 2m2s\n",
      " 16350K .......... .......... .......... .......... .......... 23% 1.05M 2m2s\n",
      " 16400K .......... .......... .......... .......... .......... 23%  322K 2m2s\n",
      " 16450K .......... .......... .......... .......... .......... 23% 1.69M 2m1s\n",
      " 16500K .......... .......... .......... .......... .......... 23%  371K 2m1s\n",
      " 16550K .......... .......... .......... .......... .......... 23% 18.8M 2m1s\n",
      " 16600K .......... .......... .......... .......... .......... 23% 43.3M 2m0s\n",
      " 16650K .......... .......... .......... .......... .......... 23%  363K 2m0s\n",
      " 16700K .......... .......... .......... .......... .......... 23% 1.46M 2m0s\n",
      " 16750K .......... .......... .......... .......... .......... 24%  478K 2m0s\n",
      " 16800K .......... .......... .......... .......... .......... 24%  344K 2m0s\n",
      " 16850K .......... .......... .......... .......... .......... 24%  620K 1m59s\n",
      " 16900K .......... .......... .......... .......... .......... 24% 2.07M 1m59s\n",
      " 16950K .......... .......... .......... .......... .......... 24%  430K 1m59s\n",
      " 17000K .......... .......... .......... .......... .......... 24% 1.42M 1m59s\n",
      " 17050K .......... .......... .......... .......... .......... 24%  577K 1m58s\n",
      " 17100K .......... .......... .......... .......... .......... 24% 3.09M 1m58s\n",
      " 17150K .......... .......... .......... .......... .......... 24%  326K 1m58s\n",
      " 17200K .......... .......... .......... .......... .......... 24%  483K 1m58s\n",
      " 17250K .......... .......... .......... .......... .......... 24%  450K 1m58s\n",
      " 17300K .......... .......... .......... .......... .......... 24% 1.28M 1m57s\n",
      " 17350K .......... .......... .......... .......... .......... 24%  234K 1m58s\n",
      " 17400K .......... .......... .......... .......... .......... 25%  151K 1m58s\n",
      " 17450K .......... .......... .......... .......... .......... 25% 37.2M 1m58s\n",
      " 17500K .......... .......... .......... .......... .......... 25% 81.5M 1m57s\n",
      " 17550K .......... .......... .......... .......... .......... 25%  103M 1m57s\n",
      " 17600K .......... .......... .......... .......... .......... 25%  247K 1m57s\n",
      " 17650K .......... .......... .......... .......... .......... 25%  348K 1m57s\n",
      " 17700K .......... .......... .......... .......... .......... 25%  420K 1m57s\n",
      " 17750K .......... .......... .......... .......... .......... 25%  617K 1m57s\n",
      " 17800K .......... .......... .......... .......... .......... 25%  365K 1m57s\n",
      " 17850K .......... .......... .......... .......... .......... 25% 1.31M 1m56s\n",
      " 17900K .......... .......... .......... .......... .......... 25%  511K 1m56s\n",
      " 17950K .......... .......... .......... .......... .......... 25%  401K 1m56s\n",
      " 18000K .......... .......... .......... .......... .......... 25%  195K 1m56s\n",
      " 18050K .......... .......... .......... .......... .......... 25% 67.2M 1m56s\n",
      " 18100K .......... .......... .......... .......... .......... 26% 69.4M 1m56s\n",
      " 18150K .......... .......... .......... .......... .......... 26%  489K 1m55s\n",
      " 18200K .......... .......... .......... .......... .......... 26%  266K 1m55s\n",
      " 18250K .......... .......... .......... .......... .......... 26%  783K 1m55s\n",
      " 18300K .......... .......... .......... .......... .......... 26%  329K 1m55s\n",
      " 18350K .......... .......... .......... .......... .......... 26% 1.21M 1m55s\n",
      " 18400K .......... .......... .......... .......... .......... 26%  502K 1m55s\n",
      " 18450K .......... .......... .......... .......... .......... 26%  415K 1m55s\n",
      " 18500K .......... .......... .......... .......... .......... 26% 1.53M 1m54s\n",
      " 18550K .......... .......... .......... .......... .......... 26%  255K 1m54s\n",
      " 18600K .......... .......... .......... .......... .......... 26%  444K 1m54s\n",
      " 18650K .......... .......... .......... .......... .......... 26%  490K 1m54s\n",
      " 18700K .......... .......... .......... .......... .......... 26%  532K 1m54s\n",
      " 18750K .......... .......... .......... .......... .......... 26%  393K 1m54s\n",
      " 18800K .......... .......... .......... .......... .......... 27%  170K 1m54s\n",
      " 18850K .......... .......... .......... .......... .......... 27%  133K 1m55s\n",
      " 18900K .......... .......... .......... .......... .......... 27% 50.0M 1m55s\n",
      " 18950K .......... .......... .......... .......... .......... 27%  163K 1m55s\n",
      " 19000K .......... .......... .......... .......... .......... 27%  363K 1m55s\n",
      " 19050K .......... .......... .......... .......... .......... 27%  333K 1m55s\n",
      " 19100K .......... .......... .......... .......... .......... 27%  605K 1m55s\n",
      " 19150K .......... .......... .......... .......... .......... 27%  357K 1m55s\n",
      " 19200K .......... .......... .......... .......... .......... 27%  611K 1m54s\n",
      " 19250K .......... .......... .......... .......... .......... 27%  468K 1m54s\n",
      " 19300K .......... .......... .......... .......... .......... 27%  520K 1m54s\n",
      " 19350K .......... .......... .......... .......... .......... 27%  885K 1m54s\n",
      " 19400K .......... .......... .......... .......... .......... 27% 1.02M 1m54s\n",
      " 19450K .......... .......... .......... .......... .......... 27%  438K 1m54s\n",
      " 19500K .......... .......... .......... .......... .......... 28% 1.13M 1m53s\n",
      " 19550K .......... .......... .......... .......... .......... 28%  617K 1m53s\n",
      " 19600K .......... .......... .......... .......... .......... 28%  398K 1m53s\n",
      " 19650K .......... .......... .......... .......... .......... 28% 1.73M 1m53s\n",
      " 19700K .......... .......... .......... .......... .......... 28% 1.03M 1m52s\n",
      " 19750K .......... .......... .......... .......... .......... 28% 1.01M 1m52s\n",
      " 19800K .......... .......... .......... .......... .......... 28%  660K 1m52s\n",
      " 19850K .......... .......... .......... .......... .......... 28%  892K 1m52s\n",
      " 19900K .......... .......... .......... .......... .......... 28%  520K 1m51s\n",
      " 19950K .......... .......... .......... .......... .......... 28%  501K 1m51s\n",
      " 20000K .......... .......... .......... .......... .......... 28%  475K 1m51s\n",
      " 20050K .......... .......... .......... .......... .......... 28%  703K 1m51s\n",
      " 20100K .......... .......... .......... .......... .......... 28%  152K 1m51s\n",
      " 20150K .......... .......... .......... .......... .......... 28% 68.7M 1m51s\n",
      " 20200K .......... .......... .......... .......... .......... 29% 67.5M 1m51s\n",
      " 20250K .......... .......... .......... .......... .......... 29% 1.44M 1m50s\n",
      " 20300K .......... .......... .......... .......... .......... 29%  400K 1m50s\n",
      " 20350K .......... .......... .......... .......... .......... 29%  322K 1m50s\n",
      " 20400K .......... .......... .......... .......... .......... 29%  521K 1m50s\n",
      " 20450K .......... .......... .......... .......... .......... 29% 1.77M 1m50s\n",
      " 20500K .......... .......... .......... .......... .......... 29%  388K 1m50s\n",
      " 20550K .......... .......... .......... .......... .......... 29%  406K 1m50s\n",
      " 20600K .......... .......... .......... .......... .......... 29%  390K 1m50s\n",
      " 20650K .......... .......... .......... .......... .......... 29%  445K 1m49s\n",
      " 20700K .......... .......... .......... .......... .......... 29% 1.12M 1m49s\n",
      " 20750K .......... .......... .......... .......... .......... 29%  522K 1m49s\n",
      " 20800K .......... .......... .......... .......... .......... 29%  419K 1m49s\n",
      " 20850K .......... .......... .......... .......... .......... 29%  620K 1m49s\n",
      " 20900K .......... .......... .......... .......... .......... 30%  290K 1m49s\n",
      " 20950K .......... .......... .......... .......... .......... 30%  624K 1m49s\n",
      " 21000K .......... .......... .......... .......... .......... 30%  290K 1m49s\n",
      " 21050K .......... .......... .......... .......... .......... 30%  649K 1m48s\n",
      " 21100K .......... .......... .......... .......... .......... 30%  637K 1m48s\n",
      " 21150K .......... .......... .......... .......... .......... 30%  315K 1m48s\n",
      " 21200K .......... .......... .......... .......... .......... 30%  857K 1m48s\n",
      " 21250K .......... .......... .......... .......... .......... 30%  421K 1m48s\n",
      " 21300K .......... .......... .......... .......... .......... 30%  502K 1m48s\n",
      " 21350K .......... .......... .......... .......... .......... 30% 1.52M 1m48s\n",
      " 21400K .......... .......... .......... .......... .......... 30%  891K 1m47s\n",
      " 21450K .......... .......... .......... .......... .......... 30%  543K 1m47s\n",
      " 21500K .......... .......... .......... .......... .......... 30%  667K 1m47s\n",
      " 21550K .......... .......... .......... .......... .......... 30%  614K 1m47s\n",
      " 21600K .......... .......... .......... .......... .......... 31%  726K 1m47s\n",
      " 21650K .......... .......... .......... .......... .......... 31% 1.45M 1m46s\n",
      " 21700K .......... .......... .......... .......... .......... 31%  705K 1m46s\n",
      " 21750K .......... .......... .......... .......... .......... 31% 1.48M 1m46s\n",
      " 21800K .......... .......... .......... .......... .......... 31%  575K 1m46s\n",
      " 21850K .......... .......... .......... .......... .......... 31% 1003K 1m45s\n",
      " 21900K .......... .......... .......... .......... .......... 31%  649K 1m45s\n",
      " 21950K .......... .......... .......... .......... .......... 31% 1.36M 1m45s\n",
      " 22000K .......... .......... .......... .......... .......... 31%  878K 1m45s\n",
      " 22050K .......... .......... .......... .......... .......... 31%  737K 1m45s\n",
      " 22100K .......... .......... .......... .......... .......... 31%  552K 1m44s\n",
      " 22150K .......... .......... .......... .......... .......... 31%  544K 1m44s\n",
      " 22200K .......... .......... .......... .......... .......... 31%  502K 1m44s\n",
      " 22250K .......... .......... .......... .......... .......... 31%  627K 1m44s\n",
      " 22300K .......... .......... .......... .......... .......... 32%  612K 1m44s\n",
      " 22350K .......... .......... .......... .......... .......... 32%  727K 1m44s\n",
      " 22400K .......... .......... .......... .......... .......... 32%  600K 1m43s\n",
      " 22450K .......... .......... .......... .......... .......... 32%  749K 1m43s\n",
      " 22500K .......... .......... .......... .......... .......... 32%  494K 1m43s\n",
      " 22550K .......... .......... .......... .......... .......... 32%  419K 1m43s\n",
      " 22600K .......... .......... .......... .......... .......... 32%  354K 1m43s\n",
      " 22650K .......... .......... .......... .......... .......... 32%  558K 1m43s\n",
      " 22700K .......... .......... .......... .......... .......... 32%  587K 1m43s\n",
      " 22750K .......... .......... .......... .......... .......... 32%  650K 1m42s\n",
      " 22800K .......... .......... .......... .......... .......... 32%  489K 1m42s\n",
      " 22850K .......... .......... .......... .......... .......... 32%  566K 1m42s\n",
      " 22900K .......... .......... .......... .......... .......... 32%  295K 1m42s\n",
      " 22950K .......... .......... .......... .......... .......... 32%  333K 1m42s\n",
      " 23000K .......... .......... .......... .......... .......... 33%  593K 1m42s\n",
      " 23050K .......... .......... .......... .......... .......... 33%  462K 1m42s\n",
      " 23100K .......... .......... .......... .......... .......... 33%  524K 1m42s\n",
      " 23150K .......... .......... .......... .......... .......... 33%  718K 1m42s\n",
      " 23200K .......... .......... .......... .......... .......... 33%  571K 1m41s\n",
      " 23250K .......... .......... .......... .......... .......... 33%  159K 1m42s\n",
      " 23300K .......... .......... .......... .......... .......... 33%  189M 1m41s\n",
      " 23350K .......... .......... .......... .......... .......... 33%  625K 1m41s\n",
      " 23400K .......... .......... .......... .......... .......... 33%  288K 1m41s\n",
      " 23450K .......... .......... .......... .......... .......... 33%  355K 1m41s\n",
      " 23500K .......... .......... .......... .......... .......... 33%  603K 1m41s\n",
      " 23550K .......... .......... .......... .......... .......... 33%  386K 1m41s\n",
      " 23600K .......... .......... .......... .......... .......... 33%  458K 1m41s\n",
      " 23650K .......... .......... .......... .......... .......... 33%  515K 1m41s\n",
      " 23700K .......... .......... .......... .......... .......... 34%  912K 1m41s\n",
      " 23750K .......... .......... .......... .......... .......... 34%  497K 1m40s\n",
      " 23800K .......... .......... .......... .......... .......... 34%  329K 1m40s\n",
      " 23850K .......... .......... .......... .......... .......... 34%  256K 1m40s\n",
      " 23900K .......... .......... .......... .......... .......... 34%  567K 1m40s\n",
      " 23950K .......... .......... .......... .......... .......... 34%  499K 1m40s\n",
      " 24000K .......... .......... .......... .......... .......... 34%  402K 1m40s\n",
      " 24050K .......... .......... .......... .......... .......... 34%  571K 1m40s\n",
      " 24100K .......... .......... .......... .......... .......... 34%  479K 1m40s\n",
      " 24150K .......... .......... .......... .......... .......... 34%  821K 1m40s\n",
      " 24200K .......... .......... .......... .......... .......... 34%  449K 99s\n",
      " 24250K .......... .......... .......... .......... .......... 34%  629K 99s\n",
      " 24300K .......... .......... .......... .......... .......... 34%  830K 99s\n",
      " 24350K .......... .......... .......... .......... .......... 34% 1.24M 99s\n",
      " 24400K .......... .......... .......... .......... .......... 35%  532K 99s\n",
      " 24450K .......... .......... .......... .......... .......... 35%  423K 99s\n",
      " 24500K .......... .......... .......... .......... .......... 35%  414K 99s\n",
      " 24550K .......... .......... .......... .......... .......... 35%  406K 98s\n",
      " 24600K .......... .......... .......... .......... .......... 35%  895K 98s\n",
      " 24650K .......... .......... .......... .......... .......... 35%  337K 98s\n",
      " 24700K .......... .......... .......... .......... .......... 35%  882K 98s\n",
      " 24750K .......... .......... .......... .......... .......... 35%  483K 98s\n",
      " 24800K .......... .......... .......... .......... .......... 35%  530K 98s\n",
      " 24850K .......... .......... .......... .......... .......... 35%  494K 98s\n",
      " 24900K .......... .......... .......... .......... .......... 35%  244K 98s\n",
      " 24950K .......... .......... .......... .......... .......... 35%  485K 98s\n",
      " 25000K .......... .......... .......... .......... .......... 35%  491K 97s\n",
      " 25050K .......... .......... .......... .......... .......... 35%  423K 97s\n",
      " 25100K .......... .......... .......... .......... .......... 36%  442K 97s\n",
      " 25150K .......... .......... .......... .......... .......... 36%  661K 97s\n",
      " 25200K .......... .......... .......... .......... .......... 36%  781K 97s\n",
      " 25250K .......... .......... .......... .......... .......... 36%  442K 97s\n",
      " 25300K .......... .......... .......... .......... .......... 36%  804K 97s\n",
      " 25350K .......... .......... .......... .......... .......... 36%  658K 96s\n",
      " 25400K .......... .......... .......... .......... .......... 36%  689K 96s\n",
      " 25450K .......... .......... .......... .......... .......... 36%  822K 96s\n",
      " 25500K .......... .......... .......... .......... .......... 36%  786K 96s\n",
      " 25550K .......... .......... .......... .......... .......... 36%  494K 96s\n",
      " 25600K .......... .......... .......... .......... .......... 36%  334K 96s\n",
      " 25650K .......... .......... .......... .......... .......... 36%  481K 96s\n",
      " 25700K .......... .......... .......... .......... .......... 36%  500K 96s\n",
      " 25750K .......... .......... .......... .......... .......... 36%  480K 95s\n",
      " 25800K .......... .......... .......... .......... .......... 37%  447K 95s\n",
      " 25850K .......... .......... .......... .......... .......... 37%  782K 95s\n",
      " 25900K .......... .......... .......... .......... .......... 37%  590K 95s\n",
      " 25950K .......... .......... .......... .......... .......... 37%  782K 95s\n",
      " 26000K .......... .......... .......... .......... .......... 37%  433K 95s\n",
      " 26050K .......... .......... .......... .......... .......... 37%  697K 95s\n",
      " 26100K .......... .......... .......... .......... .......... 37% 1.11M 94s\n",
      " 26150K .......... .......... .......... .......... .......... 37%  564K 94s\n",
      " 26200K .......... .......... .......... .......... .......... 37%  695K 94s\n",
      " 26250K .......... .......... .......... .......... .......... 37%  992K 94s\n",
      " 26300K .......... .......... .......... .......... .......... 37%  816K 94s\n",
      " 26350K .......... .......... .......... .......... .......... 37%  726K 93s\n",
      " 26400K .......... .......... .......... .......... .......... 37%  635K 93s\n",
      " 26450K .......... .......... .......... .......... .......... 37%  604K 93s\n",
      " 26500K .......... .......... .......... .......... .......... 38%  976K 93s\n",
      " 26550K .......... .......... .......... .......... .......... 38% 1.09M 93s\n",
      " 26600K .......... .......... .......... .......... .......... 38% 1.01M 93s\n",
      " 26650K .......... .......... .......... .......... .......... 38% 1.20M 92s\n",
      " 26700K .......... .......... .......... .......... .......... 38%  511K 92s\n",
      " 26750K .......... .......... .......... .......... .......... 38%  830K 92s\n",
      " 26800K .......... .......... .......... .......... .......... 38%  562K 92s\n",
      " 26850K .......... .......... .......... .......... .......... 38%  521K 92s\n",
      " 26900K .......... .......... .......... .......... .......... 38% 46.6M 91s\n",
      " 26950K .......... .......... .......... .......... .......... 38% 95.7K 92s\n",
      " 27000K .......... .......... .......... .......... .......... 38% 99.0K 93s\n",
      " 27050K .......... .......... .......... .......... .......... 38% 1.38M 92s\n",
      " 27100K .......... .......... .......... .......... .......... 38%  230K 92s\n",
      " 27150K .......... .......... .......... .......... .......... 38% 1.49M 92s\n",
      " 27200K .......... .......... .......... .......... .......... 39% 12.5M 92s\n",
      " 27250K .......... .......... .......... .......... .......... 39% 58.9M 92s\n",
      " 27300K .......... .......... .......... .......... .......... 39% 1.38M 91s\n",
      " 27350K .......... .......... .......... .......... .......... 39%  267K 91s\n",
      " 27400K .......... .......... .......... .......... .......... 39%  890K 91s\n",
      " 27450K .......... .......... .......... .......... .......... 39%  904K 91s\n",
      " 27500K .......... .......... .......... .......... .......... 39%  350K 91s\n",
      " 27550K .......... .......... .......... .......... .......... 39% 1.67M 91s\n",
      " 27600K .......... .......... .......... .......... .......... 39%  620K 91s\n",
      " 27650K .......... .......... .......... .......... .......... 39%  253K 91s\n",
      " 27700K .......... .......... .......... .......... .......... 39%  317K 91s\n",
      " 27750K .......... .......... .......... .......... .......... 39%  505K 90s\n",
      " 27800K .......... .......... .......... .......... .......... 39% 1.09M 90s\n",
      " 27850K .......... .......... .......... .......... .......... 39%  312K 90s\n",
      " 27900K .......... .......... .......... .......... .......... 40% 1.02M 90s\n",
      " 27950K .......... .......... .......... .......... .......... 40% 1016K 90s\n",
      " 28000K .......... .......... .......... .......... .......... 40%  319K 90s\n",
      " 28050K .......... .......... .......... .......... .......... 40% 1.03M 90s\n",
      " 28100K .......... .......... .......... .......... .......... 40% 1.19M 89s\n",
      " 28150K .......... .......... .......... .......... .......... 40%  309K 89s\n",
      " 28200K .......... .......... .......... .......... .......... 40% 1.02M 89s\n",
      " 28250K .......... .......... .......... .......... .......... 40% 1.22M 89s\n",
      " 28300K .......... .......... .......... .......... .......... 40% 1.02M 89s\n",
      " 28350K .......... .......... .......... .......... .......... 40%  414K 89s\n",
      " 28400K .......... .......... .......... .......... .......... 40%  547K 89s\n",
      " 28450K .......... .......... .......... .......... .......... 40%  310K 89s\n",
      " 28500K .......... .......... .......... .......... .......... 40%  145K 89s\n",
      " 28550K .......... .......... .......... .......... .......... 40% 54.3M 89s\n",
      " 28600K .......... .......... .......... .......... .......... 41% 67.2M 88s\n",
      " 28650K .......... .......... .......... .......... .......... 41%  503K 88s\n",
      " 28700K .......... .......... .......... .......... .......... 41%  353K 88s\n",
      " 28750K .......... .......... .......... .......... .......... 41%  205K 88s\n",
      " 28800K .......... .......... .......... .......... .......... 41% 5.40M 88s\n",
      " 28850K .......... .......... .......... .......... .......... 41%  189K 88s\n",
      " 28900K .......... .......... .......... .......... .......... 41% 2.24M 88s\n",
      " 28950K .......... .......... .......... .......... .......... 41%  503K 88s\n",
      " 29000K .......... .......... .......... .......... .......... 41%  333K 88s\n",
      " 29050K .......... .......... .......... .......... .......... 41%  487K 88s\n",
      " 29100K .......... .......... .......... .......... .......... 41%  325K 87s\n",
      " 29150K .......... .......... .......... .......... .......... 41%  518K 87s\n",
      " 29200K .......... .......... .......... .......... .......... 41%  312K 87s\n",
      " 29250K .......... .......... .......... .......... .......... 41%  551K 87s\n",
      " 29300K .......... .......... .......... .......... .......... 42%  313K 87s\n",
      " 29350K .......... .......... .......... .......... .......... 42% 1.29M 87s\n",
      " 29400K .......... .......... .......... .......... .......... 42%  738K 87s\n",
      " 29450K .......... .......... .......... .......... .......... 42%  134K 87s\n",
      " 29500K .......... .......... .......... .......... .......... 42% 70.9M 87s\n",
      " 29550K .......... .......... .......... .......... .......... 42% 77.6M 87s\n",
      " 29600K .......... .......... .......... .......... .......... 42%  816K 86s\n",
      " 29650K .......... .......... .......... .......... .......... 42%  498K 86s\n",
      " 29700K .......... .......... .......... .......... .......... 42%  328K 86s\n",
      " 29750K .......... .......... .......... .......... .......... 42%  979K 86s\n",
      " 29800K .......... .......... .......... .......... .......... 42%  336K 86s\n",
      " 29850K .......... .......... .......... .......... .......... 42%  894K 86s\n",
      " 29900K .......... .......... .......... .......... .......... 42%  572K 86s\n",
      " 29950K .......... .......... .......... .......... .......... 42%  329K 86s\n",
      " 30000K .......... .......... .......... .......... .......... 43%  235K 86s\n",
      " 30050K .......... .......... .......... .......... .......... 43%  873K 85s\n",
      " 30100K .......... .......... .......... .......... .......... 43%  607K 85s\n",
      " 30150K .......... .......... .......... .......... .......... 43%  332K 85s\n",
      " 30200K .......... .......... .......... .......... .......... 43%  677K 85s\n",
      " 30250K .......... .......... .......... .......... .......... 43%  704K 85s\n",
      " 30300K .......... .......... .......... .......... .......... 43%  461K 85s\n",
      " 30350K .......... .......... .......... .......... .......... 43%  309K 85s\n",
      " 30400K .......... .......... .......... .......... .......... 43%  369K 85s\n",
      " 30450K .......... .......... .......... .......... .......... 43%  436K 85s\n",
      " 30500K .......... .......... .......... .......... .......... 43%  537K 85s\n",
      " 30550K .......... .......... .......... .......... .......... 43%  576K 84s\n",
      " 30600K .......... .......... .......... .......... .......... 43%  402K 84s\n",
      " 30650K .......... .......... .......... .......... .......... 43%  628K 84s\n",
      " 30700K .......... .......... .......... .......... .......... 44% 1024K 84s\n",
      " 30750K .......... .......... .......... .......... .......... 44%  702K 84s\n",
      " 30800K .......... .......... .......... .......... .......... 44%  540K 84s\n",
      " 30850K .......... .......... .......... .......... .......... 44% 1.10M 84s\n",
      " 30900K .......... .......... .......... .......... .......... 44%  564K 83s\n",
      " 30950K .......... .......... .......... .......... .......... 44%  634K 83s\n",
      " 31000K .......... .......... .......... .......... .......... 44%  364K 83s\n",
      " 31050K .......... .......... .......... .......... .......... 44%  370K 83s\n",
      " 31100K .......... .......... .......... .......... .......... 44%  731K 83s\n",
      " 31150K .......... .......... .......... .......... .......... 44%  571K 83s\n",
      " 31200K .......... .......... .......... .......... .......... 44%  359K 83s\n",
      " 31250K .......... .......... .......... .......... .......... 44%  616K 83s\n",
      " 31300K .......... .......... .......... .......... .......... 44% 1.04M 82s\n",
      " 31350K .......... .......... .......... .......... .......... 44%  601K 82s\n",
      " 31400K .......... .......... .......... .......... .......... 45%  539K 82s\n",
      " 31450K .......... .......... .......... .......... .......... 45%  833K 82s\n",
      " 31500K .......... .......... .......... .......... .......... 45%  656K 82s\n",
      " 31550K .......... .......... .......... .......... .......... 45%  629K 82s\n",
      " 31600K .......... .......... .......... .......... .......... 45%  686K 82s\n",
      " 31650K .......... .......... .......... .......... .......... 45% 2.04M 81s\n",
      " 31700K .......... .......... .......... .......... .......... 45%  629K 81s\n",
      " 31750K .......... .......... .......... .......... .......... 45% 1.26M 81s\n",
      " 31800K .......... .......... .......... .......... .......... 45%  668K 81s\n",
      " 31850K .......... .......... .......... .......... .......... 45% 1.02M 81s\n",
      " 31900K .......... .......... .......... .......... .......... 45%  991K 81s\n",
      " 31950K .......... .......... .......... .......... .......... 45%  828K 80s\n",
      " 32000K .......... .......... .......... .......... .......... 45%  732K 80s\n",
      " 32050K .......... .......... .......... .......... .......... 45%  730K 80s\n",
      " 32100K .......... .......... .......... .......... .......... 46% 3.18M 80s\n",
      " 32150K .......... .......... .......... .......... .......... 46%  574K 80s\n",
      " 32200K .......... .......... .......... .......... .......... 46% 1.46M 80s\n",
      " 32250K .......... .......... .......... .......... .......... 46% 1.33M 79s\n",
      " 32300K .......... .......... .......... .......... .......... 46%  581K 79s\n",
      " 32350K .......... .......... .......... .......... .......... 46%  611K 79s\n",
      " 32400K .......... .......... .......... .......... .......... 46%  488K 79s\n",
      " 32450K .......... .......... .......... .......... .......... 46%  659K 79s\n",
      " 32500K .......... .......... .......... .......... .......... 46%  558K 79s\n",
      " 32550K .......... .......... .......... .......... .......... 46%  749K 79s\n",
      " 32600K .......... .......... .......... .......... .......... 46%  544K 78s\n",
      " 32650K .......... .......... .......... .......... .......... 46%  959K 78s\n",
      " 32700K .......... .......... .......... .......... .......... 46%  965K 78s\n",
      " 32750K .......... .......... .......... .......... .......... 46%  520K 78s\n",
      " 32800K .......... .......... .......... .......... .......... 47%  862K 78s\n",
      " 32850K .......... .......... .......... .......... .......... 47% 1.05M 78s\n",
      " 32900K .......... .......... .......... .......... .......... 47%  920K 78s\n",
      " 32950K .......... .......... .......... .......... .......... 47%  616K 77s\n",
      " 33000K .......... .......... .......... .......... .......... 47%  714K 77s\n",
      " 33050K .......... .......... .......... .......... .......... 47%  409K 77s\n",
      " 33100K .......... .......... .......... .......... .......... 47%  363K 77s\n",
      " 33150K .......... .......... .......... .......... .......... 47%  716K 77s\n",
      " 33200K .......... .......... .......... .......... .......... 47%  445K 77s\n",
      " 33250K .......... .......... .......... .......... .......... 47%  716K 77s\n",
      " 33300K .......... .......... .......... .......... .......... 47%  639K 77s\n",
      " 33350K .......... .......... .......... .......... .......... 47%  737K 76s\n",
      " 33400K .......... .......... .......... .......... .......... 47%  306K 76s\n",
      " 33450K .......... .......... .......... .......... .......... 47%  325K 76s\n",
      " 33500K .......... .......... .......... .......... .......... 48%  485K 76s\n",
      " 33550K .......... .......... .......... .......... .......... 48%  368K 76s\n",
      " 33600K .......... .......... .......... .......... .......... 48%  452K 76s\n",
      " 33650K .......... .......... .......... .......... .......... 48%  986K 76s\n",
      " 33700K .......... .......... .......... .......... .......... 48%  487K 76s\n",
      " 33750K .......... .......... .......... .......... .......... 48%  522K 76s\n",
      " 33800K .......... .......... .......... .......... .......... 48%  960K 76s\n",
      " 33850K .......... .......... .......... .......... .......... 48%  498K 75s\n",
      " 33900K .......... .......... .......... .......... .......... 48%  928K 75s\n",
      " 33950K .......... .......... .......... .......... .......... 48%  912K 75s\n",
      " 34000K .......... .......... .......... .......... .......... 48%  578K 75s\n",
      " 34050K .......... .......... .......... .......... .......... 48%  817K 75s\n",
      " 34100K .......... .......... .......... .......... .......... 48%  573K 75s\n",
      " 34150K .......... .......... .......... .......... .......... 48% 1.54M 74s\n",
      " 34200K .......... .......... .......... .......... .......... 49%  694K 74s\n",
      " 34250K .......... .......... .......... .......... .......... 49%  768K 74s\n",
      " 34300K .......... .......... .......... .......... .......... 49%  490K 74s\n",
      " 34350K .......... .......... .......... .......... .......... 49%  515K 74s\n",
      " 34400K .......... .......... .......... .......... .......... 49%  347K 74s\n",
      " 34450K .......... .......... .......... .......... .......... 49%  966K 74s\n",
      " 34500K .......... .......... .......... .......... .......... 49%  430K 74s\n",
      " 34550K .......... .......... .......... .......... .......... 49%  658K 74s\n",
      " 34600K .......... .......... .......... .......... .......... 49%  891K 73s\n",
      " 34650K .......... .......... .......... .......... .......... 49%  837K 73s\n",
      " 34700K .......... .......... .......... .......... .......... 49%  432K 73s\n",
      " 34750K .......... .......... .......... .......... .......... 49% 1.10M 73s\n",
      " 34800K .......... .......... .......... .......... .......... 49%  657K 73s\n",
      " 34850K .......... .......... .......... .......... .......... 50%  596K 73s\n",
      " 34900K .......... .......... .......... .......... .......... 50% 1011K 73s\n",
      " 34950K .......... .......... .......... .......... .......... 50% 1.15M 72s\n",
      " 35000K .......... .......... .......... .......... .......... 50%  733K 72s\n",
      " 35050K .......... .......... .......... .......... .......... 50%  901K 72s\n",
      " 35100K .......... .......... .......... .......... .......... 50%  445K 72s\n",
      " 35150K .......... .......... .......... .......... .......... 50%  522K 72s\n",
      " 35200K .......... .......... .......... .......... .......... 50%  366K 72s\n",
      " 35250K .......... .......... .......... .......... .......... 50%  626K 72s\n",
      " 35300K .......... .......... .......... .......... .......... 50%  195K 72s\n",
      " 35350K .......... .......... .......... .......... .......... 50% 58.6M 72s\n",
      " 35400K .......... .......... .......... .......... .......... 50% 3.47M 71s\n",
      " 35450K .......... .......... .......... .......... .......... 50% 1.02M 71s\n",
      " 35500K .......... .......... .......... .......... .......... 50%  444K 71s\n",
      " 35550K .......... .......... .......... .......... .......... 51%  535K 71s\n",
      " 35600K .......... .......... .......... .......... .......... 51%  534K 71s\n",
      " 35650K .......... .......... .......... .......... .......... 51%  356K 71s\n",
      " 35700K .......... .......... .......... .......... .......... 51% 2.04M 71s\n",
      " 35750K .......... .......... .......... .......... .......... 51%  585K 71s\n",
      " 35800K .......... .......... .......... .......... .......... 51%  663K 70s\n",
      " 35850K .......... .......... .......... .......... .......... 51%  588K 70s\n",
      " 35900K .......... .......... .......... .......... .......... 51% 1.06M 70s\n",
      " 35950K .......... .......... .......... .......... .......... 51%  305K 70s\n",
      " 36000K .......... .......... .......... .......... .......... 51%  563K 70s\n",
      " 36050K .......... .......... .......... .......... .......... 51%  391K 70s\n",
      " 36100K .......... .......... .......... .......... .......... 51%  407K 70s\n",
      " 36150K .......... .......... .......... .......... .......... 51% 1.09M 70s\n",
      " 36200K .......... .......... .......... .......... .......... 51%  406K 70s\n",
      " 36250K .......... .......... .......... .......... .......... 52%  588K 69s\n",
      " 36300K .......... .......... .......... .......... .......... 52% 1.12M 69s\n",
      " 36350K .......... .......... .......... .......... .......... 52%  542K 69s\n",
      " 36400K .......... .......... .......... .......... .......... 52%  606K 69s\n",
      " 36450K .......... .......... .......... .......... .......... 52%  674K 69s\n",
      " 36500K .......... .......... .......... .......... .......... 52% 1.04M 69s\n",
      " 36550K .......... .......... .......... .......... .......... 52%  625K 69s\n",
      " 36600K .......... .......... .......... .......... .......... 52%  130K 69s\n",
      " 36650K .......... .......... .......... .......... .......... 52% 56.2M 69s\n",
      " 36700K .......... .......... .......... .......... .......... 52% 76.8M 68s\n",
      " 36750K .......... .......... .......... .......... .......... 52% 69.1M 68s\n",
      " 36800K .......... .......... .......... .......... .......... 52%  502K 68s\n",
      " 36850K .......... .......... .......... .......... .......... 52%  331K 68s\n",
      " 36900K .......... .......... .......... .......... .......... 52% 1.30M 68s\n",
      " 36950K .......... .......... .......... .......... .......... 53%  354K 68s\n",
      " 37000K .......... .......... .......... .......... .......... 53%  618K 68s\n",
      " 37050K .......... .......... .......... .......... .......... 53%  292K 68s\n",
      " 37100K .......... .......... .......... .......... .......... 53%  633K 67s\n",
      " 37150K .......... .......... .......... .......... .......... 53%  725K 67s\n",
      " 37200K .......... .......... .......... .......... .......... 53%  360K 67s\n",
      " 37250K .......... .......... .......... .......... .......... 53%  226K 67s\n",
      " 37300K .......... .......... .......... .......... .......... 53%  260K 67s\n",
      " 37350K .......... .......... .......... .......... .......... 53%  692K 67s\n",
      " 37400K .......... .......... .......... .......... .......... 53%  273K 67s\n",
      " 37450K .......... .......... .......... .......... .......... 53%  735K 67s\n",
      " 37500K .......... .......... .......... .......... .......... 53% 1.43M 67s\n",
      " 37550K .......... .......... .......... .......... .......... 53%  331K 67s\n",
      " 37600K .......... .......... .......... .......... .......... 53%  410K 67s\n",
      " 37650K .......... .......... .......... .......... .......... 54%  244K 67s\n",
      " 37700K .......... .......... .......... .......... .......... 54%  789K 66s\n",
      " 37750K .......... .......... .......... .......... .......... 54%  318K 66s\n",
      " 37800K .......... .......... .......... .......... .......... 54%  540K 66s\n",
      " 37850K .......... .......... .......... .......... .......... 54%  247K 66s\n",
      " 37900K .......... .......... .......... .......... .......... 54% 98.3K 67s\n",
      " 37950K .......... .......... .......... .......... .......... 54% 23.7M 66s\n",
      " 38000K .......... .......... .......... .......... .......... 54% 81.4K 67s\n",
      " 38050K .......... .......... .......... .......... .......... 54% 45.1M 66s\n",
      " 38100K .......... .......... .......... .......... .......... 54% 50.6M 66s\n",
      " 38150K .......... .......... .......... .......... .......... 54%  162K 66s\n",
      " 38200K .......... .......... .......... .......... .......... 54% 1.71M 66s\n",
      " 38250K .......... .......... .......... .......... .......... 54%  182K 66s\n",
      " 38300K .......... .......... .......... .......... .......... 54%  856K 66s\n",
      " 38350K .......... .......... .......... .......... .......... 55%  250K 66s\n",
      " 38400K .......... .......... .......... .......... .......... 55%  969K 66s\n",
      " 38450K .......... .......... .......... .......... .......... 55%  257K 66s\n",
      " 38500K .......... .......... .......... .......... .......... 55%  909K 66s\n",
      " 38550K .......... .......... .......... .......... .......... 55%  920K 66s\n",
      " 38600K .......... .......... .......... .......... .......... 55%  243K 66s\n",
      " 38650K .......... .......... .......... .......... .......... 55%  122K 66s\n",
      " 38700K .......... .......... .......... .......... .......... 55% 27.6M 65s\n",
      " 38750K .......... .......... .......... .......... .......... 55% 81.5K 66s\n",
      " 38800K .......... .......... .......... .......... .......... 55% 47.3M 66s\n",
      " 38850K .......... .......... .......... .......... .......... 55% 59.5M 65s\n",
      " 38900K .......... .......... .......... .......... .......... 55%  164K 65s\n",
      " 38950K .......... .......... .......... .......... .......... 55% 4.34M 65s\n",
      " 39000K .......... .......... .......... .......... .......... 55%  192K 65s\n",
      " 39050K .......... .......... .......... .......... .......... 56%  147K 65s\n",
      " 39100K .......... .......... .......... .......... .......... 56% 22.2M 65s\n",
      " 39150K .......... .......... .......... .......... .......... 56%  163K 65s\n",
      " 39200K .......... .......... .......... .......... .......... 56% 35.8M 65s\n",
      " 39250K .......... .......... .......... .......... .......... 56%  201K 65s\n",
      " 39300K .......... .......... .......... .......... .......... 56%  368K 65s\n",
      " 39350K .......... .......... .......... .......... .......... 56%  315K 65s\n",
      " 39400K .......... .......... .......... .......... .......... 56%  171K 65s\n",
      " 39450K .......... .......... .......... .......... .......... 56%  128K 65s\n",
      " 39500K .......... .......... .......... .......... .......... 56%  198K 65s\n",
      " 39550K .......... .......... .......... .......... .......... 56%  475K 65s\n",
      " 39600K .......... .......... .......... .......... .......... 56%  195K 65s\n",
      " 39650K .......... .......... .......... .......... .......... 56%  163K 65s\n",
      " 39700K .......... .......... .......... .......... .......... 56%  160K 65s\n",
      " 39750K .......... .......... .......... .......... .......... 57% 1.98M 65s\n",
      " 39800K .......... .......... .......... .......... .......... 57%  176K 65s\n",
      " 39850K .......... .......... .......... .......... .......... 57%  439K 65s\n",
      " 39900K .......... .......... .......... .......... .......... 57%  333K 65s\n",
      " 39950K .......... .......... .......... .......... .......... 57%  944K 65s\n",
      " 40000K .......... .......... .......... .......... .......... 57%  268K 64s\n",
      " 40050K .......... .......... .......... .......... .......... 57%  832K 64s\n",
      " 40100K .......... .......... .......... .......... .......... 57%  943K 64s\n",
      " 40150K .......... .......... .......... .......... .......... 57%  360K 64s\n",
      " 40200K .......... .......... .......... .......... .......... 57%  865K 64s\n",
      " 40250K .......... .......... .......... .......... .......... 57%  967K 64s\n",
      " 40300K .......... .......... .......... .......... .......... 57%  351K 64s\n",
      " 40350K .......... .......... .......... .......... .......... 57% 3.42M 64s\n",
      " 40400K .......... .......... .......... .......... .......... 57%  167K 64s\n",
      " 40450K .......... .......... .......... .......... .......... 58% 45.6M 63s\n",
      " 40500K .......... .......... .......... .......... .......... 58% 57.9M 63s\n",
      " 40550K .......... .......... .......... .......... .......... 58% 55.3M 63s\n",
      " 40600K .......... .......... .......... .......... .......... 58%  270K 63s\n",
      " 40650K .......... .......... .......... .......... .......... 58%  792K 63s\n",
      " 40700K .......... .......... .......... .......... .......... 58%  259K 63s\n",
      " 40750K .......... .......... .......... .......... .......... 58% 4.16M 63s\n",
      " 40800K .......... .......... .......... .......... .......... 58%  486K 63s\n",
      " 40850K .......... .......... .......... .......... .......... 58%  315K 62s\n",
      " 40900K .......... .......... .......... .......... .......... 58% 38.2M 62s\n",
      " 40950K .......... .......... .......... .......... .......... 58%  680K 62s\n",
      " 41000K .......... .......... .......... .......... .......... 58%  216K 62s\n",
      " 41050K .......... .......... .......... .......... .......... 58% 34.4M 62s\n",
      " 41100K .......... .......... .......... .......... .......... 58%  122K 62s\n",
      " 41150K .......... .......... .......... .......... .......... 59% 39.7K 63s\n",
      " 41200K .......... .......... .......... .......... .......... 59% 43.6M 63s\n",
      " 41250K .......... .......... .......... .......... .......... 59% 70.9M 62s\n",
      " 41300K .......... .......... .......... .......... .......... 59% 70.6M 62s\n",
      " 41350K .......... .......... .......... .......... .......... 59% 64.1M 62s\n",
      " 41400K .......... .......... .......... .......... .......... 59%  185K 62s\n",
      " 41450K .......... .......... .......... .......... .......... 59%  187K 62s\n",
      " 41500K .......... .......... .......... .......... .......... 59%  200K 62s\n",
      " 41550K .......... .......... .......... .......... .......... 59%  889K 62s\n",
      " 41600K .......... .......... .......... .......... .......... 59%  241K 62s\n",
      " 41650K .......... .......... .......... .......... .......... 59%  522K 62s\n",
      " 41700K .......... .......... .......... .......... .......... 59%  338K 62s\n",
      " 41750K .......... .......... .......... .......... .......... 59%  956K 61s\n",
      " 41800K .......... .......... .......... .......... .......... 59%  221K 61s\n",
      " 41850K .......... .......... .......... .......... .......... 60% 1.77M 61s\n",
      " 41900K .......... .......... .......... .......... .......... 60% 1.00M 61s\n",
      " 41950K .......... .......... .......... .......... .......... 60%  268K 61s\n",
      " 42000K .......... .......... .......... .......... .......... 60% 1.83M 61s\n",
      " 42050K .......... .......... .......... .......... .......... 60%  195K 61s\n",
      " 42100K .......... .......... .......... .......... .......... 60% 11.6M 61s\n",
      " 42150K .......... .......... .......... .......... .......... 60% 1.50M 60s\n",
      " 42200K .......... .......... .......... .......... .......... 60%  538K 60s\n",
      " 42250K .......... .......... .......... .......... .......... 60%  289K 60s\n",
      " 42300K .......... .......... .......... .......... .......... 60% 27.2K 61s\n",
      " 42350K .......... .......... .......... .......... .......... 60% 55.0M 61s\n",
      " 42400K .......... .......... .......... .......... .......... 60% 66.8M 61s\n",
      " 42450K .......... .......... .......... .......... .......... 60% 78.6M 61s\n",
      " 42500K .......... .......... .......... .......... .......... 60% 78.0M 61s\n",
      " 42550K .......... .......... .......... .......... .......... 61%  165K 61s\n",
      " 42600K .......... .......... .......... .......... .......... 61%  163K 61s\n",
      " 42650K .......... .......... .......... .......... .......... 61%  162K 61s\n",
      " 42700K .......... .......... .......... .......... .......... 61%  163K 61s\n",
      " 42750K .......... .......... .......... .......... .......... 61% 52.4M 60s\n",
      " 42800K .......... .......... .......... .......... .......... 61%  160K 60s\n",
      " 42850K .......... .......... .......... .......... .......... 61%  165K 60s\n",
      " 42900K .......... .......... .......... .......... .......... 61% 1.43M 60s\n",
      " 42950K .......... .......... .......... .......... .......... 61%  183K 60s\n",
      " 43000K .......... .......... .......... .......... .......... 61%  624K 60s\n",
      " 43050K .......... .......... .......... .......... .......... 61%  220K 60s\n",
      " 43100K .......... .......... .......... .......... .......... 61%  194K 60s\n",
      " 43150K .......... .......... .......... .......... .......... 61%  824K 60s\n",
      " 43200K .......... .......... .......... .......... .......... 61%  261K 60s\n",
      " 43250K .......... .......... .......... .......... .......... 62%  853K 60s\n",
      " 43300K .......... .......... .......... .......... .......... 62%  201K 60s\n",
      " 43350K .......... .......... .......... .......... .......... 62% 33.2M 60s\n",
      " 43400K .......... .......... .......... .......... .......... 62% 1023K 59s\n",
      " 43450K .......... .......... .......... .......... .......... 62%  248K 59s\n",
      " 43500K .......... .......... .......... .......... .......... 62% 11.5M 59s\n",
      " 43550K .......... .......... .......... .......... .......... 62% 1.04M 59s\n",
      " 43600K .......... .......... .......... .......... .......... 62%  238K 59s\n",
      " 43650K .......... .......... .......... .......... .......... 62% 36.9M 59s\n",
      " 43700K .......... .......... .......... .......... .......... 62% 1.19M 59s\n",
      " 43750K .......... .......... .......... .......... .......... 62% 1.03M 58s\n",
      " 43800K .......... .......... .......... .......... .......... 62%  308K 58s\n",
      " 43850K .......... .......... .......... .......... .......... 62% 1024K 58s\n",
      " 43900K .......... .......... .......... .......... .......... 62% 1.28M 58s\n",
      " 43950K .......... .......... .......... .......... .......... 63% 10.3M 58s\n",
      " 44000K .......... .......... .......... .......... .......... 63%  297K 58s\n",
      " 44050K .......... .......... .......... .......... .......... 63% 1.25M 58s\n",
      " 44100K .......... .......... .......... .......... .......... 63% 1.26M 58s\n",
      " 44150K .......... .......... .......... .......... .......... 63% 6.54M 57s\n",
      " 44200K .......... .......... .......... .......... .......... 63%  303K 57s\n",
      " 44250K .......... .......... .......... .......... .......... 63% 1.24M 57s\n",
      " 44300K .......... .......... .......... .......... .......... 63% 1.35M 57s\n",
      " 44350K .......... .......... .......... .......... .......... 63% 6.18M 57s\n",
      " 44400K .......... .......... .......... .......... .......... 63%  959K 57s\n",
      " 44450K .......... .......... .......... .......... .......... 63%  398K 57s\n",
      " 44500K .......... .......... .......... .......... .......... 63% 1.65M 56s\n",
      " 44550K .......... .......... .......... .......... .......... 63% 1.21M 56s\n",
      " 44600K .......... .......... .......... .......... .......... 63% 1.10M 56s\n",
      " 44650K .......... .......... .......... .......... .......... 64%  742K 56s\n",
      " 44700K .......... .......... .......... .......... .......... 64%  498K 56s\n",
      " 44750K .......... .......... .......... .......... .......... 64%  596K 56s\n",
      " 44800K .......... .......... .......... .......... .......... 64%  344K 56s\n",
      " 44850K .......... .......... .......... .......... .......... 64% 2.11M 55s\n",
      " 44900K .......... .......... .......... .......... .......... 64%  629K 55s\n",
      " 44950K .......... .......... .......... .......... .......... 64%  938K 55s\n",
      " 45000K .......... .......... .......... .......... .......... 64%  361K 55s\n",
      " 45050K .......... .......... .......... .......... .......... 64%  378K 55s\n",
      " 45100K .......... .......... .......... .......... .......... 64%  358K 55s\n",
      " 45150K .......... .......... .......... .......... .......... 64%  197K 55s\n",
      " 45200K .......... .......... .......... .......... .......... 64% 27.3M 55s\n",
      " 45250K .......... .......... .......... .......... .......... 64%  617K 55s\n",
      " 45300K .......... .......... .......... .......... .......... 64%  223K 54s\n",
      " 45350K .......... .......... .......... .......... .......... 65% 47.2M 54s\n",
      " 45400K .......... .......... .......... .......... .......... 65%  163K 54s\n",
      " 45450K .......... .......... .......... .......... .......... 65% 58.2M 54s\n",
      " 45500K .......... .......... .......... .......... .......... 65%  164K 54s\n",
      " 45550K .......... .......... .......... .......... .......... 65% 32.1M 54s\n",
      " 45600K .......... .......... .......... .......... .......... 65%  161K 54s\n",
      " 45650K .......... .......... .......... .......... .......... 65%  203M 54s\n",
      " 45700K .......... .......... .......... .......... .......... 65%  166K 54s\n",
      " 45750K .......... .......... .......... .......... .......... 65% 1.04M 54s\n",
      " 45800K .......... .......... .......... .......... .......... 65%  192K 54s\n",
      " 45850K .......... .......... .......... .......... .......... 65% 1.24M 53s\n",
      " 45900K .......... .......... .......... .......... .......... 65%  990K 53s\n",
      " 45950K .......... .......... .......... .......... .......... 65%  229K 53s\n",
      " 46000K .......... .......... .......... .......... .......... 65%  163K 53s\n",
      " 46050K .......... .......... .......... .......... .......... 66%  196K 53s\n",
      " 46100K .......... .......... .......... .......... .......... 66% 69.0M 53s\n",
      " 46150K .......... .......... .......... .......... .......... 66%  140K 53s\n",
      " 46200K .......... .......... .......... .......... .......... 66% 99.2K 53s\n",
      " 46250K .......... .......... .......... .......... .......... 66% 6.82M 53s\n",
      " 46300K .......... .......... .......... .......... .......... 66%  162K 53s\n",
      " 46350K .......... .......... .......... .......... .......... 66%  516K 53s\n",
      " 46400K .......... .......... .......... .......... .......... 66%  257K 53s\n",
      " 46450K .......... .......... .......... .......... .......... 66%  498K 53s\n",
      " 46500K .......... .......... .......... .......... .......... 66%  417K 53s\n",
      " 46550K .......... .......... .......... .......... .......... 66%  597K 52s\n",
      " 46600K .......... .......... .......... .......... .......... 66%  520K 52s\n",
      " 46650K .......... .......... .......... .......... .......... 66%  481K 52s\n",
      " 46700K .......... .......... .......... .......... .......... 66%  481K 52s\n",
      " 46750K .......... .......... .......... .......... .......... 67%  261K 52s\n",
      " 46800K .......... .......... .......... .......... .......... 67%  479K 52s\n",
      " 46850K .......... .......... .......... .......... .......... 67%  465K 52s\n",
      " 46900K .......... .......... .......... .......... .......... 67%  204K 52s\n",
      " 46950K .......... .......... .......... .......... .......... 67% 1.11M 52s\n",
      " 47000K .......... .......... .......... .......... .......... 67%  241K 51s\n",
      " 47050K .......... .......... .......... .......... .......... 67% 1.12M 51s\n",
      " 47100K .......... .......... .......... .......... .......... 67%  735K 51s\n",
      " 47150K .......... .......... .......... .......... .......... 67%  355K 51s\n",
      " 47200K .......... .......... .......... .......... .......... 67% 1.11M 51s\n",
      " 47250K .......... .......... .......... .......... .......... 67%  705K 51s\n",
      " 47300K .......... .......... .......... .......... .......... 67%  364K 51s\n",
      " 47350K .......... .......... .......... .......... .......... 67% 1.14M 51s\n",
      " 47400K .......... .......... .......... .......... .......... 67%  943K 50s\n",
      " 47450K .......... .......... .......... .......... .......... 68%  486K 50s\n",
      " 47500K .......... .......... .......... .......... .......... 68%  835K 50s\n",
      " 47550K .......... .......... .......... .......... .......... 68% 1.33M 50s\n",
      " 47600K .......... .......... .......... .......... .......... 68%  745K 50s\n",
      " 47650K .......... .......... .......... .......... .......... 68%  655K 50s\n",
      " 47700K .......... .......... .......... .......... .......... 68%  696K 50s\n",
      " 47750K .......... .......... .......... .......... .......... 68% 1.35M 49s\n",
      " 47800K .......... .......... .......... .......... .......... 68%  544K 49s\n",
      " 47850K .......... .......... .......... .......... .......... 68%  387K 49s\n",
      " 47900K .......... .......... .......... .......... .......... 68%  603K 49s\n",
      " 47950K .......... .......... .......... .......... .......... 68%  569K 49s\n",
      " 48000K .......... .......... .......... .......... .......... 68%  445K 49s\n",
      " 48050K .......... .......... .......... .......... .......... 68%  944K 49s\n",
      " 48100K .......... .......... .......... .......... .......... 68% 1.05M 49s\n",
      " 48150K .......... .......... .......... .......... .......... 69%  310K 49s\n",
      " 48200K .......... .......... .......... .......... .......... 69%  361K 48s\n",
      " 48250K .......... .......... .......... .......... .......... 69%  442K 48s\n",
      " 48300K .......... .......... .......... .......... .......... 69%  576K 48s\n",
      " 48350K .......... .......... .......... .......... .......... 69%  347K 48s\n",
      " 48400K .......... .......... .......... .......... .......... 69%  761K 48s\n",
      " 48450K .......... .......... .......... .......... .......... 69%  372K 48s\n",
      " 48500K .......... .......... .......... .......... .......... 69%  541K 48s\n",
      " 48550K .......... .......... .......... .......... .......... 69%  996K 48s\n",
      " 48600K .......... .......... .......... .......... .......... 69%  721K 47s\n",
      " 48650K .......... .......... .......... .......... .......... 69%  387K 47s\n",
      " 48700K .......... .......... .......... .......... .......... 69% 1.77M 47s\n",
      " 48750K .......... .......... .......... .......... .......... 69% 1.18M 47s\n",
      " 48800K .......... .......... .......... .......... .......... 69%  332K 47s\n",
      " 48850K .......... .......... .......... .......... .......... 70% 1.52M 47s\n",
      " 48900K .......... .......... .......... .......... .......... 70%  893K 47s\n",
      " 48950K .......... .......... .......... .......... .......... 70% 1.12M 47s\n",
      " 49000K .......... .......... .......... .......... .......... 70%  391K 46s\n",
      " 49050K .......... .......... .......... .......... .......... 70% 2.03M 46s\n",
      " 49100K .......... .......... .......... .......... .......... 70% 1.75M 46s\n",
      " 49150K .......... .......... .......... .......... .......... 70% 1.57M 46s\n",
      " 49200K .......... .......... .......... .......... .......... 70%  773K 46s\n",
      " 49250K .......... .......... .......... .......... .......... 70%  472K 46s\n",
      " 49300K .......... .......... .......... .......... .......... 70% 1.87M 46s\n",
      " 49350K .......... .......... .......... .......... .......... 70%  825K 45s\n",
      " 49400K .......... .......... .......... .......... .......... 70% 1.38M 45s\n",
      " 49450K .......... .......... .......... .......... .......... 70%  360K 45s\n",
      " 49500K .......... .......... .......... .......... .......... 70%  646K 45s\n",
      " 49550K .......... .......... .......... .......... .......... 71%  900K 45s\n",
      " 49600K .......... .......... .......... .......... .......... 71%  378K 45s\n",
      " 49650K .......... .......... .......... .......... .......... 71% 1.02M 45s\n",
      " 49700K .......... .......... .......... .......... .......... 71%  954K 45s\n",
      " 49750K .......... .......... .......... .......... .......... 71%  348K 44s\n",
      " 49800K .......... .......... .......... .......... .......... 71% 4.04M 44s\n",
      " 49850K .......... .......... .......... .......... .......... 71%  550K 44s\n",
      " 49900K .......... .......... .......... .......... .......... 71% 1.50M 44s\n",
      " 49950K .......... .......... .......... .......... .......... 71%  426K 44s\n",
      " 50000K .......... .......... .......... .......... .......... 71% 1.07M 44s\n",
      " 50050K .......... .......... .......... .......... .......... 71%  911K 44s\n",
      " 50100K .......... .......... .......... .......... .......... 71% 1.07M 44s\n",
      " 50150K .......... .......... .......... .......... .......... 71%  475K 43s\n",
      " 50200K .......... .......... .......... .......... .......... 71% 1.08M 43s\n",
      " 50250K .......... .......... .......... .......... .......... 72%  994K 43s\n",
      " 50300K .......... .......... .......... .......... .......... 72% 4.07M 43s\n",
      " 50350K .......... .......... .......... .......... .......... 72% 1.16M 43s\n",
      " 50400K .......... .......... .......... .......... .......... 72%  464K 43s\n",
      " 50450K .......... .......... .......... .......... .......... 72% 1.23M 43s\n",
      " 50500K .......... .......... .......... .......... .......... 72%  382K 43s\n",
      " 50550K .......... .......... .......... .......... .......... 72% 32.3M 42s\n",
      " 50600K .......... .......... .......... .......... .......... 72% 43.9M 42s\n",
      " 50650K .......... .......... .......... .......... .......... 72%  624K 42s\n",
      " 50700K .......... .......... .......... .......... .......... 72% 1.16M 42s\n",
      " 50750K .......... .......... .......... .......... .......... 72%  312K 42s\n",
      " 50800K .......... .......... .......... .......... .......... 72% 30.6M 42s\n",
      " 50850K .......... .......... .......... .......... .......... 72% 1.86M 42s\n",
      " 50900K .......... .......... .......... .......... .......... 72% 1.77M 41s\n",
      " 50950K .......... .......... .......... .......... .......... 73%  137K 41s\n",
      " 51000K .......... .......... .......... .......... .......... 73% 25.8M 41s\n",
      " 51050K .......... .......... .......... .......... .......... 73% 98.0K 41s\n",
      " 51100K .......... .......... .......... .......... .......... 73% 32.0M 41s\n",
      " 51150K .......... .......... .......... .......... .......... 73% 10.6M 41s\n",
      " 51200K .......... .......... .......... .......... .......... 73% 1.12M 41s\n",
      " 51250K .......... .......... .......... .......... .......... 73% 52.7M 41s\n",
      " 51300K .......... .......... .......... .......... .......... 73%  195K 41s\n",
      " 51350K .......... .......... .......... .......... .......... 73% 25.1M 41s\n",
      " 51400K .......... .......... .......... .......... .......... 73% 97.9K 41s\n",
      " 51450K .......... .......... .......... .......... .......... 73% 1.42M 40s\n",
      " 51500K .......... .......... .......... .......... .......... 73% 73.5M 40s\n",
      " 51550K .......... .......... .......... .......... .......... 73% 75.8M 40s\n",
      " 51600K .......... .......... .......... .......... .......... 73% 1.14M 40s\n",
      " 51650K .......... .......... .......... .......... .......... 74%  218K 40s\n",
      " 51700K .......... .......... .......... .......... .......... 74% 48.4M 40s\n",
      " 51750K .......... .......... .......... .......... .......... 74%  185K 40s\n",
      " 51800K .......... .......... .......... .......... .......... 74% 55.6M 40s\n",
      " 51850K .......... .......... .......... .......... .......... 74% 66.3M 39s\n",
      " 51900K .......... .......... .......... .......... .......... 74% 29.1M 39s\n",
      " 51950K .......... .......... .......... .......... .......... 74% 1018K 39s\n",
      " 52000K .......... .......... .......... .......... .......... 74%  243K 39s\n",
      " 52050K .......... .......... .......... .......... .......... 74% 1.17M 39s\n",
      " 52100K .......... .......... .......... .......... .......... 74% 1019K 39s\n",
      " 52150K .......... .......... .......... .......... .......... 74%  906K 39s\n",
      " 52200K .......... .......... .......... .......... .......... 74%  527K 39s\n",
      " 52250K .......... .......... .......... .......... .......... 74%  963K 38s\n",
      " 52300K .......... .......... .......... .......... .......... 75%  479K 38s\n",
      " 52350K .......... .......... .......... .......... .......... 75% 1.12M 38s\n",
      " 52400K .......... .......... .......... .......... .......... 75%  897K 38s\n",
      " 52450K .......... .......... .......... .......... .......... 75%  328K 38s\n",
      " 52500K .......... .......... .......... .......... .......... 75% 10.8M 38s\n",
      " 52550K .......... .......... .......... .......... .......... 75% 1.21M 38s\n",
      " 52600K .......... .......... .......... .......... .......... 75%  239K 38s\n",
      " 52650K .......... .......... .......... .......... .......... 75% 8.54M 37s\n",
      " 52700K .......... .......... .......... .......... .......... 75% 1.06M 37s\n",
      " 52750K .......... .......... .......... .......... .......... 75% 1.06M 37s\n",
      " 52800K .......... .......... .......... .......... .......... 75%  320K 37s\n",
      " 52850K .......... .......... .......... .......... .......... 75% 1.38M 37s\n",
      " 52900K .......... .......... .......... .......... .......... 75% 2.68M 37s\n",
      " 52950K .......... .......... .......... .......... .......... 75% 1.45M 37s\n",
      " 53000K .......... .......... .......... .......... .......... 76%  303K 37s\n",
      " 53050K .......... .......... .......... .......... .......... 76%  600K 37s\n",
      " 53100K .......... .......... .......... .......... .......... 76%  298K 36s\n",
      " 53150K .......... .......... .......... .......... .......... 76% 1.19M 36s\n",
      " 53200K .......... .......... .......... .......... .......... 76% 1.02M 36s\n",
      " 53250K .......... .......... .......... .......... .......... 76%  306K 36s\n",
      " 53300K .......... .......... .......... .......... .......... 76%  976K 36s\n",
      " 53350K .......... .......... .......... .......... .......... 76% 1.36M 36s\n",
      " 53400K .......... .......... .......... .......... .......... 76%  302K 36s\n",
      " 53450K .......... .......... .......... .......... .......... 76% 1.14M 36s\n",
      " 53500K .......... .......... .......... .......... .......... 76% 1.37M 35s\n",
      " 53550K .......... .......... .......... .......... .......... 76% 1010K 35s\n",
      " 53600K .......... .......... .......... .......... .......... 76%  300K 35s\n",
      " 53650K .......... .......... .......... .......... .......... 76%  593K 35s\n",
      " 53700K .......... .......... .......... .......... .......... 77%  330K 35s\n",
      " 53750K .......... .......... .......... .......... .......... 77%  897K 35s\n",
      " 53800K .......... .......... .......... .......... .......... 77%  283K 35s\n",
      " 53850K .......... .......... .......... .......... .......... 77% 1.36M 35s\n",
      " 53900K .......... .......... .......... .......... .......... 77%  657K 35s\n",
      " 53950K .......... .......... .......... .......... .......... 77%  338K 34s\n",
      " 54000K .......... .......... .......... .......... .......... 77% 1.74M 34s\n",
      " 54050K .......... .......... .......... .......... .......... 77%  569K 34s\n",
      " 54100K .......... .......... .......... .......... .......... 77%  338K 34s\n",
      " 54150K .......... .......... .......... .......... .......... 77%  477K 34s\n",
      " 54200K .......... .......... .......... .......... .......... 77%  341K 34s\n",
      " 54250K .......... .......... .......... .......... .......... 77%  581K 34s\n",
      " 54300K .......... .......... .......... .......... .......... 77%  352K 34s\n",
      " 54350K .......... .......... .......... .......... .......... 77%  862K 34s\n",
      " 54400K .......... .......... .......... .......... .......... 78%  806K 33s\n",
      " 54450K .......... .......... .......... .......... .......... 78%  558K 33s\n",
      " 54500K .......... .......... .......... .......... .......... 78%  508K 33s\n",
      " 54550K .......... .......... .......... .......... .......... 78%  965K 33s\n",
      " 54600K .......... .......... .......... .......... .......... 78%  963K 33s\n",
      " 54650K .......... .......... .......... .......... .......... 78%  413K 33s\n",
      " 54700K .......... .......... .......... .......... .......... 78%  906K 33s\n",
      " 54750K .......... .......... .......... .......... .......... 78% 1.96M 33s\n",
      " 54800K .......... .......... .......... .......... .......... 78%  971K 32s\n",
      " 54850K .......... .......... .......... .......... .......... 78%  388K 32s\n",
      " 54900K .......... .......... .......... .......... .......... 78% 2.16M 32s\n",
      " 54950K .......... .......... .......... .......... .......... 78% 1.01M 32s\n",
      " 55000K .......... .......... .......... .......... .......... 78% 1.01M 32s\n",
      " 55050K .......... .......... .......... .......... .......... 78%  380K 32s\n",
      " 55100K .......... .......... .......... .......... .......... 79% 2.46M 32s\n",
      " 55150K .......... .......... .......... .......... .......... 79% 1.11M 32s\n",
      " 55200K .......... .......... .......... .......... .......... 79% 2.41M 32s\n",
      " 55250K .......... .......... .......... .......... .......... 79% 1.22M 31s\n",
      " 55300K .......... .......... .......... .......... .......... 79%  366K 31s\n",
      " 55350K .......... .......... .......... .......... .......... 79% 4.36M 31s\n",
      " 55400K .......... .......... .......... .......... .......... 79%  865K 31s\n",
      " 55450K .......... .......... .......... .......... .......... 79% 7.98M 31s\n",
      " 55500K .......... .......... .......... .......... .......... 79%  525K 31s\n",
      " 55550K .......... .......... .......... .......... .......... 79%  515K 31s\n",
      " 55600K .......... .......... .......... .......... .......... 79%  541K 31s\n",
      " 55650K .......... .......... .......... .......... .......... 79% 2.99M 30s\n",
      " 55700K .......... .......... .......... .......... .......... 79%  360K 30s\n",
      " 55750K .......... .......... .......... .......... .......... 79%  502K 30s\n",
      " 55800K .......... .......... .......... .......... .......... 80% 1.25M 30s\n",
      " 55850K .......... .......... .......... .......... .......... 80%  457K 30s\n",
      " 55900K .......... .......... .......... .......... .......... 80%  471K 30s\n",
      " 55950K .......... .......... .......... .......... .......... 80% 1.26M 30s\n",
      " 56000K .......... .......... .......... .......... .......... 80%  433K 30s\n",
      " 56050K .......... .......... .......... .......... .......... 80%  464K 30s\n",
      " 56100K .......... .......... .......... .......... .......... 80%  347K 29s\n",
      " 56150K .......... .......... .......... .......... .......... 80%  392K 29s\n",
      " 56200K .......... .......... .......... .......... .......... 80%  887K 29s\n",
      " 56250K .......... .......... .......... .......... .......... 80%  288K 29s\n",
      " 56300K .......... .......... .......... .......... .......... 80%  713K 29s\n",
      " 56350K .......... .......... .......... .......... .......... 80%  780K 29s\n",
      " 56400K .......... .......... .......... .......... .......... 80%  357K 29s\n",
      " 56450K .......... .......... .......... .......... .......... 80% 1.02M 29s\n",
      " 56500K .......... .......... .......... .......... .......... 81% 1.22M 29s\n",
      " 56550K .......... .......... .......... .......... .......... 81%  318K 28s\n",
      " 56600K .......... .......... .......... .......... .......... 81%  904K 28s\n",
      " 56650K .......... .......... .......... .......... .......... 81% 1.09M 28s\n",
      " 56700K .......... .......... .......... .......... .......... 81% 1.57M 28s\n",
      " 56750K .......... .......... .......... .......... .......... 81%  395K 28s\n",
      " 56800K .......... .......... .......... .......... .......... 81%  955K 28s\n",
      " 56850K .......... .......... .......... .......... .......... 81% 1.08M 28s\n",
      " 56900K .......... .......... .......... .......... .......... 81%  284K 28s\n",
      " 56950K .......... .......... .......... .......... .......... 81% 1.54M 28s\n",
      " 57000K .......... .......... .......... .......... .......... 81%  557K 27s\n",
      " 57050K .......... .......... .......... .......... .......... 81%  304K 27s\n",
      " 57100K .......... .......... .......... .......... .......... 81% 1.05M 27s\n",
      " 57150K .......... .......... .......... .......... .......... 81%  291K 27s\n",
      " 57200K .......... .......... .......... .......... .......... 82%  415K 27s\n",
      " 57250K .......... .......... .......... .......... .......... 82%  307K 27s\n",
      " 57300K .......... .......... .......... .......... .......... 82%  569K 27s\n",
      " 57350K .......... .......... .......... .......... .......... 82%  366K 27s\n",
      " 57400K .......... .......... .......... .......... .......... 82%  600K 27s\n",
      " 57450K .......... .......... .......... .......... .......... 82%  644K 26s\n",
      " 57500K .......... .......... .......... .......... .......... 82%  510K 26s\n",
      " 57550K .......... .......... .......... .......... .......... 82%  655K 26s\n",
      " 57600K .......... .......... .......... .......... .......... 82%  699K 26s\n",
      " 57650K .......... .......... .......... .......... .......... 82%  514K 26s\n",
      " 57700K .......... .......... .......... .......... .......... 82%  895K 26s\n",
      " 57750K .......... .......... .......... .......... .......... 82%  835K 26s\n",
      " 57800K .......... .......... .......... .......... .......... 82%  576K 26s\n",
      " 57850K .......... .......... .......... .......... .......... 82%  986K 26s\n",
      " 57900K .......... .......... .......... .......... .......... 83%  491K 25s\n",
      " 57950K .......... .......... .......... .......... .......... 83%  323K 25s\n",
      " 58000K .......... .......... .......... .......... .......... 83%  516K 25s\n",
      " 58050K .......... .......... .......... .......... .......... 83%  464K 25s\n",
      " 58100K .......... .......... .......... .......... .......... 83%  488K 25s\n",
      " 58150K .......... .......... .......... .......... .......... 83% 1003K 25s\n",
      " 58200K .......... .......... .......... .......... .......... 83%  498K 25s\n",
      " 58250K .......... .......... .......... .......... .......... 83%  503K 25s\n",
      " 58300K .......... .......... .......... .......... .......... 83% 1.11M 25s\n",
      " 58350K .......... .......... .......... .......... .......... 83%  831K 24s\n",
      " 58400K .......... .......... .......... .......... .......... 83%  472K 24s\n",
      " 58450K .......... .......... .......... .......... .......... 83%  397K 24s\n",
      " 58500K .......... .......... .......... .......... .......... 83%  329K 24s\n",
      " 58550K .......... .......... .......... .......... .......... 83%  693K 24s\n",
      " 58600K .......... .......... .......... .......... .......... 84%  313K 24s\n",
      " 58650K .......... .......... .......... .......... .......... 84%  735K 24s\n",
      " 58700K .......... .......... .......... .......... .......... 84%  245K 24s\n",
      " 58750K .......... .......... .......... .......... .......... 84%  534K 24s\n",
      " 58800K .......... .......... .......... .......... .......... 84%  287K 24s\n",
      " 58850K .......... .......... .......... .......... .......... 84%  472K 23s\n",
      " 58900K .......... .......... .......... .......... .......... 84%  168K 23s\n",
      " 58950K .......... .......... .......... .......... .......... 84% 1.46M 23s\n",
      " 59000K .......... .......... .......... .......... .......... 84%  614K 23s\n",
      " 59050K .......... .......... .......... .......... .......... 84%  294K 23s\n",
      " 59100K .......... .......... .......... .......... .......... 84%  574K 23s\n",
      " 59150K .......... .......... .......... .......... .......... 84%  497K 23s\n",
      " 59200K .......... .......... .......... .......... .......... 84%  478K 23s\n",
      " 59250K .......... .......... .......... .......... .......... 84% 1.08M 23s\n",
      " 59300K .......... .......... .......... .......... .......... 85%  715K 22s\n",
      " 59350K .......... .......... .......... .......... .......... 85%  684K 22s\n",
      " 59400K .......... .......... .......... .......... .......... 85%  505K 22s\n",
      " 59450K .......... .......... .......... .......... .......... 85% 3.84M 22s\n",
      " 59500K .......... .......... .......... .......... .......... 85%  503K 22s\n",
      " 59550K .......... .......... .......... .......... .......... 85%  630K 22s\n",
      " 59600K .......... .......... .......... .......... .......... 85%  155K 22s\n",
      " 59650K .......... .......... .......... .......... .......... 85% 57.4M 22s\n",
      " 59700K .......... .......... .......... .......... .......... 85% 41.8M 22s\n",
      " 59750K .......... .......... .......... .......... .......... 85%  498K 21s\n",
      " 59800K .......... .......... .......... .......... .......... 85%  317K 21s\n",
      " 59850K .......... .......... .......... .......... .......... 85% 1.05M 21s\n",
      " 59900K .......... .......... .......... .......... .......... 85%  318K 21s\n",
      " 59950K .......... .......... .......... .......... .......... 85%  625K 21s\n",
      " 60000K .......... .......... .......... .......... .......... 86%  403K 21s\n",
      " 60050K .......... .......... .......... .......... .......... 86%  436K 21s\n",
      " 60100K .......... .......... .......... .......... .......... 86% 3.18M 21s\n",
      " 60150K .......... .......... .......... .......... .......... 86%  779K 21s\n",
      " 60200K .......... .......... .......... .......... .......... 86%  290K 20s\n",
      " 60250K .......... .......... .......... .......... .......... 86% 3.94M 20s\n",
      " 60300K .......... .......... .......... .......... .......... 86%  908K 20s\n",
      " 60350K .......... .......... .......... .......... .......... 86%  403K 20s\n",
      " 60400K .......... .......... .......... .......... .......... 86%  741K 20s\n",
      " 60450K .......... .......... .......... .......... .......... 86% 5.71M 20s\n",
      " 60500K .......... .......... .......... .......... .......... 86%  943K 20s\n",
      " 60550K .......... .......... .......... .......... .......... 86%  301K 20s\n",
      " 60600K .......... .......... .......... .......... .......... 86% 2.19M 20s\n",
      " 60650K .......... .......... .......... .......... .......... 86% 4.76M 19s\n",
      " 60700K .......... .......... .......... .......... .......... 87%  949K 19s\n",
      " 60750K .......... .......... .......... .......... .......... 87%  582K 19s\n",
      " 60800K .......... .......... .......... .......... .......... 87%  457K 19s\n",
      " 60850K .......... .......... .......... .......... .......... 87%  518K 19s\n",
      " 60900K .......... .......... .......... .......... .......... 87%  345K 19s\n",
      " 60950K .......... .......... .......... .......... .......... 87% 7.64M 19s\n",
      " 61000K .......... .......... .......... .......... .......... 87%  211K 19s\n",
      " 61050K .......... .......... .......... .......... .......... 87% 4.29M 19s\n",
      " 61100K .......... .......... .......... .......... .......... 87%  897K 18s\n",
      " 61150K .......... .......... .......... .......... .......... 87%  269K 18s\n",
      " 61200K .......... .......... .......... .......... .......... 87% 3.52M 18s\n",
      " 61250K .......... .......... .......... .......... .......... 87% 1.11M 18s\n",
      " 61300K .......... .......... .......... .......... .......... 87%  209K 18s\n",
      " 61350K .......... .......... .......... .......... .......... 87% 6.90M 18s\n",
      " 61400K .......... .......... .......... .......... .......... 88%  984K 18s\n",
      " 61450K .......... .......... .......... .......... .......... 88% 1008K 18s\n",
      " 61500K .......... .......... .......... .......... .......... 88%  339K 18s\n",
      " 61550K .......... .......... .......... .......... .......... 88%  815K 17s\n",
      " 61600K .......... .......... .......... .......... .......... 88%  251K 17s\n",
      " 61650K .......... .......... .......... .......... .......... 88%  800K 17s\n",
      " 61700K .......... .......... .......... .......... .......... 88%  378K 17s\n",
      " 61750K .......... .......... .......... .......... .......... 88%  488K 17s\n",
      " 61800K .......... .......... .......... .......... .......... 88%  414K 17s\n",
      " 61850K .......... .......... .......... .......... .......... 88%  519K 17s\n",
      " 61900K .......... .......... .......... .......... .......... 88% 1.17M 17s\n",
      " 61950K .......... .......... .......... .......... .......... 88%  439K 17s\n",
      " 62000K .......... .......... .......... .......... .......... 88%  521K 17s\n",
      " 62050K .......... .......... .......... .......... .......... 88% 1.12M 16s\n",
      " 62100K .......... .......... .......... .......... .......... 89%  697K 16s\n",
      " 62150K .......... .......... .......... .......... .......... 89%  839K 16s\n",
      " 62200K .......... .......... .......... .......... .......... 89%  429K 16s\n",
      " 62250K .......... .......... .......... .......... .......... 89%  619K 16s\n",
      " 62300K .......... .......... .......... .......... .......... 89%  369K 16s\n",
      " 62350K .......... .......... .......... .......... .......... 89%  692K 16s\n",
      " 62400K .......... .......... .......... .......... .......... 89%  348K 16s\n",
      " 62450K .......... .......... .......... .......... .......... 89%  856K 16s\n",
      " 62500K .......... .......... .......... .......... .......... 89% 1013K 15s\n",
      " 62550K .......... .......... .......... .......... .......... 89%  276K 15s\n",
      " 62600K .......... .......... .......... .......... .......... 89%  853K 15s\n",
      " 62650K .......... .......... .......... .......... .......... 89%  258K 15s\n",
      " 62700K .......... .......... .......... .......... .......... 89% 1.15M 15s\n",
      " 62750K .......... .......... .......... .......... .......... 89%  291K 15s\n",
      " 62800K .......... .......... .......... .......... .......... 90% 1022K 15s\n",
      " 62850K .......... .......... .......... .......... .......... 90% 1.02M 15s\n",
      " 62900K .......... .......... .......... .......... .......... 90%  255K 15s\n",
      " 62950K .......... .......... .......... .......... .......... 90%  894K 14s\n",
      " 63000K .......... .......... .......... .......... .......... 90%  246K 14s\n",
      " 63050K .......... .......... .......... .......... .......... 90%  787K 14s\n",
      " 63100K .......... .......... .......... .......... .......... 90%  272K 14s\n",
      " 63150K .......... .......... .......... .......... .......... 90%  959K 14s\n",
      " 63200K .......... .......... .......... .......... .......... 90%  318K 14s\n",
      " 63250K .......... .......... .......... .......... .......... 90%  579K 14s\n",
      " 63300K .......... .......... .......... .......... .......... 90%  984K 14s\n",
      " 63350K .......... .......... .......... .......... .......... 90%  493K 14s\n",
      " 63400K .......... .......... .......... .......... .......... 90%  486K 14s\n",
      " 63450K .......... .......... .......... .......... .......... 90% 1.08M 13s\n",
      " 63500K .......... .......... .......... .......... .......... 91%  882K 13s\n",
      " 63550K .......... .......... .......... .......... .......... 91%  481K 13s\n",
      " 63600K .......... .......... .......... .......... .......... 91% 1.01M 13s\n",
      " 63650K .......... .......... .......... .......... .......... 91% 1.10M 13s\n",
      " 63700K .......... .......... .......... .......... .......... 91%  314K 13s\n",
      " 63750K .......... .......... .......... .......... .......... 91%  542K 13s\n",
      " 63800K .......... .......... .......... .......... .......... 91%  300K 13s\n",
      " 63850K .......... .......... .......... .......... .......... 91% 1.08M 13s\n",
      " 63900K .......... .......... .......... .......... .......... 91% 1.11M 12s\n",
      " 63950K .......... .......... .......... .......... .......... 91%  306K 12s\n",
      " 64000K .......... .......... .......... .......... .......... 91% 1.10M 12s\n",
      " 64050K .......... .......... .......... .......... .......... 91% 1.11M 12s\n",
      " 64100K .......... .......... .......... .......... .......... 91%  366K 12s\n",
      " 64150K .......... .......... .......... .......... .......... 91%  994K 12s\n",
      " 64200K .......... .......... .......... .......... .......... 92%  776K 12s\n",
      " 64250K .......... .......... .......... .......... .......... 92%  289K 12s\n",
      " 64300K .......... .......... .......... .......... .......... 92%  610K 12s\n",
      " 64350K .......... .......... .......... .......... .......... 92%  283K 11s\n",
      " 64400K .......... .......... .......... .......... .......... 92%  652K 11s\n",
      " 64450K .......... .......... .......... .......... .......... 92%  782K 11s\n",
      " 64500K .......... .......... .......... .......... .......... 92%  441K 11s\n",
      " 64550K .......... .......... .......... .......... .......... 92%  675K 11s\n",
      " 64600K .......... .......... .......... .......... .......... 92%  402K 11s\n",
      " 64650K .......... .......... .......... .......... .......... 92%  830K 11s\n",
      " 64700K .......... .......... .......... .......... .......... 92%  744K 11s\n",
      " 64750K .......... .......... .......... .......... .......... 92% 1.70M 11s\n",
      " 64800K .......... .......... .......... .......... .......... 92%  320K 10s\n",
      " 64850K .......... .......... .......... .......... .......... 92% 1.24M 10s\n",
      " 64900K .......... .......... .......... .......... .......... 93% 1004K 10s\n",
      " 64950K .......... .......... .......... .......... .......... 93% 1001K 10s\n",
      " 65000K .......... .......... .......... .......... .......... 93%  411K 10s\n",
      " 65050K .......... .......... .......... .......... .......... 93% 1.58M 10s\n",
      " 65100K .......... .......... .......... .......... .......... 93% 1.01M 10s\n",
      " 65150K .......... .......... .......... .......... .......... 93%  924K 10s\n",
      " 65200K .......... .......... .......... .......... .......... 93%  410K 10s\n",
      " 65250K .......... .......... .......... .......... .......... 93% 2.13M 10s\n",
      " 65300K .......... .......... .......... .......... .......... 93% 1.20M 9s\n",
      " 65350K .......... .......... .......... .......... .......... 93% 2.99M 9s\n",
      " 65400K .......... .......... .......... .......... .......... 93%  500K 9s\n",
      " 65450K .......... .......... .......... .......... .......... 93%  654K 9s\n",
      " 65500K .......... .......... .......... .......... .......... 93%  700K 9s\n",
      " 65550K .......... .......... .......... .......... .......... 93%  320K 9s\n",
      " 65600K .......... .......... .......... .......... .......... 94% 1.02M 9s\n",
      " 65650K .......... .......... .......... .......... .......... 94%  963K 9s\n",
      " 65700K .......... .......... .......... .......... .......... 94%  390K 9s\n",
      " 65750K .......... .......... .......... .......... .......... 94%  820K 8s\n",
      " 65800K .......... .......... .......... .......... .......... 94%  752K 8s\n",
      " 65850K .......... .......... .......... .......... .......... 94%  248K 8s\n",
      " 65900K .......... .......... .......... .......... .......... 94%  987K 8s\n",
      " 65950K .......... .......... .......... .......... .......... 94%  268K 8s\n",
      " 66000K .......... .......... .......... .......... .......... 94%  810K 8s\n",
      " 66050K .......... .......... .......... .......... .......... 94%  349K 8s\n",
      " 66100K .......... .......... .......... .......... .......... 94%  726K 8s\n",
      " 66150K .......... .......... .......... .......... .......... 94% 1.20M 8s\n",
      " 66200K .......... .......... .......... .......... .......... 94%  350K 7s\n",
      " 66250K .......... .......... .......... .......... .......... 94%  735K 7s\n",
      " 66300K .......... .......... .......... .......... .......... 95% 1.22M 7s\n",
      " 66350K .......... .......... .......... .......... .......... 95% 1.02M 7s\n",
      " 66400K .......... .......... .......... .......... .......... 95%  356K 7s\n",
      " 66450K .......... .......... .......... .......... .......... 95% 1.11M 7s\n",
      " 66500K .......... .......... .......... .......... .......... 95% 1.03M 7s\n",
      " 66550K .......... .......... .......... .......... .......... 95%  999K 7s\n",
      " 66600K .......... .......... .......... .......... .......... 95%  450K 7s\n",
      " 66650K .......... .......... .......... .......... .......... 95%  535K 7s\n",
      " 66700K .......... .......... .......... .......... .......... 95%  350K 6s\n",
      " 66750K .......... .......... .......... .......... .......... 95%  839K 6s\n",
      " 66800K .......... .......... .......... .......... .......... 95%  522K 6s\n",
      " 66850K .......... .......... .......... .......... .......... 95%  359K 6s\n",
      " 66900K .......... .......... .......... .......... .......... 95%  863K 6s\n",
      " 66950K .......... .......... .......... .......... .......... 95%  997K 6s\n",
      " 67000K .......... .......... .......... .......... .......... 96%  303K 6s\n",
      " 67050K .......... .......... .......... .......... .......... 96%  272K 6s\n",
      " 67100K .......... .......... .......... .......... .......... 96%  736K 6s\n",
      " 67150K .......... .......... .......... .......... .......... 96%  560K 5s\n",
      " 67200K .......... .......... .......... .......... .......... 96%  330K 5s\n",
      " 67250K .......... .......... .......... .......... .......... 96%  983K 5s\n",
      " 67300K .......... .......... .......... .......... .......... 96%  346K 5s\n",
      " 67350K .......... .......... .......... .......... .......... 96%  803K 5s\n",
      " 67400K .......... .......... .......... .......... .......... 96% 1.07M 5s\n",
      " 67450K .......... .......... .......... .......... .......... 96%  348K 5s\n",
      " 67500K .......... .......... .......... .......... .......... 96%  835K 5s\n",
      " 67550K .......... .......... .......... .......... .......... 96% 1.33M 5s\n",
      " 67600K .......... .......... .......... .......... .......... 96%  906K 5s\n",
      " 67650K .......... .......... .......... .......... .......... 96%  494K 4s\n",
      " 67700K .......... .......... .......... .......... .......... 97%  520K 4s\n",
      " 67750K .......... .......... .......... .......... .......... 97%  322K 4s\n",
      " 67800K .......... .......... .......... .......... .......... 97%  518K 4s\n",
      " 67850K .......... .......... .......... .......... .......... 97%  506K 4s\n",
      " 67900K .......... .......... .......... .......... .......... 97%  431K 4s\n",
      " 67950K .......... .......... .......... .......... .......... 97%  570K 4s\n",
      " 68000K .......... .......... .......... .......... .......... 97%  480K 4s\n",
      " 68050K .......... .......... .......... .......... .......... 97%  833K 4s\n",
      " 68100K .......... .......... .......... .......... .......... 97% 1.28M 3s\n",
      " 68150K .......... .......... .......... .......... .......... 97%  558K 3s\n",
      " 68200K .......... .......... .......... .......... .......... 97%  446K 3s\n",
      " 68250K .......... .......... .......... .......... .......... 97% 1011K 3s\n",
      " 68300K .......... .......... .......... .......... .......... 97%  843K 3s\n",
      " 68350K .......... .......... .......... .......... .......... 97% 1.24M 3s\n",
      " 68400K .......... .......... .......... .......... .......... 98%  464K 3s\n",
      " 68450K .......... .......... .......... .......... .......... 98% 1.07M 3s\n",
      " 68500K .......... .......... .......... .......... .......... 98%  851K 3s\n",
      " 68550K .......... .......... .......... .......... .......... 98% 1.24M 3s\n",
      " 68600K .......... .......... .......... .......... .......... 98%  498K 2s\n",
      " 68650K .......... .......... .......... .......... .......... 98%  965K 2s\n",
      " 68700K .......... .......... .......... .......... .......... 98% 1.37M 2s\n",
      " 68750K .......... .......... .......... .......... .......... 98%  953K 2s\n",
      " 68800K .......... .......... .......... .......... .......... 98%  600K 2s\n",
      " 68850K .......... .......... .......... .......... .......... 98%  494K 2s\n",
      " 68900K .......... .......... .......... .......... .......... 98%  339K 2s\n",
      " 68950K .......... .......... .......... .......... .......... 98%  724K 2s\n",
      " 69000K .......... .......... .......... .......... .......... 98%  550K 2s\n",
      " 69050K .......... .......... .......... .......... .......... 99%  521K 1s\n",
      " 69100K .......... .......... .......... .......... .......... 99%  767K 1s\n",
      " 69150K .......... .......... .......... .......... .......... 99%  846K 1s\n",
      " 69200K .......... .......... .......... .......... .......... 99%  403K 1s\n",
      " 69250K .......... .......... .......... .......... .......... 99% 1.10M 1s\n",
      " 69300K .......... .......... .......... .......... .......... 99%  738K 1s\n",
      " 69350K .......... .......... .......... .......... .......... 99%  965K 1s\n",
      " 69400K .......... .......... .......... .......... .......... 99%  546K 1s\n",
      " 69450K .......... .......... .......... .......... .......... 99%  777K 1s\n",
      " 69500K .......... .......... .......... .......... .......... 99%  942K 1s\n",
      " 69550K .......... .......... .......... .......... .......... 99% 1.22M 0s\n",
      " 69600K .......... .......... .......... .......... .......... 99%  522K 0s\n",
      " 69650K .......... .......... .......... .......... .......... 99% 4.33M 0s\n",
      " 69700K .......... .......... .......... .......... .......... 99%  624K 0s\n",
      " 69750K .......... .......... .......... .......... .......   100%  707K=2m26s\n",
      "\n",
      "2022-09-22 17:46:35 (479 KB/s) - ‘IGV.zip’ saved [71472977/71472977]\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "wget https://data.broadinstitute.org/igv/projects/downloads/2.14/IGV_Linux_2.14.1_WithJava.zip -O IGV.zip\n",
    "unzip IGV.zip"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "ename": "ModuleNotFoundError",
     "evalue": "No module named 'ete3'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mModuleNotFoundError\u001b[0m                       Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-1-86c0621bf9b1>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mnumpy\u001b[0m \u001b[0;32mas\u001b[0m \u001b[0mnp\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mpandas\u001b[0m \u001b[0;32mas\u001b[0m \u001b[0mpd\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 4\u001b[0;31m \u001b[0;32mfrom\u001b[0m \u001b[0mete3\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mTree\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      5\u001b[0m \u001b[0;32mif\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0mos\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mpath\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mexists\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'output'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      6\u001b[0m     \u001b[0mos\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mmakedirs\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'output'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mModuleNotFoundError\u001b[0m: No module named 'ete3'"
     ]
    }
   ],
   "source": [
    "import os, sys\n",
    "import numpy as np\n",
    "import pandas as pd\n",
    "\n",
    "if not os.path.exists('output'):\n",
    "    os.makedirs('output')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Pipeline\n",
    "### Assembly"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Create assembly file with skesa"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "skesa --memory 8 --cores 4 --reads samples/samples_AS00000197_file-0.fastq.gz samples/samples_AS00000197_file-1.fastq.gz \n",
      "\n",
      "Total reads: 4469477\n",
      "Reads acquired in  26.485391s wall, 26.100000s user + 0.370000s system = 26.470000s CPU (99.9%)\n",
      "\n",
      "Kmer len: 19\n",
      "Raw kmers: 594362962 Memory needed (GB): 11.4118 Memory available (GB): 5.64984 3 cycle(s) will be performed\n",
      "Distinct kmers: 855416\n",
      "Kmer count in  81.477610s wall, 311.240000s user + 7.470000s system = 318.710000s CPU (391.2%)\n",
      "Uniq kmers merging in  0.044238s wall, 0.040000s user + 0.020000s system = 0.060000s CPU (135.6%)\n",
      "Adapters: 0 Reads before: 4469477 Sequence before: 674813548 Reads after: 4469477 Sequence after: 674813548 Reads clipped: 0\n",
      "Adapters clipped in  81.531635s wall, 311.290000s user + 7.490000s system = 318.780000s CPU (391.0%)\n",
      "\n",
      "Kmer len: 21\n",
      "Raw kmers: 585424008 Memory needed (GB): 11.2401 Memory available (GB): 5.64984 2 cycle(s) will be performed\n",
      "Distinct kmers: 9227566\n",
      "Kmer count in  65.377001s wall, 248.640000s user + 7.590000s system = 256.230000s CPU (391.9%)\n",
      "Uniq kmers merging in  0.288712s wall, 0.440000s user + 0.200000s system = 0.640000s CPU (221.7%)\n",
      "WARNING: --min_count changed from 2 to 3 because of high coverage for genome size 4592001\n",
      "WARNING: --max_kmer_count 10 to 15 because of high coverage for genome size 4592001\n",
      "Kmers branching in  7.100643s wall, 25.840000s user + 0.030000s system = 25.870000s CPU (364.3%)\n",
      "\n",
      "Average read length: 151\n",
      "Genome size estimate: 4592001\n",
      "\n",
      "Kmer: 21 Graph size: 8935689 Contigs in: 0\n",
      "Valley: 43\n",
      "\n",
      "Mark used kmers in  0.000002s wall, 0.000000s user + 0.000000s system = 0.000000s CPU (n/a%)\n",
      "Kmers in multiple/single contigs: 0 0\n",
      "Fragments before: 2610 5339351\n",
      "Fragments after: 2585 5339331\n",
      "New seeds: 1962\n",
      "New seeds in  3.841352s wall, 14.830000s user + 0.020000s system = 14.850000s CPU (386.6%)\n",
      "Fragments before: 3922 82362\n",
      "Fragments after: 3922 82362\n",
      "Connectors: 0 Extenders: 3922\n",
      "Connections and extensions in  0.846612s wall, 3.280000s user + 0.020000s system = 3.300000s CPU (389.8%)\n",
      "Contigs out: 1962 Genome: 5259383 N50: 6769 L50: 231\n",
      "Assembled in  4.720268s wall, 18.140000s user + 0.040000s system = 18.180000s CPU (385.1%)\n",
      "\n",
      "\n",
      "Kmer len: 125\n",
      "Raw kmers: 120641976 Memory needed (GB): 5.79081 Memory available (GB): 5.48776 2 cycle(s) will be performed\n",
      "Distinct kmers: 7695652\n",
      "Kmer count in  21.640913s wall, 80.970000s user + 3.280000s system = 84.250000s CPU (389.3%)\n",
      "Uniq kmers merging in  0.542776s wall, 0.740000s user + 0.410000s system = 1.150000s CPU (211.9%)\n",
      "\n",
      "Average count: 78.2237 Max kmer: 125\n",
      "Seed kmers: 0\n",
      "Cleaned reads: 1574874\n",
      "Reads for connection: 0\n",
      "Internal reads: 0\n",
      "Reads cleaned in  14.055434s wall, 55.010000s user + 0.150000s system = 55.160000s CPU (392.4%)\n",
      "\n",
      "Kmer len: 31\n",
      "Raw kmers: 190531832 Memory needed (GB): 3.65821 Memory available (GB): 5.65134 1 cycle(s) will be performed\n",
      "Distinct kmers: 4634807\n",
      "Kmer count in  14.442203s wall, 54.370000s user + 2.280000s system = 56.650000s CPU (392.3%)\n",
      "Uniq kmers merging in  0.201397s wall, 0.170000s user + 0.240000s system = 0.410000s CPU (203.6%)\n",
      "Kmers branching in  3.455197s wall, 12.390000s user + 0.010000s system = 12.400000s CPU (358.9%)\n",
      "Kmer: 31 Graph size: 4634807 Contigs in: 1962\n",
      "Valley: 5\n",
      "\n",
      "Mark used kmers in  1.054933s wall, 4.010000s user + 0.000000s system = 4.010000s CPU (380.1%)\n",
      "Kmers in multiple/single contigs: 0 885637\n",
      "Fragments before: 3363 3635624\n",
      "Fragments after: 3340 3635624\n",
      "New seeds: 124\n",
      "New seeds in  6.011327s wall, 12.580000s user + 0.020000s system = 12.600000s CPU (209.6%)\n",
      "Fragments before: 4170 261734\n",
      "Fragments after: 2941 261734\n",
      "Connectors: 1229 Extenders: 1712\n",
      "Connections and extensions in  0.855027s wall, 3.260000s user + 0.010000s system = 3.270000s CPU (382.4%)\n",
      "Contigs out: 859 Genome: 5450889 N50: 21120 L50: 78\n",
      "Assembled in  7.939041s wall, 19.870000s user + 0.030000s system = 19.900000s CPU (250.7%)\n",
      "\n",
      "Seed kmers: 0\n",
      "Cleaned reads: 1200708\n",
      "Reads for connection: 0\n",
      "Internal reads: 0\n",
      "Reads cleaned in  11.751916s wall, 46.020000s user + 0.080000s system = 46.100000s CPU (392.3%)\n",
      "\n",
      "Kmer len: 41\n",
      "Raw kmers: 133255470 Memory needed (GB): 3.83776 Memory available (GB): 5.68781 1 cycle(s) will be performed\n",
      "Distinct kmers: 3916744\n",
      "Kmer count in  12.427244s wall, 45.150000s user + 2.550000s system = 47.700000s CPU (383.8%)\n",
      "Uniq kmers merging in  0.156325s wall, 0.250000s user + 0.070000s system = 0.320000s CPU (204.7%)\n",
      "Kmers branching in  3.502642s wall, 12.670000s user + 0.010000s system = 12.680000s CPU (362.0%)\n",
      "Kmer: 41 Graph size: 3916744 Contigs in: 859\n",
      "Valley: 4\n",
      "\n",
      "Mark used kmers in  1.067988s wall, 4.240000s user + 0.000000s system = 4.240000s CPU (397.0%)\n",
      "Kmers in multiple/single contigs: 0 375695\n",
      "Fragments before: 1922 3439110\n",
      "Fragments after: 1898 3439110\n",
      "New seeds: 62\n",
      "New seeds in  6.106883s wall, 12.540000s user + 0.010000s system = 12.550000s CPU (205.5%)\n",
      "Fragments before: 1836 126046\n",
      "Fragments after: 1609 126046\n",
      "Connectors: 227 Extenders: 1382\n",
      "Connections and extensions in  0.836663s wall, 3.090000s user + 0.010000s system = 3.100000s CPU (370.5%)\n",
      "Contigs out: 694 Genome: 5528871 N50: 30734 L50: 52\n",
      "Assembled in  8.028835s wall, 19.890000s user + 0.020000s system = 19.910000s CPU (248.0%)\n",
      "\n",
      "Seed kmers: 0\n",
      "Cleaned reads: 1097693\n",
      "Reads for connection: 0\n",
      "Internal reads: 0\n",
      "Reads cleaned in  11.114858s wall, 43.340000s user + 0.070000s system = 43.410000s CPU (390.6%)\n",
      "\n",
      "Kmer len: 51\n",
      "Raw kmers: 110850411 Memory needed (GB): 3.19249 Memory available (GB): 5.59987 1 cycle(s) will be performed\n",
      "Distinct kmers: 3750302\n",
      "Kmer count in  11.375042s wall, 40.340000s user + 2.560000s system = 42.900000s CPU (377.1%)\n",
      "Uniq kmers merging in  0.137987s wall, 0.190000s user + 0.090000s system = 0.280000s CPU (202.9%)\n",
      "Kmers branching in  3.470363s wall, 11.550000s user + 0.010000s system = 11.560000s CPU (333.1%)\n",
      "Kmer: 51 Graph size: 3750302 Contigs in: 694\n",
      "Valley: 4\n",
      "\n",
      "Mark used kmers in  1.091156s wall, 4.030000s user + 0.010000s system = 4.040000s CPU (370.2%)\n",
      "Kmers in multiple/single contigs: 0 284465\n",
      "Fragments before: 1649 3374821\n",
      "Fragments after: 1622 3374821\n",
      "New seeds: 23\n",
      "New seeds in  6.244703s wall, 12.080000s user + 0.000000s system = 12.080000s CPU (193.4%)\n",
      "Fragments before: 1428 112758\n",
      "Fragments after: 1282 112758\n",
      "Connectors: 146 Extenders: 1136\n",
      "Connections and extensions in  0.853759s wall, 3.280000s user + 0.000000s system = 3.280000s CPU (384.2%)\n",
      "Contigs out: 571 Genome: 5593700 N50: 48488 L50: 34\n",
      "Assembled in  8.206114s wall, 19.410000s user + 0.010000s system = 19.420000s CPU (236.7%)\n",
      "\n",
      "Seed kmers: 0\n",
      "Cleaned reads: 1019923\n",
      "Reads for connection: 0\n",
      "Internal reads: 0\n",
      "Reads cleaned in  10.149802s wall, 38.580000s user + 0.090000s system = 38.670000s CPU (381.0%)\n",
      "\n",
      "Kmer len: 63\n",
      "Raw kmers: 90759284 Memory needed (GB): 2.61387 Memory available (GB): 5.50473 1 cycle(s) will be performed\n",
      "Distinct kmers: 3621751\n",
      "Kmer count in  8.423702s wall, 30.800000s user + 1.650000s system = 32.450000s CPU (385.2%)\n",
      "Uniq kmers merging in  0.133562s wall, 0.200000s user + 0.070000s system = 0.270000s CPU (202.2%)\n",
      "Kmers branching in  3.070765s wall, 11.220000s user + 0.020000s system = 11.240000s CPU (366.0%)\n",
      "Kmer: 63 Graph size: 3621751 Contigs in: 571\n",
      "Valley: 3\n",
      "\n",
      "Mark used kmers in  1.112410s wall, 4.040000s user + 0.010000s system = 4.050000s CPU (364.1%)\n",
      "Kmers in multiple/single contigs: 0 221678\n",
      "Fragments before: 1703 3317492\n",
      "Fragments after: 1677 3317492\n",
      "New seeds: 22\n",
      "New seeds in  5.667326s wall, 11.270000s user + 0.020000s system = 11.290000s CPU (199.2%)\n",
      "Fragments before: 1180 87628\n",
      "Fragments after: 1114 87628\n",
      "Connectors: 66 Extenders: 1048\n",
      "Connections and extensions in  0.919667s wall, 3.520000s user + 0.000000s system = 3.520000s CPU (382.7%)\n",
      "Contigs out: 528 Genome: 5633885 N50: 55195 L50: 25\n",
      "Assembled in  7.714550s wall, 18.850000s user + 0.030000s system = 18.880000s CPU (244.7%)\n",
      "\n",
      "Seed kmers: 0\n",
      "Cleaned reads: 965589\n",
      "Reads for connection: 0\n",
      "Internal reads: 0\n",
      "Reads cleaned in  8.725668s wall, 33.150000s user + 0.080000s system = 33.230000s CPU (380.8%)\n",
      "\n",
      "Kmer len: 73\n",
      "Raw kmers: 76271154 Memory needed (GB): 2.92881 Memory available (GB): 5.41171 1 cycle(s) will be performed\n",
      "Distinct kmers: 3550928\n",
      "Kmer count in  8.362512s wall, 30.390000s user + 1.860000s system = 32.250000s CPU (385.6%)\n",
      "Uniq kmers merging in  0.181149s wall, 0.220000s user + 0.110000s system = 0.330000s CPU (182.2%)\n",
      "Kmers branching in  3.502357s wall, 12.290000s user + 0.020000s system = 12.310000s CPU (351.5%)\n",
      "Kmer: 73 Graph size: 3550928 Contigs in: 528\n",
      "Valley: 3\n",
      "\n",
      "Mark used kmers in  1.267043s wall, 4.470000s user + 0.010000s system = 4.480000s CPU (353.6%)\n",
      "Kmers in multiple/single contigs: 0 196128\n",
      "Fragments before: 2155 3278781\n",
      "Fragments after: 2118 3278781\n",
      "New seeds: 17\n",
      "New seeds in  5.890077s wall, 11.830000s user + 0.020000s system = 11.850000s CPU (201.2%)\n",
      "Fragments before: 1048 65266\n",
      "Fragments after: 1022 65266\n",
      "Connectors: 26 Extenders: 996\n",
      "Connections and extensions in  0.932789s wall, 3.490000s user + 0.010000s system = 3.500000s CPU (375.2%)\n",
      "Contigs out: 502 Genome: 5650347 N50: 55195 L50: 25\n",
      "Assembled in  8.102863s wall, 19.800000s user + 0.040000s system = 19.840000s CPU (244.9%)\n",
      "\n",
      "Seed kmers: 0\n",
      "Cleaned reads: 947955\n",
      "Reads for connection: 0\n",
      "Internal reads: 0\n",
      "Reads cleaned in  9.199953s wall, 32.820000s user + 0.100000s system = 32.920000s CPU (357.8%)\n",
      "\n",
      "Kmer len: 83\n",
      "Raw kmers: 65400829 Memory needed (GB): 2.51139 Memory available (GB): 5.29107 1 cycle(s) will be performed\n",
      "Distinct kmers: 3510662\n",
      "Kmer count in  7.392300s wall, 25.480000s user + 1.610000s system = 27.090000s CPU (366.5%)\n",
      "Uniq kmers merging in  0.183204s wall, 0.200000s user + 0.140000s system = 0.340000s CPU (185.6%)\n",
      "Kmers branching in  3.366712s wall, 11.790000s user + 0.050000s system = 11.840000s CPU (351.7%)\n",
      "Kmer: 83 Graph size: 3510662 Contigs in: 502\n",
      "Valley: 3\n",
      "\n",
      "Mark used kmers in  1.215076s wall, 4.290000s user + 0.010000s system = 4.300000s CPU (353.9%)\n",
      "Kmers in multiple/single contigs: 0 188953\n",
      "Fragments before: 3261 3221323\n",
      "Fragments after: 3220 3221323\n",
      "New seeds: 19\n",
      "New seeds in  5.699878s wall, 11.400000s user + 0.010000s system = 11.410000s CPU (200.2%)\n",
      "Fragments before: 1004 68041\n",
      "Fragments after: 966 68041\n",
      "Connectors: 38 Extenders: 928\n",
      "Connections and extensions in  1.059436s wall, 3.450000s user + 0.010000s system = 3.460000s CPU (326.6%)\n",
      "Contigs out: 469 Genome: 5665221 N50: 66619 L50: 23\n",
      "Assembled in  7.987275s wall, 19.150000s user + 0.030000s system = 19.180000s CPU (240.1%)\n",
      "\n",
      "Seed kmers: 0\n",
      "Cleaned reads: 929193\n",
      "Reads for connection: 0\n",
      "Internal reads: 0\n",
      "Reads cleaned in  7.556755s wall, 29.250000s user + 0.110000s system = 29.360000s CPU (388.5%)\n",
      "\n",
      "Kmer len: 93\n",
      "Raw kmers: 54818081 Memory needed (GB): 2.10501 Memory available (GB): 5.17022 1 cycle(s) will be performed\n",
      "Distinct kmers: 3438290\n",
      "Kmer count in  5.891445s wall, 21.450000s user + 1.320000s system = 22.770000s CPU (386.5%)\n",
      "Uniq kmers merging in  0.163709s wall, 0.190000s user + 0.120000s system = 0.310000s CPU (189.4%)\n",
      "Kmers branching in  3.248985s wall, 11.600000s user + 0.030000s system = 11.630000s CPU (358.0%)\n",
      "Kmer: 93 Graph size: 3438290 Contigs in: 469\n",
      "Valley: 0\n",
      "\n",
      "Mark used kmers in  1.122547s wall, 4.330000s user + 0.050000s system = 4.380000s CPU (390.2%)\n",
      "Kmers in multiple/single contigs: 0 181400\n",
      "Fragments before: 5066 3006230\n",
      "Fragments after: 5022 3006230\n",
      "New seeds: 19\n",
      "New seeds in  5.325419s wall, 10.970000s user + 0.010000s system = 10.980000s CPU (206.2%)\n",
      "Fragments before: 946 73264\n",
      "Fragments after: 911 73264\n",
      "Connectors: 35 Extenders: 876\n",
      "Connections and extensions in  0.931068s wall, 3.620000s user + 0.020000s system = 3.640000s CPU (390.9%)\n",
      "Contigs out: 445 Genome: 5683412 N50: 73615 L50: 22\n",
      "Assembled in  7.392741s wall, 18.930000s user + 0.080000s system = 19.010000s CPU (257.1%)\n",
      "\n",
      "Seed kmers: 0\n",
      "Cleaned reads: 881430\n",
      "Reads for connection: 0\n",
      "Internal reads: 0\n",
      "Reads cleaned in  6.544859s wall, 25.260000s user + 0.090000s system = 25.350000s CPU (387.3%)\n",
      "\n",
      "Kmer len: 103\n",
      "Raw kmers: 43186216 Memory needed (GB): 2.07294 Memory available (GB): 5.05179 1 cycle(s) will be performed\n",
      "Distinct kmers: 3287256\n",
      "Kmer count in  5.370807s wall, 19.490000s user + 1.370000s system = 20.860000s CPU (388.4%)\n",
      "Uniq kmers merging in  0.185041s wall, 0.230000s user + 0.170000s system = 0.400000s CPU (216.2%)\n",
      "Kmers branching in  3.426770s wall, 12.050000s user + 0.000000s system = 12.050000s CPU (351.6%)\n",
      "Kmer: 103 Graph size: 3287256 Contigs in: 445\n",
      "Valley: 0\n",
      "\n",
      "Mark used kmers in  1.225223s wall, 4.770000s user + 0.010000s system = 4.780000s CPU (390.1%)\n",
      "Kmers in multiple/single contigs: 0 170798\n",
      "Fragments before: 6017 2262430\n",
      "Fragments after: 5970 2262430\n",
      "New seeds: 4\n",
      "New seeds in  4.422070s wall, 10.550000s user + 0.020000s system = 10.570000s CPU (239.0%)\n",
      "Fragments before: 857 69179\n",
      "Fragments after: 838 69179\n",
      "Connectors: 19 Extenders: 819\n",
      "Connections and extensions in  0.995092s wall, 3.680000s user + 0.000000s system = 3.680000s CPU (369.8%)\n",
      "Contigs out: 421 Genome: 5689476 N50: 77530 L50: 20\n",
      "Assembled in  6.658079s wall, 19.020000s user + 0.030000s system = 19.050000s CPU (286.1%)\n",
      "\n",
      "Seed kmers: 0\n",
      "Cleaned reads: 766662\n",
      "Reads for connection: 0\n",
      "Internal reads: 0\n",
      "Reads cleaned in  5.743174s wall, 21.600000s user + 0.120000s system = 21.720000s CPU (378.2%)\n",
      "\n",
      "Kmer len: 115\n",
      "Raw kmers: 28363910 Memory needed (GB): 1.36147 Memory available (GB): 4.91326 1 cycle(s) will be performed\n",
      "Distinct kmers: 2910180\n",
      "Kmer count in  3.376286s wall, 12.290000s user + 0.730000s system = 13.020000s CPU (385.6%)\n",
      "Uniq kmers merging in  0.162668s wall, 0.180000s user + 0.130000s system = 0.310000s CPU (190.6%)\n",
      "Kmers branching in  2.835970s wall, 10.100000s user + 0.010000s system = 10.110000s CPU (356.5%)\n",
      "Kmer: 115 Graph size: 2910180 Contigs in: 421\n",
      "Valley: 0\n",
      "\n",
      "Mark used kmers in  1.210397s wall, 4.510000s user + 0.020000s system = 4.530000s CPU (374.3%)\n",
      "Kmers in multiple/single contigs: 0 151133\n",
      "Fragments before: 2312 537464\n",
      "Fragments after: 2287 537464\n",
      "New seeds: 2\n",
      "New seeds in  2.526493s wall, 8.660000s user + 0.010000s system = 8.670000s CPU (343.2%)\n",
      "Fragments before: 790 68141\n",
      "Fragments after: 767 68141\n",
      "Connectors: 23 Extenders: 744\n",
      "Connections and extensions in  0.985286s wall, 3.720000s user + 0.010000s system = 3.730000s CPU (378.6%)\n",
      "Contigs out: 391 Genome: 5694579 N50: 82083 L50: 19\n",
      "Assembled in  4.735654s wall, 16.900000s user + 0.040000s system = 16.940000s CPU (357.7%)\n",
      "\n",
      "Seed kmers: 0\n",
      "Cleaned reads: 737280\n",
      "Reads for connection: 0\n",
      "Internal reads: 0\n",
      "Reads cleaned in  4.740221s wall, 17.810000s user + 0.070000s system = 17.880000s CPU (377.2%)\n",
      "\n",
      "Kmer len: 125\n",
      "Raw kmers: 19904097 Memory needed (GB): 0.955397 Memory available (GB): 4.79299 1 cycle(s) will be performed\n",
      "Distinct kmers: 2291676\n",
      "Kmer count in  2.295672s wall, 8.660000s user + 0.270000s system = 8.930000s CPU (389.0%)\n",
      "Uniq kmers merging in  0.128996s wall, 0.160000s user + 0.090000s system = 0.250000s CPU (193.8%)\n",
      "Kmers branching in  2.277360s wall, 7.960000s user + 0.020000s system = 7.980000s CPU (350.4%)\n",
      "Kmer: 125 Graph size: 2291676 Contigs in: 391\n",
      "Valley: 0\n",
      "\n",
      "Mark used kmers in  1.213047s wall, 4.460000s user + 0.020000s system = 4.480000s CPU (369.3%)\n",
      "Kmers in multiple/single contigs: 0 134211\n",
      "Fragments before: 750 78053\n",
      "Fragments after: 735 78053\n",
      "New seeds: 1\n",
      "New seeds in  1.761761s wall, 6.570000s user + 0.020000s system = 6.590000s CPU (374.1%)\n",
      "Fragments before: 716 64634\n",
      "Fragments after: 683 64634\n",
      "Connectors: 33 Extenders: 650\n",
      "Connections and extensions in  1.034233s wall, 3.850000s user + 0.010000s system = 3.860000s CPU (373.2%)\n",
      "Contigs out: 355 Genome: 5699612 N50: 88117 L50: 16\n",
      "Assembled in  4.019683s wall, 14.890000s user + 0.050000s system = 14.940000s CPU (371.7%)\n",
      "\n",
      "Seed kmers: 0\n",
      "Cleaned reads: 724136\n",
      "Reads for connection: 0\n",
      "Internal reads: 0\n",
      "Reads cleaned in  3.982151s wall, 14.810000s user + 0.100000s system = 14.910000s CPU (374.4%)\n",
      "DONE\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "skesa --memory 8 --cores 4 --reads  samples/samples_AS00000197_file-0.fastq.gz samples/samples_AS00000197_file-1.fastq.gz >output/contigs.fasta\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Assembly file is stored in output/contigs.fasta"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">Contig_1_1002.32_Circ\n",
      "AAAAAAAAACAGGAAAGGTTAAGCAAAAATTAGGAAAGGTTAAGCAAAAATTAGTATTTCCCTCCCAAAGGTTAAGTGAAAATTAGGTAATCCACACAACCACGCAAAAGGTTATGCCAGTGGAATTGACATAACCAATACGAAAGGTTATGTTAATTTAAAATCACAACCAATGGTTTTAGTCATGGCTGAATTGGTAGTATTCAAAGCAAATGAGCTTGCGGTTAGTCGTTATGACCTAACAGAGCACGAAACGAAACTAATCCTGTTTTGTGTTGCAAAGCTAAACCCCACATTAGAAACACCAGATGAAACTGAACGAACGGTTAAATTTTCATGCTCAGAATATGCTTCATTTATGGGGATAAGTTATGAGAATGCGTGGGGACGCTTAAATACATCAACACGAGAATTATTTAAGCGGTCTATTGAGTTAATATATCCAACAGGTGCTGTATCTATGCGTGTTTTTAATTGGGCTGAATATGCAGAGTTCAATAGGGAAACACAAGAAGTAACTTTAGTATTTAGTAAATATATAATCCCGTTGCTATTTCACCTAAAGAGATTCATTAAATACAACCTCGAGCATGTTAAGTCTTTTGAAAACAAATATTCAATGCGCATTTATGAATGGCTACTAAAAGAACTAACACAAAGAAAAACACATAAAGCCAATATTTCAATAAGTATTGATGATTTTAAATTCATGCTAATGTTAGAAAAAAGCTACCCGGAATATAAATTACTAAACCATTGGGTTTTAAAACCAATATCAAAAGATTTAAATACATATAGCAATATAAAACTATCTATTGATAAGCGCGGTCGTCCTGCTGATACGCTGATCTTTCAAGTTGAACTTGATAAACAAATTGACCTTGTGACTGAGCTAACAAAAGATCCAATATCAAAAAAAGAAGATGATTCAATCCGTTTAACGCCTGAAAATCGTCTGCATGAGGGGCTAAAAACAACATTGCATGATGCTTTAACTGCGAAAATACAACTTACTGGATTTGAAGCTAAATTCCTGAGTGACATGCAGAGTAAGTATGATCTCAACGGCTCATTTACATGGCTGACTCAAAAACAACGAGCCACGCTAGAGAAAATTTTAGCCAAGTACGGACGAATCTGAGGTATCTATGGCTCTAGTATCAATAAGTGAAGCAGCAAGACTAACTGGAAAAAGCCGCACAACAGTACAGGCATACATTAAGCAGGGAAAACTGACCAAGTGTACAGATAGCGCAGGCGCTTCAAAACTAGACACATCAGAGCTTCTACGTGTTTTCGGTTCATTCACTGGTCAGCAGGTTGGCAGTGAACAATCAAGCATCATTGTGCAGCACTTGGCAGGCGAATCTGTGCAGTTAACTGACCAGAATTTGCTACATGAAAACTATGCTTTAAAAGCTGAAGTAGAACTCGTAAAGGCATTGTTGCGTGAAAAAGAGCTGCTACTCAATGAAAAAGACAAGCGCAACGAGGATCTCAAGCAAGCGTTGTTACTGATAGAGTCCAAGCTGCCAGCAACACCAAAACCTGCCACGCAGCCGGTGGAAAAAAAATCATGGCAATTCTGGAAGAAATAGAGGTTTCAGCCTGTTTACTGGCTGAAACCATACCCCCGATTTTGATAACAAACCGGTAACACCAAAACAGCCCATTTTCGGGGATTAAAACCCGCTATTTTGGACGTTCCGGCGGGTTCTGGTGATGAGTAACAGGTCAGTGGTGCGCGCAAGATCCCTTATGTTAAATACCACCTGTTTTGGTGATTTTTTCCGCATCTGCATCAGCAGTATCGCTGCCAGGTTTCTGCTCAGGACGCGATGCCCTACAATCATTTCAATCTCTAACAGTCATAAGGTAATGACATGCAGACCGAATTAATAAAAGAACTGAACAAGGTTTTAAAAGTGTTTCCGCAATTCTGGAGCGGAGAAAATTTACACCGTTCAATGGTGAGTGACGCCATCAGCCAAAAGCAGCCAGATCTAATCAAAGCACTGGTCGCTAATAAAAAAATACGTTCTGTCTACGGCACAGATATTAACGGAACTTTAATTTTCGATTTTGACAAACTTTGCAGCCTGTTAAAATATAAAGAATACTGGGAAAACAGCTTTACTAAATACCGAAATAAGATTGGATTAACCAGCTCAGGTAAATACATTGCCGATAGTACTGATATAGTGCTTGATTTTCCTTTCAAGGATTGCCTTCTTGAAGGCGGAATGACAAAAGAAGAACAAGGGAAAGACGAGGTTTATTATAACGAGATTATTGCTCGCGATGAAATAGATAATCTTCTATCCCCAAAAGCATTGGTTAATGTATCTCGCCATACAAAGGATGGGGAAGTTAAAAACATTCATAGCTTCCATGAAAAAGATAACTTGATAATAAAGGGAAATAATTTAATTGCACTTCACTCACTATTGGGTAAGTTTACTGGAAAAGTTAAATTAATATATATAGACCCACCATACAATACAGGATCTGACAGTTTTAAATACAATGACAATTTCAATCATTCATCATGGCTAACATTTATAAAAAATAGACTAGAAGTAGCAAAAGAACTTCTTAGTGAAGATGGAGTCATCTTCGTACAATCAGACGACAATGAACAAGCGTATTTAAAAGTTTTAATGGATGGCATTTTTGATAGGGAAAACTTTATCAATATGATATCGGTTAACATGAAGAATGTTGCCGGAGCCTCTGGTGGAGGTGAGGATAAAAGACTTAAAAAAAACTGTGAGTTTATTTTAATTTACTCAAAAAACTATGCCTCCCTTTCTGCATTTAAAAATGTTTTTGAATACACAGAAATTCATGAGGTAGTTGAGCAATATAGGAAGGAAGGTAAAAATTGGCACTATTCCTCCGTACTGGTAAATGCAGGGGAAAAAATATACGTAACCTCTACTGTTGATGGCAGTGGGAACGAAATAAAGATATACAAACACCCTAACGCAGTAATTAAATCAGTTAGGCAATTGATGGCGCAGGAAAACATTTCAGAAAAGGATGTTTTTTATAACTTTGGAACAAAAATATTTGAAGCGAAAGATGCTCAATCATCCATTAGAACGCGAGTAGCAAAAGCAAAAAAAGAGCATCATGTTAATGATGATATATTATCAATAGAATACACACCTAAATCAGGAAAAAATAAAGGTCAAGTATATGAACAATTCTACAAAGGGGATAAATGTAGATTGTTTGCATGGCTTAGAGATATAACTGAAAATATAGATGGCATTTTATACAAAAAAGATAAACAAGGAACGTATTGGGATTTCACATCACACATGAACAATCTAATGAAAGAAGGTGAAATATCATTTCAAAATGGAAAAAAACCAGAGGCATTAATACAAAGAATTATAGATATTTCAACAAATAAAGATGATATTGTACTAGATTTCCATCTCGGAAGCGGGACAACAGCTGCAACAGCACATAAAATGGAACGTCGTTACATTGGCATTGAACAAATGGATTACATCAATGACATAACGGTTCCTCGACTAAAAAATGTAATCGCAGCGGAAAACAATGGGATTTCAAGAGACGTGAGCTGGCAGGGCGGTGGAAGCTTTGTATACGCCGAACTGATGGAGCTGAACGCGTACTTCGTGCACGAAATCCAGAAAGCGCAGTCCACTGAAGAACTCGAAAAACTCTTTGCTGTAATGAAAACCGAGGCGCATCTCAACTATCAGGTGGCACTGGAAAACGTACTCTCTGCTGAATACGAGATGGACGGCATTCCGCGTAAAGTTGCGTTCAGCGAACTGGAACTGCATGAGCAGAAGCAGCTACTGATTGATATCCTCGATAAAAATCAGCTCTACGTGAACGTTTCTGATATGGATGACAGTGACCTGAACATCTCAGAGAGCGATAAAGCCTTTACCCGCAGCTTCTACGGAATGGAGTAACGGCATGGCGAAGAAACCAAAGTCAGCCGGCAGCGAGAAGCTGCTGTTTAACAAGCTGAAGCAGTTCGATGAACCTGGGCTTTTCCAGGATAGCTACCAGACGCCGGACTACATTCAGGCGAACCTGAAGGATGCCCTGCGCCCGTACCAGCATGGGGCGCTGCGCTACCTGCATTACACTCAGCGCAAACGTGAAGAGGCGCTGCTGCATTACCGCCATCTGTTGTTTCATATGGCAACCGGTGCAGGGAAAACGATGGTGATGGCGGGGACGATACTCTATCTGTTTAAAGAGTACGGGTATCAGAACTTCATCTTTTTCGTGCATACCGACGCGATTATCCAGAAAACCCGAGAGAACCTGCTCAACCCGCAGTCGCCGAAGTACCTGTTCAGTCAGGAGCTGGAAATCAACGGGGAAAAAATCACCATTGCACCGGTGGATACGTTCCCGTCTGCACCAGAGCGCAACACGATTTACCTCAAGTTATCAACTATCCATAAGATGCACGACGAGCTGAACAGCTATCGTGAGAACAGCATCACCTACGAGGATTTAAAAGAAATACCGCTCGTCCTGCTTGGCGACGAGGCGCATCACTTCAACGCCGGCACGAAAGCGAAAGGCAAGTCAAAGACCTCACCGGAGAACGAGGAGCAGACCTGGGAGCGCACGATTGAGAACATCCTCGCTCTGAGACCGGATAACCGACTGTTCGAGTACACCGCCACCATTGACCTGGCAAACAAAGATATCGGAAATAAATACCGTGACAAAGTGGTGTATCAGTATGACCTGAAGCAGTTCATGTCAGACGGTTACTCGAAAAAGGTCATGCTACTGGAAGCCAACCAGAACGACAGCGAAAAAATGCTTGATGCCGTGCTGTTAAGTCAGTACCGCAAACTGATAGCGATAGATAACGGCATCACCGGCTTCAAGCCGGTCATTCTGTTCAAATCGAACAAGATTGCGATATCGAAGGCGAAGCAGGAAGAGTTTTCGCAACTGATTGCGGCAATGACCCCTGAAAGCGTCAGACGTCATCTGGCGAATAAAAGAGTCCAGCTCAGCTCTGACACCAGTATCTGGCATAAAGTGATTCAGCGTTACGCTGGCAGCGATTTAGTGACGGTTATCGGGCAGATTCAGGAAGATTTTAACGACTTCAACCTGCTGAACGTCAATAAATCCGACTTGCTGGAAGAAAACCCCGTCCTGCTGAATACGCTGGAAGAAGTCGATAATCCGGTGAGAGCCGTCTTTGCCGTGGCGAAGGTCAATGAAGGCTGGGACGTGCTCAACCTCTACGACATCGTCCGCATCAGCGAACAGGCGAGTAGCAGCAAAACCGGCACGGACAGCGAGGCACAGCTTATCGGGCGCGGCGCGCGTTACTTCCCGTTCGTATATGACGGCAAGCGGAGCTTCACCCGCCGATTTGATACCAGCGCGAAGGAGTTGAGCGTGCTGGAACAACTGCACTATCACACCATCAATGAACCGGCGTACATCAAAACGCTGCACGCATCTCTGGAGCAGGCGGACATTGACGTGCATCAGGACGGCGGCGGCACGATTGAGCACGCCAGACTGAAAGAGGACTTTAAAAAGTCCTCCGTATATCAGACAGGTAAGCTGTACTTTAACGAAGTGGAAGAGATTGAGAGCAGCAGTCGCAACTGGGAAACCTACTCGCTGGAAACGCGCTTTGAGATACCGTATCAGACCGCCGGTGAGGAGTCGTTGGACAACCTGACAGGGGCAACGGCTGGCATCACAAAGTCAGAACCCCTGCTACTGGACGAGCGTTTTTATCGCAAGGCCATGCAGAGGATCAGTTTTTACGCGCTGGATAACCTTCAACGCCTCTTCCCGAAACTGACCGGTATTCGTGAATTCATCCGCAGTGAGTCTTATCTTGGGAAACTGAAAATCACCGTTACCGTACCGCAGTCTCTGGATTTCGCCATCGTACCGGCAAAAGAGAAACTGCACCTGCTGGAAACCGTTCTGCTGCGTATCGCAGAGAACATGCGCCGCAATGACCAGAAAGTGAAAGGTACGTACCGTTTTATCAGCCAGCCGGTGAAAGAGGTGATTAAAGACTACAGCCTGCACATTGACCCGTCGGTAGTCATTAACCAGAAAATCACCGCCGTACCCACCATTGGTAAAAAATGGTACGTGTACGATAACGCGATACTGAATCAGCTTGAGCACCGTCTGGTGAAAACTCTGGAAGCCTTTATACCGAAGCTCAGAGCACGTTACGACGATATTTACGTGCTGCGTAACGACGAGCAGTCAACGCGCTTTAAGCTCACGGAATTTGGCGGCGTACGTGGCTTTATGCCGGACTTCATCATGATACTGACGTGCCATGCGGATAATGCTTACTGGCAGGTCTTCCTTGAGCCGAAAGGGGATGACCGTCTACTGGATGACGCATGGAAAGAGCGGATGCTCGAAACCCTGAATGACCGGGAACGCATTGTAATTGATGAAAATGAGGACGTCCGGCTGGTCGGGATTAAATTCTTTGCCAACAGTCAGATGGATGCCTTCGTCAGTGATATGCAGAACCGGCTGAATGAGGGGGAATCGCTTGAAACCGCATCGCTCTCCCTGCCACTGTAAACGGTGACAAATACAAATAACGGCGCACAGTGCGCCGTTATTTTATTTTTTGATAATCATCCAGGTATTCTCTTTATCACCATACGTGGTGCGAGTGTCCACCTTAACGCAGGGCTTTCCGTCACAACGCGATATATCAGCTAGCGGGGCTTTCTTCTGCCAGACGGCTTCCATCCGCTGCATTTCAGCAATCTGGCTGTACCCGTCATTCATAAACCACGTGAATGCCGTCACGCAGGCAGCCAGAACGAAGAATATCGTCAGCACAAGATAAATCGCGGATTTCCACGTATAGCGCGACATCTCGCGGCGCATTTCATGGATCATTGCCTTCGCCGTATCGGCAGCCTGATTAAGCGCTTCTGTCGCTGGTTTCTGCTGTGCTAATCCGGCTTGTTTCAGTTCTTTCTCAACCTGAGTGAGCGCGGAATTAACCGATTTCCTGACGGTTTCAGCCATATTACCGGACGCGCTGTCCAGCTCACGAATGAGCCTGCTCAGCGTTTCACTCTGCTGCTGTAATTGTGCTGAGGCGTCCTGAAACTGCTCAGTCAGAGAGATAACACGCTTTTCCAGCGCCTGATGATGCCCGATAAGGGCGGCGATTTGTTTAATTTCGTCACTCATGCAAAATCCTGTCTATCGTGAAAATGACCAGCCTTTATCCGGCTTCTGTTGTGTATGTTCGGCAAGTCTCTGCCGTTCTTTCTCCTGCTGACGCTCGCGTTCTTTCTGTCGTTCCATTTCCTGATGTATCCCCTGAAACTCCGCCATTGCATCGTTAACCAGGGACTGAAGGTCGATTTCTTCCTGTATATCCTTCATGGCATCACTGACCAGTGCGTTCAGCTTGTCAGGCTCTTTTTCAAAATCAAACGTTCTGCCGGAATGGGATTCCTGCTCAGGCTCTGACTTCAGCTCCTGTTTTAGTGTCAGAGTATCCCTCTCGCTGAGGGCTTCCCGTAACGAGGTAGTCACGTCAATTACGCTGTCACGTTCATCACGGGACTGCTGCACCTGCCTTTCAGCCTCCCTGCGCTCAAGAATGGCCTGTAGCTGCTCAGTATCGAATCGCTGAACCTGACCCGCGCCCAGATGCCGCTCAGGCTCACGGTCAATGCCCTGCGCCTTCAGGGAACGGGAATCAACCCGGTCAGCGTGCTGATACCGTTCAAGGTGCTTATTCTGGAGGTCAGCCCAGCGTTCCCTCTGGGCAACAAGGTATTCTTTGCGTTCGGTCGGTGTTTCCCCGAAACGTGCCTTTTTTGCGCCACCGCGCTCCGGCTCTTTGGTGTTAGCCCGTTTAAAATACTGCTCAGGGTCACGGTGAATACCGTCATTAATGCGTTCAGAGAACATGATATGGGCGTGGGGCTGCTCGCCACCGGCTATCGCTGCTTTCGGATTATGGATAGCGAACTGATAGGCATGGCGGTCGCCAATTTCCTGTTGGACAAAATCGCGGACAAGCTCAAGACGTTGTTCGGGTTTTAACTCACGCGGCAGGGCAATCTCGATTTCACGGTAGGTACAGCCGTTGGCACGTTCAGACGTGTCAGCGGCTTTCCAGAACTCGGACGGTTTATGCGCTGCCCACGCCGGCATATTGCCGGACTCCTTGTGCTCAAGGTCGGAGTCTTTTTCACGGGCATACTTTCCCTCACGCGCAATATAATCGGCATGAGGAGAGGCACGGCCTTTCCCGCCGGTTTTTACGCTGAGATGATAGGAGGCCATCGTGTTTTTATCCCGCTGAAGGCGCGCACCGTTTCTGAACGAAGTGAAGAAACGTCTAAGTGCGCCCTGATAAATAAAAGAGTTATCAGGGATTGTAGTGGGATTTGACCTCCTCTGCCATCACTGAGCATAATCATTCCGTTAGCATTCAGGAGGTAAACAGCATGAATAAAAGCGAAAAACAGGAACAATGGGCAGCAGAAAGAGTGCAGTATATTCGCGGCTTAAAGTCGCCGAATGAGCAACAGAAACTTATGCTGATACTGACGGATAAAGCAGATAAAACAGCACAGGATATCAAAACGCTGTCCCTGCTGATGAAGGCTGAACAGGCAGCAGAGAAAGCGCAGGAAGCCAGAGCGAAAGTCATGAACCTGATACAGGCAGAAAAGCGAGCCGAAGCCAGAGCCGCCCGTAAAGCCCGTGACCATGCTCTGTACCAGTCTGCCGGCTTACTTATCATGGCGGGACTGGTAGACAGCCAGACAGGGAAGCCCGTCGATGATACCGCTGCCTTACTGGGTGCATTAGCCAGTCTGAATGACCTGTCACGGGATAATCCGAAGTGGTCAGACTGGAAAATCAGAGGGCAGGAACTGCTGAACAGCAAAAAGTCAGATAGCACCACATAGCAGACCCGCCATAAAACGCCCTGAGAAGCCCGTGACGGGCTTTTCTTGTATTATGGGTAGTTTCCTTGCATGAATCCATAAAAGGCGCCTGTAGTGCCATTTACCCCCATTCCCTGCCAGAGCCGTGAGCGAAGCGAACTGATTTTCACAGAGATAACAGCGATTCAGGTGCCGGACGGTCGGAGACAAAAGGAATGTTCAGCGGGTTGCCCGAGCTTGCGAGGGTGCTATCTCAGTGTTTTTGGGGGTTAAGGTCTGTTTTGAAGAGAAGCAAACAGCGTTTGCGACATCCTTTTATCATATCGCTCAAGTGACTCAGGCGTGCAGTTATCCACAAGGCTGGGATCTATTCTTTTTTATCTTTTTCTATTCTTTCTTTATTCTAAAGATTTTAAGTTATTGTTAAAAAAC\n",
      ">Contig_2_240.811_Circ\n",
      "AAAAAAAAATACATTAGAGCAAACAGGTTTTGCTCTAACGACTACAGGTATTAAATACTGCGAAAAGTTTTTTAGTTTTGAGGATGTGATCGAAACCAGAGCTGTAAGACAGGTGTTAGAACAGCGAACGGTTTTAGTTAATTCAACTTTTCACTATTCTATTTCTATTGTGATGATTACTAAATCAGGTGAAATGCTTCAGGTAACGGAGCAACCGACAGTGTTTTCTGATAGTAAGCTGAGTAAGGTCGAATATATTGAAGATTTATTTAAAGAAATCTCCAGAAAGACTTGGGATACTCGTCTAAAAAAATATATAAACCAAGTTAATGAAAAAGGTTTTTTTGAGTATGACAACTGGCGCTTTTATACCAATCAGAAAGTAATTCAAGATTTAAAATCAGATAAAGTTTATCCAATTGAAACAACTGAACTTGTAAAACAACCGCATAGTATTTTAGTCCAAGAAAAAAATATGGGTATTGGTATGAAATTAATGAAAAGTTTTGTTGGTAAACCTCAACAAATTACAACAATGACAGATCCAGATATATTCTTTTTTATACTTCGTCATTACTTCGGTCTTGTGTGGTCTAAGTAGGATTTTATGATGTTAATTTTGATTGTAATAACGGGCCTGATAATCATATATTTTTGGTTTAAAGCTAAAGCAAGAGTGCAAAGTACTCAATTTGGTGTTGAAGCTAGATATATTGCATGTAATGAGTTGGAAGTTCCGAATTGTTATTTTAACTATATGGTTCAAAATGAAATTGAAGATATAAAGAAAGCAGCATTAAAGTTACAAGAAATTGAACCATTTTATAGTTCATTAAGTTGGTCAAGATTATTAGCTTGGACTATTTATGGTGGATATAAATATGATTGCAATCATGCTTACTTCAATGAAGATATTCCTGCAATGCAGCGTTTAAAAACATCAGGTGTCACATACGAACAGATGGTTGAAGAAGCTAACAAAGAACATAAAGCGGAAAAACATTTAATAAATCATTAAATGGATCAAGGCAAAAGATCTAATGAAGCGTATTAAATCTTTTGCCTTTTAAAGGGTTATACTTTCAAATACAACTGTAACGGATTACTTACAGTTGATTCAAAGCCATATTTTTCATAATAAAGTTTGGCATCGTGGTTAAGTGCATCTACAACGATAAATACACATCCACCATTCTGCTTTGATATCTGTGCTCGATGGAATACTTCAGCCATCAAAATTTTGCCAAAGCCCTTATTCTTAAAGTGTTTATCAATAGCTAATCGGGACAGGCGTAAAATTGGAACCGTATTAGGGAGCTTTAACTTCTTTTGGGTTGCTTCTGCTAAATATGAACAGTCAAGTTGTCCATGAGTCAGAGAATAATATCCAATTAGCGGATGTAAAGGGTCTGTTGACGGAAGTAGAAATGAAGGTGTAATACCTTTCTTATTATGTTGCCCCATTGTTTGTACTAAATATTCATTTAGTGATGGTTCCCCACAATCAAATTTCTCACGAAAGTCATTGATCTCCTGTTTTGCCATATCGGCGTATGGAATTGCTAGTGTTACTGACATTGTTGGTTGTCTCCTGATATAGACGAAGAGCATCCTGTAAATAAGGATTCACGATTGCTGGCTCTTCTAATTTTGCTTTTAACCAACCAGCTTGATCTTCAGTCACTTTGATCCACTGCATATTGTGTTTATCAATAGTATCTTGTGCCTGTTGAAAAGCGGATTGAACCATAAAATTATTTATTGTTTGACCTGTGACTTCAGCAGCTTTGGATATAACTGCATAAATCTCATGGCTAACCCTTGCTGAAATTCTTTCTCTCGAATTGTCATGCAATTTGGTATTGATCACTTTTGAACTCCTAATTGAATTGAATGTTTATCTTGTTTAAGACTGAAACTGTTAAATCACAGTGATTAGAATGAAGTAACTGTTTTTCATCGGCGTTGGACTATTAAAAATAGTGTCACGCTAAGATGAATGGATTATACATCTTTTACCTAGTGGTGCCAAAATGGCACCACTTTAATTTGAGCATATTTTGATCTGTTGTACAATTTTTTTAATGTAAAAAAGAAAAAGCCCAATAACTCTTGCAAGTTATTGGGCTTTTCCTTTAAATTCAGTGGGTACAGTTATTTGATGCTTCCAACATCAATTAACGAAAGATCGGCAGTGCTTCCAACACGTACTGATATTCACCTGAAAGGCTAGTACCACTATCTATTTTAAAGAATTAGAACAGTTTGACAAGCCTCAATTTGAAGTCCTTCGTTAATTCAGCAGTTTAAATCATCAAATAAGCACCTAACGGTCGAGCTTTATCTACTTCTGTGCGCCCTTTTAAAAGGAAGATATGCGGTGCATGTGGAGGGAATAAGTGTCACTGGTGCATCGTAGCGATAGTCGCAAGGCTAAACCGCTTACAGGGGATCTCACATAGGCGGAGCGTTGGAAGTAGTTGTATTGGTTGGTTAAGTTGTATGTTGGAGAAATCCTCGCAGAGATATCACCATACTTCGACAGGGAGCGCGTGGGAGAAGGTGCAAAACCTTTAGTCCTTGTCTGCATGGTGTTGCAAAACCATGTGCAGATACGACCGATGCGTGGCTCCGTCAGCCCAATTACAGGTTAATCGGCAATACTTAAGGGACGTTGATTTTTGAATCATCGCCCTTAGGGAAAGTATTGCCGAAACTCTCTCAACGTTCACCATCAGCCCGAGCAAAGCGAGGGATGTGGGGACGGTGAGAAAGAGTTGAGCGGCTTTATTTTTACAAGATTCATGCCAATACTTCCTAGAGTTTATATACCGTGTCATTTGATTTAATTTTCATATAATAACTCTGTCCATTTTTTAATAAGCAAGGATACACAGCTACCCCTATTTAACTACGTTAAATGTGGTAAGCTGTGACCTTGATCAGGGGCTTCACCCCCGATACCCCCGAAAAGCTTTAAACAGAAAAATCAAAGATTTTCCAATTTAAAGCTTACAAAAGCCAAAGCAGGGTAGATTGAAATGTCAGGACAGGAACAAGAAAACAAACAGAGCAAAGCAGTTCGAGAACGAACCTTTAAAGTCAGACTTTCAGAATCAGAATATCAATTGCTTGATTCCAATAATCCTTATGGTTCGATTGCAAAGTTGCTTAGACAAGCAGCAGTAGAGAAAGCGACTAAAATTCATAATTCCAAAGCAGAAGATGACCAGAAAATTAAAGCAACAAAGGTCGTTGAATCTTCGTATAGCAAGACAGAGCGAGCGTTAATTTTAGAACTTAGTCGGATTGGTAATAATGTGAATCAGATTGCGAAAGCAGTGAATACTGATATTGCAGGTACTGGTGCTTTCGATAAAGTGAAGTTACTTCATTTATTGATTTCGATTGATCAACAGCTTCGGGAGTTACGTCCAGATGATCGTTGATTTCTTTCGACATGGTTCAGGTCTATCAAAAGGCTGCCTTGACTATTTACTGGGTGAAGATCGAGAGCGTGAACACGCACAGGTGCTTAGTGGTGATGTAGAGCTGACAGGGCAATTGATTGATAGCAGTCCCTTTGCCAAAAAATATACTTCAGGCTGTTTGTCCTTTTATGAGCATGATTTATCTGACCAGGATAAGCAGAAGATCATGCAAAATTTTGAGCAATGTCTTTTCCCTGGTCTTGATCAAGATCAGTATCAAATTTTATGGGTACAACACCAAGACAAAGTAAATCAAGACACAGGACAGACAAGGCTTGAGCTTAATTTTGTGATTCCCAATGTTGAGCTTTCAACAGGTAAACGTCTACAGCCTTTCTATGCACCAGTTGATCTCGATCGAGTCGATCTATTTAAAAAAATCACCAATACCCAACACCGACTTTATGACCCTGATGATCCTGAGCATCGACAGCTATTTTTAAATAAAAAGAACCTACCCAAAGATATTAAGGATTTTAAAGAGCAGTTACATCAACGTGTTTACCGAGCGGTTGCCAATGGTGATGTTGCTGATCGGCAAGAATTGGTGCAATGGCTTGAATCGAATCAAATCAATGTCACCCGACAAGTAAAAAACTCCATATCCATTGAAAACCCGTATGAGGGCGCAAAACGTCCTATCCGTTTAGAGGGCGAGATTTATGAGCAAAGCTTTAGAGCTACTGGCGAATATCGACAAGAAGTACAACAACGAATTGAAGAATACAGAGGAACAACTTCTGAGCGGTATCGAGCAAACGTCACAGACTATCAGAGACAGCTTGAACACAAGAGCCAATATCACAGCGACCGCTATTCAACAGTCGGACGAGAAAATAGCCCAGAGCATTCAAAACAGCACCCAGACTGTCGAGAAACAGTTAAACCAGTATCAAACCTTGCTGCAATCGAAATTGAGCCGTTTAACGCAATTAAACGAGCAAATACAGAGCCAAGAACAACAAGCCCAGAATCAAGCGGTACAGAAAAAACTTACCATTTTGAATATGGGGCTGATTTCAGCAGCAGTTATTTTACTTATAGCGATTTTCTCGCTTGGTCTCGTCACCAAAAACAAGTACAGCGAGATGCAGACGCTAAATACAGTGATCAACGAGAAGCAAATGAGGGTCGACCATTTGAATCGATCGGGGGGCAGTTTGACGATCAGCACATGCAAGCGACCAGACAGCAAGGCACAACGTCTCTGTATTCAGATCAACAAGAAAGCGGGCGAATGGCAGAACGGTTATATGATTCCAATGGGGTATTAAACGATGACCGAATTAGAAACACAGTTATTGAAGATCATCGAAGAACAACAGCAACAATTACTGCAACAACAAGCGCAATTGCAGAAGCAACAGCAAGCTTTAGACACAATGCAAACCAAGATTATCCAAGCCTTGTCAGCTCAATCAAAGGCAATCGAGAGCGATCAGCAGAGCTTACAGCAAGCACAGAAATCATTATCGACAATACAGAAGCAATTTCACGAGCTAGAACGCAGTACAGCAAGCTATACCGAGCGGATCAATGGAAAAGTTCGAGAGATGAGCCAGATCACAGCCAACAACGAAGAAGTGCTGGAGAGAATCTCAACCGTACAGCATTCAGTCTCGTTACAAGTGACCAAAGCTCAGCAGAATTTGCTCGAAACGTCAGAGATATTGAAAAAAATTTCGTCCAGATAAAAGAACGGCAAAAGGAAATGGACAAGCCACAACCTAAGCAAGATCGAGGGATAGATTTTGGGATGTAATATTTTTAAGGCAGGCTTATATGTAATCATTGAAAATGATAAAATTTAATAAATTTGTATATAAAATAGGTCTTTACCTATTAAGAGAAATATATGGCTGCACCACTTAG\n",
      ">Contig_3_186.434_Circ\n",
      "AAAAAAAACCCCGACATCCTGTCGGGGTTTTTCGTATTTGGTACTAGGGTCAACTCCTGTCATCCCTCGCGTCCTGTCGCTGATCTTATTATTGTTATCTGGAACATCCTGTTCTCAATGAAGCCATGATATGCAATCTGGTTGTGTCTGCCTATTGGTGATTTTCTTGAATGCCTGTACGCCAGAACGTACATGCAGATATCCTCAAATTATCCCAAGAATTTTCCGCAGGAAATGGGGATAATTTCAGAAATGGAGGCCCTGAGAAATCAGGAAAAATTGAAATTTTGAGCATTTTTTAACCGATTTTATTGGCAGAATATGACAAAGTTGCTTAAAAAGCAGCCGACTTATTTCTGAGTCGGCTTTTTGGTGTTTTTTGTGATTATGTAAAATTCACAAATAATTGATTTAATTAAAATAATTAAGAGTGGTTTCGCATAACGGCCATTATGTTAAATCAGCAGTCAGAGCTTGGATCTTCCAGCTGTAAATAAGTCTCGAATTTCCCTCTAATACTGTCAGGTAAAGCTAAGTCGTCTATATTTTTTTGACTTGCCAGCATGTATGTTTTTAGAGCATCAAAGGCCTTTTGATCATCTAATAACATATGGATAAATTTACTTCGATTGATTCCCATATCTTCACAAAAAACATCCAATTTCATGAGATCTGGCTCAGGTAACTTAAATGACATTGCGAGAGTTTTTACAGACTTTGCATCTAGCTCTGTGGCATGAAGGAGACCTAGTTCTTCAAAATAATTACTCATTAGCTTGCCCCTTCTTTTGTTGATTCAGTATTTCATCTGCTAAATCCGTAGCTCGGTCTATGAAGCCCATTTTTGCCAGCATGAAGATATGGAAATACTCATTTCTTGAATCTTTGAGTTTATTTTTTATCTTAATTAAATGGTCCTGTACTGCATCAATATCAGTATTTTCTTGATTTAAATCACCAATCTGATAAAGGCCTTTCAGATATGAAAGATAAGCTGATTCTATATAAGAGCTAACTATTTCTTGAAGAACTTTACTCTTGCTTTTGCCTATAGCATTTGCAACTGATTCAAGAATCATATTCTGATCTTCGGAAAGTTTAATTGACAGCCCTTTCATTGCATTAGGCTCAAGATTGATACGAGTATTCAAACCTAGTTCGACAAAAGCACTCAGCCCATCGATAGACATTTCCCTTTTAGCTCGATGTCTATAGGCCAACTCTTCTAAATCTAATTTTTCAACCATTTGTTTTATCTATCAGTTTTTGAAATTGAATAAATTATAGCACATTAAATATAAGTGTAGATACAAATGTAAATACTTTTTGTAATTAAAAAAATTACAAAAAGTATTTACAAGTTTTTGTGTGTGCTATAGGATGAAAAACATACAGGCAATAGGCAGAAATAAAATGATTAAAACGAACCATTTCATGAAGAGAACTTCTCAACGAGGAATTACCAAAGCAATGGTTAACTATGCTTTAGAGCATGGTGAAATCGATGGAGATAAGGTGATCGTGAATAGAAAATTGATCCTTCGTCATCGAGAAGAAATGCTACGTGAATTAGCTGTGATGGCTAAGCTCTTGGATAAAGGCGGAATCGTTGTTGTTAGTGAATATGAGTCTTTAATTACTGCCTACGACTTTGATAGTCGTAAAAACAAATATTCTTAAGGAATAGTGTCATGAACAAGAAATATGAAAATGAAGACAAAATCTATGTATGGCGTTCTGATGATGCTTTGAATAGATATCAGAAAGAAGAATCCTCTGATATTCCATACGATGATGATGAGCCATACGAAAATCAAACAGAACCTAATAACCCCCCTCCCAGAAAACCACTCACACAATCATTAACAGGGGCTAACAGAGGATTTATATCTAATCATTCTGGATCGGATTATGAAGGTTCTGAAAACTATGAACATGATGATAATGGAGATCTTATGAACTACTCAGGTGTATACGATAGTGTAATTAATGACCTTAGTGAGCATGAGGGTTATCAGGAAGGCTATGATTATTGAGTTTTGCTACAATTAACATAATACGCCTTATGCGAAGTCAATTTGTTAAATTAATTTAAATCAATACCTTAAGTAATTTTACTCAGCCCAATCAATCCAGATAGGGCTTTTTTATGGATTCTTCACGTTCCACGCCTAGTATTTAACCATTGTTCCACAATTTTGATTAGCTACAGTCAACAATGCTCTCACCCATTCGGTAAGAAAAAGCGACTGAGAAAGCCTCTTTTCGGCTGCTCTGGTTCTACGTGCTCTGGCACTGGAATACGCTTATTTTCTGTTTCGACTGGAGTAGTCAATCCGTCATCTTTCGGCTCGGGTTCTGACCGAGGATCTGTTGCTATATCTGGAGCAGATTTCTGATCGGTAAACGTAGTCATATTGGATTGTGGGGGCTTCATTTCAATGAGTGTCAGCAGTGTATCTATACGAGCATTTGCCTTAGCTTCACGATCTTCGGCTTTTTCTAACTGTTTTTTTAGAAAATCAATCTGGTTTTTGAGTTCTACAACGGTTTTATCAGTGTATTCAGATGTATTTTGATTTTGTATTGGAGACGTATCGATACGTTTTACTTGTTTTACTCCAAAAGGTTCTCCATAGACACGTAGAAGTTCTGAAAAATCTATAAAACCATCATGGTCACGGCTTAACACACCGTTTTTTATATCTTTATATAGCGTAGCTCGAGATTTTTTATACAGGGTTGCAGCATCACTAACACTGTATTTTGTAGCTGTATTCATAGTGTATTGCTGTGTATTTTGAAAACGTATTGGAGAATACAATATGTGTATTCTCCAATACACTCAAGCTTTAAAGCCTAAGTTTTTTAGATGAGGCAAAAACTGTTTTTGCTGTATTGGATCTCTTAGCATTTCTTTGATTTTCGCGGCTAGGCTCTCATAGCTTTCATTACCATTTGCTAAGTTTGAAAATTCAGGTAGTCGTGAAAGCTGATTGCCAAACATATTGATTTGAGCATCTGTCATTTGTTGGAATGCTGTTTGGGGTGTTCTAGTTAATTCTTGTTGAGATGGCCTAACTTTCGATTTAATTCTGAATTGAATGTCAGTGATAACTCGTCCTTTTTTGAATTGGTCATAGTCAACGGTAATATCAGTGTGTTTATTAATTTCACTTACTGCTCTGTCTAATACGTTGGATTTAAAGTTAGACATTGTTGAATATTCAGTCGCGTCAAGACCTAACTTTGATCTAAGATCGTTAATACTATATTTCTCAGTTTTTGCTACATTCAACCATTTAATTAATAGCTCATATAGTCTTATGCTATATTTACTTCTAAAATCAGATACTTGCTCTATATGATATTTAGTAAATTGTTCACTTAATCGGCTAATTAAGCTGATAACTTCGTTGCTAAGGTATAAAACAATAAGACCATTAGAATCTACATAGGTCACTTTGTTTACCCAATTTGCTGTAGTAAGAGCCTCTAGGCCCTTATATCTATCTAAATAACTGAACTCCCTTCTTTTTAGTGTTTTTGCTGCCTCTCTGAGCAAGCCATAGGCATTATTACTATCCACTTTAAATTGATGAATGTAGTCGCTTACAAGCACTTCTATTGGTGTATCTGGCGTGAGATTAGAAGACTCTCTAGCTCCAACTATGGCCAAGAGCATAATACGTTGCTCTATGAGTGATAAGTTAAAAGAAGCATCGATAAGAGCATTATCTTTAACAACTAATTTATTGTTATTTGTCATGATATGAAGCTTATTAATTCATTTAACTAGGTTATATGTCATATATTTTAAATAGTCAATAATCTATCTATTTATATACCCCTTAAACCTATCTTTATCACCCCTTAAACCTATCTTTATCACCCCTTAAACCTATCTTTATCACCCCTTAAACCTATCTTTATCACCCCTTAAACTATTGCAAACAAAGGATTTCAGCCTGTCTAAAAGCATTAAAAACTATTAAAAATAATTAAAAGCCTCATCTGGTCAAAATTGCCCTTGGTTTTCGCTACGCTCAAACTCTATTGAATCTCGCTTTCGCTCGATTCGGGGGGGCAATTTTTTGGCTCTTTTCGTTGTGAATTTGAGAAAAAGCAAAGGCAAGAGCAAAAGCAACTTGGAATTCGCTTCGCTCATAAAGCTTTATGCTCGCTTCGCTCGGTCTAGGCTCAAATTTTTGCATCTAAAATTCACTTTTTTAGGTTAATCAACCTAAAAAAGCGGTTTTTCAGAGGGCCGAGGCGCGAATTTTGGGGATGTCGCTCGTAGACACTCGCTTGGTTAAGCCGTTCTTATCGCTTATAGATGTAGTGTTATTTGCTTCTATTTGCGTTTAGAGACACTTTTTTGTTTTTTGGGTACTGATCTCTAGTTCGGTTTGAAATGATCGCACTATGGCGAAAATACAGCGTTTAAACGTGATATTTGTTCATTGCTCTGGTGTTCTGTTGAATGGGTTTCTTGGATCTGTTTTTGCTGTTTCAATAAGTCGATTCATATTTTTTGTGGTTGGCATACGTTGAACGAAAGCTTCGATTTCTTCAATTGTTCCAAGATTCCTTTTGAGCTGTCTAAAACTGAGATATGCATGAATCATGTCGGCCAGTTTATCTAAGTCGCTATTATCCTGTGAAAAACTAATCGATGCCTTTCTCAAAGATTCGATATAAAGCGATCCTATAAAGTATTTGATCTGGTCTTCTCGTTTTTTCTGTTGTTTCTGGCTTTGTTTTTCCTTTTGTTGTTCTGGATTCAGGGCTTTTTCTATTTTCTGATTCAGGTTTTTCGCTGCGGTCTGGTGGCGTTGCGTTTTGAGGATCTCATCTAGTGCATCGTGCATGGCAAAGTAGGCATCGATGTCATGTTTTTGATGGAGCTTGAGGATGTTTTGTTCCAGTGCATTGCGTTTGGCAGGTTCTGTTTGTTGGATCAGTTGCAGCTGAAGCATGCGTTTGTTGTAGGCATTGCGTTCATAGCGACATTTTTGGATCAGTTGTCGGTCTTGCTCACTGAGATCCTGAAGACTTTTTTTGTTTTTGAGCATGTGCAAGCAACGTTCTTCAACTCGGCGTTTGAACGGTAGATTGAGGATGTGCGTGCCGAGTTCGTTGATTTCTTTTGCTGTGAGAACTAACGCTTTGCGTCCTGCCATTTGGATCTCATTTTGCCGTGAATTGACTCCATGTTAGCGTCATGTTGTGGGCGTTTAAACTATTTTTTCTGCTATGCATAAAAAATAGTTTATCTCGTTTCCGTGTCTTTTTTTGACAACAACCGCTAAAGCGATTATTGTCAAAAAAAGACACTCCTCCTCGCTCCTGCGGAGAGCCTCCTTTGTGAATTGGATCTCAGGTGAATTCGGCATGTTTTACATGAATATTCGGCACAATGTGAAGAAGGATTCCAGTTCCAAGAATACCTTCCATTATCTGACTCGAACCGCGCATTTTGCCGAGCAAAAGGCCGATGAGGAATTGGAGTTTGTGCGTTCTGGAAATATGCCTGCTTGGGCAGTCGATGCGCCTGCAAGCTTTTGGCATAGTGCGGATCGTTATGAGATTGCTCGAGGTCGAACCAGTTCAACGTTGACTGTGGCTTTACCAAAAGAATTGTCGAAACAGCAGCGTAAAGAATTGGTCGAGGCCTTTATTCAGGAATTTGCTGATCAGTATCAGTTCCCTTATACCGCAGCGGTGCATAATCATCCGAGTGAGCTGACAGGTGAAGATCAACCGCATTTGCATCTGATGTATTCTGAACGGAGTATGGCCGATGGGATTGAACGACCACCAGAGCAGTTTTTTAAGCAGTATCGTCCAAAGAATCCCACCAAAGGTGGTGCACAGAAATTAACGGCTGATGCGTTGGGGTTTGGCCGTGATCAGGTTAAAGCCTTTCGAGAAAAAACTGAGCAGCTGATTAATCAGGCTTTGGAAATGCATGCACCGATGAAAACCGTAATTCTGGAGGGTATGGAGATTGTTGTTCCGAATCGAGTGAGTCATTTGTCCAATCAGGAATTTAACCAGAAGTACGGAACGCAATTGCAGGATGTTCCGCAGATCTACCGATGGAAGTTGAAAAGTGCCGATCCGCTGATTCAGCTTGAGGTTGAGGCGCAAAAGCAAACCATTCAACAGATTCGGCAACTGAATAAGCTCCAGATCTACCAAAAGGAGTACAACCAAGCTTTAGCGCAGCGTAAAAATCAAGATCTAGATCGTGATGATAGCTATACGCCATCATGGTTCTAGGCATATTGTGAGGTTTTAAATAAAGCTTTTGGAGGTCGTATGAAGACTGGAAAACAAATTCTTGAGGGTTTATTTGACGATCAACGTGCTGCTGCACATGATCCCCATCATCCAGTGAATGATCAGCTTCGGGCTAGTCTATTGTGGAAGAAACATTTACTTGAGTTGATTCAGATCCCACGTCATCAACTGAAATCAGATGATAGCTATGTTTTGGTGTCTGATATCCCTGAGTATTTTAAGGCACAGTTTGAGCAATATTTTTTAGGTCAGAAGCTGTTACAGCATGTGGATCGTCCTGAAGAATCGAGACTGGATAGCAAACGTTTAGACAGTTTTATTGCCTATCTCAAAGATCATCCAAGTTATAAATCACAGCAGCGTGGCCAAGTATGAAAAAGAACGTAGATGCAGATCGAATTCGAGCTGCTTTTAAGGCAACGCAAGGGATTGTGGAGCATTCATTTGAGCAGCATGTAATTGATGCCAGTCAAGCTTATGTCAATGCGATGATTGAGCGGATTGCGTCTGGGGAGTCGACCTTTGAGGAGGAAGATGAGGTGTTAATGATGAAGTGGCGAGCTATGAATGCCTTACAGGATAAGGATTCATAGCTTCGTATAAGGCGTATTATGTTAATTTTAGAAAAAATTGATAATTAATGAGTAAAATATGCATAAATTTTGGAAAATTTTTATCGGATTAAATATATTTGGATTCATAATTTCAATATCAAATTTCTTTTTCATGCTATTTTACCCAATTATATTTTGTATAAATTCAGCATCAAGTTTGCTCATTTATATAATCTTTAGGTTTATAAAAAAACATAAAGTTAATAATTTTTTGAATATCCTTTTTTCTACAATTTTAGTAATTTTTTTAAATATTTTAATAAGCTACACTTTTTTTAGTTTAGATAAAATGTCTCAGATAATGATATATGCATCAATCATATATCTATGTTTTCAGCTTTTAATGATAGTTCTGCTAGAAGCTGCTCTTTAACTCATTTATTTTCATTTAACATAATGGCCGTTATATGCAATGCAATTGTATATTAATATAAATTACTTAAAAAATCAATCCTGATCATAAAAACCGCAAAAAGCCGATTTTGAAACAAGTCGGCTTTTTTATGAGCAGTTTTGATACTTTCTGCCAATAAAATCGGTAAAAAAATTGACAATATAATTTATACAAATTATATTATTTATATAAATTTGATTATTTGGATAATTTAGCTATGAATGAACAACAAATATATGACCGCATCCGATTGGCTATGAATGAAGCACCTCGAAACAGGCAAACTGCTGAATTACATTTACAGATGATTAAATATGCAGATGAATTAAAAAGTATAACTTCTAAAGAGTTTTGTGAGGGAGTGGGTCTACCACTTAGTTACGGTACCGAATTTAGTAAAATGCGAAATATTACAGAGCGTTTAAAATTGGCTGGACTTAAGGTAAATATGATTTAAAAGTTAAAAAGGCTTAGTCATGACTAAGCCTTTTTAATTTCATATAACGTGTATTATGTTAATTTTAGCGAAACTCAATAATAAATTATTTCTTCTTTGGCTGTGGTGGCGATGCTTTTTTCTGAGCTTTTACTGATTGCTCAACTTTATTTTGTTTACTTGTAGGCTTGTTTATTACGGCTGTTTTTAGAGACTTTAGCCATGAGTATGCGTGAGGATCTAACTTCAATAAATGTTCAATCATTTCATTACGTTTAGCTTTATCCATCTTAGAGATAATGCTTTTAAGATCAGCATTAATTGATTGATTTGTTGCACCTCGTTCACTAGCTTTTTCATACCACTCATGCCCAAGTGTATATTGATACGTTTCCATATAAATAGCGCCTAATAAAGTACAAGGACGATAATTATCTGGAGTTAAATTATGAGCTTTGGATGCATTATCAATGGCGACGTTAACCTTGCGAAGATCACGTCTAACTCCACCAAGAGTAGTGAAATACGCAGATAATAATTTTTTATCCTTGATACCATTAGTGGAAATTTTTTCTAAAAATTCTTCTGCTTCAACACTTTCCTTACACTTTCTTAATTGACTGCTAGCATTAATAGCATTCCAAATGTTCTTGGTGTTCTTTGCATATTCCTGCAAATAATAATTAGCTTCTAATCGATGAAATTTATGACGAACTTTGGCAGTAAAGAATTTTCTACCTATTGTATTTAACCAAACAGCCTCATCTTGAGAAAGTCTTGTTTCATTTTCTAGCTTATTCAAAATTTCAGAAAGCTCTGTAGGTGATGCCCCTGATAGATATTCATTAATGTCGTATTTTTTTAGTAATGCTTTTTCTTTTTGCTTTGCAATATAAACAGGATCTGACTCAAGCAACTTTTGTTTTTCAATAGCTTCCAGTCTCAACTTCTCCTGACGAATTTTTTCCTCAGCTACTCTTTCTAATTCAACTTTATGTCTAATTTTTTTCTGAAAGTCTGGCAGGAAACCGCTTAAAGATCTGAGCATATACTCATGAATATATACTTCTTCAAATTTACTTAGGCTTTGTTTTTCTTCATATTTTTTTATAATGTCTTTTAAACGGCTAGGTATTAAAAGATCTGACTTTAATGGATCTAAAAAATATTGTTGGCACAAATTTAAGAGTTTGCTCTCGAAAGTATCTAAATCTTTATCAGTAACTTTTGCCTGTAACAGTAAAGCCTTTCCCTCAGTCATAATAGAAATCTTATTCGAATCAAAAAATTACTTCAGTATAAAGAAAAAATGTCTGTATCTCACTTATGTTTATTTCCCATTTAACATAATGGGCGTTATATGAAAGGCCCTATTCATGGGCCTTTTGATCATCTGATTAACATAAAATCTCTTATGCGAATTCTAGCCCTGCATAGGGCTAGATATAGCGATTAGACCCAATTCTCTAGCTTCAGACCATCAATCCGTTCAAATTCTTTTGTATTGTTCGTTACTAATACAACATCCAATGCCAAAGCATGAGCAGCGATCAGCATATCTAAGCTGCCAATCGGTTTACCTGTGGATTGAAGATCTTGGCGTAATTTGGCGTAATGCCAAATTGCTTGTTCATCGTAAGGAAGAATCTCTAAAGGTGACAAAAACTTGTTTAACGCCTGTTTATTGCGTTCAGATCCAGACTTCTCTACACCAAAAGCCAGCTCAGATGCCGTGATACTTGAAATCGCAAGCTGGCCTAGTTGATATTGCTTAAAGCGTTCAAAAACGTGTTGAGGCTTATTGTTGATGACATAGATACAAATATTTGTGTCTAAGAGGTAAATCATTTGAATTCCTCACGGAACTGTTCACCTTGATCAGCTCGCTCCAACTGGAATCCAACCTCAAACTCATTGATGGCTTCTAACATGACATCCCAAGGATTGTCGATTGGCATCAACAAAACGCCACGTCCAAAGCTTTTGATGGCCACTTCCGTGCCATTAAAACGAAAGGCTTTAGGCAAGCGAACAGCCTGACTGCGACCAGTTTGAAATACTTTCGCAATTTCCATAATTACCTCCCGAAACCAAGATATGTATCATTGATATATATCATATCATCGAAGGGTATTCTGTTAAAGCATAATTAAAATCATGAGACAAT\n",
      ">Contig_4_115.076_Circ\n",
      "AAAAAAAATGTCCCTTCCAGAGGGAAAAAATCAATTTTATATGTCATATAAGAACCATTAAAGCACATAAGAGTTTTTTTGAGACACTTTATAATACACAACTTTCTATTTAATTCGGAAATAAAACAATGTTTCTATATATTTAAACGTAATTTTTATTTAATTTAGCTTTGGAAAGATATTTAACATAAAATCTCTTATGCGAAATTCAAAAAAGCGCGGATCTAGTGTCCACGCTTCATATGATTTATACAATGGAAAGTGTAGTGAATTGGATAAGAAAAGCATAACCCTCTAATTGTTTATTTGCTTCTTTGAGCATGCTTTTAATAACAAAGAATTAGAGGGTTAAAATTTCCGCTGCACTTGTTTCTTTATACTTATTTATTGCTATTGTGAGGCTTATTTAATAAGGCTTAAAAAACAACATATAAGATATTGATATAAATAAAAAATATGAGTTATGAGCGAATTTTAGATGACAATTTTAGAAGATTGTCATCTAAAGTTCGCCACTAATCTTATAAACTATTGATTTATAAATAATATATTACTTTTTCTTCAAGAAACTAAAATTCGCCAGTAATCCCAATAAAACTATATTAATCATAGCCTTACTATAGTAACCTCCCTAGAAATAAAAGAATCTAAAAGTAGAATGATCTATTAAAATACAAGTAGAAGATTCTGCATTTGGCTTAATAGTTGGAATTCTAGGTTCTGTCACATTAGGGAATTAGATATCAAATAATTCGGTTAACCCCTTATAGGAAGTATTTTCCATGAAAAAATTCAGTACAAGTGAGTATCTCAAAGAAATGGATCGTAAGGATTGGGCAACAGCCCAATCTATCACCACCTTTGGCATGAACCCCAAAGAAAAAAAAGCCCATGAAAAAAGATCCTGGGAACGCTTTGAGCGGATCCAGTCTGGAGAAACCACAATTAAAGATGAAGAAGCAGTACTGCTTATGCAGTGGCGTGCCTTAAATAATCGGCTTAAAAGCGAGGACTAAAACCATTATCCTTATCTTGATTCCTTGTCTTTTCAGGAGCGATAGCCTTTTCCCTCATTTGCAAAGGCACACGTTCGATGACTTGCCGCTGAATAAATGCCAATTCTTCATCAATAGAGGCGATCTGAATAACTGAACCCTCTAATTGTTGGCCAAGATCCCTTAAGTATTCGAACCTCTCGACTCGATCGAAGGACTCGGTATTCAAAAGGTCTATATAGGTCTGTAAATCATGCTGAATCAACCTAAGCGGTTGGGTCGTTTGACGATCTTGCAATAAGGTTTGCAGCATCTTCTGTTGTAAAAACAATAACTTTTCTTGGGCATACCATCGCTGCCGTGTCCGTGGAGTCTCGACTTGCTTTTGCTGTTCTTCAATGGTTTTTCGATTCTCTAATTCAGCCTCATAAAATGGACGATACAACTCATAATTATTTTCAGCACGTAACCGGTTAATTTCAGCAGTCATTCTTTCCCGTTCGGCCACTAACGCTGGCTCTTTCTCACGCGCAAAACGGACGGCCTTATTCATCACAGGCACATCCTTCAGCTGTGTGCCATGCTTCTTATTGTATTCCTGATTGGATAAACAGCTGACCAGGCTAGGGACCTCGACCTGAATGCCCCGGATTTCCACCATCTTGGTGGGAGCATAATACCTTAGACTTTCATTAATAAGCTCTTCAGTCTTCTGCCGATACAGCTGCAGCTGCGCCTTTCCCATGCCTAACACATCAGCTGTTAACTTCTGTGCCCCGCCTTTATGCGGATCCTTCGGATTATAACGCTTAAAAAACTGCTCTGCTGTCCGCTCGATGCCATCCACATGCCGCTCACTGTACATGAAATGCAAATGCGGTTGCTCTTTCCCTCCCAAGGACCCAGCATGATTATGAATGGCACAGGTGAATGGATAGCGGTAACTGTCTGCAAACTCACTAATGAATTGCTCGGCCAGTTCAATCCGCTGTGCCTGACTCAGCTCTTTGGGCAAGGCCACCACCATACTGGAACAGGTGCGGCCATTACTTCGTTCATAAATATCAGCAGCTTCCCAAAACTCAGCCGGCTTCCCTTCAGCAAAATTCGGCATATTCCCCGATCTGACAAACTCAATCTGTTCTGAAGCACTTTCTTTGTAATTTCTGAAATGCGAAGTCCGGCTAATGTAATGAAACCGGTTCTTGGCCGTGGCACCATTTTTCACTTTTCCGGATCGGGTCTTCACGGCCTTTTTTGTTGTGTGTTCCAGATTGCAGTAGAACATATTCCCACCTTCAAATTCACGGAGCCTCGCAGAGCCTGCGGAGCACGAGGAGAAGTGTTTTTAATTCGATATAATTGCGTAGTAATTATATCGAATTAAAAACACGGAAACGAGCTTAAGAATTAAAAAAAATGTTCCCACATTTTTTTTAATTCTTCCCGTGCCGGCCTATGCTAAACTGAAAACACAAAAACCTCACCCATTTTATGCTGGAGAATGCGATGGCGGCCCGTGGAAGACCTGCGAAAATACTCACTTCAAATGACCTCATGACATTACAGCAGTTCCTTAAAAATCTCCCTTTCCTCAAGAAAAACCAGCAGCAGGCTTTGACCTTGCTTCAACAAGCAAATGGCATATTCAATGACAAACAGCTCAATCTACTGAAAGCTGCAGACCGTGATAAAAACCAGTTTTTAAAAAGACAGGCCCTCATTGAACAGATCAAGCTAAAGGATAAAAACCAGCAGCCCCTTCTGGCCAACGAAACAGAAATTCTGGCATTACTGACACAGGAAATGGATCAGGATAATTTCTTCCGCCTGGATCGTGCCCTGGAGTCTTACCAGAAAATTGAAAAAGCAGCCTTGGAAAACCGGATCCGGCTGGAGAATGAACACAAGCGCGAAATTTTGCAAAAAAGTCATAAAGAACTGACAGACGCCCAAAAGAAACGTAATGCAGAAAATCAGCTCAAATATGCTTTAGGTGGAGCAGTACTAGCCGCCTGGAACAAACTTAATTTATCAACCGAAAATATAGATCCAGAAAAAGTTAAAAATACCATCGTCAATAACCAAAATTTTTTCAGAATGGTCAGTAACACCACGCTTTATCAATATATTCATCCAAGAACTGAAAACTACTTTCGAAGCAGAGAATTGTTTATCAAAGTTATTGAAGGATTATGTGAATATGATGACCACGGTACAGAACTTTATATCTTTGAAGCAGATAGTCACTTAAAACCGCAATAAAATGCGATTACAGCCACTTTTAAACCAATTTTTGAGTTAGCTACGTCTAAATCAATAAAGTGGCTGTAATCGCAAAACAAGAGCATTTAGCTTATTTTGCAAAAATACAAATCAAGATCCATTTCAAAACTCACAATATACCTAGAAGGTATATACCGAGCGAGTGTCTACGAGCGGCATAACTAAAATTCGCGCTTCGACTCTTTGAAAAACTGTTTTTTAGTATGTGAAATCTAAAAGAGCGTAAATCAAATGCCAAAATTTGATTCTAGAACGAGCGAAAGCGAGTAAAAAAAGCACTATAAGCGAAGCGAATTCCGAATTGTTTGTGCTTTTTTTAAAGTCATAACGAGATTTACTAAAAATTGCCCCAAGAATCGAGCGATAGCGAGATTCAATAGAGTTTGAGCGTAGCGAAAACATAGGGCAATTTTTCATTCTTTCCTTTTTAATATATTTTAAATCTTTTAAATCTTTTAGGCTATCTGAAAGGCTTTAACAGAAAGGCTTTCAGCTCCTAACTATGAGGGATTGACTACTAACTATGAGGGATTGACTACTAACTATGAGGGATTGACTACTTAACTATGAGGGATTGACTACCAAATATGAGAGAATTATATTACTCATAGTTAGTTTATCGAATTAAAAATGATGAGTACATCTAAAAAAGAATTAGTAGTTAAATCAAATCAAGTAATTGAGGCTTCATATCAACTCAGCTCAACAGAACAACGGATTGTCTTAGCAGCTATTAGTAAGATTAGTCGGGCTGAAGATATTACAGATGATGAAATTTATCGCGTAACTATTGATGATTTAAAAAAACTTGGGGTTCATGAAAAAACAGCTTATAGAGATCTAAAAGATGGTGTAAACCGCTTATATGACAGATCAATTAATTTAGCTATTGATGATGAGTCAATCAAAATGAGATGGATTCAATCCATACGCTTTTTAGAAAGTAAAAGCGTTGTAGGAATTCGGTTTTCTAAAGAAATTTTGCCGTTCATTTCTAATCTAAGCCGTGAATTCACTAAGTATTCGCTATCTGATATTGCTGGTATGAGTAGCGCATATGCTATTCGTATTTATGAGCTACTAAGCCAATATCGTTCCATAGGAAAACGAGAGATTTCCATCGAATCTCTAAGAAGTATGTTGGAGTTAGGGAAAAGATATCCATTGTCTGCTGATTTAAAGAGATGGGTTATTGATACTGCTGTAGTTCAAATCAATGAACATAGCCCTTTAAATGTATCTTACGAGCAAATAAAGACAGGTCGAAAAGTCACACACATTCACTTCACGTTTAAAGAAAAATCAAAAAGTATTGAGCATAAATCCGAACAGAATAATTTTTATAAATTGACAGATTCACAAATTAATATGTTCGGTAATCAGCTTTCACGCTTACATGAAGTATCTCACCTAGCACAACAAGGTGAAAGTTATGACGACTTAGCTATTAAAATTAAAGATATGCTGCGAGATCCAATAAAACAAAAACAACTTATTCCTCATCTTAAAAATTTAGGCTTTAAGGCATAAAGGTCGATTGTGTATGTAGCAGTTGACTGTTTGTTAATGTATTTTAGTTCACATAATGGACAACCAAAATTACAAGATGAAATCACTATCTGTTATTGAGCTTTCAAATCTCTATGGAATAACTCGTCAAGCCATTTATAAGCAAGTTAATAAAGGTAATTTAAGTAAAAATAGTGACGGTAAAATTGATCTTTCCGAAGCTATTCGAGTCTTTGGTGAACCTTCTAGAAATGTAAACAGTTCACAGACAACCGAAACACGAAAGTTGTCAGAAGTTCACCTATTAGAACAGCAGGTTTACATGCTTCAAAAACAGCTTGAACAGGCTCATGAGCGTGAGCAGTTCCAACGTGAGGAACTAAAAGCTAAGAATGATCAGTTACACGTTAAAGATGAGCAAATTGAAGCAATACAACGCCTACTAGAAGCTCCAAAAGCTTATACAACTAGTTCTATCGACCCAAAATTAGATATAGCAACAGATACTAGGTCAAAAAGTGAGTCGAACTATGATGTATTGACTACTCAACCAAGAGAAACTCCTGTAGAAGTAGAGATCCAAACTCAGCCGAAAGATGACGGATTGACTACTCCGGAGCAGGTGGAAAATAAGCGGATTCCAGTACCGGAGCACATTGACCAGGAACCGGAGAAAAGGGGCTTTTGGAGCCGTTTTTTTAGACCCTATGATTAACTACGATTGTTTAACTGATATGAACAAAGTCGTGAAACATTGATCAAATAATAAGCGTGGAACGTGAAAAATCATAAAAAAAGCCCGGTCATGGAAACTCGGGCTATCAACTAGGAACTACAGCGTTGAATTATAAGGATAATTATAACGCATTTCGTATAAGGTGTATTATGTTAATTTTAGATAAACTATAGAATCCATTAGTTTTTAAACCAATAAATTAAATATTTTATGGAACTCCAGAATATTTATTCAGGCGATACTTAAGCAATAAGGGTAGGGTGATTACATAAAGTATTGCAGCTAATGCCCATACCATTCCAGGAGCTGATGCCTGAGTCATAAAATAGAGGCTAGTAACTAATACAGGACCAATTATAGCCCCCAGACTATTAAAACTGCTTAATAGCCCTTGTAATTCACCTTGTCTTTCATCACTAATACCTCTGGACATCATTGCTTGCAAAGCAGGAACGGCTACTCCACCAAGGGCAAACAATGGTAACAACGCAAAAGGCACCCATTCTTCAACAGCAATAGATAAAAGAAGTAAACCAATTGCATCGCAAGCTATTCCACATAACAACGTATTTTTCTCTCCAAACCTCTTTACCATAGGGGCGATAGCAAAAGCCTGTGCAATAGAATGGCATATTCCATAACTAGCAAGGGAAACACCTGCAATAAATATACTCCAGCCATAACGATCCTGCCCATAGATAACCCATAAAGTTGCGGGGACTTGTGAAACCAATGTGATAATTAAAAAGATACCAAGCAATGGAGCCATATTAGGTTGAGTGATCAAATAGGATAATTTCTTTAATATTGATTGCTCCTGAACCTTCTCAGTCATTTTATTGCTGTGTTTTGATTCTTTTAATAAGACTGCTGTCATTATTAAATTAATCCCATTCATAAAAGCAGCAGCAAAGAACGGTGCATGTAAGTTCCATTCACCCAAAACTCCACCTATTACCGGACCCGCTATAAACCCAATACCAAAGACAGCTCCAAGGAGACCAAACCTTTTTGCACGCTCATGGGCTGGAGTAATATCTGAAACATAAGCTGTTGCAACAGCCATGTTGGCACCTGTAATTCCCGCAAAAATACGACCAATATATAGCCACAATAGAGAAGGAGCAGCAGCCATTAGGAGATAATCAGCCGTTGCACCAGCAATTGAAATAATTAATACAGGTCGACGTCCAAATCGGTCACTCAACGCTCCAAGGATAGGTGCAAAAATGAACTGCATCAGAGCATACACAGCTAATAAAGCACCATAGTGAACACCGCCTGCTTCAGCTCCAGCCAATGACCGTAATAATTCAGGCAAGATCGGCATAATTAAACCAATCCCAACAGCATCCAGAAATATAGTGATTAAAATCACGCTCAATGATTTCTTCACAAATGCCTTCTCTAGCAATGATAGAATTAATATTCTCTCACACACTCCCTATCAGTGATAGATTTATGAGTAAAAAAGATATTGCACCTCAACGTCTGACACGTGAAATAGTCCTGCGTACCGCATTGGATATGTTGAATGAAGAAGGCATAGATTCAATAACCACACGCAAGCTAGCTCAACGATTGGGTATTAAATCTCCAACTCTATATTGGCATTTCAAGAATAAATCCTTGTTAATGGAAGCAATGGCGGAAACCATTATAAATGAACACCATTTAGTTTCTTTACCCATCGATGGGATGACATGGCAAGATTGGTTGCTCGCTAATTCTGTAAGCTTTCGTCGAGCACTACTAGCTTACCGTGATGGTGCTCGTCTACATGCAAGAACTAGCCCCAGTCAAGGACATTTCAACACGATTGAGGCCCAAGTTGCGCTGTTATCTCATGCTGGGTTTTCTCCTGTTGAAGCAGTTGCATTACTTATGACATTAGGAAGGTTTATTGTTGGGTGGGTCCTTGAGGAGCAGCAAGAAGAAATAAGATCAGACCCACCATTCGAAGCCGATCCAACTATATATCCACTCATGCTCCAAGGAGTGAATACCTTACAAAATATGAATGCGGATGACATTTTTGAAAATGGGATTCGTATGGTGATTATCGGTGCTGAAAGACAACTTGATATTAAAATGCAAACTTAACTCTATACTGCTTTAAGCAACTGTTCTATTTAGTAATAAAGCTTTTATCCTTAATTATTTAACATAATAGAGCTTATACGAAATGCTTTTTAAAATATCTTTGTTTTTCATATAGTTAAAAAATTTTCATCCCCCTAAAAAACCACAAAAAACCGACTTGGAAACAAGTCGGTTTCTTTTTGAGCAATTCTGATACTTCTTGCCAATAAAATTGATCAATATTTAATCAATTTGTCATTTTTCTCTGCGTTATCAAGCCAGCGCTTGCTTAAATTATCCCCAATTTCTGCGGACAACTCTCGGGATAAAAATTCACCAATTTTGTACGCCTGAGCGCACATGGATTTAGGGATTTTGCGGATATTCATAACCCCTTGTATTCAATATCATGAAGTCATAAGAGAACAGGACGTTCCGCAGAATAAAAATAAATAATCGCACAAGGATCGGCGGCACGACAGGAGTTATGCCAAGCGAACCAATACGAAAAAGCCCGACAGGATGTCGGGCTTTTTTATTTAATTCATTGGCTTAATTAAAAACTAAAACTTCATGTTTATTGAGTAAAATGATAAATAGTTCAATATAAGACCATTATTAAGGGGACTATTCTAAAATGACACATGCTTTTCAGACACTTTCAACAAATATAGAAAATAATCCGAAACTAAGAGAACCACAGATTAATTCATATCTTGCTCTAAAAGATTTTTATAAAAATCACTTTCAGACCTCTACCAATAAAGAGGTTGGAATAGTGCTTCCTGTTGGTTGTGGTAAATCTGGTTGTATAACTCTTACTCCTTTTGCATTTGAGTCAAAAAGATCTTTAGTAATTGCTCCTGGTGTTTCTATTGCAAATCAGCTACTTGAAGATTTTGAACCATTAACATCTTTTTATAAAAAATGTTCAGTGTTAAAAGGTAATGAATTTCCAGAACCAGTAGAAATTCGTAGCAAAGATACAAACCTATCTGATTTAGAAGAGTCTGATGTAGTAATTACCAATATTCAGCAACTACAAGGAGACAATAATCGCTGGTTAAATAATCTACCTGATGACTTTTTTGATGTCATAATTTTTGATGAAGGGCATCATAGTGTTGCTGAAACTTGGGAAAATTTGAAAACTAAGTTTTCCCAAGCTGCAATTGTAAATTACAGTGCAACACCTGAACGATCTGATGGAAAAATTATGTCAGGAGAAGTTATTTATACCTTCCCTGTTCAAACAGCTATTCAAAAAGGATATGTAAAAGAGTTAACCGCTATAAGGCTTAATCCTAAAACATTAAAATATATAAGTTATAAAGACAGTCAAGAAGTCGAAGTATCTCTTGAAGAAGTAATAAGACTAGGAGAGGAGGATGCAGATTTCCGTAGAAGTATTGTCACATCTAAGGAAACTTTAGACACAATTGTTGATGCTTCTATTCATCAACTTATGGAGCTACGTAATGCTAACAATGATAATAGATTAAAGATTATTGCCTCTGCTCTAAACTATGAACACTGTATCCAAGTAGTTCAAGCATATAGGGAAAGAGGTTTCTCTGCTGATTATGTGCATACAAGAGAAGATAGTAAAAAGAATGATGCCGTTTTAAAAAAATTAAAAGCTCATGAATTAGATGTAATTGTACAAGTAAGAAAACTAACAGAAGGTTTTGACCATCCATATTTATCTGTTGCAGCTGTATTTAGTGTCTTTGCCAACTTATCTCCATTTATTCAATTTGTTGGCAGAATCATGCGTGTTATTGAACAAGACTCTCCTCGTAGTATGATTAACCAAGGGATAGTTGTATATCATGCAGGTTCAAATATAGCTAAACGTTGGGCCGACTTTAAGGAATTTAGTTCAGCTGATAAAGCATATTTTGACCAACTATTACCAGAAAAAGAGTTCACTCCTTCAGGTTCTACAAACTCTGAAATCCTAAAACCAAGAATAGAAAACTCTGAGAAAGTTGATATTAGAGAACAGTCTGATGTCTCATTAGATACATCAACTTTAATAGATGATCCTAAAGCAATCGAAGCATTACGCTATCTGCGTAATGCCGGAGTATCTGGCAATATTGATGACTTATTACAAGCACTAGAATTAGAGAGAATTCCAGTTACTAAAGTTAAAAAACGTCAAGCCCAACGCCAAGCTTTGAATGAAAAAGTACAACATGCAACTGGAAAAATATTAGCTATTTTAGGAAAAAATCCAACAAGTAAAAATTTAGACAGATCATTTCGGCAGAACAACTATGAGTTTGTATGTTCTCTATTAAATAAACATATAAACCAACATGTTGAAAGGAAATCAGATGAACGACATGAATTTACAGCCCACGAACTAAACACTATAGAAAGCGTATTTGATCATATAGTATCTAAGGTAGAGCAGGAATTAAGTTCAAATGAGCACTAGAGCATTGATGATACCTGCTCGTGTAGCTATCGCTCAACGGGCTCACAAATGTAAGGCAAATCAAAAACATCTTATATCTCAAGGCCAATATAGATTTGAGGTTAAAAATGAGAGGAATTGGGATAAGTATTGTTTAACTTGCGGTAATAAAATCATGCTCAAATTAATTAGTGATTCTTCAGCATTAATGGAAGATTTTAAAAAATAGAGGGTGTAAATAATTCTGTGTAAATACAGTTTTAAAAATACTTGGTTACACGCTCTTCAAACATGATACTAAAGCGGTTCATAGCAGATTTCCAATTTTGGATAGGCATTGTCCATTTTTTGCTTGCTGCCATAATCGCTAAATATACCACTTTTTTCACTGAATCATCAGTCGGGAAGATTTTTCTTTTCTTGGTCGCATGACGGATCACGCTGTTCAATGATTCAATCGCATTCGTTGTGTAGATTGCTTTACGAATATCCATAGGATAAGCAAAGAACGTGTTGAGATTCGGCCAATGCCCGAGCCATGATTTGCTGATTTGAGGGTATTTGTCATCCCATTTCTGACCAAATTCATCAAGAGCGTGTAAGGCTTCATCTTCGGTTACCGACTGGTAGATTTTCTTCAAATCAGATGTAACGGCCTTATAATCTTTCCAACTGACATACTTCAAGCTATTGCGTACCATATGGATAATACACAACTGGATATGGGTATCAGGATAGACCGTATTAATCGCATCAGGAAAGCCTTTTAATCCATCCACACATGCAATCAGAATATCTTGCAAGCCACGATTCTTCAGTTCGGTGAGAACTGATAACCAGAACTTCGCCCCTTCGTTTTCAGCCATCCACAGTCCAAGTAAGTCTTTCTGCCCCTCTGTATTGACACCCAAAGCAAGAAATACAGCCTTGTTGATGACGGTGTTGTTATGTCGCACCTTCACGACAATACAGTCTAGGTAAACGATTGGATAGAATGCATCCAAAGGGCGGTTTTGCCATTGAGTGACTTGGTCGATCACAGCATCTGTCACCTTGGAAATTAAGGTAGCAGAGACATCTGCATCGTACATTTCCTTGAACGTGGCAACGATATCTCGTGTCGTCATGCCTTTAGCATAAAGGGATAGAATTTGACTATCCATTTGCGTGATTCGAGTTTGACTTTTCTTGATGAACTGAGGCTCAAATGTACCATCACGATCTCTAGGCGTGTCTAGCTCAAACTCACCATCATCAGTCAGTAGGCGTTTAGTTGTATAGCCATTACGAGCATTTTTACCTACTCTAGGACTATGCTTCTCATAACCTAAATGGTCAGTGAGTTCAGCATTTAGAATAGTTTCCACCGTTAATTTCTTTAGAGCTTTACTGAAATTATTCAGGTCAGCTTCTGTCTTTAAGTTCTTGGCAAATTCTTTGGCTAATGCCAGCATTGCTTCATCTTTCATGGTAAACACCTGTGTTGCCCCTAACAGCGTTGAGGGATAGTCTAAGCGTTTACACAGTTTTATTTACAGTCTCAAAAAATACCACACGTGTTGAAATCTAAATATGTTATTAAAAAATCCCAGTCTTACTACTGGGATTTTTTAATAACCAAACATATAAACTGGCTTTATCAATACATACTACTTCAGTAGTTTATTACCATCTAAAGCCATGTTAATTAAGTTAAAATCATGAGAACTTAAGTTTTGAGAATAAAGTTCTCTTAACTCATCTTCCCACGAAGCAACTTCATCTAATATTTCAGCTGCTTGAGTTTCTGTTAATAAAAATTGTGAACACTGAGATAAAAAATTTTCCCTGTTTATTAAACTATCTTGTACACCAATTCTCATTGCTAATCTTGGGGGTGAATATCCATAGTGCGGAACAACATCATATAATGGAGCTAATCTCCATTGTTCTTCAATATATATAAAGGCATGATTTTTTGGATGATCATCATCATTACCTATTAATGCATTAAAAGCCATTCTTCGATATAGCTCATGTATGTCTTCTAATGGTATGTTTTTCCTTAACATTTCATTAGCCAACAATGGATAACTCCAACGATTACTGTTATCAATAGGCCAATCAGCATCTAAAAGAGTTAAAGCACTAACCATTGGATATCTTTCAAAGATTTGATGTTCTTTACTATAAGTACGATCAAACCTTTTCAAAACAAGCAAACTCTTTTCACTACCTTGATGAGATATGATTTCCAATGAGTGATCTGCGACATTTAACCCTTTAGTAGCTGCAAAATGTAAACATAAGAACTCTACTTTAGGAATATCTACTAATTCATCTCTATCCTTAGGTTTAGCTAAATATAAGTCATGTTCATCCCTTAATGTTATTTTAGGTCTTGCACCACCTGAAGATGTTTGGAAAAGAGCTAAATTAATATTTTCAACCGCCTGACCAGATCGAATTAGTTCTACTAACTGTATGAAATCACCAGCTCTTTCAAACTTAGGAAAATTTTCAATAGTCAAATTTTTAGGTGGTCTTTTATTACCTCCAACAGCTAAGTTCCCTGATCTATCTTTATTTTGTGCTAAAAGTAAATAATCTAAGTCCGTAAATTCATGCCCAATATCTTTTCTTATTACTTTTTTAATTAATGACTTACCCCAATTATCAGGTACTAAATCTAAAATAAAACCAGGAAGACCATTGTTATTTTTAACTGGAAAAATTTTTCCTCGATCTTGAAGTGAAGGTAAAGGGTAATGGACTTCATCTGGAACCCAAGGTCCCTCATAAATAGGGGAATAGACCCATTCGCCTTCTCCATTTTCAACTGTTAACCTACCTAAAGTCTCAAACTCAAGAGCATCTGGCTTTTGAGCATAAATATAAATGCTCTGTTTCAATTAAAAATCCTCCCAGGAACTTTTTTTCAGTCTTACTCTTTGTAAATCAGTCATCTTTTTAGATTTAGGTAAAACTTCTGCTAAGCCATCTAAGATATTTAAATGCCATAAAACCTTTATAAAAGACCCTAGTTCAACGTTGGGATTCCCCTGCTCAATCCTTTTATATGTAGATAATGAAATACCAATCATTTCAGAAATATCCTGCTGACGTAATTTCTTCTCTAATCTTTTTTCTTTAATGAGTGCAGCACAATTTTCT\n",
      ">Contig_5_2245.16_Circ\n",
      "AAAAAAACAACCACTTATAAATCGGCGTAAAAGATCTATTCTAGGCAAGGGTGTTGCGTGCAACGCAACGCCCTTTGGGGGTGTCGGGGCGAAGCCCTGACCCAGTCACGTAGCGATAGCGGAGTGTATACTGGCTTACTCCTATGCAAGCCTCGCCAGCCGTGATTTTTGCCAGATTTCGTGGTTTTCACGTTCAACAGCTCCATCGTGTTTTTGCCGTCCGTTCACCGTTAAAACGCCTCAGAAATCGGGACAGGATGTGTAATCGTATAACCGCGCATGCACGCCGATACGATTACCACTTGGTCAGGGCTTCACCCCGACACCCCGCATCAGTGCGTAAGTGCTCAATGTGGCTTCCGGTTCTCTTCCGCTTCCTCGTCGATTCGCTCGCTACGCTCGGTCATCGACTTGCGGCGAGCGGGATCACCTCTCGCATGTTGTCGGTGATCCCGGTGCTGTTCTTTTGGGGATGAGTCGCCAGATAGCCCCTCAAACGCGCTCAAAAGCTCTCAGACGCACGAAACGGGTTTTAGATGCCCGCCAGGGCATGGAAGGCGCGAAAAACGCTTACAGAGGCTCTCAGGAGGTCAAGCGCGCCGATACCCCTAACCAGGGGCTTTCAGGTCGCCAGGAACCGCGCCAGCCCCCTGCAAAGCCCATTTGCACGGAACAGGCAGCCATGCGGTTGGTAGGTGGTCGTAGACCGTCGGTGTTGCAGCGATGGGGGTCTGGGGTTCTCCCCAGACGGTTTCCGGTAGCGCTCAAGGCGCTGCCGGAACCGTGGCGGCTCGGTTTTTTCTGGAGAACGGGGAGCAAAAACACGCAAAGGATTTTGAGGCGGCAAAAGAGCAATGCAGGATAGACAATCATGGTCAGAACGTTAACACGTAGCCCCTGATTGTCCGCCTGCCGCTAGCTCAATTTGAAAACATGGCCATGATTCGGCGCTGAATATCACCGTCTTTGTTGACCAGTGTTTTAGTGACAAACGCTGCATGGTTTTTCGTAGTGTTGTATCCCTTCCAAACTTCATGTGGATCGAGGGCATACGCACAAACCTTACCAGCAACGGGGATTCTAACTAATGCATTTTTTGATTCTAAGTAATTGAGCCACCGAATAATTGTGCTTCTGCTAACTCCAAGTTTTTCAGCTAGGAATTGCTGATCCGCAACGACAGCGCCACACGTAGGATCGATATGCTCCGCTAAAAATGAGTACAGCTTCAGTGCTGATATGCCTTGGCTATCCTTCAGCAGCTCTCGAACACGTTCCCAACCTTTTGGGGATACCTGTGTAAACCTTGAATTTTTCTTTGCTTCTTCGAGATCCTCGGTGTTTTTCTTTGCCTCATCTGAAAGTTGTTTCAGTCGTTTTTCTGAATCAATAAACATGATTTTTTCCCTCTGAAAATCCCATGCACTAACGTTGGTTCAAATCTAAGCCATCATGACACATCTGTGCAACACCTTGTTGCGTGTTGTCGCATCCTGGATGCGACAACGTTGTGTCATCCTGGATGCGACAGGTAAATAAAAAACACACT\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "head output/contigs.fasta"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Sequence typing\n",
    "Using mslt and pubmlst"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "output/contigs.fasta\tecoli\t-\tadk(10)\tfumC(11)\tgyrB(4)\ticd(8)\tmdh(8)\tpurA(~13)\trecA(2)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[19:48:19] This is mlst 2.19.0 running on linux with Perl 5.026002\n",
      "[19:48:19] Checking mlst dependencies:\n",
      "[19:48:19] Found 'blastn' => /home/quang/miniconda3/envs/amrtraining/bin/blastn\n",
      "[19:48:19] Found 'any2fasta' => /home/quang/miniconda3/envs/amrtraining/bin/any2fasta\n",
      "[19:48:21] Found blastn: 2.13.0+ (002013)\n",
      "[19:48:21] Excluding 2 schemes: ecoli_2 abaumannii\n",
      "[19:48:24] Excluding ecoli_2.dinB.131 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.69 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.42 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.148 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.134 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.50 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.159 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.33 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.23 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.133 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.113 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.95 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.54 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.46 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.43 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.8 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.153 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.135 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.128 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.89 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.83 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.78 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.58 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.24 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.21 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.141 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.97 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.79 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.72 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.41 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.6 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.152 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.140 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.64 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.56 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.28 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.22 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.20 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.10 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.7 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.156 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.149 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.144 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.121 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.103 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.86 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.81 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.61 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.57 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.38 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.36 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.35 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.29 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.19 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.11 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.5 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.158 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.157 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.151 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.142 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.109 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.108 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.105 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.87 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.77 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.75 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.71 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.48 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.32 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.17 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.146 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.139 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.104 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.18 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.154 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.150 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.117 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.84 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.82 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.76 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.59 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.49 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.25 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.136 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.119 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.106 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.85 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.68 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.60 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.47 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.44 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.26 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.155 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.132 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.9 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.66 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.88 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.65 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.27 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.14 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.15 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.118 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.99 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.2 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.129 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.4 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.145 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.122 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.111 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.73 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.63 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.55 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.53 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.51 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.16 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.1 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.130 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.112 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.110 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.107 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.74 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.62 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.dinB.40 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.2 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.205 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.202 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.201 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.190 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.187 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.183 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.167 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.162 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.159 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.147 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.146 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.133 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.132 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.126 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.102 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.98 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.92 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.86 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.79 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.78 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.77 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.66 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.65 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.64 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.55 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.51 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.170 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.166 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.68 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.21 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.20 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.200 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.163 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.90 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.88 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.23 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.177 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.114 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.110 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.26 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.22 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.191 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.168 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.131 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.96 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.171 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.82 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.67 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.44 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.4 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.180 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.179 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.161 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.104 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.61 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.38 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.17 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.3 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.194 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.185 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.108 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.100 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.94 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.48 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.42 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.40 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.204 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.199 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.181 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.178 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.164 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.158 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.156 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.151 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.149 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.113 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.107 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.69 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.58 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.57 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.47 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.45 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.30 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.18 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.16 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.184 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.182 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.135 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.120 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.116 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.89 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.85 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.52 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.50 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.43 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.35 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.13 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.5 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.198 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.195 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.160 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.150 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.148 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.134 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.125 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.106 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.103 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.101 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.71 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.70 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.46 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.33 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.32 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.14 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.203 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.196 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.193 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.173 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.169 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.157 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.128 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.121 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.93 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.91 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.80 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.54 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.37 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.36 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.15 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.12 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.10 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.206 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.197 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.189 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.174 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.142 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.109 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.105 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.73 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.27 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.7 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.186 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.172 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.154 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.141 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.139 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.115 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.99 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.56 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.24 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.11 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.9 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.8 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.188 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.176 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.153 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.137 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.136 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.130 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.129 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.127 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.112 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.84 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.81 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.74 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.72 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.62 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.53 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.49 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.34 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.19 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.6 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.97 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.75 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.63 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.60 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.41 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.31 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.1 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.175 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.140 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.95 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.76 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.59 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.39 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.28 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.25 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.124 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.118 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.123 due to --exclude option\n",
      "[19:48:24] Found exact allele match cfreundii.fadD-112\n",
      "[19:48:24] Excluding ecoli_2.uidA.29 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.29 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.165 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.165 due to --exclude option\n",
      "[19:48:24] Found exact allele match ecoli.fumC-11\n",
      "[19:48:24] Excluding ecoli_2.pabB.7 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.183 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.167 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.166 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.164 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.148 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.100 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.64 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.165 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.66 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.189 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.179 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.88 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.51 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.44 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.21 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.186 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.162 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.159 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.107 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.98 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.91 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.53 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.49 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.39 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.38 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.22 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.6 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.182 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.178 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.134 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.106 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.70 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.69 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.56 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.47 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.45 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.41 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.17 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.3 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.194 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.191 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.181 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.168 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.151 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.149 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.135 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.128 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.118 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.117 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.87 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.67 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.62 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.61 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.50 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.48 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.30 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.19 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.18 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.4 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.193 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.190 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.188 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.150 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.147 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.138 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.136 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.133 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.130 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.114 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.110 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.108 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.105 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.94 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.85 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.83 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.65 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.52 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.31 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.20 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.109 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.80 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.46 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.35 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.175 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.123 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.115 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.96 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.75 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.74 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.58 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.174 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.161 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.157 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.156 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.102 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.60 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.37 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.36 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.23 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.187 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.127 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.111 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.103 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.76 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.15 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.13 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.10 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.5 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.1 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.196 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.173 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.131 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.116 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.113 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.99 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.97 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.73 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.71 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.59 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.55 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.16 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.14 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.12 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.11 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.172 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.158 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.139 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.86 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.84 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.34 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.33 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.32 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.8 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.2 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.197 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.176 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.171 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.160 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.137 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.126 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.104 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.89 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.82 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.81 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.77 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.57 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.42 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.9 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.125 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.72 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.54 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.170 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.93 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.40 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.29 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.28 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.24 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.120 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.92 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.79 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.180 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.132 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.27 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.195 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.129 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.121 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.26 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.101 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.90 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.68 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.184 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.163 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.112 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.pabB.25 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.83 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.83 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.87 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.87 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.138 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.uidA.138 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.7 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.202 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.198 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.197 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.192 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.186 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.89 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.61 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.60 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.59 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.58 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.52 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.22 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.185 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.152 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.113 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.81 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.107 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.135 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.80 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.28 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.166 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.42 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.37 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.35 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.201 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.191 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.153 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.131 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.23 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.6 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.5 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.193 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.154 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.126 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.125 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.109 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.90 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.49 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.46 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.41 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.33 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.203 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.160 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.136 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.97 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.82 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.78 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.67 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.63 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.43 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.38 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.31 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.29 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.16 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.12 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.194 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.189 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.179 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.178 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.177 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.157 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.151 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.133 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.132 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.118 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.110 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.105 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.85 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.71 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.50 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.25 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.24 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.15 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.9 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.184 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.176 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.175 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.159 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.144 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.142 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.128 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.127 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.123 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.111 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.77 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.74 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.65 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.54 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.51 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.44 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.39 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.27 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.26 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.19 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.18 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.200 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.195 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.187 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.183 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.158 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.146 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.145 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.137 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.134 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.121 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.116 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.108 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.100 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.75 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.40 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.34 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.30 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.17 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.13 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.10 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.3 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.2 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.188 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.181 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.161 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.156 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.130 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.129 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.124 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.122 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.119 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.103 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.98 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.92 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.86 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.84 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.73 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.70 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.55 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.48 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.47 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.32 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.21 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.20 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.11 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.4 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.1 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.190 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.182 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.165 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.117 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.112 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.104 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.102 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.101 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.96 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.95 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.93 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.87 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.83 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.76 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.64 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.45 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.36 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.8 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.120 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.106 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.69 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.155 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.99 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.94 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.88 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.72 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.66 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.143 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.91 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.57 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.53 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.14 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.68 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.150 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.149 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.115 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.162 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.141 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.62 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.180 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.putP.180 due to --exclude option\n",
      "[19:48:24] Found exact allele match senterica.aroC-621\n",
      "[19:48:24] Found exact allele match senterica.thrA-636\n",
      "[19:48:24] Found exact allele match senterica.dnaN-559\n",
      "[19:48:24] Found exact allele match ecoli.gyrB-4\n",
      "[19:48:24] Found exact allele match cronobacter.fusA-92\n",
      "[19:48:24] Found exact allele match cfreundii.mdh-172\n",
      "[19:48:24] Found exact allele match ecoli.mdh-8\n",
      "[19:48:24] Found exact allele match cfreundii.clpX-193\n",
      "[19:48:24] Found exact allele match ecoli.adk-10\n",
      "[19:48:24] Found exact allele match cfreundii.aspC-171\n",
      "[19:48:24] Found exact allele match senterica.sucA-581\n",
      "[19:48:24] Found exact allele match ecloacae.rplB-123\n",
      "[19:48:24] Excluding ecoli_2.polB.3 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.203 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.190 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.152 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.124 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.106 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.103 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.69 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.67 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.2 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.191 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.186 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.166 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.105 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.62 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.24 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.23 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.83 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.199 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.187 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.197 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.64 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.61 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.47 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.201 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.176 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.66 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.63 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.99 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.22 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.49 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.189 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.123 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.101 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.91 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.48 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.42 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.39 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.185 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.175 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.169 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.96 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.92 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.57 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.17 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.200 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.198 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.173 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.171 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.168 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.167 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.114 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.54 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.44 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.205 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.138 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.204 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.137 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.70 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.46 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.45 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.18 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.163 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.172 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.153 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.121 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.178 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.108 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.100 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.94 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.40 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.20 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.15 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.11 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.181 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.136 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.122 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.113 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.109 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.88 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.71 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.60 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.41 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.36 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.19 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.16 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.14 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.13 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.202 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.183 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.140 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.135 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.128 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.126 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.119 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.98 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.87 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.85 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.82 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.43 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.38 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.35 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.30 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.21 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.12 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.5 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.55 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.208 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.207 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.145 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.115 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.95 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.90 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.37 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.32 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.31 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.9 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.7 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.6 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.86 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.182 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.165 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.162 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.146 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.141 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.134 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.112 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.93 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.79 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.75 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.73 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.65 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.50 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.34 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.1 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.59 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.8 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.206 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.164 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.159 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.154 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.107 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.97 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.81 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.52 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.51 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.33 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.26 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.25 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.4 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.174 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.144 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.143 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.195 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.193 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.192 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.184 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.180 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.177 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.170 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.104 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.80 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.76 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.74 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.72 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.68 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.56 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.53 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.28 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.27 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.10 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.polB.58 due to --exclude option\n",
      "[19:48:24] Found exact allele match cfreundii.dnaG-84\n",
      "[19:48:24] Excluding ecoli_2.trpB.4 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.98 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.180 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.179 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.158 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.149 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.135 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.132 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.110 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.80 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.72 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.58 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.57 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.54 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.47 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.23 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.17 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.16 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.15 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.184 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.140 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.60 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.52 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.2 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.186 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.116 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.111 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.109 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.97 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.96 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.73 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.62 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.59 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.53 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.48 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.79 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.178 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.144 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.133 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.99 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.95 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.63 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.101 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.64 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.55 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.50 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.18 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.191 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.78 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.141 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.88 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.84 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.75 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.134 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.92 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.183 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.160 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.112 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.108 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.105 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.61 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.139 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.66 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.40 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.1 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.192 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.185 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.127 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.216 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.215 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.212 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.193 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.192 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.173 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.164 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.140 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.137 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.123 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.118 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.117 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.116 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.114 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.113 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.108 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.90 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.89 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.85 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.75 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.65 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.57 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.157 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.74 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.39 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.209 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.150 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.79 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.64 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.56 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.41 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.26 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.49 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.43 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.224 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.220 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.204 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.202 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.187 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.129 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.106 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.46 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.37 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.27 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.187 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.176 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.174 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.89 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.30 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.29 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.20 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.19 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.206 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.126 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.120 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.95 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.69 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.53 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.162 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.143 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.113 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.86 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.38 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.36 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.34 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.28 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.8 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.161 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.115 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.87 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.85 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.65 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.221 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.214 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.190 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.184 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.146 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.111 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.86 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.78 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.36 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.7 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.181 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.164 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.45 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.7 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.223 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.222 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.185 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.177 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.121 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.119 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.112 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.99 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.83 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.76 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.31 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.30 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.29 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.177 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.156 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.136 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.121 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.103 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.11 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.218 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.205 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.179 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.147 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.133 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.109 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.62 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.58 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.55 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.54 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.43 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.188 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.107 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.102 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.145 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.110 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.47 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.190 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.125 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.114 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.68 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.56 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.35 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.28 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.14 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.12 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.9 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.82 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.175 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.138 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.67 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.41 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.32 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.31 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.126 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.91 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.33 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.25 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.10 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.5 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.119 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.24 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.82 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.44 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.174 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.48 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.171 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.93 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.213 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.201 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.39 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.166 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.131 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.27 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.21 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.101 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.51 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.189 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.46 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.13 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.6 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.219 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.139 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.173 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.172 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.167 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.76 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.51 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.1 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.84 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.66 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.169 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.142 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.189 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.191 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.132 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.94 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.67 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.87 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.33 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.23 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.157 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.178 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.141 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.100 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.59 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.50 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.42 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.45 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.24 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.80 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.25 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.186 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpA.156 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.77 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.77 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.83 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.trpB.83 due to --exclude option\n",
      "[19:48:24] Found exact allele match ecoli.recA-2\n",
      "[19:48:24] Found exact allele match senterica.hemD-494\n",
      "[19:48:24] Found exact allele match ecoli.icd-8\n",
      "[19:48:24] Excluding ecoli_2.icdA.2 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.292 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.235 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.159 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.155 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.90 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.177 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.191 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.103 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.80 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.265 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.86 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.307 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.149 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.239 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.29 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.28 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.118 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.104 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.269 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.240 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.85 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.117 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.83 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.192 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.127 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.88 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.87 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.64 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.5 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.260 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.168 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.163 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.91 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.81 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.38 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.294 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.160 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.305 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.206 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.162 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.158 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.144 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.34 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.31 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.30 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.301 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.282 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.254 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.123 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.92 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.79 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.67 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.13 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.310 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.293 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.106 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.14 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.271 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.181 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.164 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.152 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.141 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.110 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.66 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.46 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.36 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.15 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.6 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.3 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.1 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.309 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.250 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.219 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.201 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.200 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.172 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.153 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.134 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.49 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.39 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.287 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.277 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.273 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.272 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.262 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.256 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.234 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.213 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.174 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.146 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.131 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.125 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.119 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.116 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.107 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.105 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.93 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.89 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.73 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.50 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.40 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.37 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.35 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.16 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.311 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.238 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.208 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.176 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.161 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.150 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.95 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.72 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.55 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.45 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.17 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.308 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.290 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.270 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.230 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.225 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.207 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.196 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.187 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.173 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.136 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.120 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.113 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.111 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.108 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.100 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.97 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.47 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.44 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.33 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.25 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.306 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.275 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.242 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.224 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.195 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.194 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.179 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.170 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.145 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.128 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.124 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.112 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.84 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.65 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.62 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.56 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.43 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.26 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.298 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.285 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.283 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.280 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.276 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.223 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.218 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.217 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.209 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.183 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.154 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.151 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.140 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.139 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.122 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.102 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.48 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.42 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.32 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.24 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.20 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.4 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.268 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.266 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.255 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.243 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.228 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.211 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.205 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.198 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.156 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.132 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.77 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.75 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.63 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.58 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.23 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.19 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.18 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.297 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.236 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.216 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.215 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.148 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.138 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.135 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.130 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.101 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.98 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.71 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.68 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.22 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.313 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.288 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.279 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.278 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.253 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.249 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.220 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.212 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.210 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.199 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.184 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.137 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.133 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.109 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.99 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.60 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.21 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.221 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.202 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.193 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.147 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.114 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.94 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.61 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.51 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.41 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.7 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.303 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.302 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.257 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.121 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.76 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.69 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.12 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.312 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.289 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.267 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.264 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.222 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.214 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.203 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.142 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.284 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.281 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.263 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.261 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.258 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.247 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.185 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.27 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.9 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.304 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.190 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.53 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.52 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.274 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.237 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.197 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.143 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.70 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.54 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.8 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.299 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.296 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.96 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.78 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.74 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.59 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.252 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.204 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.186 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.10 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.291 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.251 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.226 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.157 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.129 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.314 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.300 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.259 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.115 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.11 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.188 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.178 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.126 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.189 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.171 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.82 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.175 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.246 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.182 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.180 due to --exclude option\n",
      "[19:48:24] Excluding ecoli_2.icdA.248 due to --exclude option\n",
      "[19:48:24] If you like MLST, you're going to absolutely love cgMLST!\n",
      "[19:48:24] Done.\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "mlst output/contigs.fasta"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Annotation genome\n",
    "Using prokka to annotate genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[tbl2asn-forever] Found Prokka input, correcting dates in .gbf|.sqn files.\n",
      "[tbl2asn-forever] Correcting dates in output/sample.gbf\n",
      "[tbl2asn-forever] Correcting dates in output/sample.sqn\n",
      "[tbl2asn-forever] Dates changed from 01-JAN-2019 to 22-SEP-2022\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[17:57:45] This is prokka 1.14.6\n",
      "[17:57:45] Written by Torsten Seemann <torsten.seemann@gmail.com>\n",
      "[17:57:45] Homepage is https://github.com/tseemann/prokka\n",
      "[17:57:45] Local time is Thu Sep 22 17:57:45 2022\n",
      "[17:57:45] You are quang\n",
      "[17:57:45] Operating system is linux\n",
      "[17:57:45] You have BioPerl 1.007002\n",
      "[17:57:45] System has 4 cores.\n",
      "[17:57:45] Option --cpu asked for 8 cores, but system only has 4\n",
      "[17:57:45] Will use maximum of 4 cores.\n",
      "[17:57:45] Annotating as >>> Bacteria <<<\n",
      "[17:57:45] Re-using existing --outdir output\n",
      "[17:57:45] Using filename prefix: sample.XXX\n",
      "[17:57:45] Setting HMMER_NCPU=1\n",
      "[17:57:45] Writing log to: output/sample.log\n",
      "[17:57:45] Command: /home/quang/miniconda3/envs/amrtraining/bin/prokka --force --addgenes --prefix sample --locus sample --outdir output output/contigs.fasta\n",
      "[17:57:45] Appending to PATH: /home/quang/miniconda3/envs/amrtraining/bin\n",
      "[17:57:45] Looking for 'aragorn' - found /home/quang/miniconda3/envs/amrtraining/bin/aragorn\n",
      "[17:57:45] Determined aragorn version is 001002 from 'ARAGORN v1.2.41 Dean Laslett'\n",
      "[17:57:46] Looking for 'barrnap' - found /home/quang/miniconda3/envs/amrtraining/bin/barrnap\n",
      "[17:57:46] Determined barrnap version is 000009 from 'barrnap 0.9'\n",
      "[17:57:46] Looking for 'blastp' - found /home/quang/miniconda3/envs/amrtraining/bin/blastp\n",
      "[17:57:48] Determined blastp version is 002013 from 'blastp: 2.13.0+'\n",
      "[17:57:48] Looking for 'cmpress' - found /home/quang/miniconda3/envs/amrtraining/bin/cmpress\n",
      "[17:57:48] Determined cmpress version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\n",
      "[17:57:48] Looking for 'cmscan' - found /home/quang/miniconda3/envs/amrtraining/bin/cmscan\n",
      "[17:57:48] Determined cmscan version is 001001 from '# INFERNAL 1.1.4 (Dec 2020)'\n",
      "[17:57:48] Looking for 'egrep' - found /bin/egrep\n",
      "[17:57:48] Looking for 'find' - found /usr/bin/find\n",
      "[17:57:48] Looking for 'grep' - found /bin/grep\n",
      "[17:57:48] Looking for 'hmmpress' - found /home/quang/miniconda3/envs/amrtraining/bin/hmmpress\n",
      "[17:57:48] Determined hmmpress version is 003003 from '# HMMER 3.3.2 (Nov 2020); http://hmmer.org/'\n",
      "[17:57:48] Looking for 'hmmscan' - found /home/quang/miniconda3/envs/amrtraining/bin/hmmscan\n",
      "[17:57:48] Determined hmmscan version is 003003 from '# HMMER 3.3.2 (Nov 2020); http://hmmer.org/'\n",
      "[17:57:48] Looking for 'java' - found /home/quang/miniconda3/envs/amrtraining/bin/java\n",
      "[17:57:48] Looking for 'makeblastdb' - found /home/quang/miniconda3/envs/amrtraining/bin/makeblastdb\n",
      "[17:57:48] Determined makeblastdb version is 002013 from 'makeblastdb: 2.13.0+'\n",
      "[17:57:48] Looking for 'minced' - found /home/quang/miniconda3/envs/amrtraining/bin/minced\n",
      "[17:57:50] Determined minced version is 004002 from 'minced 0.4.2'\n",
      "[17:57:50] Looking for 'parallel' - found /home/quang/miniconda3/envs/amrtraining/bin/parallel\n",
      "[17:57:50] Determined parallel version is 20220722 from 'GNU parallel 20220722'\n",
      "[17:57:50] Looking for 'prodigal' - found /home/quang/miniconda3/envs/amrtraining/bin/prodigal\n",
      "[17:57:50] Determined prodigal version is 002006 from 'Prodigal V2.6.3: February, 2016'\n",
      "[17:57:50] Looking for 'prokka-genbank_to_fasta_db' - found /home/quang/miniconda3/envs/amrtraining/bin/prokka-genbank_to_fasta_db\n",
      "[17:57:50] Looking for 'sed' - found /bin/sed\n",
      "[17:57:50] Looking for 'tbl2asn' - found /home/quang/miniconda3/envs/amrtraining/bin/tbl2asn\n",
      "[17:57:50] Determined tbl2asn version is 025007 from 'tbl2asn 25.7   arguments:'\n",
      "[17:57:50] Using genetic code table 11.\n",
      "[17:57:50] Loading and checking input file: output/contigs.fasta\n",
      "[17:57:51] Wrote 323 contigs totalling 5694587 bp.\n",
      "[17:57:51] Predicting tRNAs and tmRNAs\n",
      "[17:57:51] Running: aragorn -l -gc11  -w output\\/sample\\.fna\n",
      "[17:57:51] 1 tRNA-Thr c[21904,21979] 34 (cgt)\n",
      "[17:57:51] 1 tRNA-Gly [31549,31624] 34 (gcc)\n",
      "[17:57:51] 2 tRNA-Cys [31678,31751] 33 (gca)\n",
      "[17:57:51] 3 tRNA-Leu [31764,31850] 35 (taa)\n",
      "[17:57:51] 4 tRNA-Val c[264197,264273] 35 (gac)\n",
      "[17:57:51] 5 tRNA-Val c[264278,264354] 35 (gac)\n",
      "[17:57:51] 1 tRNA-Leu [47937,48023] 35 (cag)\n",
      "[17:57:51] 2 tRNA-Leu [48052,48138] 35 (cag)\n",
      "[17:57:51] 3 tRNA-Leu [48172,48258] 35 (cag)\n",
      "[17:57:51] 1 tRNA-Met c[19708,19783] 34 (cat)\n",
      "[17:57:51] 1 tRNA-Leu [63882,63966] 35 (caa)\n",
      "[17:57:51] 1 tRNA-Arg c[19436,19512] 35 (tct)\n",
      "[17:57:51] 1 tRNA-Arg [161299,161373] 34 (cct)\n",
      "[17:57:51] 1 tRNA-Pro [11940,12016] 35 (cgg)\n",
      "[17:57:51] 1 tRNA-Gly c[40064,40139] 34 (gcc)\n",
      "[17:57:51] 2 tRNA-Gly c[40175,40250] 34 (gcc)\n",
      "[17:57:51] 3 tRNA-Gly c[40408,40483] 34 (gcc)\n",
      "[17:57:51] 4 tRNA-Phe [70223,70298] 34 (gaa)\n",
      "[17:57:51] 1 tRNA-Ser c[19553,19642] 35 (cga)\n",
      "[17:57:51] 2 tRNA-Asn [20634,20709] 34 (gtt)\n",
      "[17:57:51] 1 tRNA-Thr [35,110] 34 (ggt)\n",
      "[17:57:51] 1 tRNA-Arg c[282,357] 35 (ccg)\n",
      "[17:57:51] 1 tRNA-Met [32438,32514] 35 (cat)\n",
      "[17:57:51] 2 tRNA-Leu [32524,32608] 35 (tag)\n",
      "[17:57:51] 3 tRNA-Gln [32632,32706] 33 (ttg)\n",
      "[17:57:51] 4 tRNA-Gln [32741,32815] 33 (ttg)\n",
      "[17:57:51] 5 tRNA-Met [32831,32907] 35 (cat)\n",
      "[17:57:52] 6 tRNA-Gln [32955,33029] 33 (ctg)\n",
      "[17:57:52] 1 tRNA-Gly c[520,593] 33 (ccc)\n",
      "[17:57:52] 2 tRNA-Phe [111899,111974] 34 (gaa)\n",
      "[17:57:52] 1 tRNA-Ser c[85,172] 35 (gga)\n",
      "[17:57:52] 1 tRNA-Asp c[23146,23222] 35 (gtc)\n",
      "[17:57:52] 1 tRNA-Pro [154971,155047] 35 (ggg)\n",
      "[17:57:52] 1 tRNA-Thr c[249,324] 34 (ggt)\n",
      "[17:57:52] 2 tRNA-Gly c[331,405] 34 (tcc)\n",
      "[17:57:52] 3 tRNA-Tyr c[522,606] 35 (gta)\n",
      "[17:57:52] 4 tRNA-Thr c[615,690] 34 (tgt)\n",
      "[17:57:52] 1 tRNA-Ile [19,95] 35 (gat)\n",
      "[17:57:52] 2 tRNA-Ala [138,213] 34 (tgc)\n",
      "[17:57:52] 1 tmRNA [74868,75227] 96,128 ANDETYALAA*\n",
      "[17:57:52] 1 tRNA-Met c[33435,33511] 35 (cat)\n",
      "[17:57:52] 2 tRNA-Leu c[37294,37380] 35 (gag)\n",
      "[17:57:52] 1 tRNA-Lys c[197694,197769] 34 (ttt)\n",
      "[17:57:52] 2 tRNA-Val c[197774,197849] 34 (tac)\n",
      "[17:57:52] 3 tRNA-Val c[197896,197971] 34 (tac)\n",
      "[17:57:52] 4 tRNA-Val c[198016,198091] 34 (tac)\n",
      "[17:57:52] 5 tRNA-Ala [200791,200866] 34 (ggc)\n",
      "[17:57:52] 6 tRNA-Ala [200906,200981] 34 (ggc)\n",
      "[17:57:52] 1 tRNA-Asn c[271,346] 34 (gtt)\n",
      "[17:57:52] 2 tRNA-Asn [2095,2170] 34 (gtt)\n",
      "[17:57:52] 3 tRNA-Asn [4504,4579] 34 (gtt)\n",
      "[17:57:52] 1 tRNA-Ser c[16479,16559] 38 (gga)\n",
      "[17:57:52] 1 tRNA-SeC [18518,18612] 35 (tca)\n",
      "[17:57:52] 1 tRNA-Ser [18623,18710] 35 (tga)\n",
      "[17:57:52] 2 tRNA-Ser [124366,124453] 35 (gga)\n",
      "[17:57:52] 3 tRNA-Lys c[268666,268741] 34 (ttt)\n",
      "[17:57:52] 4 tRNA-Lys c[268874,268949] 34 (ttt)\n",
      "[17:57:52] 5 tRNA-Lys c[269096,269171] 34 (ttt)\n",
      "[17:57:52] 6 tRNA-Val c[269175,269250] 34 (tac)\n",
      "[17:57:53] 7 tRNA-Lys c[269400,269475] 34 (ttt)\n",
      "[17:57:53] 8 tRNA-Val c[269478,269553] 34 (tac)\n",
      "[17:57:53] 9 tRNA-Lys c[269689,269764] 34 (ttt)\n",
      "[17:57:53] 1 tRNA-Met [122180,122255] 34 (cat)\n",
      "[17:57:53] 1 tRNA-Tyr c[16853,16937] 35 (gta)\n",
      "[17:57:53] 2 tRNA-Tyr c[17147,17231] 35 (gta)\n",
      "[17:57:53] 1 tmRNA [52528,52890] 90,125 ANDENYALAA**\n",
      "[17:57:53] 2 tRNA-Met c[60667,60742] 34 (cat)\n",
      "[17:57:53] 3 tRNA-Arg c[92696,92772] 35 (acg)\n",
      "[17:57:53] 4 tRNA-Arg c[92971,93047] 35 (acg)\n",
      "[17:57:53] 5 tRNA-Arg c[93110,93186] 35 (acg)\n",
      "[17:57:53] 6 tRNA-Arg [93370,93441] 34 (tcg)\n",
      "[17:57:53] 7 tRNA-Arg c[93385,93461] 35 (acg)\n",
      "[17:57:53] 8 tRNA-Ser c[93465,93557] 35 (gct)\n",
      "[17:57:53] 9 tRNA-Met [222178,222254] 35 (cat)\n",
      "[17:57:53] 10 tRNA-Met [222288,222364] 35 (cat)\n",
      "[17:57:53] 11 tRNA-Met [222398,222474] 35 (cat)\n",
      "[17:57:53] 1 tRNA-Pro c[52255,52331] 35 (tgg)\n",
      "[17:57:53] 2 tRNA-Leu c[52374,52460] 35 (cag)\n",
      "[17:57:53] 3 tRNA-His c[52481,52556] 34 (gtg)\n",
      "[17:57:53] 4 tRNA-Arg c[52615,52691] 35 (ccg)\n",
      "[17:57:53] 5 tRNA-Trp c[88036,88111] 34 (cca)\n",
      "[17:57:53] 1 tRNA-Ile [1836,1912] 35 (gat)\n",
      "[17:57:53] 2 tRNA-Ala [1983,2058] 34 (tgc)\n",
      "[17:57:53] Found 83 tRNAs\n",
      "[17:57:53] Predicting Ribosomal RNAs\n",
      "[17:57:53] Running Barrnap with 4 threads\n",
      "[17:57:56] 1 Contig_311_176.481 243 16S ribosomal RNA\n",
      "[17:57:56] 2 Contig_311_176.481 2441 23S ribosomal RNA\n",
      "[17:57:56] 3 Contig_311_176.481 5506 5S ribosomal RNA (partial)\n",
      "[17:57:56] 4 Contig_51_544.22 1 16S ribosomal RNA (partial)\n",
      "[17:57:56] 5 Contig_56_562.784 2 16S ribosomal RNA (partial)\n",
      "[17:57:56] 6 Contig_58_86.5211 152 5S ribosomal RNA\n",
      "[17:57:56] Found 6 rRNAs\n",
      "[17:57:56] Skipping ncRNA search, enable with --rfam if desired.\n",
      "[17:57:56] Total of 87 tRNA + rRNA features\n",
      "[17:57:56] Searching for CRISPR repeats\n",
      "[17:57:57] CRISPR1 Contig_254_65.1262 19 with 5 spacers\n",
      "[17:57:57] CRISPR2 Contig_283_46.7228 19 with 10 spacers\n",
      "[17:57:57] CRISPR3 Contig_305_84.0088 152614 with 12 spacers\n",
      "[17:57:57] CRISPR4 Contig_305_84.0088 178866 with 6 spacers\n",
      "[17:57:57] Found 4 CRISPRs\n",
      "[17:57:57] Predicting coding sequences\n",
      "[17:57:57] Contigs total 5694587 bp, so using single mode\n",
      "[17:57:57] Running: prodigal -i output\\/sample\\.fna -c -m -g 11 -p single -f sco -q\n",
      "[17:58:03] Excluding CDS which overlaps existing RNA (tRNA) at Contig_23_89.5642:63761..63985 on - strand\n",
      "[17:58:03] Excluding CDS which overlaps existing RNA (rRNA) at Contig_56_562.784:147..401 on + strand\n",
      "[17:58:03] Excluding CDS which overlaps existing RNA (tRNA) at Contig_79_83.8174:32940..33032 on - strand\n",
      "[17:58:05] Excluding CDS which overlaps existing RNA (tRNA) at Contig_245_98.386:16244..16489 on - strand\n",
      "[17:58:06] Excluding CDS which overlaps existing RNA (tRNA) at Contig_298_76.069:16647..17240 on + strand\n",
      "[17:58:06] Excluding CDS which overlaps existing RNA (tRNA) at Contig_305_84.0088:92835..93305 on + strand\n",
      "[17:58:06] Excluding CDS which overlaps existing RNA (rRNA) at Contig_311_176.481:956..1063 on + strand\n",
      "[17:58:07] Found 5427 CDS\n",
      "[17:58:07] Connecting features back to sequences\n",
      "[17:58:07] Not using genus-specific database. Try --usegenus to enable it.\n",
      "[17:58:07] Annotating CDS, please be patient.\n",
      "[17:58:07] Will use 4 CPUs for similarity searching.\n",
      "[17:58:08] There are still 5427 unannotated CDS left (started with 5427)\n",
      "[17:58:08] Will use blast to search against /home/quang/miniconda3/envs/amrtraining/db/kingdom/Bacteria/IS with 4 CPUs\n",
      "[17:58:08] Running: cat output\\/sample\\.IS\\.tmp\\.21213\\.faa | parallel --gnu --plain -j 4 --block 208238 --recstart '>' --pipe blastp -query - -db /home/quang/miniconda3/envs/amrtraining/db/kingdom/Bacteria/IS -evalue 1e-30 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > output\\/sample\\.IS\\.tmp\\.21213\\.blast 2> /dev/null\n",
      "[17:58:48] Modify product: Tn3 family transposase Tn5403 => Tn3 family transposase\n",
      "[17:58:49] Cleaned 1 /product names\n",
      "[17:58:49] Deleting unwanted file: output/sample.IS.tmp.21213.faa\n",
      "[17:58:49] Deleting unwanted file: output/sample.IS.tmp.21213.blast\n",
      "[17:58:50] There are still 5376 unannotated CDS left (started with 5427)\n",
      "[17:58:50] Will use blast to search against /home/quang/miniconda3/envs/amrtraining/db/kingdom/Bacteria/AMR with 4 CPUs\n",
      "[17:58:50] Running: cat output\\/sample\\.AMR\\.tmp\\.21213\\.faa | parallel --gnu --plain -j 4 --block 206436 --recstart '>' --pipe blastp -query - -db /home/quang/miniconda3/envs/amrtraining/db/kingdom/Bacteria/AMR -evalue 9.99999999999999e-301 -qcov_hsp_perc 90 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > output\\/sample\\.AMR\\.tmp\\.21213\\.blast 2> /dev/null\n",
      "[17:59:22] Deleting unwanted file: output/sample.AMR.tmp.21213.faa\n",
      "[17:59:22] Deleting unwanted file: output/sample.AMR.tmp.21213.blast\n",
      "[17:59:23] There are still 5374 unannotated CDS left (started with 5427)\n",
      "[17:59:23] Will use blast to search against /home/quang/miniconda3/envs/amrtraining/db/kingdom/Bacteria/sprot with 4 CPUs\n",
      "[17:59:23] Running: cat output\\/sample\\.sprot\\.tmp\\.21213\\.faa | parallel --gnu --plain -j 4 --block 206309 --recstart '>' --pipe blastp -query - -db /home/quang/miniconda3/envs/amrtraining/db/kingdom/Bacteria/sprot -evalue 1e-09 -qcov_hsp_perc 80 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > output\\/sample\\.sprot\\.tmp\\.21213\\.blast 2> /dev/null\n",
      "[18:02:08] Modify product: Uncharacterized oxidoreductase Rv0945 => putative oxidoreductase\n",
      "[18:02:08] Modify product: Probable diguanylate cyclase DgcE => putative diguanylate cyclase DgcE\n",
      "[18:02:08] Modify product: Probable diguanylate cyclase DgcE => putative diguanylate cyclase DgcE\n",
      "[18:02:08] Modify product: UPF0339 protein YegP => hypothetical protein\n",
      "[18:02:08] Modify product: 23S rRNA 5-hydroxycytidine C2501 synthase => 23S rRNA 5-hydroxycytidine synthase\n",
      "[18:02:08] Modify product: Uncharacterized fimbrial-like protein YehA => putative fimbrial-like protein YehA\n",
      "[18:02:08] Modify product: Probable fimbrial chaperone YehC => putative fimbrial chaperone YehC\n",
      "[18:02:08] Modify product: Uncharacterized fimbrial-like protein YehD => putative fimbrial-like protein YehD\n",
      "[18:02:08] Modify product: Uncharacterized protein YehL => putative protein YehL\n",
      "[18:02:08] Modify product: Uncharacterized lipoprotein YehR => putative lipoprotein YehR\n",
      "[18:02:08] Modify product: UPF0387 membrane protein YohO => hypothetical protein\n",
      "[18:02:08] Modify product: Uncharacterized oxidoreductase YohF => putative oxidoreductase YohF\n",
      "[18:02:08] Modify product: UPF0299 membrane protein YohJ => hypothetical protein\n",
      "[18:02:08] Modify product: UPF0324 inner membrane protein YeiH => hypothetical protein\n",
      "[18:02:08] Modify product: UPF0153 protein YeiW => hypothetical protein\n",
      "[18:02:08] Modify product: Probable cyclic di-GMP phosphodiesterase PdeN => putative cyclic di-GMP phosphodiesterase PdeN\n",
      "[18:02:08] Modify product: Uncharacterized ABC transporter ATP-binding protein YejF => putative ABC transporter ATP-binding protein YejF\n",
      "[18:02:08] Modify product: Uncharacterized protein YejG => putative protein YejG\n",
      "[18:02:08] Modify product: UPF0352 protein YejL => hypothetical protein\n",
      "[18:02:08] Modify product: Probable autotransporter ROD_p1121 => putative autotransporter\n",
      "[18:02:08] Modify product: UPF0597 protein YhaM => hypothetical protein\n",
      "[18:02:08] Modify product: Probable serine transporter => putative serine transporter\n",
      "[18:02:08] Modify product: Protein SlyX homolog => Protein SlyX \n",
      "[18:02:08] Modify product: Uncharacterized ABC transporter ATP-binding protein YheS => putative ABC transporter ATP-binding protein YheS\n",
      "[18:02:08] Modify product: UPF0270 protein PA3463 => hypothetical protein\n",
      "[18:02:08] Modify product: Probable protein adenylyltransferase Fic => putative protein adenylyltransferase Fic\n",
      "[18:02:08] Modify product: Uncharacterized protein YhfG => putative protein YhfG\n",
      "[18:02:08] Modify product: Probable fructoselysine/psicoselysine transporter FrlA => putative fructoselysine/psicoselysine transporter FrlA\n",
      "[18:02:08] Modify product: Probable fructoselysine utilization operon transcriptional repressor => putative fructoselysine utilization operon transcriptional repressor\n",
      "[18:02:08] Modify product: Uncharacterized protein YhfS => putative protein YhfS\n",
      "[18:02:08] Modify product: Uncharacterized protein YhfX => putative protein YhfX\n",
      "[18:02:08] Modify product: Putative membrane protein IgaA homolog => Putative membrane protein IgaA \n",
      "[18:02:08] Modify product: Probable [Fe-S]-dependent transcriptional repressor FeoC => putative [Fe-S]-dependent transcriptional repressor FeoC\n",
      "[18:02:08] Modify product: UPF0056 inner membrane protein YhgN => hypothetical protein\n",
      "[18:02:08] Modify product: Uncharacterized oxidoreductase YhhX => putative oxidoreductase YhhX\n",
      "[18:02:08] Modify product: Uncharacterized protein YhhA => putative protein YhhA\n",
      "[18:02:08] Modify product: Uncharacterized protein YhhA => putative protein YhhA\n",
      "[18:02:08] Modify product: Uncharacterized MFS-type transporter YhhS => putative MFS-type transporter YhhS\n",
      "[18:02:08] Modify product: Probable chromate transport protein => putative chromate transport protein\n",
      "[18:02:08] Modify product: Transcriptional regulator Rv3488 => Transcriptional regulator\n",
      "[18:02:08] Modify product: Uncharacterized HTH-type transcriptional regulator CBU_1416 => putative HTH-type transcriptional regulator\n",
      "[18:02:08] Modify product: Probable multidrug resistance protein EmrY => putative multidrug resistance protein EmrY\n",
      "[18:02:08] Modify product: Probable multidrug resistance protein EmrK => putative multidrug resistance protein EmrK\n",
      "[18:02:08] Modify product: Probable TonB-dependent receptor NMB0964 => putative TonB-dependent receptor\n",
      "[18:02:08] Modify product: Probable pyridine nucleotide-disulfide oxidoreductase RclA => putative pyridine nucleotide-disulfide oxidoreductase RclA\n",
      "[18:02:08] Modify product: Probable pyridine nucleotide-disulfide oxidoreductase RclA => putative pyridine nucleotide-disulfide oxidoreductase RclA\n",
      "[18:02:08] Modify product: Uncharacterized protein YhcH => putative protein YhcH\n",
      "[18:02:08] Modify product: Uncharacterized GST-like protein YibF => putative GST-like protein YibF\n",
      "[18:02:08] Modify product: UPF0401 protein YkfF => hypothetical protein\n",
      "[18:02:08] Modify product: Uncharacterized metal-dependent hydrolase YcfH => putative metal-dependent hydrolase YcfH\n",
      "[18:02:08] Modify product: Probable multidrug ABC transporter permease YbhR => putative multidrug ABC transporter permease YbhR\n",
      "[18:02:09] Modify product: Putative universal stress protein SA1532 => Putative universal stress protein\n",
      "[18:02:09] Modify product: UPF0255 protein VV1_0328 => hypothetical protein\n",
      "[18:02:09] Modify product: Uncharacterized N-acetyltransferase YafP => putative N-acetyltransferase YafP\n",
      "[18:02:09] Modify product: UPF0394 inner membrane protein YedE => hypothetical protein\n",
      "[18:02:09] Modify product: Probable transcriptional regulatory protein YebC => putative transcriptional regulatory protein YebC\n",
      "[18:02:09] Modify product: Uncharacterized protein YebV => putative protein YebV\n",
      "[18:02:09] Modify product: Probable manganese efflux pump MntP => putative manganese efflux pump MntP\n",
      "[18:02:09] Modify product: UPF0266 membrane protein YobD => hypothetical protein\n",
      "[18:02:09] Modify product: UPF0053 inner membrane protein YoaE => hypothetical protein\n",
      "[18:02:09] Modify product: Probable cyclic di-GMP phosphodiesterase PdeD => putative cyclic di-GMP phosphodiesterase PdeD\n",
      "[18:02:09] Modify product: Probable ATP-dependent DNA helicase YoaA => putative ATP-dependent DNA helicase YoaA\n",
      "[18:02:09] Modify product: Probable diguanylate cyclase DgcJ => putative diguanylate cyclase DgcJ\n",
      "[18:02:09] Modify product: Probable diguanylate cyclase CdgI => putative diguanylate cyclase CdgI\n",
      "[18:02:09] Modify product: Stress response UPF0229 protein YhbH => hypothetical protein\n",
      "[18:02:09] Modify product: Uncharacterized zinc-type alcohol dehydrogenase-like protein YdjJ => putative zinc-type alcohol dehydrogenase-like protein YdjJ\n",
      "[18:02:09] Modify product: Uncharacterized protein YdjI => putative protein YdjI\n",
      "[18:02:09] Modify product: Uncharacterized sugar kinase YdjH => putative sugar kinase YdjH\n",
      "[18:02:09] Modify product: Uncharacterized HTH-type transcriptional regulator YdjF => putative HTH-type transcriptional regulator YdjF\n",
      "[18:02:09] Modify product: Probable ketoamine kinase lp_1983 => putative ketoamine kinase\n",
      "[18:02:09] Modify product: Uncharacterized protein YniD => putative protein YniD\n",
      "[18:02:09] Modify product: Uncharacterized protein YdiE => putative protein YdiE\n",
      "[18:02:09] Modify product: Uncharacterized protein HI_1163 => putative protein\n",
      "[18:02:09] Modify product: Probable ATP-dependent transporter SufC => putative ATP-dependent transporter SufC\n",
      "[18:02:09] Modify product: Probable L,D-transpeptidase YnhG => putative L,D-transpeptidase YnhG\n",
      "[18:02:09] Modify product: Uncharacterized ferredoxin-like protein YdhY => putative ferredoxin-like protein YdhY\n",
      "[18:02:09] Modify product: Uncharacterized oxidoreductase YdhV => putative oxidoreductase YdhV\n",
      "[18:02:09] Modify product: Uncharacterized protein YdhW => putative protein YdhW\n",
      "[18:02:09] Modify product: Uncharacterized ferredoxin-like protein YdhX => putative ferredoxin-like protein YdhX\n",
      "[18:02:09] Modify product: Uncharacterized protein YdhT => putative protein YdhT\n",
      "[18:02:09] Modify product: Probable autotransporter ROD_p1121 => putative autotransporter\n",
      "[18:02:09] Modify product: Uncharacterized HTH-type transcriptional regulator YahB => putative HTH-type transcriptional regulator YahB\n",
      "[18:02:09] Modify product: Uncharacterized protein YnhF => putative protein YnhF\n",
      "[18:02:09] Modify product: Uncharacterized oxidoreductase YdgJ => putative oxidoreductase YdgJ\n",
      "[18:02:09] Modify product: Probable dimethyl sulfoxide reductase chain YnfF => putative dimethyl sulfoxide reductase chain YnfF\n",
      "[18:02:09] Modify product: Uncharacterized protein YnfD => putative protein YnfD\n",
      "[18:02:09] Modify product: UPF0482 protein YnfB => hypothetical protein\n",
      "[18:02:09] Modify product: UPF0060 membrane protein YnfA => hypothetical protein\n",
      "[18:02:09] Modify product: Probable amino-acid metabolite efflux pump => putative amino-acid metabolite efflux pump\n",
      "[18:02:09] Modify product: UPF0056 inner membrane protein MarC => hypothetical protein\n",
      "[18:02:09] Modify product: Uncharacterized HTH-type transcriptional regulator HI_1476 => putative HTH-type transcriptional regulator\n",
      "[18:02:09] Modify product: UPF0442 protein YjjB => hypothetical protein\n",
      "[18:02:09] Modify product: Probable HTH-type transcriptional regulator LgoR => putative HTH-type transcriptional regulator LgoR\n",
      "[18:02:09] Modify product: Probable L-galactonate transporter => putative L-galactonate transporter\n",
      "[18:02:09] Modify product: Probable L,D-transpeptidase YcfS => putative L,D-transpeptidase YcfS\n",
      "[18:02:09] Modify product: UPF0227 protein YcfP => hypothetical protein\n",
      "[18:02:09] Modify product: Uncharacterized metal-dependent hydrolase YcfH => putative metal-dependent hydrolase YcfH\n",
      "[18:02:09] Modify product: 23S rRNA 5-hydroxycytidine C2501 synthase => 23S rRNA 5-hydroxycytidine synthase\n",
      "[18:02:09] Modify product: Probable mRNA interferase toxin HicA => putative mRNA interferase toxin HicA\n",
      "[18:02:09] Modify product: Uncharacterized protein YdcY => putative protein YdcY\n",
      "[18:02:09] Modify product: Uncharacterized protein YncE => putative protein YncE\n",
      "[18:02:09] Modify product: Uncharacterized protein YtfJ => putative protein YtfJ\n",
      "[18:02:09] Modify product: UPF0053 inner membrane protein YtfL => hypothetical protein\n",
      "[18:02:09] Modify product: UPF0307 protein YjgA => hypothetical protein\n",
      "[18:02:09] Modify product: Uncharacterized protein YjgL => putative protein YjgL\n",
      "[18:02:09] Modify product: Uncharacterized protein YjgL => putative protein YjgL\n",
      "[18:02:09] Modify product: UPF0167 protein CbrC => hypothetical protein\n",
      "[18:02:09] Modify product: Probable glucosamine-6-phosphate deaminase 2 => putative glucosamine-6-phosphate deaminase 2\n",
      "[18:02:09] Modify product: Probable transport protein HsrA => putative transport protein HsrA\n",
      "[18:02:09] Modify product: Probable RNA polymerase sigma factor FecI => putative RNA polymerase sigma factor FecI\n",
      "[18:02:09] Modify product: UPF0401 protein YkfF => hypothetical protein\n",
      "[18:02:09] Modify product: Probable DNA endonuclease SmrA => putative DNA endonuclease SmrA\n",
      "[18:02:09] Modify product: Uncharacterized protein YcjY => putative protein YcjY\n",
      "[18:02:09] Modify product: UPF0283 membrane protein YcjF => hypothetical protein\n",
      "[18:02:09] Modify product: Uncharacterized protein YcjX => putative protein YcjX\n",
      "[18:02:09] Modify product: UPF0509 protein YciZ => hypothetical protein\n",
      "[18:02:09] Modify product: Uncharacterized protein YciH => putative protein YciH\n",
      "[18:02:09] Modify product: Probable galactarate transporter => putative galactarate transporter\n",
      "[18:02:09] Modify product: Uncharacterized protein HI_0933 => putative protein\n",
      "[18:02:09] Modify product: Probable glutamate/gamma-aminobutyrate antiporter => putative glutamate/gamma-aminobutyrate antiporter\n",
      "[18:02:09] Modify product: Probable D,D-dipeptide-binding periplasmic protein DdpA => putative D,D-dipeptide-binding periplasmic protein DdpA\n",
      "[18:02:09] Modify product: Probable D,D-dipeptide transport system permease protein DdpB => putative D,D-dipeptide transport system permease protein DdpB\n",
      "[18:02:09] Modify product: Probable D,D-dipeptide transport system permease protein DdpC => putative D,D-dipeptide transport system permease protein DdpC\n",
      "[18:02:09] Modify product: Probable D,D-dipeptide transport ATP-binding protein DdpD => putative D,D-dipeptide transport ATP-binding protein DdpD\n",
      "[18:02:09] Modify product: Probable D,D-dipeptide transport ATP-binding protein DdpF => putative D,D-dipeptide transport ATP-binding protein DdpF\n",
      "[18:02:09] Modify product: Uncharacterized HTH-type transcriptional regulator YddM => putative HTH-type transcriptional regulator YddM\n",
      "[18:02:09] Modify product: Probable nitrate reductase molybdenum cofactor assembly chaperone NarW => putative nitrate reductase molybdenum cofactor assembly chaperone NarW\n",
      "[18:02:09] Modify product: Uncharacterized isomerase YddE => putative isomerase YddE\n",
      "[18:02:09] Modify product: Prophage tail fiber assembly protein homolog TfaE => Prophage tail fiber assembly protein TfaE\n",
      "[18:02:09] Modify product: Probable 2-dehydro-3-deoxygalactonokinase DgoK1 => putative 2-dehydro-3-deoxygalactonokinase DgoK1\n",
      "[18:02:09] Modify product: Uncharacterized symporter YidK => putative symporter YidK\n",
      "[18:02:09] Modify product: Uncharacterized signaling protein PA1727 => putative signaling protein\n",
      "[18:02:09] Modify product: Uncharacterized protein YqfG => putative protein YqfG\n",
      "[18:02:09] Modify product: UPF0073 inner membrane protein YqfA => hypothetical protein\n",
      "[18:02:09] Modify product: UPF0267 protein YqfB => hypothetical protein\n",
      "[18:02:09] Modify product: Uncharacterized oxidoreductase YgfF => putative oxidoreductase YgfF\n",
      "[18:02:09] Modify product: UPF0149 protein HI_0817 => hypothetical protein\n",
      "[18:02:09] Modify product: UPF0301 protein YqgE => hypothetical protein\n",
      "[18:02:09] Modify product: Uncharacterized protein YggR => putative protein YggR\n",
      "[18:02:09] Modify product: UPF0235 protein YggU => hypothetical protein\n",
      "[18:02:09] Modify product: Uncharacterized protein HI_0341 => putative protein\n",
      "[18:02:09] Modify product: Probable Fe(2+)-trafficking protein => putative Fe(2+)-trafficking protein\n",
      "[18:02:09] Modify product: Uncharacterized protein YbdM => putative protein YbdM\n",
      "[18:02:09] Modify product: Probable cystine transporter YijE => putative cystine transporter YijE\n",
      "[18:02:09] Modify product: UPF0381 protein YiiS => hypothetical protein\n",
      "[18:02:09] Modify product: Probable phosphatase YcdX => putative phosphatase YcdX\n",
      "[18:02:09] Modify product: Uncharacterized protein YmdA => putative protein YmdA\n",
      "[18:02:09] Modify product: UPF0176 protein YceA => hypothetical protein\n",
      "[18:02:09] Modify product: Cytochrome b561 homolog 2 => Cytochrome b561 \n",
      "[18:02:09] Modify product: Uncharacterized protein YceO => putative protein YceO\n",
      "[18:02:09] Modify product: Uncharacterized lipoprotein YceB => putative lipoprotein YceB\n",
      "[18:02:09] Modify product: UPF0502 protein YceH => hypothetical protein\n",
      "[18:02:09] Modify product: Uncharacterized methyltransferase YcgJ => putative methyltransferase YcgJ\n",
      "[18:02:09] Modify product: Probable 9-O-acetyl-N-acetylneuraminic acid deacetylase => putative 9-O-acetyl-N-acetylneuraminic acid deacetylase\n",
      "[18:02:09] Modify product: Probable N-acetylneuraminic acid outer membrane channel protein NanC => putative N-acetylneuraminic acid outer membrane channel protein NanC\n",
      "[18:02:09] Modify product: Uncharacterized ABC transporter ATP-binding protein YheS => putative ABC transporter ATP-binding protein YheS\n",
      "[18:02:10] Modify product: Probable thiol:disulfide interchange protein DsbC => putative thiol:disulfide interchange protein DsbC\n",
      "[18:02:10] Modify product: Uncharacterized fimbrial-like protein SfmF => putative fimbrial-like protein SfmF\n",
      "[18:02:10] Modify product: Uncharacterized fimbrial-like protein SfmH => putative fimbrial-like protein SfmH\n",
      "[18:02:10] Modify product: Probable fimbrial chaperone SfmC => putative fimbrial chaperone SfmC\n",
      "[18:02:10] Modify product: Uncharacterized fimbrial-like protein SfmA => putative fimbrial-like protein SfmA\n",
      "[18:02:10] Modify product: Bifunctional protein FolD => Bifunctional protein FolD protein\n",
      "[18:02:10] Modify product: Uncharacterized protein YbcJ => putative protein YbcJ\n",
      "[18:02:10] Modify product: Uncharacterized protein YbbC => putative protein YbbC\n",
      "[18:02:10] Modify product: Uncharacterized protein YccJ => putative protein YccJ\n",
      "[18:02:10] Modify product: Probable malonic semialdehyde reductase RutE => putative malonic semialdehyde reductase RutE\n",
      "[18:02:10] Modify product: Probable diguanylate cyclase DgcT => putative diguanylate cyclase DgcT\n",
      "[18:02:10] Modify product: Probable fimbrial chaperone YadV => putative fimbrial chaperone YadV\n",
      "[18:02:10] Modify product: Uncharacterized protein YjiK => putative protein YjiK\n",
      "[18:02:10] Modify product: Probable RNA 2'-phosphotransferase => putative RNA 2'-phosphotransferase\n",
      "[18:02:10] Modify product: Probable autotransporter YfaL => putative autotransporter YfaL\n",
      "[18:02:10] Modify product: Uncharacterized HTH-type transcriptional regulator RhmR => putative HTH-type transcriptional regulator RhmR\n",
      "[18:02:10] Modify product: Uncharacterized HTH-type transcriptional regulator RhmR => putative HTH-type transcriptional regulator RhmR\n",
      "[18:02:10] Modify product: Probable 4-deoxy-4-formamido-L-arabinose-phosphoundecaprenol deformylase ArnD => putative 4-deoxy-4-formamido-L-arabinose-phosphoundecaprenol deformylase ArnD\n",
      "[18:02:10] Modify product: Probable 4-amino-4-deoxy-L-arabinose-phosphoundecaprenol flippase subunit ArnE => putative 4-amino-4-deoxy-L-arabinose-phosphoundecaprenol flippase subunit ArnE\n",
      "[18:02:10] Modify product: Probable 4-amino-4-deoxy-L-arabinose-phosphoundecaprenol flippase subunit ArnF => putative 4-amino-4-deoxy-L-arabinose-phosphoundecaprenol flippase subunit ArnF\n",
      "[18:02:10] Modify product: UPF0304 protein YfbU => hypothetical protein\n",
      "[18:02:10] Modify product: UPF0208 membrane protein YfbV => hypothetical protein\n",
      "[18:02:10] Modify product: Uncharacterized Nudix hydrolase YfcD => putative Nudix hydrolase YfcD\n",
      "[18:02:10] Modify product: Epimerase family protein SA0724 => Epimerase family protein\n",
      "[18:02:10] Modify product: Uncharacterized MFS-type transporter YfcJ => putative MFS-type transporter YfcJ\n",
      "[18:02:10] Modify product: Probable membrane transporter protein YfcA => putative membrane transporter protein YfcA\n",
      "[18:02:10] Modify product: Uncharacterized protein YfcO => putative protein YfcO\n",
      "[18:02:10] Modify product: Uncharacterized fimbrial-like protein YfcP => putative fimbrial-like protein YfcP\n",
      "[18:02:10] Modify product: Uncharacterized fimbrial-like protein YfcQ => putative fimbrial-like protein YfcQ\n",
      "[18:02:10] Modify product: Uncharacterized fimbrial-like protein YfcR => putative fimbrial-like protein YfcR\n",
      "[18:02:10] Modify product: Probable fimbrial chaperone YfcS => putative fimbrial chaperone YfcS\n",
      "[18:02:10] Modify product: Uncharacterized fimbrial-like protein YfcV => putative fimbrial-like protein YfcV\n",
      "[18:02:10] Modify product: UPF0381 protein YiiS => hypothetical protein\n",
      "[18:02:10] Modify product: Probable lipoprotein YiaD => putative lipoprotein YiaD\n",
      "[18:02:10] Modify product: Uncharacterized MFS-type transporter YhjX => putative MFS-type transporter YhjX\n",
      "[18:02:10] Modify product: Probable cyclic di-GMP phosphodiesterase PdeK => putative cyclic di-GMP phosphodiesterase PdeK\n",
      "[18:02:10] Modify product: Probable L-ascorbate-6-phosphate lactonase UlaG => putative L-ascorbate-6-phosphate lactonase UlaG\n",
      "[18:02:10] Modify product: UPF0719 inner membrane protein YjfL => hypothetical protein\n",
      "[18:02:10] Modify product: UPF0716 protein FxsA => hypothetical protein\n",
      "[18:02:10] Modify product: Probable diguanylate cyclase DgcQ => putative diguanylate cyclase DgcQ\n",
      "[18:02:10] Modify product: Uncharacterized inner membrane transporter YedA => putative inner membrane transporter YedA\n",
      "[18:02:10] Modify product: Uncharacterized protein YedJ => putative protein YedJ\n",
      "[18:02:10] Modify product: Cytochrome b561 homolog 1 => Cytochrome b561 \n",
      "[18:02:10] Modify product: Cytochrome b561 homolog 1 => Cytochrome b561 \n",
      "[18:02:10] Modify product: Probable transcriptional regulatory protein YeeN => putative transcriptional regulatory protein YeeN\n",
      "[18:02:10] Modify product: Probable inner membrane protein Smp => putative inner membrane protein Smp\n",
      "[18:02:10] Modify product: Uncharacterized protein YjjI => putative protein YjjI\n",
      "[18:02:10] Modify product: Uncharacterized metal-dependent hydrolase YjjV => putative metal-dependent hydrolase YjjV\n",
      "[18:02:10] Modify product: Uncharacterized protein YbfE => putative protein YbfE\n",
      "[18:02:10] Modify product: Uncharacterized lipoprotein ChiQ => putative lipoprotein ChiQ\n",
      "[18:02:10] Modify product: UPF0250 protein YbeD => hypothetical protein\n",
      "[18:02:10] Modify product: Uncharacterized HTH-type transcriptional regulator YbdO => putative HTH-type transcriptional regulator YbdO\n",
      "[18:02:10] Modify product: Probable zinc-binding alcohol dehydrogenase Rv1895 => putative zinc-binding alcohol dehydrogenase\n",
      "[18:02:10] Modify product: Uncharacterized HTH-type transcriptional regulator YbdO => putative HTH-type transcriptional regulator YbdO\n",
      "[18:02:10] Modify product: Uncharacterized protein YbdM => putative protein YbdM\n",
      "[18:02:10] Modify product: Probable multidrug resistance protein EmrK => putative multidrug resistance protein EmrK\n",
      "[18:02:10] Modify product: Epimerase family protein SA0724 => Epimerase family protein\n",
      "[18:02:10] Modify product: Uncharacterized protein HI_0431 => putative protein\n",
      "[18:02:10] Modify product: Uncharacterized fimbrial-like protein YraH => putative fimbrial-like protein YraH\n",
      "[18:02:10] Modify product: Probable fimbrial chaperone YraI => putative fimbrial chaperone YraI\n",
      "[18:02:10] Modify product: Uncharacterized fimbrial-like protein YraK => putative fimbrial-like protein YraK\n",
      "[18:02:10] Modify product: UPF0213 protein BH0048 => hypothetical protein\n",
      "[18:02:10] Modify product: Uncharacterized protease YhbU => putative protease YhbU\n",
      "[18:02:10] Modify product: Uncharacterized inner membrane transporter YhbE => putative inner membrane transporter YhbE\n",
      "[18:02:10] Modify product: Uncharacterized protein YrbL => putative protein YrbL\n",
      "[18:02:10] Modify product: Uncharacterized tRNA/rRNA methyltransferase YfiF => putative tRNA/rRNA methyltransferase YfiF\n",
      "[18:02:10] Modify product: Uncharacterized protein YphB => putative protein YphB\n",
      "[18:02:10] Modify product: Probable 3-phenylpropionic acid transporter => putative 3-phenylpropionic acid transporter\n",
      "[18:02:10] Modify product: Uncharacterized protein Rv2307c => putative protein\n",
      "[18:02:10] Modify product: UPF0070 protein HI_0370 => hypothetical protein\n",
      "[18:02:10] Modify product: Uncharacterized protein YfgJ => putative protein YfgJ\n",
      "[18:02:10] Modify product: Uncharacterized protein YfgD => putative protein YfgD\n",
      "[18:02:10] Modify product: Probable formate transporter 2 => putative formate transporter 2\n",
      "[18:02:10] Modify product: Uncharacterized protein Rv2575 => putative protein\n",
      "[18:02:10] Modify product: Uncharacterized protein YfeY => putative protein YfeY\n",
      "[18:02:10] Modify product: Probable cyclic di-GMP phosphodiesterase PdeA => putative cyclic di-GMP phosphodiesterase PdeA\n",
      "[18:02:10] Modify product: UPF0761 membrane protein YihY => hypothetical protein\n",
      "[18:02:10] Modify product: Uncharacterized protein YihR => putative protein YihR\n",
      "[18:02:10] Modify product: Uncharacterized protein YshB => putative protein YshB\n",
      "[18:02:10] Modify product: Probable GTP-binding protein EngB => putative GTP-binding protein EngB\n",
      "[18:02:10] Modify product: Probable acyltransferase YihG => putative acyltransferase YihG\n",
      "[18:02:10] Modify product: UPF0401 protein YkfF => hypothetical protein\n",
      "[18:02:10] Modify product: Uncharacterized protein YnbC => putative protein YnbC\n",
      "[18:02:10] Modify product: Uncharacterized protein YnbD => putative protein YnbD\n",
      "[18:02:10] Modify product: Uncharacterized protein Rv3292 => putative protein\n",
      "[18:02:10] Modify product: Probable FMN/FAD exporter YeeO => putative FMN/FAD exporter YeeO\n",
      "[18:02:10] Modify product: Probable FMN/FAD exporter YeeO => putative FMN/FAD exporter YeeO\n",
      "[18:02:10] Modify product: Probable L,D-transpeptidase ErfK/SrfK => putative L,D-transpeptidase ErfK/SrfK\n",
      "[18:02:10] Modify product: Probable protein adenylyltransferase Fic => putative protein adenylyltransferase Fic\n",
      "[18:02:10] Modify product: Uncharacterized membrane protein YhdT => putative membrane protein YhdT\n",
      "[18:02:10] Modify product: Probable glycerol-3-phosphate acyltransferase => putative glycerol-3-phosphate acyltransferase\n",
      "[18:02:10] Modify product: Uncharacterized protein YqjD => putative protein YqjD\n",
      "[18:02:11] Modify product: UPF0225 protein YchJ => hypothetical protein\n",
      "[18:02:11] Modify product: Uncharacterized NTE family protein Rv1063c => putative NTE family protein\n",
      "[18:02:11] Modify product: UPF0056 inner membrane protein MarC => hypothetical protein\n",
      "[18:02:11] Modify product: UPF0263 protein YciU => hypothetical protein\n",
      "[18:02:11] Modify product: Uncharacterized acyl-CoA thioester hydrolase HI_0827 => putative acyl-CoA thioester hydrolase\n",
      "[18:02:11] Modify product: Probable intracellular septation protein A => putative intracellular septation protein A\n",
      "[18:02:11] Modify product: UPF0259 membrane protein YciC => hypothetical protein\n",
      "[18:02:11] Modify product: Uncharacterized protein YciO => putative protein YciO\n",
      "[18:02:11] Modify product: Uncharacterized oxidoreductase YciK => putative oxidoreductase YciK\n",
      "[18:02:11] Modify product: Probable protease SohB => putative protease SohB\n",
      "[18:02:11] Modify product: Uncharacterized FAD-linked oxidoreductase Rv2280 => putative FAD-linked oxidoreductase\n",
      "[18:02:11] Modify product: Probable membrane transporter protein YfcA => putative membrane transporter protein YfcA\n",
      "[18:02:11] Modify product: Uncharacterized protein YbiI => putative protein YbiI\n",
      "[18:02:11] Modify product: UPF0053 inner membrane protein YfjD => hypothetical protein\n",
      "[18:02:11] Modify product: UPF0125 protein RatB => hypothetical protein\n",
      "[18:02:11] Modify product: Probable autotransporter ROD_p1121 => putative autotransporter\n",
      "[18:02:11] Modify product: Probable HTH-type transcriptional regulator YgaV => putative HTH-type transcriptional regulator YgaV\n",
      "[18:02:11] Modify product: Uncharacterized protein YgaM => putative protein YgaM\n",
      "[18:02:11] Modify product: Probable transporter SCO4007 => putative transporter\n",
      "[18:02:11] Modify product: Uncharacterized protein YgcP => putative protein YgcP\n",
      "[18:02:11] Modify product: Uncharacterized protein YqcG => putative protein YqcG\n",
      "[18:02:11] Modify product: Probable glucarate transporter => putative glucarate transporter\n",
      "[18:02:11] Modify product: Uncharacterized protein PA3435 => putative protein\n",
      "[18:02:11] Modify product: Uncharacterized protein YqcC => putative protein YqcC\n",
      "[18:02:11] Modify product: Uncharacterized HTH-type transcriptional regulator YdjF => putative HTH-type transcriptional regulator YdjF\n",
      "[18:02:11] Modify product: UPF0382 inner membrane protein YgdD => hypothetical protein\n",
      "[18:02:11] Modify product: Uncharacterized lipoprotein YgdR => putative lipoprotein YgdR\n",
      "[18:02:11] Modify product: UPF0053 inner membrane protein YgdQ => hypothetical protein\n",
      "[18:02:11] Modify product: Uncharacterized lipoprotein YgdR => putative lipoprotein YgdR\n",
      "[18:02:11] Modify product: Uncharacterized response regulatory protein YgeK => putative response regulatory protein YgeK\n",
      "[18:02:11] Modify product: Probable protein kinase UbiB => putative protein kinase UbiB\n",
      "[18:02:11] Modify product: Probable transport protein YifK => putative transport protein YifK\n",
      "[18:02:11] Modify product: Probable ECA polymerase => putative ECA polymerase\n",
      "[18:02:11] Modify product: UPF0438 protein YifE => hypothetical protein\n",
      "[18:02:11] Modify product: Uncharacterized protein YggR => putative protein YggR\n",
      "[18:02:11] Modify product: UPF0265 protein YeeX => hypothetical protein\n",
      "[18:02:11] Modify product: UPF0394 inner membrane protein YeeE => hypothetical protein\n",
      "[18:02:11] Modify product: Uncharacterized protein YcgM => putative protein YcgM\n",
      "[18:02:11] Modify product: UPF0757 protein YmgG => hypothetical protein\n",
      "[18:02:11] Modify product: Uncharacterized protein YmgD => putative protein YmgD\n",
      "[18:02:11] Modify product: Uncharacterized protein YmgD => putative protein YmgD\n",
      "[18:02:11] Modify product: Probable cyclic di-GMP phosphodiesterase PdeG => putative cyclic di-GMP phosphodiesterase PdeG\n",
      "[18:02:11] Modify product: Uncharacterized protein YmgC => putative protein YmgC\n",
      "[18:02:11] Modify product: Probable two-component-system connector protein AriR => putative two-component-system connector protein AriR\n",
      "[18:02:11] Modify product: Probable two-component-system connector protein YcgZ => putative two-component-system connector protein YcgZ\n",
      "[18:02:11] Modify product: Uncharacterized protein YdfO => putative protein YdfO\n",
      "[18:02:11] Modify product: Probable acrylyl-CoA reductase AcuI => putative acrylyl-CoA reductase AcuI\n",
      "[18:02:11] Modify product: Uncharacterized protein YhcO => putative protein YhcO\n",
      "[18:02:11] Modify product: Uncharacterized protein YhcH => putative protein YhcH\n",
      "[18:02:11] Modify product: Probable iron export permease protein FetB => putative iron export permease protein FetB\n",
      "[18:02:11] Modify product: Probable iron export ATP-binding protein FetA => putative iron export ATP-binding protein FetA\n",
      "[18:02:11] Modify product: Uncharacterized HTH-type transcriptional regulator YbaQ => putative HTH-type transcriptional regulator YbaQ\n",
      "[18:02:11] Modify product: Probable cyclic di-GMP phosphodiesterase PdeB => putative cyclic di-GMP phosphodiesterase PdeB\n",
      "[18:02:11] Modify product: Uncharacterized protein YbaA => putative protein YbaA\n",
      "[18:02:11] Modify product: Uncharacterized lipoprotein YbaY => putative lipoprotein YbaY\n",
      "[18:02:11] Modify product: Probable multidrug resistance ABC transporter ATP-binding/permease protein YheI => putative multidrug resistance ABC transporter ATP-binding/permease protein YheI\n",
      "[18:02:11] Modify product: Uncharacterized protein HI_1008 => putative protein\n",
      "[18:02:11] Modify product: UPF0234 protein YajQ => hypothetical protein\n",
      "[18:02:11] Modify product: Uncharacterized lipoprotein YajI => putative lipoprotein YajI\n",
      "[18:02:11] Modify product: Uncharacterized protein YaiA => putative protein YaiA\n",
      "[18:02:11] Modify product: UPF0178 protein YaiI => hypothetical protein\n",
      "[18:02:11] Modify product: Probable diguanylate cyclase DgcC => putative diguanylate cyclase DgcC\n",
      "[18:02:11] Modify product: Uncharacterized transporter YycB => putative transporter YycB\n",
      "[18:02:11] Modify product: Uncharacterized HTH-type transcriptional regulator YahB => putative HTH-type transcriptional regulator YahB\n",
      "[18:02:11] Modify product: Pentapeptide repeat protein VPA0095 => Pentapeptide repeat protein\n",
      "[18:02:11] Modify product: UPF0758 protein CT0611 => hypothetical protein\n",
      "[18:02:11] Modify product: UPF0126 inner membrane protein YicG => hypothetical protein\n",
      "[18:02:11] Modify product: Uncharacterized inner membrane transporter YicL => putative inner membrane transporter YicL\n",
      "[18:02:11] Modify product: UPF0701 protein YicC => hypothetical protein\n",
      "[18:02:11] Modify product: UPF0758 protein SP_1088 => hypothetical protein\n",
      "[18:02:11] Modify product: Uncharacterized protein YibN => putative protein YibN\n",
      "[18:02:11] Modify product: Uncharacterized protein YibL => putative protein YibL\n",
      "[18:02:11] Modify product: Uncharacterized protein YcbX => putative protein YcbX\n",
      "[18:02:11] Modify product: UPF0401 protein YkfF => hypothetical protein\n",
      "[18:02:11] Modify product: Uncharacterized lipoprotein GfcB => putative lipoprotein GfcB\n",
      "[18:02:11] Modify product: Probable Ni/Fe-hydrogenase 1 B-type cytochrome subunit => putative Ni/Fe-hydrogenase 1 B-type cytochrome subunit\n",
      "[18:02:11] Modify product: Uncharacterized protein YccU => putative protein YccU\n",
      "[18:02:11] Modify product: UPF0319 protein HI_1681 => hypothetical protein\n",
      "[18:02:11] Modify product: Uncharacterized protein YcbX => putative protein YcbX\n",
      "[18:02:11] Modify product: Uncharacterized fimbrial chaperone YcbF => putative fimbrial chaperone YcbF\n",
      "[18:02:11] Modify product: Uncharacterized fimbrial-like protein YcbV => putative fimbrial-like protein YcbV\n",
      "[18:02:11] Modify product: Uncharacterized fimbrial-like protein YcbU => putative fimbrial-like protein YcbU\n",
      "[18:02:11] Modify product: Uncharacterized fimbrial-like protein ElfG => putative fimbrial-like protein ElfG\n",
      "[18:02:11] Modify product: Probable outer membrane usher protein ElfC => putative outer membrane usher protein ElfC\n",
      "[18:02:11] Modify product: Probable fimbrial chaperone protein ElfD => putative fimbrial chaperone protein ElfD\n",
      "[18:02:11] Modify product: Probable L,D-transpeptidase YcbB => putative L,D-transpeptidase YcbB\n",
      "[18:02:11] Modify product: UPF0434 protein YcaR => hypothetical protein\n",
      "[18:02:11] Modify product: Probable formate transporter 1 => putative formate transporter 1\n",
      "[18:02:11] Modify product: Uncharacterized MFS-type transporter YcaD => putative MFS-type transporter YcaD\n",
      "[18:02:11] Modify product: Probable hydrolase YcaC => putative hydrolase YcaC\n",
      "[18:02:11] Modify product: UPF0145 protein YbjQ => hypothetical protein\n",
      "[18:02:11] Modify product: Uncharacterized lipoprotein YbjP => putative lipoprotein YbjP\n",
      "[18:02:11] Modify product: Uncharacterized protein YbjN => putative protein YbjN\n",
      "[18:02:11] Modify product: Probable cyclic di-GMP phosphodiesterase PdeI => putative cyclic di-GMP phosphodiesterase PdeI\n",
      "[18:02:11] Modify product: Probable cyclic di-GMP phosphodiesterase PdeI => putative cyclic di-GMP phosphodiesterase PdeI\n",
      "[18:02:11] Modify product: Uncharacterized protein YbiU => putative protein YbiU\n",
      "[18:02:11] Modify product: Uncharacterized ABC transporter ATP-binding protein YbiT => putative ABC transporter ATP-binding protein YbiT\n",
      "[18:02:11] Modify product: Probable L,D-transpeptidase YbiS => putative L,D-transpeptidase YbiS\n",
      "[18:02:11] Modify product: Uncharacterized protein McbA => putative protein McbA\n",
      "[18:02:11] Modify product: PKHD-type hydroxylase Sbal_3634 => PKHD-type hydroxylase\n",
      "[18:02:11] Modify product: Uncharacterized protein YbiI => putative protein YbiI\n",
      "[18:02:11] Modify product: Uncharacterized protein YbiB => putative protein YbiB\n",
      "[18:02:11] Modify product: UPF0194 membrane protein YbhG => hypothetical protein\n",
      "[18:02:11] Modify product: Probable multidrug ABC transporter ATP-binding protein YbhF => putative multidrug ABC transporter ATP-binding protein YbhF\n",
      "[18:02:11] Modify product: Probable multidrug ABC transporter permease YbhS => putative multidrug ABC transporter permease YbhS\n",
      "[18:02:11] Modify product: Probable multidrug ABC transporter permease YbhR => putative multidrug ABC transporter permease YbhR\n",
      "[18:02:11] Modify product: UPF0098 protein YbhB => hypothetical protein\n",
      "[18:02:11] Modify product: Uncharacterized fimbrial-like protein YbgD => putative fimbrial-like protein YbgD\n",
      "[18:02:11] Modify product: GTP cyclohydrolase 1 type 2 homolog => GTP cyclohydrolase 1 type 2 \n",
      "[18:02:11] Modify product: Uncharacterized HTH-type transcriptional regulator Rv1395 => putative HTH-type transcriptional regulator\n",
      "[18:02:12] Modify product: Probable bifunctional chitinase/lysozyme => putative bifunctional chitinase/lysozyme\n",
      "[18:02:12] Modify product: Uncharacterized protein YaeQ => putative protein YaeQ\n",
      "[18:02:12] Modify product: UPF0253 protein YaeP => hypothetical protein\n",
      "[18:02:12] Modify product: UPF0126 inner membrane protein YadS => hypothetical protein\n",
      "[18:02:12] Modify product: Uncharacterized fimbrial-like protein YadN => putative fimbrial-like protein YadN\n",
      "[18:02:12] Modify product: Probable fimbrial chaperone YadV => putative fimbrial chaperone YadV\n",
      "[18:02:12] Modify product: Uncharacterized fimbrial-like protein YadM => putative fimbrial-like protein YadM\n",
      "[18:02:12] Modify product: Uncharacterized fimbrial-like protein YadL => putative fimbrial-like protein YadL\n",
      "[18:02:12] Modify product: Uncharacterized fimbrial-like protein YadK => putative fimbrial-like protein YadK\n",
      "[18:02:12] Modify product: Uncharacterized fimbrial-like protein YadC => putative fimbrial-like protein YadC\n",
      "[18:02:12] Modify product: Uncharacterized ABC transporter ATP-binding protein YadG => putative ABC transporter ATP-binding protein YadG\n",
      "[18:02:12] Modify product: UPF0231 protein YacL => hypothetical protein\n",
      "[18:02:12] Modify product: Probable peptidoglycan glycosyltransferase FtsW => putative peptidoglycan glycosyltransferase FtsW\n",
      "[18:02:12] Modify product: Probable cadaverine/lysine antiporter => putative cadaverine/lysine antiporter\n",
      "[18:02:12] Modify product: Uncharacterized protein YjdJ => putative protein YjdJ\n",
      "[18:02:12] Modify product: Probable transcriptional regulator PhnF => putative transcriptional regulator PhnF\n",
      "[18:02:12] Modify product: Uncharacterized ferredoxin-like protein YdhX => putative ferredoxin-like protein YdhX\n",
      "[18:02:12] Modify product: Probable cyclic di-GMP phosphodiesterase PdeC => putative cyclic di-GMP phosphodiesterase PdeC\n",
      "[18:02:12] Modify product: Uncharacterized protein YjbR => putative protein YjbR\n",
      "[18:02:12] Modify product: UPF0337 protein YjbJ => hypothetical protein\n",
      "[18:02:12] Modify product: Uncharacterized lipoprotein GfcB => putative lipoprotein GfcB\n",
      "[18:02:12] Modify product: Uncharacterized FAD-linked oxidoreductase Rv2280 => putative FAD-linked oxidoreductase\n",
      "[18:02:12] Modify product: Uncharacterized protein YtfJ => putative protein YtfJ\n",
      "[18:02:12] Modify product: Probable Ni/Fe-hydrogenase 2 b-type cytochrome subunit => putative Ni/Fe-hydrogenase 2 b-type cytochrome subunit\n",
      "[18:02:12] Modify product: Uncharacterized oxidoreductase YghA => putative oxidoreductase YghA\n",
      "[18:02:12] Modify product: Probable deoxycholate-binding periplasmic protein YgiS => putative deoxycholate-binding periplasmic protein YgiS\n",
      "[18:02:12] Modify product: Probable quinol monooxygenase YgiN => putative quinol monooxygenase YgiN\n",
      "[18:02:12] Modify product: Uncharacterized fimbrial-like protein YbgD => putative fimbrial-like protein YbgD\n",
      "[18:02:12] Cleaned 386 /product names\n",
      "[18:02:12] Deleting unwanted file: output/sample.sprot.tmp.21213.faa\n",
      "[18:02:12] Deleting unwanted file: output/sample.sprot.tmp.21213.blast\n",
      "[18:02:12] There are still 1701 unannotated CDS left (started with 5427)\n",
      "[18:02:12] Will use hmmer3 to search against /home/quang/miniconda3/envs/amrtraining/db/hmm/HAMAP.hmm with 4 CPUs\n",
      "[18:02:12] Running: cat output\\/sample\\.HAMAP\\.hmm\\.tmp\\.21213\\.faa | parallel --gnu --plain -j 4 --block 51737 --recstart '>' --pipe hmmscan --noali --notextw --acc -E 1e-09 --cpu 1 /home/quang/miniconda3/envs/amrtraining/db/hmm/HAMAP.hmm /dev/stdin > output\\/sample\\.HAMAP\\.hmm\\.tmp\\.21213\\.hmmer3 2> /dev/null\n",
      "[18:02:43] Modify product: Uncharacterized Nudix hydrolase NudL => putative Nudix hydrolase NudL\n",
      "[18:02:43] Cleaned 1 /product names\n",
      "[18:02:43] Deleting unwanted file: output/sample.HAMAP.hmm.tmp.21213.faa\n",
      "[18:02:43] Deleting unwanted file: output/sample.HAMAP.hmm.tmp.21213.hmmer3\n",
      "[18:02:43] Labelling remaining 1594 proteins as 'hypothetical protein'\n",
      "[18:02:43] Possible /pseudo 'IS3 family transposase ISAba14' at Contig_16_843.989 position 535\n",
      "[18:02:43] Possible /pseudo 'putative protein YjgL' at Contig_23_89.5642 position 49309\n",
      "[18:02:43] Possible /pseudo 'Hydroxypyruvate isomerase' at Contig_24_82.4185 position 48264\n",
      "[18:02:43] Possible /pseudo 'putative HTH-type transcriptional regulator RhmR' at Contig_37_79.7725 position 56877\n",
      "[18:02:43] Possible /pseudo 'Cellulose biosynthesis protein BcsQ' at Contig_49_89.7964 position 24863\n",
      "[18:02:43] Possible /pseudo 'Transcriptional regulatory protein RcsA' at Contig_53_75.6927 position 472\n",
      "[18:02:43] Possible /pseudo 'Cytochrome b561 ' at Contig_53_75.6927 position 18621\n",
      "[18:02:43] Possible /pseudo 'Inducible ornithine decarboxylase' at Contig_79_83.8174 position 9807\n",
      "[18:02:43] Possible /pseudo 'Inducible ornithine decarboxylase' at Contig_79_83.8174 position 10599\n",
      "[18:02:43] Possible /pseudo 'Deaminated glutathione amidase' at Contig_79_83.8174 position 70002\n",
      "[18:02:43] Possible /pseudo 'Cryptic outer membrane porin BglH' at Contig_80_84.3214 position 3028\n",
      "[18:02:43] Possible /pseudo 'Formate dehydrogenase, nitrate-inducible, major subunit' at Contig_99_77.5417 position 22520\n",
      "[18:02:43] Possible /pseudo 'Respiratory nitrate reductase 2 alpha chain' at Contig_99_77.5417 position 31067\n",
      "[18:02:43] Possible /pseudo 'Regulatory protein rop' at Contig_105_2074.04 position 3229\n",
      "[18:02:43] Possible /pseudo '6-phospho-alpha-glucosidase' at Contig_108_88.3561 position 36931\n",
      "[18:02:43] Possible /pseudo 'Propionyl-CoA:succinate CoA transferase' at Contig_111_84.3536 position 66566\n",
      "[18:02:43] Possible /pseudo 'putative diguanylate cyclase DgcE' at Contig_134_77.6005 position 11478\n",
      "[18:02:43] Possible /pseudo 'putative protein YhhA' at Contig_137_86.0435 position 115097\n",
      "[18:02:43] Possible /pseudo '(6-4) photolyase' at Contig_180_17.8124 position 59007\n",
      "[18:02:43] Possible /pseudo 'D-inositol-3-phosphate glycosyltransferase' at Contig_181_74.7781 position 12758\n",
      "[18:02:43] Possible /pseudo 'D-inositol-3-phosphate glycosyltransferase' at Contig_181_74.7781 position 13950\n",
      "[18:02:43] Possible /pseudo '3-methylmercaptopropionyl-CoA dehydrogenase' at Contig_199_18.4146 position 4411\n",
      "[18:02:43] Possible /pseudo 'Formate dehydrogenase-O major subunit' at Contig_202_88.9789 position 42781\n",
      "[18:02:43] Possible /pseudo 'Putative 2,3-dihydroxypropane-1-sulfonate exporter' at Contig_202_88.9789 position 62201\n",
      "[18:02:43] Possible /pseudo 'putative FMN/FAD exporter YeeO' at Contig_230_80.4106 position 912\n",
      "[18:02:43] Possible /pseudo 'D-alanyl-D-alanine carboxypeptidase DacC' at Contig_270_79.6856 position 168984\n",
      "[18:02:43] Possible /pseudo 'putative cyclic di-GMP phosphodiesterase PdeI' at Contig_270_79.6856 position 176727\n",
      "[18:02:43] Possible /pseudo 'Clamp-binding protein CrfC' at Contig_282_93.0876 position 31707\n",
      "[18:02:43] Possible /pseudo 'Formate dehydrogenase H' at Contig_282_93.0876 position 61415\n",
      "[18:02:43] Possible /pseudo 'Type II secretion system protein E' at Contig_288_86.1701 position 9467\n",
      "[18:02:43] Possible /pseudo 'Threonine/homoserine exporter RhtA' at Contig_304_18.4016 position 5109\n",
      "[18:02:43] Possible /pseudo 'Surface presentation of antigens protein SpaS' at Contig_305_84.0088 position 275370\n",
      "[18:02:43] Possible /pseudo 'putative protein YmgD' at Contig_322_76.7805 position 46626\n",
      "[18:02:43] Found 3219 unique /gene codes.\n",
      "[18:02:43] Fixed 2 duplicate /gene - anmK_1 anmK_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - crfC_1 crfC_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - fdnH_1 fdnH_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - fixA_1 fixA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - flhA_1 flhA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - pgrR_1 pgrR_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - baeR_1 baeR_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - dnaG_1 dnaG_2\n",
      "[18:02:43] Fixed 3 duplicate /gene - epsE_1 epsE_2 epsE_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - mltC_1 mltC_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - topA_1 topA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ybgD_1 ybgD_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - arcC2_1 arcC2_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - fecE_1 fecE_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - lptG_1 lptG_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - yibA_1 yibA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ssuA_1 ssuA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - opgE_1 opgE_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - kdsB_1 kdsB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - virB4_1 virB4_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - smc_1 smc_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ymgD_1 ymgD_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - slp_1 slp_2\n",
      "[18:02:43] Fixed 6 duplicate /gene - intA_1 intA_2 intA_3 intA_4 intA_5 intA_6\n",
      "[18:02:43] Fixed 2 duplicate /gene - cra_1 cra_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - icsA_1 icsA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - tauA_1 tauA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - radA_1 radA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - nuoL_1 nuoL_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - aaeB_1 aaeB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - srmB_1 srmB_2\n",
      "[18:02:43] Fixed 3 duplicate /gene - srlR_1 srlR_2 srlR_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - ccmH_1 ccmH_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - astE_1 astE_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - spoT_1 spoT_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - lacY_1 lacY_2\n",
      "[18:02:43] Fixed 7 duplicate /gene - umuC_1 umuC_2 umuC_3 umuC_4 umuC_5 umuC_6 umuC_7\n",
      "[18:02:43] Fixed 2 duplicate /gene - dinG_1 dinG_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ydjE_1 ydjE_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - rbsC_1 rbsC_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - psiE_1 psiE_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - uxaA_1 uxaA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - chuR_1 chuR_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - soj_1 soj_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - malP_1 malP_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - gyrB_1 gyrB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - hcpA_1 hcpA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - yfbR_1 yfbR_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - intS_1 intS_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - nagB_1 nagB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - rhtB_1 rhtB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - narZ_1 narZ_2\n",
      "[18:02:43] Fixed 3 duplicate /gene - ompX_1 ompX_2 ompX_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - iadA_1 iadA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - yggS_1 yggS_2\n",
      "[18:02:43] Fixed 3 duplicate /gene - arnC_1 arnC_2 arnC_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - psuK_1 psuK_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - mipA_1 mipA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - yadV_1 yadV_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - purE_1 purE_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - groS_1 groS_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - acs_1 acs_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - yedK_1 yedK_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - spaS_1 spaS_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - yabJ_1 yabJ_2\n",
      "[18:02:43] Fixed 4 duplicate /gene - rhaR_1 rhaR_2 rhaR_3 rhaR_4\n",
      "[18:02:43] Fixed 3 duplicate /gene - rhtA_1 rhtA_2 rhtA_3\n",
      "[18:02:43] Fixed 3 duplicate /gene - yhdJ_1 yhdJ_2 yhdJ_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - clsA_1 clsA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - hyi_1 hyi_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ftnA_1 ftnA_2\n",
      "[18:02:43] Fixed 3 duplicate /gene - fixX_1 fixX_2 fixX_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - ftsA_1 ftsA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - agaC_1 agaC_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - pdhR_1 pdhR_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - scrK_1 scrK_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - purR_1 purR_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - yodB_1 yodB_2\n",
      "[18:02:43] Fixed 6 duplicate /gene - dmlR_1 dmlR_2 dmlR_3 dmlR_4 dmlR_5 dmlR_6\n",
      "[18:02:43] Fixed 2 duplicate /gene - lutA_1 lutA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - greA_1 greA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - truB_1 truB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - yhhT_1 yhhT_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - pspB_1 pspB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - bglF_1 bglF_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - dmsB_1 dmsB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ndhB_1 ndhB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - hha_1 hha_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - actP_1 actP_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - metH_1 metH_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - yggR_1 yggR_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - glnE_1 glnE_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - gltB_1 gltB_2\n",
      "[18:02:43] Fixed 4 duplicate /gene - btuD_1 btuD_2 btuD_3 btuD_4\n",
      "[18:02:43] Fixed 2 duplicate /gene - sgrR_1 sgrR_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - recD2_1 recD2_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - kefC_1 kefC_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - mlc_1 mlc_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - hxpB_1 hxpB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - rep_1 rep_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - hpt_1 hpt_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ispH_1 ispH_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ynaI_1 ynaI_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - rnpA_1 rnpA_2\n",
      "[18:02:43] Fixed 3 duplicate /gene - phrB_1 phrB_2 phrB_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - astA_1 astA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - dacC_1 dacC_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - azoR_1 azoR_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - cutA_1 cutA_2\n",
      "[18:02:43] Fixed 3 duplicate /gene - rop_1 rop_2 rop_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - narL_1 narL_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - emrK_1 emrK_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - yihP_1 yihP_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - sbmC_1 sbmC_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - mtlA_1 mtlA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - yrdA_1 yrdA_2\n",
      "[18:02:43] Fixed 3 duplicate /gene - groL_1 groL_2 groL_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - ccmF_1 ccmF_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ydgA_1 ydgA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ygcS_1 ygcS_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - rffG_1 rffG_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - fruA_1 fruA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - bcsQ_1 bcsQ_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - rhmT_1 rhmT_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - caiA_1 caiA_2\n",
      "[18:02:43] Fixed 4 duplicate /gene - klcA_1 klcA_2 klcA_3 klcA_4\n",
      "[18:02:43] Fixed 2 duplicate /gene - ybhR_1 ybhR_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - rpsP_1 rpsP_2\n",
      "[18:02:43] Fixed 3 duplicate /gene - lacI_1 lacI_2 lacI_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - scpC_1 scpC_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - traI_1 traI_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - fosA_1 fosA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - manX_1 manX_2\n",
      "[18:02:43] Fixed 3 duplicate /gene - clpP_1 clpP_2 clpP_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - yjiG_1 yjiG_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - lptB_1 lptB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ydjZ_1 ydjZ_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - gstB_1 gstB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - dsbD_1 dsbD_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - hupB_1 hupB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - fecD_1 fecD_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - gspK_1 gspK_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - emrB_1 emrB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - bfr_1 bfr_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - rfaH_1 rfaH_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - aaeA_1 aaeA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - astB_1 astB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - dcuD_1 dcuD_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - aroB_1 aroB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ubiJ_1 ubiJ_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - malG_1 malG_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - guaD_1 guaD_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - mbeC_1 mbeC_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - bioD1_1 bioD1_2\n",
      "[18:02:43] Fixed 3 duplicate /gene - rusA_1 rusA_2 rusA_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - ppiA_1 ppiA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - gmk_1 gmk_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - modA_1 modA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ybdO_1 ybdO_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - prfB_1 prfB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ygiC_1 ygiC_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - osmY_1 osmY_2\n",
      "[18:02:43] Fixed 5 duplicate /gene - umuD_1 umuD_2 umuD_3 umuD_4 umuD_5\n",
      "[18:02:43] Fixed 2 duplicate /gene - ydjF_1 ydjF_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - rlhA_1 rlhA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - cbeA_1 cbeA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - xylB_1 xylB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - acrB_1 acrB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ulaE_1 ulaE_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - iclR_1 iclR_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ais_1 ais_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - yheS_1 yheS_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - cmoA_1 cmoA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - por_1 por_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - aspT_1 aspT_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - gyrA_1 gyrA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - rhsD_1 rhsD_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - prs_1 prs_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - lpxL_1 lpxL_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - lysA_1 lysA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ssuB_1 ssuB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - rplS_1 rplS_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - tyrS_1 tyrS_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - yhhA_1 yhhA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - astC_1 astC_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - hdfR_1 hdfR_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - leuC_1 leuC_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - cysH_1 cysH_2\n",
      "[18:02:43] Fixed 3 duplicate /gene - pitA_1 pitA_2 pitA_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - dnaB_1 dnaB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - rpmH_1 rpmH_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - mdtL_1 mdtL_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - pdeI_1 pdeI_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - cusC_1 cusC_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - tauB_1 tauB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - astD_1 astD_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - rclA_1 rclA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - acpP_1 acpP_2\n",
      "[18:02:43] Fixed 3 duplicate /gene - glnQ_1 glnQ_2 glnQ_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - galE_1 galE_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - deoB_1 deoB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - yhcH_1 yhcH_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - mtlK_1 mtlK_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - dnaA_1 dnaA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - paaF_1 paaF_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - murR_1 murR_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - gltC_1 gltC_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ybdM_1 ybdM_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - parM_1 parM_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - yhaJ_1 yhaJ_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - xdhA_1 xdhA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ttdT_1 ttdT_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - rbsA_1 rbsA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - mnmE_1 mnmE_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - epsF_1 epsF_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - lldR_1 lldR_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - yeeO_1 yeeO_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - manP_1 manP_2\n",
      "[18:02:43] Fixed 5 duplicate /gene - hyfA_1 hyfA_2 hyfA_3 hyfA_4 hyfA_5\n",
      "[18:02:43] Fixed 3 duplicate /gene - ydcV_1 ydcV_2 ydcV_3\n",
      "[18:02:43] Fixed 3 duplicate /gene - hin_1 hin_2 hin_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - dmsC_1 dmsC_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - trmD_1 trmD_2\n",
      "[18:02:43] Fixed 3 duplicate /gene - speF_1 speF_2 speF_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - asnC_1 asnC_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - mdh_1 mdh_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - cycA_1 cycA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - melR_1 melR_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - aglB_1 aglB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ndhI_1 ndhI_2\n",
      "[18:02:43] Fixed 3 duplicate /gene - papC_1 papC_2 papC_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - araB_1 araB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - sacA_1 sacA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ftsH_1 ftsH_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - serA_1 serA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - dcm_1 dcm_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - aphA_1 aphA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - rcsA_1 rcsA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ybeM_1 ybeM_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - rpoZ_1 rpoZ_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - rimM_1 rimM_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - betI_1 betI_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - nlpD_1 nlpD_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - tap_1 tap_2\n",
      "[18:02:43] Fixed 3 duplicate /gene - potA_1 potA_2 potA_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - rhmR_1 rhmR_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - papD_1 papD_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - yfcA_1 yfcA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - yahB_1 yahB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - dadA_1 dadA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - htrE_1 htrE_2\n",
      "[18:02:43] Fixed 3 duplicate /gene - traD_1 traD_2 traD_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - erpA_1 erpA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ydhC_1 ydhC_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - tdh_1 tdh_2\n",
      "[18:02:43] Fixed 3 duplicate /gene - dnaK_1 dnaK_2 dnaK_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - pndA_1 pndA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - mpl_1 mpl_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ssuC_1 ssuC_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - clpV1_1 clpV1_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - tmk_1 tmk_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - coaA_1 coaA_2\n",
      "[18:02:43] Fixed 4 duplicate /gene - rcsC_1 rcsC_2 rcsC_3 rcsC_4\n",
      "[18:02:43] Fixed 2 duplicate /gene - kdgK_1 kdgK_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - gfcB_1 gfcB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - dsbC_1 dsbC_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - fic_1 fic_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - aspS_1 aspS_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - blc_1 blc_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - cbtA_1 cbtA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - arfA_1 arfA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - csbX_1 csbX_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - tetR_1 tetR_2\n",
      "[18:02:43] Fixed 3 duplicate /gene - manZ_1 manZ_2 manZ_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - mglA_1 mglA_2\n",
      "[18:02:43] Fixed 3 duplicate /gene - pncC_1 pncC_2 pncC_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - gltD_1 gltD_2\n",
      "[18:02:43] Fixed 3 duplicate /gene - fixB_1 fixB_2 fixB_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - dnaN_1 dnaN_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - proA_1 proA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ybiI_1 ybiI_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - gdhA_1 gdhA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - yjgL_1 yjgL_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - lolD_1 lolD_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - grpE_1 grpE_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - def_1 def_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - fdhF_1 fdhF_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - fdoG_1 fdoG_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - dgcE_1 dgcE_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - fdnG_1 fdnG_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - aroK_1 aroK_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - mltF_1 mltF_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - dedA_1 dedA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - modB_1 modB_2\n",
      "[18:02:43] Fixed 3 duplicate /gene - norR_1 norR_2 norR_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - argE_1 argE_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - gpFI_1 gpFI_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ycbX_1 ycbX_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - cutC_1 cutC_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - fryA_1 fryA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - alx_1 alx_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - cpdA_1 cpdA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - yraJ_1 yraJ_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - nanR_1 nanR_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - folP_1 folP_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - marA_1 marA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - rrrD_1 rrrD_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - mepS_1 mepS_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - galF_1 galF_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ygiW_1 ygiW_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - lapB_1 lapB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - yehB_1 yehB_2\n",
      "[18:02:43] Fixed 3 duplicate /gene - bhsA_1 bhsA_2 bhsA_3\n",
      "[18:02:43] Fixed 3 duplicate /gene - ssb_1 ssb_2 ssb_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - purK_1 purK_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - topB_1 topB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ycfH_1 ycfH_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ydhX_1 ydhX_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - tsaR_1 tsaR_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - rffH_1 rffH_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - flu_1 flu_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - fryB_1 fryB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - hemB_1 hemB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - yidC_1 yidC_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ilvE_1 ilvE_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - oxyR_1 oxyR_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - lon_1 lon_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - oleD_1 oleD_2\n",
      "[18:02:43] Fixed 4 duplicate /gene - rhaS_1 rhaS_2 rhaS_3 rhaS_4\n",
      "[18:02:43] Fixed 2 duplicate /gene - recF_1 recF_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - tauD_1 tauD_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - malK_1 malK_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - fdrA_1 fdrA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - gltF_1 gltF_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - bioF_1 bioF_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ygdR_1 ygdR_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - dmdC_1 dmdC_2\n",
      "[18:02:43] Fixed 13 duplicate /gene - xerC_1 xerC_2 xerC_3 xerC_4 xerC_5 xerC_6 xerC_7 xerC_8 xerC_9 xerC_10 xerC_11 xerC_12 xerC_13\n",
      "[18:02:43] Fixed 3 duplicate /gene - cynR_1 cynR_2 cynR_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - artM_1 artM_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - ugd_1 ugd_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - pal_1 pal_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - lamB_1 lamB_2\n",
      "[18:02:43] Fixed 4 duplicate /gene - mshA_1 mshA_2 mshA_3 mshA_4\n",
      "[18:02:43] Fixed 3 duplicate /gene - lexA_1 lexA_2 lexA_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - hycG_1 hycG_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - bfd_1 bfd_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - malI_1 malI_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - dapF_1 dapF_2\n",
      "[18:02:43] Fixed 3 duplicate /gene - bglH_1 bglH_2 bglH_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - msbA_1 msbA_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - mngR_1 mngR_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - tus_1 tus_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - rhtC_1 rhtC_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - lsrK_1 lsrK_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - gadX_1 gadX_2\n",
      "[18:02:43] Fixed 4 duplicate /gene - noc_1 noc_2 noc_3 noc_4\n",
      "[18:02:43] Fixed 2 duplicate /gene - dinI_1 dinI_2\n",
      "[18:02:43] Fixed 4 duplicate /gene - hcaR_1 hcaR_2 hcaR_3 hcaR_4\n",
      "[18:02:43] Fixed 2 duplicate /gene - trmH_1 trmH_2\n",
      "[18:02:43] Fixed 3 duplicate /gene - pld_1 pld_2 pld_3\n",
      "[18:02:43] Fixed 2 duplicate /gene - ytfJ_1 ytfJ_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - yiaW_1 yiaW_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - psiB_1 psiB_2\n",
      "[18:02:43] Fixed 2 duplicate /gene - kduD_1 kduD_2\n",
      "[18:02:43] Fixed 365 colliding /gene names.\n",
      "[18:02:43] Adding /locus_tag identifiers\n",
      "[18:02:44] Assigned 5516 locus_tags to CDS and RNA features.\n",
      "[18:02:44] Writing outputs to output/\n",
      "[18:02:47] Generating annotation statistics file\n",
      "[18:02:47] Generating Genbank and Sequin files\n",
      "[18:02:47] Running: tbl2asn -V b -a r10k -l paired-ends -M n -N 1 -y 'Annotated using prokka 1.14.6 from https://github.com/tseemann/prokka' -Z output\\/sample\\.err -i output\\/sample\\.fsa 2> /dev/null\n",
      "[18:02:59] Deleting unwanted file: output/errorsummary.val\n",
      "[18:02:59] Deleting unwanted file: output/sample.dr\n",
      "[18:02:59] Deleting unwanted file: output/sample.fixedproducts\n",
      "[18:02:59] Deleting unwanted file: output/sample.ecn\n",
      "[18:02:59] Deleting unwanted file: output/sample.val\n",
      "[18:02:59] Repairing broken .GBK output that tbl2asn produces...\n",
      "[18:02:59] Running: sed 's/COORDINATES: profile/COORDINATES:profile/' < output\\/sample\\.gbf > output\\/sample\\.gbk\n",
      "[18:02:59] Deleting unwanted file: output/sample.gbf\n",
      "[18:02:59] Output files:\n",
      "[18:02:59] output/sample.fsa\n",
      "[18:02:59] output/sample.ffn\n",
      "[18:02:59] output/sample.gbk\n",
      "[18:02:59] output/sample.sqn\n",
      "[18:02:59] output/sample.tbl\n",
      "[18:02:59] output/sample.log\n",
      "[18:02:59] output/sample.faa\n",
      "[18:02:59] output/sample.fna\n",
      "[18:02:59] output/sample.tsv\n",
      "[18:02:59] output/sample.txt\n",
      "[18:02:59] output/sample.gff\n",
      "[18:02:59] output/sample.err\n",
      "[18:02:59] Annotation finished successfully.\n",
      "[18:02:59] Walltime used: 5.23 minutes\n",
      "[18:02:59] If you use this result please cite the Prokka paper:\n",
      "[18:02:59] Seemann T (2014) Prokka: rapid prokaryotic genome annotation. Bioinformatics. 30(14):2068-9.\n",
      "[18:02:59] Type 'prokka --citation' for more details.\n",
      "[18:02:59] Thank you, come again.\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "prokka --force --addgenes --prefix sample --locus sample --outdir output output/contigs.fasta"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Detect AMR genes\n",
    "Using abricate with ncbi database to find AMR genes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "abricate --quiet  --nopath --db ncbi output/contigs.fasta > output/sample_resistome.tsv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "head output/sample_resistome.tsv"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Variant calling\n",
    "Alignment with reference genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[bwa_index] Pack FASTA... 0.10 sec\n",
      "[bwa_index] Construct BWT for the packed sequence...\n",
      "[bwa_index] 1.50 seconds elapse.\n",
      "[bwa_index] Update BWT... 0.05 sec\n",
      "[bwa_index] Pack forward-only FASTA... 0.04 sec\n",
      "[bwa_index] Construct SA from BWT and Occ... 0.54 sec\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa index samples/GCF_000005845.2_ASM584v2_genomic.fna.gz\n",
      "[main] Real time: 3.218 sec; CPU: 2.223 sec\n",
      "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19012, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (390, 453, 526)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (118, 798)\n",
      "[M::mem_pestat] mean and std.dev: (460.45, 104.01)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 934)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.212 CPU sec, 4.059 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19250, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 458, 533)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (119, 809)\n",
      "[M::mem_pestat] mean and std.dev: (465.83, 104.25)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 947)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.260 CPU sec, 4.103 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19093, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (396, 460, 535)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (118, 813)\n",
      "[M::mem_pestat] mean and std.dev: (467.91, 105.32)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 952)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.941 CPU sec, 3.786 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19225, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 457, 531)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (123, 803)\n",
      "[M::mem_pestat] mean and std.dev: (465.41, 103.92)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 939)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.880 CPU sec, 3.728 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19325, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 458, 533)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (119, 809)\n",
      "[M::mem_pestat] mean and std.dev: (465.72, 104.27)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 947)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.854 CPU sec, 3.701 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19331, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (396, 458, 534)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (120, 810)\n",
      "[M::mem_pestat] mean and std.dev: (467.58, 105.01)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 948)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.915 CPU sec, 3.762 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19412, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (397, 459, 534)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (123, 808)\n",
      "[M::mem_pestat] mean and std.dev: (467.74, 104.35)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 945)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.829 CPU sec, 3.677 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19332, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 457, 534)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (114, 814)\n",
      "[M::mem_pestat] mean and std.dev: (466.43, 106.60)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 954)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.827 CPU sec, 3.678 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19313, 0, 1)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 460, 536)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (113, 818)\n",
      "[M::mem_pestat] mean and std.dev: (468.14, 106.48)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 959)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.803 CPU sec, 3.653 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19328, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 457, 536)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (113, 818)\n",
      "[M::mem_pestat] mean and std.dev: (467.09, 106.57)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 959)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.798 CPU sec, 3.647 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19258, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (396, 460, 536)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (116, 816)\n",
      "[M::mem_pestat] mean and std.dev: (468.45, 105.76)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 956)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.829 CPU sec, 3.675 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19462, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 458, 533)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (116, 811)\n",
      "[M::mem_pestat] mean and std.dev: (465.89, 105.65)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 950)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.960 CPU sec, 3.809 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19392, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 459, 534)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (117, 812)\n",
      "[M::mem_pestat] mean and std.dev: (467.71, 105.72)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 951)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.848 CPU sec, 3.675 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19291, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (396, 459, 536)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (116, 816)\n",
      "[M::mem_pestat] mean and std.dev: (467.38, 106.08)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 956)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.921 CPU sec, 3.751 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 18253, 0, 1)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (390, 453, 527)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (116, 801)\n",
      "[M::mem_pestat] mean and std.dev: (460.50, 102.47)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 938)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.460 CPU sec, 4.308 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19162, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 460, 535)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (115, 815)\n",
      "[M::mem_pestat] mean and std.dev: (467.93, 106.67)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 955)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.538 CPU sec, 4.375 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19200, 1, 1)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (393, 456, 534)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (111, 816)\n",
      "[M::mem_pestat] mean and std.dev: (466.18, 106.26)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 957)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.136 CPU sec, 3.898 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19316, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 459, 536)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (113, 818)\n",
      "[M::mem_pestat] mean and std.dev: (467.37, 106.35)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 959)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.834 CPU sec, 3.684 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19235, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 459, 536)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (110, 820)\n",
      "[M::mem_pestat] mean and std.dev: (468.15, 106.46)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 962)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.848 CPU sec, 3.695 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19341, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (397, 462, 538)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (115, 820)\n",
      "[M::mem_pestat] mean and std.dev: (470.12, 105.64)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 961)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.875 CPU sec, 3.681 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19428, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 459, 534)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (114, 814)\n",
      "[M::mem_pestat] mean and std.dev: (467.16, 105.99)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 954)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.064 CPU sec, 3.913 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19178, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 458, 534)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (117, 812)\n",
      "[M::mem_pestat] mean and std.dev: (466.36, 106.18)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 951)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.884 CPU sec, 3.736 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19414, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 460, 536)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (110, 820)\n",
      "[M::mem_pestat] mean and std.dev: (467.50, 106.52)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 962)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.795 CPU sec, 3.635 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19254, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (396, 461, 538)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (112, 822)\n",
      "[M::mem_pestat] mean and std.dev: (469.31, 106.46)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 964)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.780 CPU sec, 3.632 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19271, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 458, 536)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (110, 820)\n",
      "[M::mem_pestat] mean and std.dev: (467.62, 106.95)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 962)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.782 CPU sec, 3.635 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19436, 1, 1)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 458, 536)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (110, 820)\n",
      "[M::mem_pestat] mean and std.dev: (466.66, 107.15)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 962)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.738 CPU sec, 3.590 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19209, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 459, 537)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (111, 821)\n",
      "[M::mem_pestat] mean and std.dev: (467.92, 107.16)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 963)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.747 CPU sec, 3.596 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19242, 1, 1)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (396, 458, 535)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (118, 813)\n",
      "[M::mem_pestat] mean and std.dev: (467.51, 106.28)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 952)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.903 CPU sec, 3.723 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19284, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 456, 534)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (114, 814)\n",
      "[M::mem_pestat] mean and std.dev: (465.76, 107.42)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 954)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.863 CPU sec, 3.718 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19246, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (396, 459, 537)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (114, 819)\n",
      "[M::mem_pestat] mean and std.dev: (468.45, 107.17)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 960)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.837 CPU sec, 3.670 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19296, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 461, 537)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (111, 821)\n",
      "[M::mem_pestat] mean and std.dev: (468.83, 106.89)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 963)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.863 CPU sec, 3.687 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19333, 1, 1)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 459, 536)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (113, 818)\n",
      "[M::mem_pestat] mean and std.dev: (467.92, 106.94)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 959)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.814 CPU sec, 3.664 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19151, 0, 1)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (398, 462, 538)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (118, 818)\n",
      "[M::mem_pestat] mean and std.dev: (471.16, 106.64)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 958)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.830 CPU sec, 3.674 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19164, 4, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (392, 456, 532)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (112, 812)\n",
      "[M::mem_pestat] mean and std.dev: (464.30, 106.20)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 952)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.836 CPU sec, 3.674 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 18873, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (389, 452, 526)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (115, 800)\n",
      "[M::mem_pestat] mean and std.dev: (459.64, 104.40)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 937)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.874 CPU sec, 3.723 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19097, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 457, 532)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (118, 808)\n",
      "[M::mem_pestat] mean and std.dev: (465.68, 104.87)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 946)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.744 CPU sec, 3.593 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19073, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 457, 534)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (114, 814)\n",
      "[M::mem_pestat] mean and std.dev: (465.78, 105.70)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 954)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.778 CPU sec, 3.624 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19134, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (393, 456, 531)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (117, 807)\n",
      "[M::mem_pestat] mean and std.dev: (464.37, 105.17)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 945)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.833 CPU sec, 3.674 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19217, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (393, 455, 530)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (119, 804)\n",
      "[M::mem_pestat] mean and std.dev: (463.69, 103.50)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 941)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.888 CPU sec, 3.721 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 19256, 3, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 457, 532)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (118, 808)\n",
      "[M::mem_pestat] mean and std.dev: (465.33, 104.95)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 946)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.062 CPU sec, 3.904 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19289, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 457, 534)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (114, 814)\n",
      "[M::mem_pestat] mean and std.dev: (466.58, 106.16)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 954)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.041 CPU sec, 3.866 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19129, 3, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 457, 533)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (116, 811)\n",
      "[M::mem_pestat] mean and std.dev: (465.90, 105.27)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 950)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.273 CPU sec, 4.104 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19315, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 459, 536)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (110, 820)\n",
      "[M::mem_pestat] mean and std.dev: (467.58, 106.35)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 962)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.989 CPU sec, 3.826 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19266, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 459, 536)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (113, 818)\n",
      "[M::mem_pestat] mean and std.dev: (467.26, 106.16)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 959)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.783 CPU sec, 3.631 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19196, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 457, 533)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (119, 809)\n",
      "[M::mem_pestat] mean and std.dev: (466.13, 105.52)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 947)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.863 CPU sec, 3.698 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19360, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 458, 532)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (118, 808)\n",
      "[M::mem_pestat] mean and std.dev: (465.62, 105.55)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 946)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.294 CPU sec, 4.143 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19103, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 458, 534)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (117, 812)\n",
      "[M::mem_pestat] mean and std.dev: (466.63, 105.24)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 951)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.263 CPU sec, 4.037 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19089, 3, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (396, 459, 533)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (122, 807)\n",
      "[M::mem_pestat] mean and std.dev: (466.63, 103.69)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 944)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.887 CPU sec, 3.707 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19262, 0, 1)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (393, 457, 535)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (109, 819)\n",
      "[M::mem_pestat] mean and std.dev: (466.35, 106.30)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 961)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.073 CPU sec, 3.919 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19312, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 457, 533)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (119, 809)\n",
      "[M::mem_pestat] mean and std.dev: (465.70, 104.59)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 947)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.906 CPU sec, 3.753 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19274, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 458, 535)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (115, 815)\n",
      "[M::mem_pestat] mean and std.dev: (467.35, 105.07)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 955)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.206 CPU sec, 4.044 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19346, 3, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 460, 536)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (113, 818)\n",
      "[M::mem_pestat] mean and std.dev: (468.06, 106.55)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 959)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.886 CPU sec, 3.732 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19307, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (396, 459, 534)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (120, 810)\n",
      "[M::mem_pestat] mean and std.dev: (466.83, 105.47)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 948)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.968 CPU sec, 3.804 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19140, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (393, 457, 535)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (109, 819)\n",
      "[M::mem_pestat] mean and std.dev: (466.75, 107.08)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 961)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.963 CPU sec, 3.813 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19328, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 460, 537)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (111, 821)\n",
      "[M::mem_pestat] mean and std.dev: (468.26, 106.90)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 963)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.754 CPU sec, 3.602 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19415, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 458, 537)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (111, 821)\n",
      "[M::mem_pestat] mean and std.dev: (467.68, 106.77)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 963)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.844 CPU sec, 3.688 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19215, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (396, 459, 539)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (110, 825)\n",
      "[M::mem_pestat] mean and std.dev: (468.98, 107.17)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 968)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.818 CPU sec, 3.656 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19367, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 458, 534)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (117, 812)\n",
      "[M::mem_pestat] mean and std.dev: (466.74, 105.75)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 951)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.048 CPU sec, 3.901 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19288, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 459, 535)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (112, 817)\n",
      "[M::mem_pestat] mean and std.dev: (466.58, 105.77)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 958)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.899 CPU sec, 3.737 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19449, 0, 1)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 459, 536)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (110, 820)\n",
      "[M::mem_pestat] mean and std.dev: (467.80, 106.94)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 962)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.839 CPU sec, 3.688 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19332, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 459, 536)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (113, 818)\n",
      "[M::mem_pestat] mean and std.dev: (468.00, 106.59)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 959)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.401 CPU sec, 4.251 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19292, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (391, 455, 530)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (113, 808)\n",
      "[M::mem_pestat] mean and std.dev: (463.01, 105.23)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 947)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 5.350 CPU sec, 5.105 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19341, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 458, 536)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (113, 818)\n",
      "[M::mem_pestat] mean and std.dev: (467.01, 106.13)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 959)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.033 CPU sec, 3.880 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19193, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 459, 536)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (110, 820)\n",
      "[M::mem_pestat] mean and std.dev: (466.58, 106.06)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 962)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.093 CPU sec, 3.927 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19252, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 457, 533)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (116, 811)\n",
      "[M::mem_pestat] mean and std.dev: (466.05, 106.32)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 950)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.078 CPU sec, 3.923 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19354, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 459, 537)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (111, 821)\n",
      "[M::mem_pestat] mean and std.dev: (468.21, 107.16)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 963)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.352 CPU sec, 4.204 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19236, 0, 1)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (392, 456, 533)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (110, 815)\n",
      "[M::mem_pestat] mean and std.dev: (465.12, 106.35)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 956)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.863 CPU sec, 3.693 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19189, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (391, 457, 532)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (109, 814)\n",
      "[M::mem_pestat] mean and std.dev: (464.12, 105.61)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 955)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.201 CPU sec, 4.033 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19188, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (396, 459, 534)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (120, 810)\n",
      "[M::mem_pestat] mean and std.dev: (467.03, 104.35)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 948)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.555 CPU sec, 4.404 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19107, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 459, 533)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (116, 811)\n",
      "[M::mem_pestat] mean and std.dev: (466.47, 104.69)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 950)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.000 CPU sec, 3.837 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19157, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 457, 533)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (119, 809)\n",
      "[M::mem_pestat] mean and std.dev: (465.48, 104.64)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 947)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.287 CPU sec, 4.135 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19275, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (396, 460, 535)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (118, 813)\n",
      "[M::mem_pestat] mean and std.dev: (467.73, 104.42)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 952)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.159 CPU sec, 4.006 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19271, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 458, 533)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (119, 809)\n",
      "[M::mem_pestat] mean and std.dev: (466.17, 104.77)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 947)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.256 CPU sec, 4.068 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19199, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 458, 531)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (120, 805)\n",
      "[M::mem_pestat] mean and std.dev: (465.76, 104.44)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 942)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.446 CPU sec, 4.275 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19201, 0, 1)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 459, 535)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (112, 817)\n",
      "[M::mem_pestat] mean and std.dev: (467.64, 106.31)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 958)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.338 CPU sec, 4.173 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19251, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (396, 459, 534)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (120, 810)\n",
      "[M::mem_pestat] mean and std.dev: (467.86, 105.55)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 948)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.499 CPU sec, 4.340 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19127, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (393, 458, 534)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (111, 816)\n",
      "[M::mem_pestat] mean and std.dev: (465.83, 105.94)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 957)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.842 CPU sec, 4.667 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19248, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (393, 458, 535)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (109, 819)\n",
      "[M::mem_pestat] mean and std.dev: (466.23, 105.70)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 961)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 5.234 CPU sec, 5.005 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19358, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (392, 456, 533)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (110, 815)\n",
      "[M::mem_pestat] mean and std.dev: (464.47, 105.49)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 956)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 5.171 CPU sec, 4.988 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19244, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 457, 534)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (114, 814)\n",
      "[M::mem_pestat] mean and std.dev: (465.85, 105.60)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 954)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.750 CPU sec, 4.564 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19323, 3, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 457, 534)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (114, 814)\n",
      "[M::mem_pestat] mean and std.dev: (466.17, 106.11)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 954)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.542 CPU sec, 4.366 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19148, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 457, 533)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (119, 809)\n",
      "[M::mem_pestat] mean and std.dev: (465.30, 105.14)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 947)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.179 CPU sec, 4.030 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19261, 3, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 459, 537)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (111, 821)\n",
      "[M::mem_pestat] mean and std.dev: (468.35, 106.88)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 963)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.457 CPU sec, 4.303 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19158, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (393, 456, 532)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (115, 810)\n",
      "[M::mem_pestat] mean and std.dev: (464.86, 105.81)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 949)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.616 CPU sec, 4.462 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19388, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 458, 534)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (117, 812)\n",
      "[M::mem_pestat] mean and std.dev: (466.44, 105.17)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 951)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.244 CPU sec, 4.091 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19256, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (396, 461, 536)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (116, 816)\n",
      "[M::mem_pestat] mean and std.dev: (468.38, 106.03)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 956)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.548 CPU sec, 4.397 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19181, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 460, 536)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (113, 818)\n",
      "[M::mem_pestat] mean and std.dev: (467.84, 105.82)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 959)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.543 CPU sec, 4.377 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19349, 0, 1)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (396, 460, 536)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (116, 816)\n",
      "[M::mem_pestat] mean and std.dev: (468.31, 106.22)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 956)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.839 CPU sec, 4.648 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19162, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (396, 459, 535)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (118, 813)\n",
      "[M::mem_pestat] mean and std.dev: (468.26, 106.24)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 952)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.284 CPU sec, 4.133 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19160, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 459, 535)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (112, 817)\n",
      "[M::mem_pestat] mean and std.dev: (466.87, 105.78)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 958)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.195 CPU sec, 4.040 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19221, 3, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 458, 537)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (111, 821)\n",
      "[M::mem_pestat] mean and std.dev: (467.90, 108.22)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 963)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.461 CPU sec, 4.312 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19287, 1, 1)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 458, 535)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (112, 817)\n",
      "[M::mem_pestat] mean and std.dev: (466.83, 106.59)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 958)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.982 CPU sec, 3.821 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19163, 0, 1)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 458, 534)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (117, 812)\n",
      "[M::mem_pestat] mean and std.dev: (467.51, 105.89)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 951)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.861 CPU sec, 3.703 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19284, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 458, 533)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (119, 809)\n",
      "[M::mem_pestat] mean and std.dev: (466.43, 105.55)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 947)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.773 CPU sec, 3.626 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19323, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 458, 535)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (112, 817)\n",
      "[M::mem_pestat] mean and std.dev: (467.02, 107.11)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 958)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.856 CPU sec, 3.705 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19153, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 457, 535)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (112, 817)\n",
      "[M::mem_pestat] mean and std.dev: (466.29, 107.16)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 958)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.806 CPU sec, 3.653 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19281, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 459, 536)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (110, 820)\n",
      "[M::mem_pestat] mean and std.dev: (467.63, 107.49)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 962)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.748 CPU sec, 3.602 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 19331, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (393, 456, 532)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (115, 810)\n",
      "[M::mem_pestat] mean and std.dev: (465.61, 106.17)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 949)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.879 CPU sec, 3.727 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19211, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 457, 533)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (116, 811)\n",
      "[M::mem_pestat] mean and std.dev: (466.15, 106.30)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 950)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.728 CPU sec, 3.582 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19221, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 459, 535)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (112, 817)\n",
      "[M::mem_pestat] mean and std.dev: (466.72, 106.47)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 958)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.766 CPU sec, 3.605 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19263, 0, 1)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (390, 454, 530)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (110, 810)\n",
      "[M::mem_pestat] mean and std.dev: (461.92, 106.29)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 950)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.809 CPU sec, 3.654 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19106, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 458, 535)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (112, 817)\n",
      "[M::mem_pestat] mean and std.dev: (466.98, 105.89)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 958)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.865 CPU sec, 3.713 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19149, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 459, 534)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (114, 814)\n",
      "[M::mem_pestat] mean and std.dev: (466.41, 104.93)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 954)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.912 CPU sec, 3.756 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19296, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 458, 532)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (121, 806)\n",
      "[M::mem_pestat] mean and std.dev: (465.93, 103.87)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 943)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.886 CPU sec, 3.725 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19209, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (393, 457, 532)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (115, 810)\n",
      "[M::mem_pestat] mean and std.dev: (465.24, 104.79)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 949)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.900 CPU sec, 3.749 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19137, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (393, 456, 532)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (115, 810)\n",
      "[M::mem_pestat] mean and std.dev: (464.34, 104.43)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 949)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.795 CPU sec, 3.643 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19064, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (393, 455, 529)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (121, 801)\n",
      "[M::mem_pestat] mean and std.dev: (463.31, 103.08)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 937)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.872 CPU sec, 3.703 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19231, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 456, 532)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (118, 808)\n",
      "[M::mem_pestat] mean and std.dev: (465.39, 104.39)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 946)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.798 CPU sec, 4.649 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19073, 3, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (392, 455, 529)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (118, 803)\n",
      "[M::mem_pestat] mean and std.dev: (462.72, 104.74)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 940)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.691 CPU sec, 4.495 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19321, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 456, 530)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (122, 802)\n",
      "[M::mem_pestat] mean and std.dev: (464.50, 103.95)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 938)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.396 CPU sec, 4.244 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19197, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 457, 532)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (118, 808)\n",
      "[M::mem_pestat] mean and std.dev: (465.12, 105.50)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 946)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 5.066 CPU sec, 4.908 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 18856, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (393, 455, 530)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (119, 804)\n",
      "[M::mem_pestat] mean and std.dev: (463.56, 103.97)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 941)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.548 CPU sec, 4.396 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19364, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (392, 456, 531)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (114, 809)\n",
      "[M::mem_pestat] mean and std.dev: (464.40, 104.95)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 948)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.118 CPU sec, 3.953 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19222, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 459, 532)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (118, 808)\n",
      "[M::mem_pestat] mean and std.dev: (466.29, 104.46)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 946)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.180 CPU sec, 4.034 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19096, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 457, 533)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (116, 811)\n",
      "[M::mem_pestat] mean and std.dev: (466.28, 106.21)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 950)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.008 CPU sec, 3.820 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19078, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 458, 533)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (119, 809)\n",
      "[M::mem_pestat] mean and std.dev: (466.00, 104.50)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 947)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.917 CPU sec, 3.756 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19224, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 459, 534)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (117, 812)\n",
      "[M::mem_pestat] mean and std.dev: (466.42, 104.33)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 951)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.088 CPU sec, 3.934 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19294, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (391, 454, 531)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (111, 811)\n",
      "[M::mem_pestat] mean and std.dev: (464.06, 106.11)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 951)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.124 CPU sec, 3.974 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19249, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 459, 536)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (113, 818)\n",
      "[M::mem_pestat] mean and std.dev: (467.83, 106.93)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 959)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.085 CPU sec, 3.934 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19289, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (396, 459, 537)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (114, 819)\n",
      "[M::mem_pestat] mean and std.dev: (467.49, 105.93)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 960)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.171 CPU sec, 3.965 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19405, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 457, 534)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (114, 814)\n",
      "[M::mem_pestat] mean and std.dev: (466.15, 105.33)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 954)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.112 CPU sec, 3.946 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19309, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (392, 457, 533)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (110, 815)\n",
      "[M::mem_pestat] mean and std.dev: (466.17, 106.57)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 956)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.332 CPU sec, 4.145 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19184, 0, 1)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 457, 530)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (125, 800)\n",
      "[M::mem_pestat] mean and std.dev: (465.13, 104.10)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 935)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.938 CPU sec, 3.790 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 19145, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (396, 459, 535)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (118, 813)\n",
      "[M::mem_pestat] mean and std.dev: (467.88, 105.80)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 952)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.960 CPU sec, 3.810 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19263, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (396, 459, 534)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (120, 810)\n",
      "[M::mem_pestat] mean and std.dev: (467.48, 106.17)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 948)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.215 CPU sec, 4.062 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19298, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 460, 537)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (111, 821)\n",
      "[M::mem_pestat] mean and std.dev: (468.51, 106.81)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 963)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.075 CPU sec, 3.934 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19187, 6, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (396, 460, 538)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (112, 822)\n",
      "[M::mem_pestat] mean and std.dev: (469.25, 107.78)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 964)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.947 CPU sec, 3.797 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19311, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 459, 538)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (106, 826)\n",
      "[M::mem_pestat] mean and std.dev: (468.57, 108.04)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 970)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.965 CPU sec, 3.816 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19320, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 459, 535)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (115, 815)\n",
      "[M::mem_pestat] mean and std.dev: (467.63, 106.13)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 955)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.933 CPU sec, 3.783 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19577, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 458, 534)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (117, 812)\n",
      "[M::mem_pestat] mean and std.dev: (466.50, 106.37)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 951)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.067 CPU sec, 3.909 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19262, 2, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 459, 536)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (113, 818)\n",
      "[M::mem_pestat] mean and std.dev: (468.14, 107.07)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 959)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.040 CPU sec, 3.874 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19242, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 458, 535)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (115, 815)\n",
      "[M::mem_pestat] mean and std.dev: (466.77, 106.32)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 955)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.949 CPU sec, 3.760 real sec\n",
      "[M::process] read 66226 sequences (10000126 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 19340, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (395, 459, 536)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (113, 818)\n",
      "[M::mem_pestat] mean and std.dev: (468.37, 107.10)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 959)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 3.940 CPU sec, 3.781 real sec\n",
      "[M::process] read 64670 sequences (9765170 bp)...\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 19236, 1, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (394, 458, 537)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (108, 823)\n",
      "[M::mem_pestat] mean and std.dev: (467.60, 108.47)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 966)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 66226 reads in 4.036 CPU sec, 3.894 real sec\n",
      "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 18641, 0, 0)\n",
      "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
      "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
      "[M::mem_pestat] (25, 50, 75) percentile: (393, 456, 533)\n",
      "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (113, 813)\n",
      "[M::mem_pestat] mean and std.dev: (464.58, 106.40)\n",
      "[M::mem_pestat] low and high boundaries for proper pairs: (1, 953)\n",
      "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
      "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
      "[M::mem_process_seqs] Processed 64670 reads in 4.106 CPU sec, 4.078 real sec\n",
      "[main] Version: 0.7.17-r1188\n",
      "[main] CMD: bwa mem samples/GCF_000005845.2_ASM584v2_genomic.fna.gz samples/samples_AS00000197_file-0.fastq.gz samples/samples_AS00000197_file-1.fastq.gz\n",
      "[main] Real time: 532.534 sec; CPU: 552.400 sec\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "bwa index samples/GCF_000005845.2_ASM584v2_genomic.fna.gz\n",
    "bwa mem samples/GCF_000005845.2_ASM584v2_genomic.fna samples/samples_AS00000197_file-0.fastq.gz samples/samples_AS00000197_file-1.fastq.gz  > output/aln-pe.sam\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "head output/aln-pe.sam"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Variant calling with bcftools"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[bam_sort_core] merging from 4 files and 1 in-memory blocks...\n",
      "Note: none of --samples-file, --ploidy or --ploidy-file given, assuming all sites are diploid\n",
      "[mpileup] 1 samples in 1 input files\n",
      "[mpileup] maximum number of reads per input file set to -d 250\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "samtools view -S -b output/aln-pe.sam > output/aln-pe.bam\n",
    "samtools sort  output/aln-pe.bam > output/aln-sorted.bam\n",
    "samtools index output/aln-sorted.bam output/aln-sorted.bam.bai\n",
    "bcftools mpileup -f samples/GCF_000005845.2_ASM584v2_genomic.fna output/aln-sorted.bam | bcftools call -mv  -Ov > output/variants.vcf"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "##fileformat=VCFv4.2\n",
      "##FILTER=<ID=PASS,Description=\"All filters passed\">\n",
      "##bcftoolsVersion=1.10+htslib-1.10.2\n",
      "##bcftoolsCommand=mpileup -f samples/GCF_000005845.2_ASM584v2_genomic.fna output/aln-sorted.bam\n",
      "##reference=file://samples/GCF_000005845.2_ASM584v2_genomic.fna\n",
      "##contig=<ID=NC_000913.3,length=4641652>\n",
      "##ALT=<ID=*,Description=\"Represents allele(s) other than observed.\">\n",
      "##INFO=<ID=INDEL,Number=0,Type=Flag,Description=\"Indicates that the variant is an INDEL.\">\n",
      "##INFO=<ID=IDV,Number=1,Type=Integer,Description=\"Maximum number of raw reads supporting an indel\">\n",
      "##INFO=<ID=IMF,Number=1,Type=Float,Description=\"Maximum fraction of raw reads supporting an indel\">\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "head output/variants.vcf"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Put all together with IGV viewer\n",
    "Using IGV to display all ouput files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Process is terminated.\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "./IGV_Linux_2.14.1/igv.sh"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Phylogeny\n",
    "Make phylogeny tree with parsnp"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Ref samples/GCF_000005845.2_ASM584v2_genomic.fna\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "22:21:17 - \u001b[1;37mINFO\u001b[0m - |--Parsnp 1.7.4--|\n",
      "\n",
      "22:21:29 - \u001b[1;37mINFO\u001b[0m - \n",
      "***********************\n",
      "SETTINGS:\n",
      "|-refgenome:\tsamples/GCF_000005845.2_ASM584v2_genomic.fna\n",
      "|-genomes:\t\n",
      "\tsamples/GCF_000005845.2_ASM584v2_genomic.fna\n",
      "\tsamples/GCF_018499845.2_ASM1849984v2_genomic.fna\n",
      "\t...1 more file(s)...\n",
      "\tsamples/GCF_021397815.1_ASM2139781v1_genomic.fna\n",
      "\toutput/contigs.fasta\n",
      "|-aligner:\tmuscle\n",
      "|-outdir:\toutput\n",
      "|-OS:\tLinux\n",
      "|-threads:\t1\n",
      "***********************\n",
      "    \n",
      "22:21:29 - \u001b[1;37mINFO\u001b[0m - <<Parsnp started>>\n",
      "22:21:29 - \u001b[1;37mINFO\u001b[0m - No genbank file provided for reference annotations, skipping..\n",
      "22:21:29 - \u001b[1;37mINFO\u001b[0m - Running Parsnp multi-MUM search and libMUSCLE aligner...\n",
      "22:22:12 - \u001b[1;37mINFO\u001b[0m - Reconstructing core genome phylogeny...\n",
      "22:22:13 - \u001b[1;37mINFO\u001b[0m - Aligned 5 genomes in 55.95 seconds\n",
      "22:22:13 - \u001b[1;37mINFO\u001b[0m - Parsnp finished! All output available in output\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "parsnp -r samples/GCF_000005845.2_ASM584v2_genomic.fna -d samples/*.fna output/contigs.fasta -o output -c"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAi0AAAEGCAYAAACpaqUaAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADh0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uMy4xLjIsIGh0dHA6Ly9tYXRwbG90bGliLm9yZy8li6FKAAAgAElEQVR4nOzdd3xUZfb48c8JQaoEpKyEoBCCAmlDEqqUUEQERUFKUPyiqCBiwf2J4MIKy4IUC7qKoLAqawnNAgKyKhBAUUOAhFAUjERpUiIEQyAQcn5/zOTuBEhBjThy3q/XvLz3uU859w7OnDz3uYmoKsYYY4wxf3R+FzsAY4wxxpiSsKTFGGOMMT7BkhZjjDHG+ARLWowxxhjjEyxpMcYYY4xP8L/YAXirUaOG1qtX72KHYYwxPmPDhg2HVbXmxY7DmN/DHyppqVevHklJSRc7DGOM8Rki8v3FjsGY34vdHjLGGGOMT7CkxRhjjDE+wZIWY4wxxvgES1qMMcYY4xMsaTHGGGOMT7CkxRhjjDE+wZIWY4wxxvgES1qMMcYY4xMsaTHGGGOMT7CkxRhjjDE+wZIWY4wxxvgES1qMMcYY4xMsaTHGGGOMT7CkxRhjjDE+wZIWY4wxxvgES1qMMcYY4xMsaTHGGGOMT7CkxRhjjDE+wZIWY4wxxvgES1qMMcYY4xMsaTHGGGOMT7CkxRhjjDE+wZIWY4wxxvgES1qMMcYY4xMsaTHGGGOMT7CkxRhjjDE+odSSFhF5TUQOisiW0hqjJLKyshg6dCgNGjQgKiqK6OhoZs2a5RzfsWMH3bp1o2HDhkRFRdG3b18OHDhAQkICAQEBuFwuXC4XnTt3LnSMnJwc+vXrR0hICC1atCA9PR2AjIwMOnToQOXKlXnwwQcLtImPjyc8PJyIiAi6du3K4cOHCxx/9tlnERGn/MiRI/Ts2ZOIiAiaN2/Oli3/u6wvvPACYWFhhIaG8vzzzzvlycnJtGzZEpfLRUxMDImJiQXGWL9+Pf7+/ixcuNApK1OmjHPOPXr0OO/5PvfcczRp0oSIiAg6derE999/X+i1yff8889Tvnx5MjMznbLs7GzuuOMOwsPDCQsLo02bNmRlZQEgIgwYMMCpm5ubS82aNbnpppsA+Prrr2nVqhXlypXjmWeeKTDWtGnTCA0NJSwsjP79+3Py5MkCxx9++GEqV67s7H///fd06tSJiIgIYmNj2bNnj3Ns5MiRhIWFERYWxrx585zyFStWEBUVhcvlok2bNnz77bcFxnj33XcREZKSkoq8Lm+//TYRERGEh4fTunVrUlJSiqz/RzFz5kz+85//lLj+ggULaNy4MR06dCjFqIwxf3qqWiovoB0QBWwpaZvo6Gj9rfXr10+feOIJPXPmjKqqHjx4UCdPnqyqqidOnNCQkBBdvHixU3/VqlWampqqq1at0u7du5dojOnTp+uQIUNUVTU+Pl779u2rqqpZWVm6du1anTFjhg4bNsypf/r0aa1Zs6YeOnRIVVVHjBihY8eOdY7/8MMP2qVLF73qqqucOo899piOGzdOVVW3b9+uHTt2VFXV1NRUDQ0N1ePHj+vp06e1U6dOunPnTlVVvf7663XZsmWqqrp06VJt3769M0Zubq526NBBb7zxRl2wYIFTXqlSpWLPd+XKlXr8+HFVVX355Zed8y1K8+bNtU2bNvraa685ZU899ZQ++uijzv7XX3+tJ0+edOKIjIzU7OxsVVVdtmyZRkZGOu/JgQMHNDExUf/2t7/p008/7fSxZ88erVevntOuT58++vrrrzvH169frwMGDChwnr1799Y33nhDVVVXrFihAwYMUFXVJUuWaOfOnfX06dOalZWlMTExmpmZqaqqDRs21G3btqmq+/0fOHCg09+xY8e0bdu22qJFC12/fn2R1+Xzzz/Xn376yTnH5s2bF3cpfdINN9yga9euvdhh/CkBSVpKn+P2stcf7VVqMy2qugb4qbT6B4iNjT3n5S0tLY3ExEQmTJiAn5/7VGvWrMnIkSMBeOedd2jVqhU333xzgT7DwsIuKI5FixYxcOBAAHr37s2KFStQVSpVqkSbNm0oX758gfr5F//48eOoKseOHSMwMNA5/uijjzJ16lRExCnbtm0bHTt2BKBRo0akp6dz4MABtm/fTosWLahYsSL+/v60b9+e9957D3DPVhw7dgyAzMzMAmO8+OKL3HbbbdSqVeuCzhWgQ4cOVKxYEYCWLVsWmJk4n7S0NLKyspgwYQLx8fFO+f79+6lTp46zf+2111KuXDlnv1u3bixduhRwz0z179/fOVarVi2aNWtG2bJlzxkvNzeXEydOkJubS3Z2tnPeZ86cYcSIEUydOrVAfe9r26FDBxYtWuSUt2vXDn9/fypVqkRERATLly8Hir62f//73xk5cmSB971ly5Zs3brV2Y+NjSUpKYnWrVtTrVq1El/Lf/7zn1x77bW0adOG/v37O7NMaWlpdO3alejoaNq2bcvXX38NwF133cXDDz9M69atCQ4OdmbVVJURI0YQFhZGeHi4M4uUkJBA+/btueWWWwgODmbUqFG8/fbbNG/enPDwcNLS0gAYN26cM/a3335L586diYyMJCoqyqmTb/z48Xz22Wfcc889jBgxgjfeeINevXrRtWtXGjZsyOOPP+7UHTp0KDExMYSGhjJ27Ngir4Ux5tJz0de0iMhgEUkSkaRDhw6VuF1sbCzJyclF1tm6dSuRkZFOwnK2LVu2EB0dXWj7tWvXOrdKJk6cWGi9vXv3UrduXQD8/f0JCAggIyOj0Pply5ZlxowZhIeHExgYyLZt27jnnnsAdwJUp04dIiMjC7SJjIx0kpHExES+//579uzZQ1hYGGvXriUjI4Ps7GyWLVvG7t27AfctmREjRlC3bl0ee+wxJk2a5MT7/vvvM3To0HNiO3nyJDExMbRs2ZIPPvig0HPI9+9//5sbb7yxyDpz584lLi6Otm3b8s0333DgwAEABg0axJQpU2jVqhVjxoxh586dBdrFxcUxd+5cTp48yebNm2nRokWx8dSpU4fHHnuMq666itq1axMQEECXLl0AeOmll+jRowe1a9cu0Mb72r7//vv8/PPPZGRkEBkZyfLly8nOzubw4cOsWrXKubazZ8+mW7duBAUF8eabbzJq1CgANm7cyO7du+nevXuBMfr168f8+fMBd7K2f/9+YmJiLuharl+/nnfffZeUlBQ++uijAreeBg8ezIsvvsiGDRt45plneOCBB5xj+/fv57PPPmPJkiVOnO+99x7JycmkpKTw6aefMmLECPbv3w9ASkoKM2fOZPv27bz55pvs2LGDxMRE7r33Xl588cVz4rrjjjsYNmwYKSkprFu37pzr++STTxITE8Pbb7/N008/DbhvXc6bN4/U1FTmzZvnXNeJEyeSlJTE5s2bWb16NZs3by70ehhjLj0XPWlR1VdVNUZVY2rWrHlBbV0uFwkJCQVeRZk4cSIul6vAT8VFadu2LcnJySQnJzN69OgLiq0op0+fZsaMGWzatIl9+/YRERHBpEmTyM7O5qmnnmL8+PHntBk1ahRHjx7F5XLx4osv0rRpU8qUKUPjxo0ZOXIkXbp0oWvXrrhcLsqUKQPAjBkzmDZtGrt372batGlOYjR8+HCmTJly3mTu+++/JykpiXfeeYfhw4ef81Ozt7feeoukpCRGjBhR5PnGx8cTFxeHn58ft912GwsWLADc7993333HiBEj+Omnn2jWrBnbt2932kVERJCenk58fDzdunUr/sLiXvuzaNEidu3axb59+zh+/DhvvfUW+/btY8GCBTz00EPntHnmmWdYvXo1TZs2ZfXq1dSpU4cyZcrQpUsXunXrRuvWrenfvz+tWrVyru20adNYtmwZe/bs4e677+avf/0reXl5/PWvf+XZZ589Z4y+ffs6sxzz58+nd+/eBY6vWrWKf//730yZMqXQc/v888+55ZZbKF++PJdffrkzQ5iVlcW6devo06cPLpeLIUOGOAkIwK233oqfnx9NmjRxEsbPPvuM/v37U6ZMGf7yl7/Qvn171q9fD0CzZs2oXbs25cqVo0GDBk7SFx4e7qzXyvfzzz+zd+9eevbsCUD58uWdWbiidOrUiYCAAMqXL0+TJk2cdVHz588nKiqKpk2bsnXrVrZt21ZsX8aYS4f/xQ6gNDVp0oSUlBTy8vLw8/Nj9OjRjB492lmEGRoayurVq3/1OHXq1GH37t0EBQWRm5tLZmYm1atXL7R+/gxRgwYNAPcX2uTJk7nlllvYtWuXM8uyZ88eoqKiSExM5Morr+T1118H3FP79evXJzg4GIB77rnHSUj+9re/ERQUBMCcOXN44YUXAOjTpw/33nsvAElJScTFxQFw+PBhli1bhr+/P7feeqtzuyY4OJjY2Fg2bdrkxOnt008/ZeLEiaxevbrALZ2zpaamsnPnTq6//noATp06Rf369Z2FyZUrV6ZXr1706tULPz8/li1bRuPGjZ32PXr04LHHHiMhIaHI2SvvuOrXr09+AtyrVy/WrVtHtWrV+PbbbwkJCQHci4BDQkL49ttvCQwMdGZasrKyePfdd6latSqA828G4Pbbb+eaa67h0KFDpKSkODM//fr1o2vXrvz8889s2bLFuU35448/0qNHDxYvXkxMTAzVq1dn8+bNzJs3j5kzZzoxb968mXvvvZePPvqoyH83hcnLy6Nq1aqFzjx6vz+qWmx/3vX9/PycfT8/P3Jzcy84vuLGKFOmDLm5uezatYtnnnmG9evXU61aNe66665zFlEbYy5tF32mpTSFhIQQExPDmDFjOHPmDOC+/ZH/wX377bezbt06Z90EwJo1awo8mVMSPXr0YM6cOQAsXLiQjh07FliPcrY6deqwbds28m+HffLJJzRu3Jjw8HAOHjxIeno66enpBAUFsXHjRq688kqOHj3KqVOnAPetiXbt2lGlShUADh48CMAPP/zAe++9x+233w5AYGCgk5StXLmShg0bArBr1y5njN69e/Pyyy9z6623cuTIEXJycgB3MvP555/TpEmTc+LftGkTQ4YMYfHixcWuiYmPj2fcuHHOePv27WPfvn18//33fP755xw5cgRwJzPbtm3j6quvLtB+0KBBjB07lvDw8CLHyXfVVVfx5Zdfkp2djaqyYsUKGjduTPfu3fnxxx+dOCpWrOg88XP48GHy8vIAmDRpEoMGDQLca2DyE6XNmzezefNmunTpQrVq1cjMzGTHjh0F3r+AgAAOHz7sjNGyZUsnYQF3cjN16lQyMzOJiIhw3rNevXrx5ptvcs011xR5btdddx0ffvghJ0+eJCsriyVLlgBQpUoV6tev78xgqWqxTyG1bduWefPmcebMGQ4dOsSaNWto3rx5ia6xt8svv5ygoCDnVmJOTg7Z2dkX3A/AsWPHqFSpEgEBARw4cICPPvroF/VjjPnzKrWZFhGJB2KBGiKyBxirqv8urfEKM3v2bEaMGEFISAjVq1enQoUKzkLMChUqsGTJEoYPH87w4cMpW7YsERERvPDCC+c8glyUe+65hzvvvJOQkBCuuOIK5s6d6xyrV68ex44d49SpU3zwwQd8/PHHNGnShLFjx9KuXTvKli3L1VdfzRtvvFHkGNu3b2fgwIGICKGhofz73/+7lLfddhsZGRmULVuW6dOnO7MEs2bN4pFHHiE3N5fy5cvz6quvFjvGkCFD8PPzIy8vj1GjRjlJS/66hB49ejBixAiysrLo06cP4E4UFi9efN4+586dy7JlywqU9ezZk7lz51K7dm2GDh2KqpKXl0f37t257bbbCtQNCgri4YcfPqffH3/8kZiYGI4dO4afnx/PP/8827Zto0WLFvTu3ZuoqCj8/f1p2rQpgwcPLvK8ExISeOKJJxAR2rVrx/Tp0wH3bby2bdsC7sTgrbfewt/f37m2t912G35+flSrVo3XXnutyDHAvUj7kUce4e9//7tTNn78eDIyMpw1KP7+/oU+Jt2sWTN69OhBREQEf/nLXwgPDycgIABwPzo9dOhQJkyYwOnTp4mLiztnXZS3nj178sUXXxAZGYmIMHXqVK688kpnAe+FePPNNxkyZAhPPvkkZcuWZcGCBQQHB+NyuYpdd+YtMjKSpk2b0qhRI+rWrct11113wbEYY/7cpCTTxb+XmJgYLe73WuTLn4Ivbh2LMX8mWVlZVK5cmezsbNq1a8err75KVFTUxQ7LXEQiskFVY4qvaYzv+1OvaTHmz2bw4MFs27aNkydPMnDgQEtYjDGXFEtaLsDEiROddQP5+vTp85s+WeSrUlNTufPOOwuUlStXjq+++uoiReS7MjIy6NSp0znlK1as4J133rkIERljzB+D3R4yxhgfZreHzKXkT/30kDHGGGP+PCxpMcYYY4xPsKTFGGOMMT7BkhZjjDHG+ARLWowxxhjjEyxpMcYYY4xPsKTFGGOMMT7BkhZjjDHG+ARLWowxxhjjEyxpMcYYY4xPsKTFGGOMMT7BkhZjjDHG+ARLWowxxhjjEyxpMcYYY4xPsKTFGGOMMT7BkhZjjDHG+ARLWowxxhjjEyxpMcYYY4xPsKTFGGOMMT7BkhZjjDHG+ARLWowxxhjjEyxpMcYYY4xPsKTFGGOMMT7BkhZjjDHG+ARLWowxxhjjEyxpMcYYY4xPsKTFGGOMMT6h1JIWEakrIqtEZJuIbBWRR0prrMJkZWUxdOhQGjRoQFRUFNHR0cyaNcs5vmPHDrp160bDhg2Jioqib9++HDhwgISEBAICAnC5XLhcLjp37lzoGDk5OfTr14+QkBBatGhBenq6c2zSpEmEhIRw7bXX8t///tcpX758Oddeey0hISFMnjzZKd+1axctWrQgJCSEfv36cerUqSLHSE9Pp0KFCk6c999/v9NXfHw84eHhRERE0LVrVw4fPlwg7meffRYRccrPPufx48ef93zvuOMOrr32WsLCwhg0aBCnT58u5l2AW2+9lZYtWxYo++abb4iNjcXlctG4cWMGDx7sxCEizJ4926mbnJyMiPDMM88AMG7cOOrUqePEumzZMgBOnz7NwIEDCQ8Pp3HjxkyaNKnAmGfOnKFp06bcdNNNRcabnZ1N9+7dadSoEaGhoYwaNarYc/yjaN26dYnr5uTk0LlzZ1wuF/PmzSvFqIwx5jeiqqXyAmoDUZ7ty4EdQJOi2kRHR2tJtW/fXtu3b19knX79+ukTTzyhZ86cUVXVgwcP6uTJk1VV9cSJExoSEqKLFy926q9atUpTU1N11apV2r179xLFMX36dB0yZIiqqsbHx2vfvn1VVXXr1q0aERGhJ0+e1O+++06Dg4M1NzdXc3NzNTg4WNPS0jQnJ0cjIiJ069atqqrap08fjY+PV1XVIUOG6Msvv1zkGLt27dLQ0NBzYjp9+rTWrFlTDx06pKqqI0aM0LFjxzrHf/jhB+3SpYteddVVTp2SnvPSpUs1Ly9P8/LyNC4uzomxMEeOHNGgoCBt1KiRpqWlOeVdunTRDz74wNnfvHmzE0dYWJhef/31zrHHH39cIyMj9emnn1ZV1bFjxzrb3t5++23t16+fqqoeP35cr776at21a5dz/Nlnn9X+/fsXe57Hjx/XlStXqqpqTk6OtmnTRpctW1ZkG1/0xRdfaKdOnS52GOZXApK0lD7H7WWvP9qr1GZaVHW/qm70bP8MbAfq/JZjJCcnExsb67y8paWlkZiYyIQJE/Dzc59mzZo1GTlyJADvvPMOrVq14uabb3baxMbGEhYWdkExLFq0iIEDBwLQu3dvVqxYgaqyaNEi4uLiKFeuHPXr1yckJITExEQSExMJCQkhODiYyy67jLi4OBYtWoSqsnLlSnr37g3AwIED+eCDD4ocozD5b+7x48dRVY4dO0ZgYKBz/NFHH2Xq1KmIyAWdK0C3bt0QEUSE5s2bs2fPniLrv/fee9x8883ExcUxd+5cp3z//v0EBQU5++Hh4c721VdfzcmTJzlw4ACqyvLly7nxxhuLjU1EOH78OLm5uZw4cYLLLruMKlWqALBnzx6WLl3Kvffe69Rfvnw5ffr0cfYTEhK46aabqFixIh06dADgsssuIyoqqsjzTEtLo2XLloSHhzNmzBgqV67sHHv66adp1qwZERERjB07FnDPkDVu3Jj77ruP0NBQunTpwokTJwD3v+mWLVsSERFBz549OXLkCOD+t/noo48SExND48aNWb9+Pb169aJhw4aMGTPGGc977ClTphAeHk5kZOQ5s0UHDx5kwIABrF+/HpfLRVpaGvXq1WPs2LFERUURHh7O119/DUBiYiKtWrWiadOmtG7dmm+++abY98IYY0rD77KmRUTqAU2Br85zbLCIJIlI0qFDh36zMbdu3UpkZKSTsJxty5YtREdHF9p+7dq1zu2HiRMnFlpv79691K1bFwB/f38CAgLIyMgoUA4QFBTE3r17Cy3PyMigatWq+Pv7Fygvagxw31Jq2rQp7du3Z+3atQCULVuWGTNmEB4eTmBgINu2beOee+4B3AlQnTp1iIyMPOdcvvjiCyIjI7nxxhvZunVroecM7lsxb775Jl27di2yXnx8PP3796d///7Ex8c75Y8++igdO3bkxhtvZNq0aRw9erRAu969e7NgwQLWrVtHVFQU5cqVK3D8pZdeIiIigkGDBjlf7L1796ZSpUrUrl2bq666iscee4wrrrgCgOHDhzN16tQC/x46d+7MV199xfHjxwGYN28ecXFxBcY5evQoH374IZ06dSr0HB955BEeeeQRUlNTCyRiH3/8MTt37iQxMZHk5GQ2bNjAmjVrANi5cyfDhg1j69atVK1alXfffReA//u//2PKlCls3ryZ8PBw/vGPfzj9XXbZZSQlJXH//fdzyy23MH36dLZs2cIbb7zh/HvI99FHH7Fo0SK++uorUlJSePzxxwscr1WrFrNnz6Zt27YkJyfToEEDAGrUqMHGjRsZOnSoczuuUaNGrF27lk2bNjF+/Hj+9re/FXotjDGmNJV60iIilYF3geGqeuzs46r6qqrGqGpMzZo1L6hvl8tFQkKC8yrKxIkTcblcBWYcipL/YZ6cnMzo0aMvKK7fS+3atfnhhx/YtGkTzz33HLfffjvHjh3j9OnTzJgxg02bNrFv3z4iIiKYNGkS2dnZPPXUU+ddrxIVFcX3339PSkoKDz30ELfeemuRYz/wwAO0a9eOtm3bFlrnwIED7Ny5kzZt2nDNNddQtmxZtmzZAsDdd9/N9u3b6dOnDwkJCbRs2ZKcnBynbd++fVmwYIGT9HgbOnQoaWlpJCcnU7t2bf7f//t/gHtGoEyZMuzbt49du3bx7LPP8t1337FkyRJq1ap1TpLq7+9P165d+fDDD8nNzWXp0qXccsstzvHc3Fz69+/Pww8/THBwcKHn+cUXXzgzNrfffrtT/vHHH/Pxxx/TtGlToqKi+Prrr9m5cycA9evXx+VyARAdHU16ejqZmZkcPXqU9u3bA+7ZtvwkB6BHjx6Ae1YqNDSU2rVrU65cOYKDg9m9e3eBmD799FPuvvtuKlasCOAkb8Xp1atXgZgAMjMz6dOnD2FhYTz66KPFJrTGGFNaSjVpEZGyuBOWt1X1vdIc62xNmjQhJSWFvLw8AEaPHk1ycjLHjrnzptDQUDZs2PCrx6lTp47zhZGbm0tmZibVq1cvUA7u2xN16tQptLx69eocPXqU3NzcAuVFjVGuXDmqV68OuL9kGjRowI4dO0hOTgagQYMGiAh9+/Zl3bp1pKWlsWvXLiIjI6lXrx579uwhKiqKH3/8kSpVqji3Frp168bp06fPWbyb7x//+AeHDh3iueeeK/LazJ8/nyNHjlC/fn3q1atHenp6gdmWwMBABg0axKJFi/D393cSGoArr7ySsmXL8sknn5wzy/GXv/yFMmXK4Ofnx3333UdiYiLgvuXXtWtXypYtS61atbjuuutISkri888/Z/HixdSrV4+4uDhWrlzJgAEDAIiLi2P+/PmsXLmSmJgYLr/8cmecwYMH07BhQ4YPH17keRZGVXniiSec5Pfbb791Zry8Z47KlCnjvO9FyW/j5+dXoL2fn1+J2pdEfr/eMf3973+nQ4cObNmyhQ8//JCTJ0/+JmMZY8yFKs2nhwT4N7BdVYv+disFISEhxMTEMGbMGM6cOQPAyZMnnbUgt99+O+vWrWPp0qVOmzVr1hT44iyJHj16MGfOHAAWLlxIx44dERF69OjB3LlzycnJYdeuXezcuZPmzZvTrFkzdu7cya5duzh16hRz586lR48eiAgdOnRg4cKFAMyZM8f5qb+wMQ4dOuSc23fffcfOnTsJDg6mTp06bNu2jfzbbZ988gmNGzcmPDycgwcPkp6eTnp6OkFBQWzcuJErr7ySH3/80bk2iYmJ5OXlOQmRt9mzZ/Pf//6X+Pj4Qm+95YuPj2f58uXOeBs2bHDWtSxfvtx58ujHH38kIyPDSdLyjR8/nilTplCmTJkC5fv373e233//fWcd0lVXXcXKlSsBOH78OF9++SWNGjVi0qRJ7Nmzh/T0dObOnUvHjh156623AGjfvj0bN25k1qxZBW4NjRkzhszMTJ5//vkizxGgZcuWzu0d73U7N9xwA6+99hpZWVmA+zbfwYMHC+0nICCAatWqObf53nzzTWfW5UJdf/31vP7662RnZwPw008//aJ+wD3Tkv/evPHGG7+4H2OM+bX8S7Hv64A7gVQRSfaU/U1Vl5XimAXMnj2bESNGEBISQvXq1alQoQJTp04FoEKFCixZsoThw4czfPhwypYtS0REBC+88EKhMwznc88993DnnXcSEhLCFVdc4XxphYaG0rdvX5o0aYK/vz/Tp093vnxfeuklbrjhBs6cOcOgQYMIDQ0F3Asn4+LiGDNmDE2bNnV+Ki9sjDVr1vDkk09StmxZ/Pz8mDlzpnMbYOzYsbRr146yZcty9dVXF/tls3DhQmbMmIG/vz8VKlRg7ty5zkLdbt26MXv2bAIDA7n//vu5+uqradWqFeC+nfDkk0+e0196ejrff/99gUed69evT0BAAF999RUff/wxjzzyCOXLlwfcC1avvPJKZ/EnFBaT/dEAACAASURBVP747uOPP+48Bl2vXj1eeeUVAIYNG8bdd99NaGgoqsrdd99NREREkeddpkwZbrrpJt544w0nMdyzZw8TJ06kUaNGREVFAfDggw8WWMTr7fnnn2fAgAFMnDiRrl27EhAQAECXLl3Yvn27c60qV67MW2+9dU4S5m3OnDncf//9ZGdnExwczOuvv15k/IXp2rUrycnJxMTEcNlll9GtWzeeeuopZs6cCVDg8fjiPP744wwcOJAJEybQvXv3XxSPMcb8FqSop1B+bzExMZqUlFSiuvlPCxW3lsWY0padnU2FChUQEebOnUt8fDyLFi262GGZS4SIbFDVmIsdhzG/h9KcaTHmkrBhwwYefPBBVJWqVavy2muvXeyQjDHmT8mSlhKaOHEiCxYsKFDWp0+fP+yTRb+n119/nRdeeKFA2XXXXcf06dMvUkSlo6h/AykpKRcpKmOMuXTY7SFjjPFhdnvIXErsDyYaY4wxxidY0mKMMcYYn2BJizHGGGN8giUtxhhjjPEJlrQYY4wxxidY0mKMMcYYn2BJizHGGGN8giUtxhhjjPEJlrQYY4wxxidY0mKMMcYYn2BJizHGGGN8giUtxhhjjPEJlrQYY4wxxidY0mKMMcYYn2BJizHGGGN8giUtxhhjjPEJlrQYY4wxxidY0mKMMcYYn2BJizHGGGN8giUtxhhjjPEJlrQYY4wxxidY0mKMMcYYn+Bf0ooiUgson7+vqj+USkTGGGOMMedR7EyLiPQQkZ3ALmA1kA58VMpxGWOMMcYUUJLbQ/8EWgI7VLU+0An4slSjMsYYY4w5S0mSltOqmgH4iYifqq4CYko5LmOMMcaYAkqStBwVkcrAGuBtEXkBOF66Yf3xHD16lJdfftnZ37dvH7179/7V/ebk5NC5c2dcLhfz5s27oLYJCQmsW7fuV8dgjDHG+IKSJC23ACeAR4HlQBpwc3GNRKS8iCSKSIqIbBWRf/y6UC+us5OWwMBAFi5c+Kv73bRpEwDJycn069fvgtpa0mKMMeZSUpKk5WpVPaOquao6R1X/BYSXoF0O0FFVIwEX0FVEWv6aYH+p5cuXExUVRWRkJJ06dQLgp59+4tZbbyUiIoKWLVuyefNmAMaNG8egQYOIjY0lODiYf/3rXwCMGjWKtLQ0XC4XI0aMID09nbCwMACys7Pp27cvTZo0oWfPnrRo0YKkpCTOnDnDXXfdRVhYGOHh4UybNq1AXAcPHmTAgAGsX78el8tFWloa48ePp1mzZoSFhTF48GBUFYB//etfNGnShIiICOLi4khPT2fmzJlMmzYNl8vF2rVr+fDDD2nRogVNmzalc+fOHDhw4Pe6xMYYY0zpU9UiX8AWYCQgQAXgReCL4tqd1UdFYCPQoqh60dHRWlLt27fX9u3bF9j2Lst38OBBDQoK0u+++05VVTMyMlRV9cEHH9Rx48apquqKFSs0MjJSVVXHjh2rrVq10pMnT+qhQ4f0iiuu0FOnTumuXbs0NDTU6dd7/+mnn9bBgwerqmpqaqqWKVNG169fr0lJSdq5c2enzZEjR845j1WrVmn37t2d/fz4VFUHDBigixcvVlXV2rVr68mTJwv0M3bsWH366aed+j/99JPm5eWpquqsWbP0r3/9a/EX0hjj04AkvYDPY3vZy5dfJZlpaQHUBdYB64F9wHUlSYhEpIyIJAMHgU9U9avz1BksIkkiknTo0KGSdFtAbGwsycnJhR7/8ssvadeuHfXr1wfgiiuuAOCzzz7jzjvvBKBjx45kZGRw7NgxALp37065cuWoUaMGtWrVKnbG4rPPPiMuLg6AsLAwIiIiAAgODua7777joYceYvny5VSpUqXY81m1ahUtWrQgPDyclStXsnXrVgAiIiK44447eOutt/D3P/+v19mzZw833HAD4eHhPP30005bY4wx5s+gRE8P4V7TUgH3L5fbpap5Jelc3beVXEAQ0FxEws5T51VVjVHVmJo1a15A6P/jcrlISEhwXr9WuXLlnO0yZcqQm5v7i/qpVq0aKSkpxMbGMnPmTO69994i6588eZIHHniAhQsXkpqayn333cfJkycBWLp0KcOGDWPjxo00a9bsvDE99NBDPPjgg6SmpvLKK684bY0xxpg/g5IkLetxJy3NgLZAfxFZcCGDqOpRYBXQ9YIj/JVatmzJmjVr2LVrF+BeywLQtm1b3n77bcC9oLVGjRpFzoRcfvnl/Pzzz+c9dt111zF//nwAtm3bRmpqKgCHDx8mLy+P2267jQkTJrBx48YiY81PMmrUqEFWVpaz0DcvL4/du3fToUMHpkyZQmZmJllZWefElJmZSZ06dQCYM2dO0RfGGGOM8TEl+TX+96hqkmd7P3CLiNxZXCMRqYn7d7wcFZEKwPXAlF8e6i9Ts2ZNXn31VXr16kVeXh61atXik08+cRbcRkREULFixWK/5KtXr851111HWFgYN954I8OGDXOOPfDAAwwcOJAmTZrQqFEjQkNDCQgIYO/evdx9993k5bknpiZNmgTAzJkzAbj//vsLjFG1alXuu+8+wsLCuPLKK2nWrBkAZ86cYcCAAWRmZqKqPPzww1StWpWbb76Z3r17s2jRIl588UXGjRtHnz59qFatGh07dnQSNWOMMebPQFS1ZBUv8G8PiUgEMAcog3tGZ76qji+qTUxMjCYlJRVVxREbG1tg/7e4LfRLnTlzhtOnT1O+fHnS0tLo3Lkz33zzDZdddtlFi8kYc2kQkQ2qar/w01wSip1pEZGbgeeAQNwLaq8GtgOhRbVT1c1A098gxj+87OxsOnTowOnTp1FVXn75ZUtYjDHGmN9YSW4PTcD9t4c+VdWmItIBGFC6YfmWyy+/nJLOEBljjDHml7G/PWSMMcYYn1CSmZaz//bQQSCrdMMyxhhjjCmoJElLCpCN+28P3QEEAJVLMyhjjDHGmLOVJGnp4Pllcnm4nwZCRDaXalTGGGOMMWcpNGkRkaHAA0CDs5KUy4HPSzswY4wxxhhvRc20vAN8BEwCRnmV/6yqP5VqVMYYY4wxZyk0aVHVTCAT6P/7hWOMMcYYc34leeTZGGOMMeais6TFGGOMMT7BkhZjjDHG+ARLWowxxhjjEyxpMcYYY4xPsKTFGGOMMT7BkhZjjDHG+ARLWowxxhjjEyxpMcYYY4xPsKTFGGOMMT7BkhZjjDHG+ARLWowxxhjjEyxpMcYYY4xPsKTFGGOMMT7Bp5OW5ORkkpOTL3YYxhhjjPkd+GzSkpCQgMvluthhGGOMMeZ34rNJC1jiYowxxlxKfDppMcYYY8ylw5IWY4wxxvgES1qMMcYY4xMsaTHGGGOMT7CkxRjzp5KVlcXQoUNp0KABUVFRREdHM2vWLOf4jh076NatGw0bNiQqKoq+ffty4MABEhISCAgIwOVy4XK56Ny5c6Fj5OTk0K9fP0JCQmjRogXp6ekAfPLJJ0RHRxMeHk50dDQrV6502owePZq6detSuXLlAn3NnDmT8PBwXC4Xbdq0Ydu2bQC8/fbbTiwulws/Pz/nVzzEx8cTHh5OREQEQEMRqQEgIv8Ukc0ikiwiH4tIoKe8kYh8ISI5IvKY9/giki4iqZ42Sec736LaF0ZEbhURFZFGXmV+IvIvEdniGXO9iNT3imPtWX0ki8gWz/b1IrLB026DiHT0qjdRRHaLSNZZ7e/3OrfPRKSJp/wOT1n+K09EXJ5j/T1tNovI8l9zbYu4NstF5KiILClJ/T8CEYkRkX9dQP1Gnmu1SUQa/GaBqGqpvoAywCZgSXF1o6Oj9UK1b99e27dvf8HtjDF/Tv369dMnnnhCz5w5o6qqBw8e1MmTJ6uq6okTJzQkJEQXL17s1F+1apWmpqbqqlWrtHv37iUaY/r06TpkyBBVVY2Pj9e+ffuqqurGjRt17969qqqampqqgYGBTpsvvvhC9+3bp5UqVSrQV2ZmprO9aNEiveGGG84Zb/PmzRocHKyqqqdPn9aaNWvqoUOHVFUV+BEY596kiv7vs/dhYKZnuxbQDJgIPKYFP6PTgRpa9Od4oe2LaDMPWAv8w6usP7AQ8PPsBwHVvOJIBup69ht79rd49psCgZ7tMGCvV78tgdpA1lkxeF+PHsDy88QZDqR5tv2Bg/nXA5j6a65tEdemE3BzSb4XffUFjALG/Nb9/h4zLY8A23+HcYwxl7i0tDQSExOZMGECfn7uj7eaNWsycuRIAN555x1atWrFzTff7LSJjY0lLCzsgsZZtGgRAwcOBKB3796sWLECVaVp06YEBgYCEBoayokTJ8jJyQGgZcuW1K5d+5y+qlSp4mwfP34cETmnTnx8PHFxccD/ftA8fvx4/peDH7DPc+yYV7NKgHrKD6rqeuD0BZ2ox4W2F5HKQBvgHiDO61BtYL+q5nn63aOqR7yOzwf6ebb7A/FeMWxS1X2e3a1ABREp5zn2paruP0/c570eZ+kPzM0P3fOqJO43ogq/8NqKyGQRGea1Py5/JkZVVwA/nyeWc4hINxH52jO79K/82RkRqSQir4lIomc24xZP+V0i8p5nNmeniEz16it/FmmLiEzxKs8SkadFZKuIfCoizUUkQUS+E5EenjqxXmNXFpHXvWakbjs7ZmA4MFREVolIPRHZLiKzPGN8LCIVPHXv88y4pYjIuyJSsajr4V+Si/ZLiUgQ0B13BvrX0hzLGHNxxcbGXpRxExISnO2tW7cSGRnpJCxn27JlC9HR0YX2tXbtWud3P/Xp04fRo0eft97evXupW7cuAP7+/gQEBJCRkUGNGjWcOu+++y5RUVGUK1eu2HOYPn06zz33HKdOnSpwSynfvHnzWLRoEQBly5ZlxowZhIeHU6lSJYAKwL/z64rIROD/gEygQ7GDu798PxYRBV5R1VdL0KY4t+Ce1dghIhkiEq2qG3AnJZ+JSFtgBfCWqm7yavcu8DrwDO6ZiDuAO8/T/23ARlXNKS4QT+LwV+AyoON5qvTzxIuqnhaRoUAqcBzYCXgnHhdybecBzwPTPft9gRuKi/es2MsDrwDtVHWXiMR7HR4NrFTVQSJSFUgUkU89x1y4Z6ZygG9E5EXgDDAFiAaO4H7Pb1XVD3AnYStVdYSIvA9MAK4HmgBzgMVnhfZ3IFNVwz1xVvM+qKrLRGQm7pmvZ0SkHtAQ6K+q94nIfNzv4VvAe6o6y9PPBNyJ7ouFXZPSnml5HngcyCusgogMFpEkEUk6dOhQKYdjjCkNsbGxf8g/qTFx4kRcLpcz+1Gctm3bOn8epLCEpSS2bt3KyJEjeeWVV0pUf9iwYaSlpTFlyhQmTJhQ4NhXX31FxYoVndmg06dPM2PGDDZt2sS+ffsATgBP5NdX1dGqWhd4G3iwBMO3UdUo4EZgmIi0K1HQRfOevZjr2UdV9wDXeuLNA1aISCevdhnAERGJwz1Dn312xyISivvLd0hJAlHV6araABgJjDmrrxZAtqrmr5spCwzFcysK2MwvvLaeZKyWiASKSCRwRFV3lyRmL42A71R1l2ffO2npAowSkWQgASgPXOU5tkJVM1X1JLANuBr3LawEVT2kqrmec8h/r08Byz3bqcBqVT3t2a53nrg6879kjLNmywqzS1XzPyQ2ePUbJiJrRSQVd5IaWlQnpTbTIiI3AQdVdYOIxBZWz5PVvwoQExNzvqk7Y4wPcLlcBWY9LoYmTZqQkpJCXl4efn5+jB49mtGjRzuLX0NDQ1m9evWvHqdOnTrs3r2boKAgcnNzyczMpHr16gDs2bOHnj178p///IcGDS5s/WFcXBxDhw4tUDZ37lz69+/v7Ocnh159/wS0Pk93bwPLgLFFjamqez3/Pej5Kbs5sOaCAvciIlfgntEI98zelAFUREaoWw7wEfCRiBwAbsU965JvHu4vxLvO03cQ8D7wf6qadoGhzQVmnFUWR8FEwAWQ37dnRmDUefoq0bUFFgC9gStxn9dvSYDbVPWbAoXuRMx7BuoMxX/Xn1bPvUbcyWQOgKrmichvlSecHVMFz/YbwK2qmiIidwGxRXVSmjMt1wE9RCQd9z+WjiLyVimOZ4y5xIWEhBATE8OYMWM4c+YMACdPnsxf+8Htt9/OunXrWLp0qdNmzZo1bNmy5YLG6dGjB3PmzAFg4cKFdOzYERHh6NGjdO/encmTJ3PdddeVqK+dO3c620uXLqVhw4bOfl5eHvPnz3fWs4A7Ydq2bRteM9NV8KwbFJH/NXbf8vi6qLE96yIuz9/G/dP7hV2Mc/UG3lTVq1W1nmdmYhfQVkSivJ668QMigO/Pav8+7gWw/z0r1qrAUmCUqn5ekkDOuh7dcd/uyT/mh/uWzVyvOnuBJiJS07N/Pb/w2nrMw50Y9cadwFyob4Bgz+0V+N96H3Bfn4c8a28QkabF9JUItBeRGiJSBvfs1y/N4D+h4G2zakXULc7lwH7PLNcdxVUutaRFVZ9Q1SBVrYf7TVupqgNKazxjjAGYPXs2GRkZTgJz/fXXM3Wqey1ihQoVWLJkCS+++CINGzakSZMmvPzyy9SsWbOYXgu65557nDGee+45Jk+eDMBLL73Et99+y/jx451HlQ8ePAjA448/TlBQENnZ2QQFBTFu3DinTWhoKC6Xi+eee85JhsCdUNWtW5fg4GCnLDAwkLFjx9KuXbv8R54rAk95Dk/2LLLcjDsBeQRARK4UkT2413aMEZE9IlIF+AvuNSYpuL/Ulqrqck+b+0Xk/mLan09/3ImHt3c95bWAD8X9GPNmIBd4ybuiqv6sqlNU9dRZfTwIhABPyv8eVa7liW+qJ76KntjG5bfxLPxM9sQ+0Ku/dsBuVf3Oa+x9wD+ANZ5r6PoV1xZV3Yr7S3mv90JhcT/avQDo5Kl/3rUuqnoCeABYLiIbcC/ezfQc/idQFtgsIls9+4XyjD8KWAWkABtUdVFRbYowAajmuR4peNb3iMhsEYm5wL7+DnwFfE4JEkH534xQ6fHcHnpMVW8qql5MTIwmJZ331wQUKn/x38WeljbmUmb/H148IrJBVS/0i8L4CBGprKpZnhmV6cBOVZ12seO6WH6XXy6nqgnFJSzGGGOMOcd9npmirUAA7qeJLlml+sizMcb4sokTJ7JgQcGlCEU9Cn0pEZHqFFxAm6+Tqmb83vH4Os8i6PpnFY/0zKpcsjMrZ7OkxRhjCpH/9JE5lycxcV3sOP4sVLXnxY7BF9jfHjLGGGOMT7CkxRhjjDE+wZIWY4wxxvgES1qMMcYY4xMsaTHGGGOMT7CkxRhjjDE+wZIWY4wxxvgES1qMMcYY4xMsaTHGGGOMT7CkxRhjjDE+4U/xa/yTk5OdvzJrjPn9JScn43LZb3Q3xpQun59pSUhIsA9LY4wx5hLwp5hpSUhIuNghGHNJs5lOY8zvwednWowxxhhzabCkxRhjjDE+wZIWY4wxxvgES1qMMcYY4xMsaTHGGGOMT7CkxRhjjDE+wZIWY4wxxvgES1qMMcYY4xMsaTHGGGOMT7CkxRhjjDE+wZIWY4wxxvgES1qMMcYY4xMsaTHGGGOMT7CkxRhjjDE+wZIWY8yfQlZWFkOHDqVBgwZERUURHR3NrFmznOM7duygW7duNGzYkKioKPr27cuBAwdISEggICAAl8uFy+Wic+fOhY6Rk5NDv379CAkJoUWLFqSnpwPwySefEB0dTXh4ONHR0axcudJpM3r0aOrWrUvlypUL9DVz5kzCw8NxuVy0adOGbdu2AXDq1CnuvvtuwsPDiYyMJCEhAYCff/7ZidHlclGjRg2GDx8OgIhcJSKrRGSTiGwWkW6e8uqe8iwRecl7fBFJEJFvRCTZ86p19vkW1b4wIuISERWRrmeVjxaRrZ74kkWkhVccP4iIeNX9QESyvPr7wqttP696D4rIt57xaniV3+I1TpKItPE6NkVEtnhe3n2t9boW+0TkA095gIh8KCIpnhju9mqzXESOisiSEl6biSKyO//cfIWIrLuAuuVE5FPPdexXfIsLpKql9gLSgVQgGUgqrn50dLQaY3xP+/bttX379hc1hn79+ukTTzyhZ86cUVXVgwcP6uTJk1VV9cSJExoSEqKLFy926q9atUpTU1N11apV2r179xKNMX36dB0yZIiqqsbHx2vfvn1VVXXjxo26d+9eVVVNTU3VwMBAp80XX3yh+/bt00qVKhXoKzMz09letGiR3nDDDaqq+tJLL+ldd92lqqoHDhzQqKgo55y8RUVF6erVqxVIAl4Fhqr7c7cJkO7ZrgS0Ae4HXtKCn88JQIwW/RleaPsi2kwB1gJzvMpaAV8A5Tz7NYBArzg2A208+1WBr4Asz/41QEPPdiCwH6jq2W8K1PN819TwGq8yIJ7tCOBrz3Z34BPA33Nu64Eq5zmHd4H/82z/DZji2a4J/ARc5tnvBNwMLCnhtWkJ1M4/tz/jy3OOn5ZW/7/HTEsHVXWpaszvMJYx5hKUlpZGYmIiEyZMwM/P/bFWs2ZNRo4cCcA777xDq1atuPnmm502sbGxhIWFXdA4ixYtYuDAgQD07t2bFStWoKo0bdqUwMBAAEJDQzlx4gQ5OTkAtGzZktq1a5/TV5UqVZzt48ePkz/RsG3bNjp27AhArVq1qFq1KklJSQXa7tixg4MHD9K2bdv8IgXyOwwA9gGo6nFV/Qw4eUEnmt/pBbb3zJb0Ae4CrheR8p5DtYHDqprj6fewqu7zajoXiPNs9wLe84phh6ru9GzvAw7iTh5Q1U2qmn6euLPU8w2KOznJ324CrFHVXFU9jjtZOntGqArQEfggvzvgcs+5VcadtOR6xlkB/HxW+64issBrPzZ/JkZVv1TV/ee5dOcQkQYi8qWIpIrIBO/ZGREZISLrPbNJ//CU1ROR7SIyyzMj9LGIVPAcc3n62iwi74tINU95gohM88xGbReRZiLynojsFJEJXuN5jz3SE1OKiEw+K+ZawFtAM89MSwMRSReRf4jIRk+7Rp66zT0zaJtEZJ2IXFuS6+JfkkrGGFOc5ORkYmNjf7fx8m+bAGzdupXIyEgnYTnbli1biI6OLrSvtWvX4nK5AOjTpw+jR48+b729e/dSt25dAPz9/QkICCAjI4MaNZw7E7z77rtERUVRrly5Ys9h+vTpPPfcc5w6dcq5pRQZGcnixYvp378/u3fvZsOGDezevZvmzZs77ebOnUu/fv2cRAcYB3wsIg/h/pIu/B5XQa+LyBncMwsTvL7of6nWwC5VTRORBNwzG+8CHwNPisgO4FNgnqqu9mq3ApglImVwJy+Dgb+f3bmINAcuA9KKC0REegKTgFqeOABSgLEi8ixQEegAbDur6a3AClU95tl/CViMOxG8HOinqnlFDP0p8KqIVPIkRv1wJ2UX6gXgBVWNF5H7vc6rC9AQaA4IsFhE2gE/eMr7q+p9IjIfuA13EvEf4CFVXS0i44GxwHBPl6dUNUZEHgEWAdG4E7M0EZmmqhleY98I3AK0UNVsEbnCO2BVPSgi9wKPqepNnjbgTlijROQB4DHgXuBroK2q5opIZ+ApT7xFKu2ZFsX9P9IGERl8vgoiMtiT5SUdOnSolMMxxpSGhIQE50v/j2DixIm4XC5n9qM4bdu2JTk5meTk5EITlpLYunUrI0eO5JVXXilR/WHDhpGWlsaUKVOYMMH9g+2gQYMICgoiJiaG4cOH07p1a8qUKVOg3dy5c+nfv793UX/gDVUNAroBb4pIcZ/vd6hqONDW87qzREEXrT//+4Ke69lHVbNwfxkOBg4B80TkLq92Z4DPcCcsFc43eyIitYE3gbuLSRrwjPm+qjbCnYT801P2MbAMWAfE475ldeY85xDvtX8D7iUOgYALeMkzG1PYuLnAcuBmEfHHnTAtKi7e82gF5M/YvONV3sXz2gRsBBrhTlbAnTAme7Y3APVEJAD37bT8JHEO0M6rv8We/6YCW1V1v2dG7Dug7lkxdQZeV9Vsz7n+VMJzyZ8524D7dh64ZwQXiMgWYBoQWpKOSnumpY2q7vVMGX0iIl+r6hrvCqr6Ku77scTExPzaLN8Yc5F4z3z83po0aUJKSgp5eXn4+fkxevRoRo8e7Sx+DQ0NZfXq1cX0Urw6deqwe/dugoKCyM3NJTMzk+rVqwOwZ88eevbsyX/+8x8aNGhwQf3GxcUxdOhQwD2DM23aNOdY69atueaaa5z9lJQUcnNzz545ugfPbQ5V/cJzW6YG7lsp56Wqez3//VlE3sH9k/t/LihwL55ZktuAW+T/t3fvQVbWdRzH358BzFCzEseazVwUVLARlcVSrNmR0rwM5IQDY5q3NCsjm3R0yrLRKKqZnDG6jilaho5UDoalziRBMt4ghJX1gkITjpfyghfQAr798fwOPp72cnZhz9nf2c9rZofnPM/v+T3f33nY3e/+LueRvkHRC7CXpD0i4tWI2Eoxf2WxpNXAmcC8UhU3A3+g6DWqrvtdwCLgGxFxX1/iioglkvaXNCoNS80GZqd6fws8XrrOKIr34ZRSFWcDc1Iv1FpJ6ygShQd6uOzNwIUUPRYPRcSrPZTtKwHfi4i3ZcaSWoE3S7u2Au+sob7KOduqzt/GzssRKvVuLdV5FXBPRJySYl9cS0UD2tNS+qZ4nuI/45E9n2Fm1ndjxoyhra2Nyy+/nK1biz+c33jjjcrEQE477TSWLVvGokWLtp+zZMkSOjo6+nSdqVOncsMNNwCwYMECjj32WCTx8ssvc9JJJzFnzhwmT55cU11PPPHE9u1FixYxdmzxx/KmTZt4/fXXgWJV0vDhwxk/fvz2svPnz6/uZYFiaGAKgKRxwK4UPRpdkjQ8/YJG0gjgZKBvb8b/mwKsioh9I6I1IvajGBo6RdJBksaWyh4G/KPq/KUUwznlXg4k7ULx++PGiFhQSyCSxqQ5KEg6AngH8IKkYZL2SvsPpZike1fp1OkUk2rLc3jK7+0+wEEUvRA9+StwBHAe/RsarZy1igAACWNJREFUAriPt4ZLZpb23wmcI2n3FFOLulj5VRERG4GXJFUmQJ2R4uuPu4GzJY1M135vL+V7sifwdNo+q+azBmqGL8W46h6l7WXAJ3s6x6uHzKy/Nm7cGOeff360trbGxIkT45hjjom5c+duP97Z2RnHH398jBkzJsaNGxczZsyIZ599tk+rhzZv3hzTp0+PAw44ICZNmhRPPvlkRERcddVVMXLkyJgwYcL2r+eeey4iIi655JJoaWkJSdHS0hJXXHFFRETMmjUrxo8fHxMmTIj29vbo6OiIiIh169bFgQceGAcffHBMmTIl1q9f/7YYRo8eHZ2dndtfU6weGg/cSzFnYyVwXLz1s3g9xV/8rwEbUtndKLrqVwGPUMyfGJbKTwWu7On86Ppn/vXABVX7pgJ/ohgaWkYxf2QVxXDBqFRmMV2sYuKt1UOnA/9N7ap8HZaOzUoxbaGYc3Jt2n9patdKiiGgysqkXVMMayiSgsOqrrm4+vcUxbDQXRTDJx3A6aVjSymSw80pjuNLx+am92xkad8PUrlt6d9vd/VeprJjKVZRrUrnPV069pUUz+rUvgMohl06SmUurtRPkSTel+q6DXhP9XsPtFNaBVV17LXS/svS+7cS+G7ad0Hl3ndRz/rSvW4DFqftoyh6uf4OfIe04q23r8qSsJ1O0v4U2TEU3UG/jaJbrlttbW1RPUvezMy6J2l5eHVm00m9GZsjIiTNpJhgO63RcTXagM1piYingAkDVb+ZmVkTm0gx6VfAy8A5DY5nUPCSZzOzKrNnz+bWW299276elkIPNZLup5gnUnZGRKxuRDw5S5OWT63afWsamfAf/lUGbHioPzw8ZGbWNx4esqHEzx4yMzOzLDhpMTMzsyw4aTEzM7MsOGkxMzOzLDhpMTMzsyw4aTEzM7MsOGkxMzOzLDhpMTMzsyw4aTEzM7MsOGkxMzOzLDhpMTMzsyw4aTEzM7MsOGkxMzOzLDhpMTMzsyw4aTEzM7MsOGkxMzOzLDhpMTMzsyw4aTEzM7MsOGkxMzOzLDhpMTMzsyw4aTEzM7MsOGkxMzOzLDhpMTMzsywoIhodw3aS/gX8ow+njAL+PUDhDAZuX96auX3N3DbIq337RcTejQ7CrB4GVdLSV5Ieioi2RscxUNy+vDVz+5q5bdD87TPLlYeHzMzMLAtOWszMzCwLuSctv2x0AAPM7ctbM7evmdsGzd8+syxlPafFzMzMho7ce1rMzMxsiHDSYmZmZlnIImmR9ElJj0laK+myLo6/Q9It6fj9klrrH2X/1dC+syT9S9LK9PW5RsTZH5Kuk/S8pI5ujkvSNantqyQdUe8Yd0QN7WuXtLF0775V7xj7S9K+ku6RtEbSI5K+0kWZbO9fje3L9v6ZNaPhjQ6gN5KGAT8BPgFsAB6UtDAi1pSKnQu8FBFjJM0Evg/MqH+0fVdj+wBuiYgL6x7gjpsHzAVu7Ob4CcDY9PVh4Gfp31zMo+f2ASyNiJPrE85OtQX4WkSskLQHsFzS3VX/N3O+f7W0D/K9f2ZNJ4eeliOBtRHxVET8B7gZmFZVZhpwQ9peAEyRpDrGuCNqaV+2ImIJ8GIPRaYBN0bhPuDdkt5fn+h2XA3ty1ZEPBMRK9L2q0An0FJVLNv7V2P7zGwQySFpaQH+WXq9gf//wbK9TERsATYCe9Uluh1XS/sAPp263xdI2rc+odVFre3P2VGSHpb0J0mHNDqY/khDrocD91cdaor710P7oAnun1mzyCFpMbgdaI2IQ4G7eatXyQa/FRTPhpkA/Bi4rcHx9Jmk3YHfARdFxCuNjmdn66V92d8/s2aSQ9LyNFDuWfhA2tdlGUnDgT2BF+oS3Y7rtX0R8UJEvJleXgtMrFNs9VDL/c1WRLwSEa+l7TuAEZJGNTismkkaQfEL/aaI+H0XRbK+f721L/f7Z9ZsckhaHgTGShotaRdgJrCwqsxC4My0PR34S+TzqXm9tq9qjsBUirH3ZrEQ+GxahfIRYGNEPNPooHYWSe+rzK+SdCTF91wWCXWK+1dAZ0T8qJti2d6/WtqX8/0za0aDfvVQRGyRdCFwJzAMuC4iHpF0JfBQRCyk+MHza0lrKSZFzmxcxH1TY/tmSZpKsdrhReCshgXcR5LmA+3AKEkbgCuAEQAR8XPgDuBEYC2wCTi7MZH2Tw3tmw58QdIWYDMwM6OEejJwBrBa0sq07+vAB6Ep7l8t7cv5/pk1HX+Mv5mZmWUhh+EhMzMzMyctZmZmlgcnLWZmZpYFJy1mZmaWBSctZmZmlgUnLTZoSWrt7unJA3jNxZLaeikzIHGlJwofXXo9T9L0nX0dM7NcOWmx7KUnZTeDduDo3gqZmQ1VTlpssBsu6SZJnelhkSMBJK2X9H1JK4BTJZ0n6cH0YLvflcrNk3SNpGWSnir3XEi6VNLqdM6c0jVPlfSApMclfbSn4CQNk/TDdO1Vkj6f9renXpsFkh5Nbah8suqJad/yFNsf0wP7LgC+Kmll6bof6yp2M7OhyEmLDXYHAT+NiHHAK8AXS8deiIgjIuJm4PcRMSk92K4TOLdU7v3AMcDJwBwASScA04APp3N+UCo/PCKOBC6i+ITbnpxL8dH1k4BJwHmSRqdjh6c6xgP7A5Ml7Qr8AjghIiYCewNExHrg58DVEXFYRCztLnYzs6HKSYsNdv+MiHvT9m8ofoFX3FLa/pCkpZJWA58BDikduy0itkXEGmCftO/jwPURsQkgIl4sla88OG850NpLfMdRPHtnJXA/sBcwNh17ICI2RMQ2YGWq62DgqYhYl8rM76X+rmI3MxuSBv2zh2zIq37ORPn166XtecCnIuJhSWdRzA+peLO0rRquWSm/ld6/RwR8OSLufNtOqb3qurXU1VMslWuZmQ1Z7mmxwe6Dko5K26cBf+um3B7AM5JGUPS09OZu4OzS3Jf39jO+OykeqDci1XOgpN16KP8YsH+awwIwo3TsVYp2mJlZF5y02GD3GPAlSZ3Ae4CfdVPumxTDM/cCj/ZWaUT8GVgIPJSGdi7uZ3zXAmuAFWkZ9C/ooUclIjZTzMv5s6TlFInKxnT4duCUqom4ZmaW+CnPZnUmafeIeC2tJvoJ8EREXN3ouMzMBjv3tJjV33mpd+cRYE+K3hkzM+uFe1rMzMwsC+5pMTMzsyw4aTEzM7MsOGkxMzOzLDhpMTMzsyw4aTEzM7Ms/A8EUFRiYa4XOgAAAABJRU5ErkJggg==\n",
      "text/plain": [
       "<Figure size 432x288 with 1 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "from Bio import Phylo\n",
    "tree = Phylo.read('output/parsnp.tree', \"newick\")\n",
    "tree.ladderize()\n",
    "Phylo.draw(tree)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
